<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Atlas of clinically distinct cell states and ecosystems across human solid tumors</title><meta name="author" content="Bogdan A. Luca"/><meta name="description" content="Cell, 184 (2021) 5482-5524. doi:10.1016/j.cell.2021.09.014"/><style type="text/css"> * {margin:0; padding:0; text-indent:0; }
 .s1 { color: #BCBEC0; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 18pt; }
 .s2 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19pt; }
 .s3 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s4 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s5 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s6 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .p, p { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; margin:0pt; }
 .s7 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .h2, h2 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 16pt; }
 .s8 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s9 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s10 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s11 { color: #0097CF; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s12 { color: #BFBFBF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s13 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 20pt; }
 .s15 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt; }
 .s16 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s17 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s18 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s19 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: 2pt; }
 .s20 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s21 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s22 { color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s23 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s24 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s26 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s27 { color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s28 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .a, a { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 h4 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 9pt; }
 .s29 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s30 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s31 { color: #AB4D4C; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s32 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s33 { color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 .s34 { color: #AB4D4C; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s35 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s36 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s37 { color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s38 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s39 { color: #0097CF; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s40 { color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s41 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s42 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s43 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s44 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s45 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s46 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 3pt; }
 .s47 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -5pt; }
 .s48 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s49 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -5pt; }
 .s50 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -2pt; }
 .s51 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -1pt; }
 .s52 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 9pt; }
 .s53 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s54 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -1pt; }
 .s55 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 2pt; }
 .s56 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -2pt; }
 .s57 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -8pt; }
 .s58 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -4pt; }
 .s59 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 6pt; }
 .s60 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 7pt; }
 .s61 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 1pt; }
 .s62 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 2pt; }
 .s63 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -4pt; }
 .s64 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -6pt; }
 .s65 { color: black; font-family:"Palatino Linotype", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s66 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s67 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s68 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 2pt; }
 .s69 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 5pt; }
 .s70 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 4pt; }
 .s71 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 5pt; }
 .s72 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -2pt; }
 .s73 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 12pt; }
 .s74 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 8.5pt; }
 .s75 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 8.5pt; }
 .s76 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s78 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 4pt; }
 .s79 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -6pt; }
 .s80 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 5.5pt; }
 .s82 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -3pt; }
 .s83 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -3pt; }
 .s84 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: underline; font-size: 8.5pt; vertical-align: 2pt; }
 .s85 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: underline; font-size: 5.5pt; vertical-align: 1pt; }
 .s86 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: underline; font-size: 5.5pt; }
 .s87 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 8.5pt; vertical-align: 2pt; }
 .s88 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -5pt; }
 .s89 { color: black; font-family:"Arial Black", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s90 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: -9pt; }
 .s91 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: underline; font-size: 8.5pt; }
 .s92 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 8.5pt; }
 .s93 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; vertical-align: -1pt; }
 .s94 { color: #BFBFBF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 24pt; }
 .s95 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s96 { color: #0097CF; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s97 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16pt; }
 .s98 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s99 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s100 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s101 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s102 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s103 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 1pt; }
 .s104 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; vertical-align: -1pt; }
 .s105 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s106 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s107 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s108 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s109 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s110 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; vertical-align: -1pt; }
 .s111 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s113 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s114 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s115 { color: #2D2D2D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s116 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 65.5pt; vertical-align: -9pt; }
 .s117 { color: #4D82B1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 65.5pt; }
 .s118 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s119 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s120 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s121 { color: #B6B6B6; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s122 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s123 { color: #1A1A1A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s124 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s125 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s126 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s127 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s128 { color: #2D2D2D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s129 { color: #2D2D2D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s130 { color: #1A1A1A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s131 { color: #B6B6B6; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; vertical-align: -1pt; }
 .s132 { color: #B6B6B6; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s133 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s134 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s135 { color: #4D4B4B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s137 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s138 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s139 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s140 { color: #1A1A1A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 3pt; }
 .s141 { color: #1A1A1A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s142 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s143 { color: #4D82B1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 24.5pt; }
 .s144 { color: #1A1A1A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s149 { color: #D4362D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s150 { color: #9C9C9C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s152 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 13pt; }
 .s153 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 22.5pt; }
 .s154 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s155 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s156 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s157 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s158 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s159 { color: #59AA48; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s160 { color: #563D6B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s161 { color: #856228; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 20pt; }
 .s163 { color: #82493B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; vertical-align: 4pt; }
 .s164 { color: #7B4DA1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; }
 .s166 { color: #775D44; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17pt; vertical-align: 2pt; }
 .s167 { color: #A55228; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; vertical-align: -8pt; }
 .s168 { color: #625B5B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; }
 .s169 { color: #3A9CB5; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; vertical-align: -7pt; }
 .s170 { color: #4D4B4B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; }
 .s171 { color: #44705D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; }
 .s172 { color: #E21A70; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 28pt; vertical-align: -2pt; }
 .s173 { color: #4D4B4B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 28pt; }
 .s174 { color: #EB8E0A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s175 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s176 { color: #F480E6; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; vertical-align: -1pt; }
 .s177 { color: #A55228; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s178 { color: #7B4DA1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s179 { color: #59AA48; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s180 { color: #855D91; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s181 { color: #E21A70; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s182 { color: #563D6B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s183 { color: #46876B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s184 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s185 { color: #1A1A1A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 1pt; }
 .s186 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s187 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; vertical-align: 2pt; }
 .s188 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s189 { color: #7B4DA1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 26pt; }
 .s190 { color: #E21A70; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 26pt; }
 .s191 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s192 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 33.5pt; }
 .s193 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s196 { color: #7E706E; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 48pt; }
 .s197 { color: #878287; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s198 { color: #9EB5C1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; vertical-align: -3pt; }
 .s199 { color: #F99A99; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s200 { color: #E1A805; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s201 { color: #895D66; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s202 { color: #269A75; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s203 { color: #726EAF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s204 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 2pt; }
 .s205 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 1pt; }
 .s206 { color: #A5CDE2; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 26pt; }
 .s207 { color: #E62D8C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 27pt; }
 .s208 { color: #625B5B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s210 { color: #7E706E; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s211 { color: #7E706E; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s212 { color: #625B5B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s214 { color: #F680BD; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; vertical-align: -5pt; }
 .s215 { color: #895D66; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s216 { color: #7E706E; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s218 { color: #7E706E; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17.5pt; }
 .s228 { color: #878287; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 29pt; }
 .s229 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s230 { color: #9C9C9C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 28pt; vertical-align: 3pt; }
 .s231 { color: #F680BD; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 26pt; vertical-align: 11pt; }
 .s232 { color: #F680BD; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s233 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17pt; }
 .s235 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s237 { color: #3A9CB5; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 27pt; vertical-align: -2pt; }
 .s238 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s239 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s240 { color: #2D2D2D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s241 { color: #2D2D2D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s242 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s243 { color: #2D2D2D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s244 { color: #2D2D2D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s245 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s246 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21pt; }
 .s247 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s248 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s249 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s250 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 68pt; }
 .s251 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s252 { color: #3F3F3D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s253 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s254 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s255 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; vertical-align: -1pt; }
 .s256 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s257 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s258 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s259 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s260 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -1pt; }
 .s261 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s262 { color: #2A2A2A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s263 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s264 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 70pt; }
 .s265 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s266 { color: #2A2A2A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s267 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s268 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s269 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s270 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s271 { color: #3A2849; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 30pt; }
 .s272 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 24pt; vertical-align: 3pt; }
 .s273 { color: #67443F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 24pt; }
 .s274 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s275 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; }
 .s276 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s277 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 1pt; }
 .s278 { color: #2A2A2A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s279 { color: #57416B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 29.5pt; }
 .s280 { color: #5BAA49; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 25.5pt; vertical-align: -14pt; }
 .s281 { color: #77544F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 25.5pt; vertical-align: -18pt; }
 .s282 { color: #77544F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 25.5pt; }
 .s284 { color: #5BAA49; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 24pt; }
 .s285 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -3pt; }
 .s286 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 27.5pt; }
 .s287 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; vertical-align: 4pt; }
 .s288 { color: #77544F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16.5pt; }
 .s289 { color: #57416B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s290 { color: #57416B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s291 { color: #2A2A2A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s292 { color: #389CB6; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23.5pt; }
 .s293 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s294 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s295 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s296 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -2pt; }
 .s297 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
 .s298 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: 1pt; }
 .s299 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s300 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s301 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; vertical-align: -1pt; }
 .s302 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 1pt; }
 .s303 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s304 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -3pt; }
 .s305 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -1pt; }
 .s306 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 1pt; }
 .s307 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s308 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s309 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; vertical-align: 2pt; }
 .s310 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; }
 .s311 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 29pt; }
 .s312 { color: #E6563F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s313 { color: #4170AF; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; vertical-align: -8pt; }
 .s314 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s316 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: 1pt; }
 .s317 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s318 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: -2pt; }
 .s319 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s320 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 18pt; }
 .s321 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s322 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s323 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s324 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: -3pt; }
 .s325 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s326 { color: #2A2A2A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s327 { color: #2A2A2A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s328 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 4pt; }
 .s329 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 44pt; vertical-align: -15pt; }
 .s330 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s331 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 1pt; }
 .s332 { color: #545454; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -1pt; }
 .s333 { color: #E6563F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
 .s334 { color: #F28E7B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s335 { color: #9AB1D4; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s336 { color: #4170AF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s337 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s338 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -4pt; }
 .s339 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s340 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s341 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 20.5pt; }
 .s342 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 17pt; }
 .s343 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17pt; }
 .s345 { color: #3F3F3D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s346 { color: #3F3F3D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s348 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 4pt; }
 .s349 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 4pt; }
 .s350 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s351 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: -1pt; }
 .s352 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s353 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: 4pt; }
 .s355 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s356 { color: #8A0101; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s357 { color: #93180F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s358 { color: #8A0101; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s359 { color: #77544F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s361 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s365 { color: #CF8979; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s366 { color: #E4B5AA; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; vertical-align: -3pt; }
 .s367 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 1pt; }
 .s368 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s369 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s370 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 2pt; }
 .s371 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 1pt; }
 .s372 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 1pt; }
 .s373 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: -1pt; }
 .s374 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: -1pt; }
 .s375 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s376 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -1pt; }
 .s377 { color: #3F3F3D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s378 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: line-through; font-size: 27.5pt; }
 .s379 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s380 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s381 { color: #2A2A2A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s382 { color: #B55944; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s384 { color: #2A2A2A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s385 { color: #3B133D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s390 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 1pt; }
 .s391 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s392 { color: #752849; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s393 { color: #334D54; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s394 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 1pt; }
 .s395 { color: #57416B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s396 { color: #67443F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s397 { color: #BFBFBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21pt; }
 .s398 { color: #BFBFBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s399 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 34.5pt; }
 .s400 { color: #3F3F3D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s401 { color: #BFBFBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s403 { color: #BFBFBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s404 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s405 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 1pt; }
 .s406 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s407 { color: #BFBFBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; }
 .s408 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: 1pt; }
 .s409 { color: #BFBFBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s410 { color: #CF8979; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s411 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 h3 { color: #BFBFBF; font-family:"Microsoft JhengHei", sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 12pt; }
 .s412 { color: #3F3F3D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; vertical-align: -4pt; }
 .s413 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s414 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s415 { color: #AA4436; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s416 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s417 { color: #080808; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s418 { color: #181818; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s419 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s420 { color: #E4B5AA; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; vertical-align: 1pt; }
 .s421 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s422 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; }
 .s423 { color: #AF182A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17.5pt; }
 .s424 { color: #2A2A2A; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s425 { color: #2A2A2A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s426 { color: #181818; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s427 { color: #937579; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s428 { color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s429 { color: #9C9C9C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s430 { color: #8C8C8C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s431 { color: #080808; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 51pt; }
 .s432 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s433 { color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
 .s434 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; vertical-align: 3pt; }
 .s435 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s436 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s437 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s438 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s439 { color: #385B70; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s441 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 35pt; }
 .s442 { color: #42423D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 55pt; }
 .s443 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 55pt; }
 .s444 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s445 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s446 { color: #D4382D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21pt; }
 .s447 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s448 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21pt; }
 .s449 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s450 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s451 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s452 { color: #D4382D; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s453 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s454 { color: #F2A199; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s455 { color: #4D8CB6; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s456 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s457 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17pt; }
 .s458 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s459 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -2pt; }
 .s460 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s461 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -1pt; }
 .s462 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 1pt; }
 .s463 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 1pt; }
 .s464 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: -1pt; }
 .s465 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; vertical-align: 2pt; }
 .s466 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: 1pt; }
 .s467 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s468 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s469 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s470 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; vertical-align: -1pt; }
 .s471 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s472 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s473 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s474 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s475 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; vertical-align: 2pt; }
 .s476 { color: #42423D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s477 { color: #42423D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s478 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; vertical-align: -1pt; }
 .s479 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s480 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s481 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s483 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 1pt; }
 .s484 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s486 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s487 { color: #D4382D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s488 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: 2pt; }
 .s489 { color: #4D8CB6; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12.5pt; }
 .s490 { color: #E48EBF; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s491 { color: #9A999A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s492 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; vertical-align: -2pt; }
 .s493 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s494 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s495 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s496 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 1pt; }
 .s497 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s498 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: 1pt; }
 .s499 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -1pt; }
 .s500 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s501 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s502 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 1pt; }
 .s503 { color: #42423D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s504 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s505 { color: #42423D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt; }
 .s506 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s507 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s508 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 71pt; }
 .s512 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s513 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; }
 .s514 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23pt; }
 .s517 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: 13pt; }
 .s518 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s519 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s520 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s521 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: -2pt; }
 .s522 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s523 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s524 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s525 { color: #BCBCBC; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s526 { color: #BCBCBC; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 66.5pt; }
 .s527 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s528 { color: #595B4F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 46.5pt; }
 .s529 { color: #FD0001; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 47.5pt; }
 .s530 { color: #0C0C0C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s531 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s532 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s533 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19pt; }
 .s534 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s535 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s536 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12.5pt; vertical-align: -1pt; }
 .s537 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16pt; }
 .s538 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s539 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12.5pt; }
 .s540 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s541 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s542 { color: #0C0C0C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s543 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 26.5pt; }
 .s544 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 25pt; }
 .s545 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 32pt; }
 .s546 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 25pt; vertical-align: -5pt; }
 .s547 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s548 { color: #1F1F1F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; }
 .s549 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s550 { color: #1F1F1F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s551 { color: #1F1F1F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s552 { color: #FD0001; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s553 { color: #FDC603; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 30.5pt; }
 .s554 { color: #87569C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 26.5pt; }
 .s555 { color: #A05D77; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s557 { color: #4D8CB6; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s558 { color: #80757B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17.5pt; }
 .s559 { color: #AA388C; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s560 { color: #6D8764; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 17.5pt; }
 .s561 { color: #FD7E01; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s562 { color: #956E64; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s563 { color: #956E64; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s564 { color: #FDD134; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s565 { color: #A575B1; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s566 { color: #C6503D; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s567 { color: #6E7CA3; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s568 { color: #87569C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
 .s569 { color: #AE9526; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s570 { color: #389CB6; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; vertical-align: -6pt; }
 .s571 { color: #87569C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s572 { color: #FDC603; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s573 { color: #64AF54; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; vertical-align: -6pt; }
 .s574 { color: #FD7E01; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 h1 { color: #0C0C0C; font-family:"Microsoft JhengHei", sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 47.5pt; }
 .s575 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; vertical-align: -1pt; }
 .s576 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s577 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s578 { color: #0C0C0C; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; vertical-align: -2pt; }
 .s579 { color: #F680EB; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23.5pt; }
 .s580 { color: #9A999A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23.5pt; }
 .s582 { color: #070707; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s583 { color: #2F2F2F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s584 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s585 { color: #676969; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s586 { color: #525252; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s587 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s588 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s589 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s590 { color: #070707; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s591 { color: #825467; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s592 { color: #2F2F2F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s593 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s595 { color: #5074A1; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 20pt; }
 .s596 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 22pt; }
 .s597 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s598 { color: #214FF7; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s599 { color: #36F224; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s600 { color: #F42424; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s601 { color: #F42424; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s603 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s604 { color: #2F2F2F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s605 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s606 { color: #414141; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s607 { color: #2F2F2F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s608 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s609 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s610 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s611 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13.5pt; }
 .s613 { color: #214FF7; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s614 { color: #36F224; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s615 { color: #F42424; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s616 { color: #F43838; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s617 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s618 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 70.5pt; }
 .s619 { color: #070707; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 16pt; }
 .s620 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s622 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s623 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s624 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s625 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s626 { color: #525252; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s627 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 25pt; }
 .s628 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s629 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 20.5pt; }
 .s630 { color: #3A62F9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s631 { color: #214FF7; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s632 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s633 { color: #525252; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s634 { color: #525252; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s635 { color: #2F2F2F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s636 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s637 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s638 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s639 { color: #36F224; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s640 { color: #F42424; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s641 { color: #525252; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s642 { color: #070707; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s643 { color: #794D85; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s644 { color: #5074A1; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s645 { color: #2393BF; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s646 { color: #70B160; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s647 { color: #525252; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s648 { color: #F9DB4B; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: -3pt; }
 .s649 { color: #F9DB4B; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s650 { color: #525252; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: -3pt; }
 .s651 { color: #214FF7; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s652 { color: #214FF7; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: 2pt; }
 .s653 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 9pt; }
 .s654 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 4.5pt; }
 .s655 { color: #E6D6C1; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 2pt; }
 .s656 { color: #36F224; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s657 { color: #F42424; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s658 { color: #E18C36; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s659 { color: #A79E9A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: -3pt; }
 .s660 { color: #525252; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s661 { color: #5B85BA; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s662 { color: #525252; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s663 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s664 { color: #525252; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s666 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: -1pt; }
 .s667 { color: #414141; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: 2pt; }
 .s668 { color: #676969; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s669 { color: #676969; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 2pt; }
 .s671 { color: #525252; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s672 { color: #525252; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s673 { color: #525252; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s674 { color: #070707; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s675 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: 1pt; }
 .s676 { color: #676969; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 52.5pt; vertical-align: 24pt; }
 .s677 { color: #676969; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 52.5pt; vertical-align: 23pt; }
 .s678 { color: #676969; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 52.5pt; vertical-align: 22pt; }
 .s679 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s680 { color: #676969; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 72pt; }
 .s681 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s683 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s684 { color: #070707; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s685 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -1pt; }
 .s686 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s687 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 1pt; }
 .s688 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: 1pt; }
 .s689 { color: #414141; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s690 { color: #070707; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: 1pt; }
 .s691 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s692 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -1pt; }
 .s693 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s694 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s695 { color: #440E56; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -2pt; }
 .s696 { color: #440E56; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s697 { color: #070707; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s699 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 37.5pt; }
 .s701 { color: #7B7E75; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 31.5pt; vertical-align: 5pt; }
 .s702 { color: #2677B1; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 39.5pt; }
 .s703 { color: #A5C4D6; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 47.5pt; vertical-align: 7pt; }
 .s704 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 47.5pt; }
 .s705 { color: #B8CAB3; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s706 { color: #B8CAB3; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s707 { color: #B8CAB3; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s708 { color: #3F9C38; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 35.5pt; vertical-align: -5pt; }
 .s710 { color: #3F9C38; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 35.5pt; vertical-align: 1pt; }
 .s711 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 35.5pt; }
 .s712 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 35.5pt; vertical-align: -4pt; }
 .s713 { color: #246B97; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 39.5pt; }
 .s714 { color: #C36B5E; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
 .s718 { color: #676969; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s719 { color: #181818; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s720 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1pt; }
 .s721 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 19.5pt; }
 .s722 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -2pt; }
 .s723 { color: #B8CAB3; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15pt; }
 .s725 { color: #2F2F2F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s726 { color: #2F2F2F; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: 3pt; }
 .s727 { color: #2F2F2F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 1pt; }
 .s729 { color: #070707; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 li {display: block; }
 #l1 {padding-left: 0pt;counter-reset: c1 1; }
 #l1> li:before {counter-increment: c1; content: counter(c1, upper-latin)" "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 9pt; }
 #l1> li:first-child:before {counter-increment: c1 0;  }
 li {display: block; }
 #l2 {padding-left: 0pt;counter-reset: d1 1; }
 #l2> li:before {counter-increment: d1; content: "("counter(d1, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l2> li:first-child:before {counter-increment: d1 0;  }
 li {display: block; }
 #l3 {padding-left: 0pt;counter-reset: e1 1; }
 #l3> li:before {counter-increment: e1; content: "("counter(e1, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l3> li:first-child:before {counter-increment: e1 0;  }
 #l4 {padding-left: 0pt;counter-reset: e2 1; }
 #l4> li:before {counter-increment: e2; content: "("counter(e2, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l4> li:first-child:before {counter-increment: e2 0;  }
 li {display: block; }
 #l5 {padding-left: 0pt;counter-reset: f1 1; }
 #l5> li:before {counter-increment: f1; content: "("counter(f1, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l5> li:first-child:before {counter-increment: f1 0;  }
 li {display: block; }
 #l6 {padding-left: 0pt;counter-reset: g1 2; }
 #l6> li:before {counter-increment: g1; content: counter(g1, upper-latin)" "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 9pt; }
 #l6> li:first-child:before {counter-increment: g1 0;  }
 li {display: block; }
 #l7 {padding-left: 0pt;counter-reset: h1 1; }
 #l7> li:before {counter-increment: h1; content: "("counter(h1, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l7> li:first-child:before {counter-increment: h1 0;  }
 li {display: block; }
 #l8 {padding-left: 0pt;counter-reset: i1 19; }
 #l8> li:before {counter-increment: i1; content: counter(i1, upper-latin)" "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l8> li:first-child:before {counter-increment: i1 0;  }
 #l9 {padding-left: 0pt;counter-reset: i2 18; }
 #l9> li:before {counter-increment: i2; content: counter(i1, upper-latin)"."counter(i2, upper-latin)". "; color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l9> li:first-child:before {counter-increment: i2 0;  }
 #l10 {padding-left: 0pt; }
 #l10> li:before {content: "• "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 #l11 {padding-left: 0pt; }
 #l11> li:before {content: "• "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 #l12 {padding-left: 0pt;counter-reset: l1 0; }
 #l12> li:before {counter-increment: l1; content: counter(l1, decimal)" "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l12> li:first-child:before {counter-increment: l1 0;  }
 #l13 {padding-left: 0pt;counter-reset: l2 25; }
 #l13> li:before {counter-increment: l2; content: counter(l1, decimal)"."counter(l2, decimal)" "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 #l13> li:first-child:before {counter-increment: l2 0;  }
 #l14 {padding-left: 0pt;counter-reset: l3 1; }
 #l14> li:before {counter-increment: l3; content: "("counter(l3, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l14> li:first-child:before {counter-increment: l3 0;  }
 #l15 {padding-left: 0pt;counter-reset: m1 8; }
 #l15> li:before {counter-increment: m1; content: counter(m1, decimal)" "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l15> li:first-child:before {counter-increment: m1 0;  }
 #l16 {padding-left: 0pt;counter-reset: m2 4; }
 #l16> li:before {counter-increment: m2; content: counter(m1, decimal)"."counter(m2, decimal)" "; color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 #l16> li:first-child:before {counter-increment: m2 0;  }
 #l17 {padding-left: 0pt; }
 #l17> li:before {content: "• "; color: #FBB362; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 #l18 {padding-left: 0pt; }
 #l18> li:before {content: "• "; color: #FBB362; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 #l19 {padding-left: 0pt; }
 #l19> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l20 {padding-left: 0pt; }
 #l20> li:before {content: "• "; color: #E1A805; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 #l21 {padding-left: 0pt; }
 #l21> li:before {content: "• "; color: #E62D8C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 #l22 {padding-left: 0pt; }
 #l22> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l23 {padding-left: 0pt; }
 #l23> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l24 {padding-left: 0pt; }
 #l24> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l25 {padding-left: 0pt; }
 #l25> li:before {content: "• "; color: #99261F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 #l26 {padding-left: 0pt; }
 #l26> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l27 {padding-left: 0pt;counter-reset: t1 1; }
 #l27> li:before {counter-increment: t1; content: "("counter(t1, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l27> li:first-child:before {counter-increment: t1 0;  }
 #l28 {padding-left: 0pt; }
 #l28> li:before {content: "• "; color: #080808; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 #l29 {padding-left: 0pt; }
 #l29> li:before {content: "• "; color: #804FA3; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; vertical-align: -3pt; }
 #l30 {padding-left: 0pt; }
 #l30> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l31 {padding-left: 0pt;counter-reset: w1 1; }
 #l31> li:before {counter-increment: w1; content: counter(w1, decimal)". "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l31> li:first-child:before {counter-increment: w1 0;  }
 #l32 {padding-left: 0pt;counter-reset: w2 1; }
 #l32> li:before {counter-increment: w2; content: "("counter(w2, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l32> li:first-child:before {counter-increment: w2 0;  }
 #l33 {padding-left: 0pt; }
 #l33> li:before {content: "■ "; color: #E48EBF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l34 {padding-left: 0pt; }
 #l34> li:before {content: "■ "; color: #64AF54; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12.5pt; }
 #l35 {padding-left: 0pt; }
 #l35> li:before {content: "■ "; color: #EF8E36; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12.5pt; }
 #l36 {padding-left: 0pt; }
 #l36> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l37 {padding-left: 0pt; }
 #l37> li:before {content: "· "; color: #A05D77; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 li {display: block; }
 #l38 {padding-left: 0pt; }
 #l38> li:before {content: "■ "; color: #F27901; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 li {display: block; }
 #l39 {padding-left: 0pt; }
 #l39> li:before {content: "■ "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 li {display: block; }
 #l40 {padding-left: 0pt; }
 #l40> li:before {content: "■ "; color: #B83128; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 li {display: block; }
 #l41 {padding-left: 0pt; }
 #l41> li:before {content: "■ "; color: #DF8CBD; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: -4pt; }
 li {display: block; }
 #l42 {padding-left: 0pt; }
 #l42> li:before {content: "■ "; color: #B83128; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 li {display: block; }
 #l43 {padding-left: 0pt; }
 #l43> li:before {content: "■ "; color: #975DA7; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: -3pt; }
 li {display: block; }
 #l44 {padding-left: 0pt; }
 #l44> li:before {content: "• "; color: #7B7E75; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 li {display: block; }
 #l45 {padding-left: 0pt; }
 #l45> li:before {content: "• "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 li {display: block; }
 #l46 {padding-left: 0pt;counter-reset: j1 1; }
 #l46> li:before {counter-increment: j1; content: "("counter(j1, upper-latin)") "; color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 #l46> li:first-child:before {counter-increment: j1 0;  }
</style></head><body><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="30" alt="image" src="1-s2.0-S0092867421010618-main/Image_001.png"/></span></p><p class="s1" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">Resource</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s2" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Atlas of clinically distinct cell states and ecosystems across human solid tumors</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s3" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Graphical abstract</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><span><img width="386" height="386" alt="image" src="1-s2.0-S0092867421010618-main/Image_002.png"/></span></p><p class="s3" style="padding-top: 10pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Highlights</p><p class="s4" style="padding-top: 4pt;padding-left: 65pt;text-indent: -12pt;line-height: 121%;text-align: left;">d <span class="s5">EcoTyper enables large-scale profiling of cell states and multicellular ecosystems</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s4" style="padding-left: 65pt;text-indent: -12pt;line-height: 121%;text-align: left;">d <span class="s5">Applicable to bulk, single-cell, and spatially resolved gene expression data</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s4" style="padding-left: 65pt;text-indent: -12pt;line-height: 121%;text-align: left;">d <span class="s5">A reference atlas of 69 cell states and 10 ecosystems across 16 types of carcinoma</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s4" style="padding-left: 65pt;text-indent: -12pt;line-height: 121%;text-align: left;">d <span class="s5">Carcinoma ecosystems have distinct biology, clinical outcomes, and spatial topology</span></p><p class="s3" style="padding-top: 3pt;padding-left: 10pt;text-indent: 0pt;text-align: left;">Authors</p><p class="s5" style="padding-top: 3pt;padding-left: 10pt;text-indent: 0pt;line-height: 121%;text-align: left;">Bogdan A. Luca, Chloe´ B. Steen, Magdalena Matusiak, ..., Matt van de Rijn, Andrew J. Gentles, Aaron M. Newman</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s5" style="padding-left: 10pt;text-indent: 0pt;line-height: 111%;text-align: left;"><a href="mailto:andrewg@stanford.edu" style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt;" target="_blank">Correspondence </a><a href="mailto:amnewman@stanford.edu" class="s6" target="_blank">andrewg@stanford.edu (A.J.G.), </a>amnewman@stanford.edu (A.M.N.)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s3" style="padding-left: 10pt;text-indent: 0pt;text-align: left;">In brief</p><p class="s5" style="padding-top: 4pt;padding-left: 10pt;text-indent: 0pt;line-height: 121%;text-align: left;">EcoTyper, a machine learning framework for identifying and characterizing cell states and ecosystems from gene expression data, yields insights into the cellular landscape and community structure of human carcinoma, the leading cause of cancer-related mortality.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 100pt;text-indent: 0pt;line-height: 7pt;text-align: left;">Luca et al., 2021, Cell <i>184</i>, 5482–5496</p><h2 style="text-indent: 0pt;text-align: left;">ll</h2><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;line-height: 11pt;text-align: left;"><span><img width="21" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_003.png"/></span><span class="s8">             </span>October 14, 2021 <span class="s9">ª </span>2021 Elsevier Inc.</p><p style="padding-left: 100pt;text-indent: 0pt;line-height: 13pt;text-align: left;"><a href="https://doi.org/10.1016/j.cell.2021.09.014" class="s10" target="_blank">https://doi.org/10.1016/j.cell.2021.09.014 </a><span><img width="20" height="20" alt="image" src="1-s2.0-S0092867421010618-main/Image_004.png"/></span><a href="https://doi.org/10.1016/j.cell.2021.09.014" class="s11" target="_blank">    </a><span><img width="25" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_005.png"/></span><a href="https://doi.org/10.1016/j.cell.2021.09.014" class="s11" target="_blank">   </a><span><img width="49" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_006.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="48" height="24" alt="image" src="1-s2.0-S0092867421010618-main/Image_007.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_008.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="20" alt="image" src="1-s2.0-S0092867421010618-main/Image_009.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="25" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_010.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="49" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_011.png"/></span></p><p class="s12" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 15pt;text-align: left;"><a name="bookmark0">Resource</a><a name="bookmark1">&zwnj;</a></p><p class="s13" style="padding-left: 59pt;text-indent: 0pt;line-height: 21pt;text-align: left;">Atlas of clinically distinct cell states</p><p class="s13" style="padding-left: 59pt;text-indent: 0pt;line-height: 22pt;text-align: left;">and ecosystems across human solid tumors</p><p class="s16" style="padding-top: 11pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark89" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Bogdan A. Luca,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">1,4,9</span> <a href="#bookmark89" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Chloe´ B. Steen,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">2,3,4,9</span> <a href="#bookmark85" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Magdalena Matusiak,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">5</span> <a href="#bookmark81" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Armon Azizi,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">1</span> <a href="#bookmark85" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Sushama Varma,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">5 </span><a href="#bookmark85" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Chunfang Zhu,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">5</span> <a href="#bookmark85" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Joanna Przybyl,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">5</span> <a href="#bookmark81" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Almudena Espı´n-Pe´ rez,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">1</span> <a href="#bookmark87" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Maximilian Diehn,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">3,6,7</span> <a href="#bookmark88" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Ash A. Alizadeh,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">2,3,7,8</span> <a href="#bookmark85" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Matt van de Rijn,</a><a href="#bookmark85" class="s15">5</a></p><p class="s17" style="padding-left: 59pt;text-indent: 0pt;line-height: 10pt;text-align: left;"><a href="#bookmark87" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" name="bookmark81">Andrew J. Gentles,</a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; vertical-align: 2pt;">1,4,7,</span>* <a href="#bookmark90" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and Aaron M. Newman</a><a href="#bookmark91" class="s15">3,4,7,10,</a><a href="#bookmark91" class="s18">*</a></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">1<span class="s20">Stanford Center for Biomedical Informatics Research, Department of Medicine, Stanford University, Stanford, CA 94305, USA</span></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark82">2</a><span class="s20">Division of Oncology, Department of Medicine, Stanford University, Stanford, CA 94305, USA</span></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark83">3</a><span class="s20">Institute for Stem Cell Biology and Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark84">4</a><span class="s20">Department of Biomedical Data Science, Stanford University, Stanford, CA 94305, USA</span></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;line-height: 108%;text-align: left;"><a name="bookmark85">5</a><span class="s20">Department of Pathology, Stanford University, Stanford, CA 94305, USA </span>6<span class="s20">Department of Radiation Oncology, Stanford University, Stanford, CA 94305, USA </span>7<span class="s20">Stanford Cancer Institute, Stanford University, Stanford, CA 94305, USA</span><a name="bookmark86">&zwnj;</a><a name="bookmark87">&zwnj;</a></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark88">8</a><span class="s20">Division of Hematology, Department of Medicine, Stanford University, Stanford, CA 94305, USA</span></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark89">9</a><span class="s20">These authors contributed equally</span></p><p class="s19" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark90">10</a><span class="s20">Lead contact</span></p><p style="padding-left: 62pt;text-indent: -2pt;line-height: 108%;text-align: left;"><a href="mailto:andrewg@stanford.edu" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;" target="_blank" name="bookmark91">*Correspondence: </a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">andrewg@stanford.edu </span><a href="mailto:amnewman@stanford.edu" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;" target="_blank">(A.J.G.), </a><span style=" color: #0097CF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">amnewman@stanford.edu </span><a href="https://doi.org/10.1016/j.cell.2021.09.014" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;" target="_blank">(A.M.N.) </a><a href="https://doi.org/10.1016/j.cell.2021.09.014" class="s21" target="_blank">https://doi.org/10.1016/j.cell.2021.09.014</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">SUMMARY</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s23" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Determining how cells vary with their local signaling environment and organize into distinct cellular commu- nities is critical for understanding processes as diverse as development, aging, and cancer. Here we intro- duce EcoTyper, a machine learning framework for large-scale identification and validation of cell states and multicellular communities from bulk, single-cell, and spatially resolved gene expression data. When applied to 12 major cell lineages across 16 types of human carcinoma, EcoTyper identified 69 transcription- ally defined cell states. Most states were specific to neoplastic tissue, ubiquitous across tumor types, and significantly prognostic. By analyzing cell-state co-occurrence patterns, we discovered ten clinically distinct multicellular communities with unexpectedly strong conservation, including three with myeloid and stromal elements linked to adverse survival, one enriched in normal tissue, and two associated with early cancer development. This study elucidates fundamental units of cellular organization in human carcinoma and pro- vides a framework for large-scale profiling of cellular ecosystems in any tissue.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">INTRODUCTION</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark105" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In cancer, complex communities of interacting cell types form powerful signaling networks that shape tumorigenesis (</a><a href="#bookmark105" class="s10">Bin- </a>newies et al., 2018<a href="#bookmark130" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Jackson et al., 2020<a href="#bookmark134" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Keren et al., 2018<a href="#bookmark171" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Schu¨ rch et al., 2020<a href="#bookmark173" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Smith and Hodges, 2019<a href="#bookmark165" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Targeted strate- gies for manipulating these networks are transforming cancer care, as exemplified by the success of immune checkpoint inhibitors (ICIs) in multiple advanced solid cancers (</a><a href="#bookmark165" class="s10">Ribas and </a>Wolchok, 2018<a href="#bookmark122" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). However, response rates to current ICIs are het- erogeneous and most patients with solid tumors fail to achieve long-term clinical benefit (</a>Galon and Bruni, 2019<span style=" color: #000;">). A comprehen- sive understanding of tumor-associated cell states, their pat- terns of interaction, and their impact on clinical outcomes could facilitate new opportunities for disease management and thera- peutic intervention.</span></p><p style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark105" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Previous studies have revealed broad phenotypic classes in human tumors, ranging from tumors that are T cell inflamed (‘‘hot’’) to those that are T cell depleted (‘‘cold’’) (</a><a href="#bookmark105" class="s10">Binnewies</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;">et al., 2018<a href="#bookmark105" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Such classifications can inform disease character- istics, including response to ICI, but oversimplify the cell types and cellular states of the tumor microenvironment (TME). In recent years, single-cell genomics, spatial transcriptomics, and multiplexed imaging have emerged as powerful technologies for obtaining high-resolution portraits of tumor cellular ecosys- tems directly from primary tissue specimens (</a><a href="#bookmark105" class="s10">Binnewies et al., </a>2018<a href="#bookmark130" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Jackson et al., 2020<a href="#bookmark135" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Keren et al., 2019<a href="#bookmark171" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark171" class="s10">Schu¨ rch et al., </a>2020<a href="#bookmark173" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Smith and Hodges, 2019<span style=" color: #000;">). However, practical consider- ations have largely limited these assays to single tumor types, modestly sized sample cohorts, or small sets of phenotypic markers.</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Here, we present EcoTyper, a machine learning framework for large-scale delineation of cell states and multicellular commu- nities from bulk tissue specimens. Our approach combines statistical learning techniques with recent advances in gene expression deconvolution (</a>Newman et al., 2019<span style=" color: #000;">) to illuminate multicellular ecosystems from bulk, single-cell, and spatially resolved gene expression data. To demonstrate the utility of</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="21" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_012.png"/></span></p><p class="s20" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">5482 Cell <i>184</i>, 5482–5496, October 14, 2021 <span class="s26">ª </span>2021 Elsevier Inc.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="641" height="324" alt="image" src="1-s2.0-S0092867421010618-main/Image_013.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="144" height="153" alt="image" src="1-s2.0-S0092867421010618-main/Image_014.png"/></span></p><p class="s27" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark2">Figure 1. High-throughput characterization of tumor cell states and ecosystems</a><a name="bookmark3">&zwnj;</a><a name="bookmark92">&zwnj;</a></p><p class="s38" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Schematic depicting the EcoTyper framework and its application to 16 types of human carcinoma (TCGA discovery cohort, </a>Table S1<span style=" color: #000;">). In this study, EcoTyper was applied within a multi-phase workflow, consisting of purification of cell-type-specific gene expression profiles from bulk tissue transcriptomic data, identification of transcriptional states for each purified cell type, and determination of co-occurrence patterns between cell states that define multicellular communities, termed ecotypes. Once cell states and ecotypes are defined, they can be queried in external expression datasets, including bulk transcriptomes, scRNA-seq data, and spatial transcriptomic arrays, allowing validation and integrative characterization.</span></p><p class="s38" style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">See also </a>Figure S1<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;">this framework, we constructed a global atlas of transcriptionally distinct cell states from 16 types of human carcinoma. We then defined cell-state co-occurrence patterns across nearly 6,000 tumors, identifying ten multicellular communities with wide- spread representation. We characterized our findings at the sin- gle-cell level; verified them in independent bulk tissue samples; and investigated their associations with genomic features, over- all survival, and ICI response. Finally, we interrogated the spatial organization of multicellular communities, including two with proinflammatory properties. This work reveals fundamental units of cellular organization in human carcinoma, with implications for novel diagnostics and individualized therapies.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">RESULTS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">The EcoTyper framework</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We designed EcoTyper as a broadly applicable framework for high-throughput identification of cell states and multicellular communities from primary tissue specimens. It consists of three key steps: digital purification of cell-type-specific gene expres- sion profiles from bulk tissue transcriptomes, identification and quantitation of transcriptionally defined cell states, and co- assignment of cell states into multicellular communities (</a><a href="#bookmark92" class="s10">Figures </a>1 <a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S1<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>STAR Methods<span style=" color: #000;">).</span></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">EcoTyper starts by applying CIBERSORTx, a recently described approach for ‘‘digital cytometry,’’ to determine the abundance and gene expression profiles of individual cell types</p><p class="s24" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">within bulk tissue transcriptomes (</a>Newman et al., 2019<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). By imputing the composition of major cell types within a collection of related tissue specimens, CIBERSORTx can mathematically purify gene expression profiles for multiple cell types of interest without single-cell sequencing or physical cell isolation. Notably, if cell-type-specific expression profiles are available from another source (e.g., single-cell or bulk-sorted transcriptomes), this step can be omitted. Second, EcoTyper employs statistical learning algorithms, including variants of unsupervised and su- pervised non-negative matrix factorization (NMF), to identify cell-type-specific transcriptional programs (‘‘cell states’’), quan- tify their relative abundance in each sample, and recover them in external expression datasets. As part of this workflow, EcoTyper implements an analytical strategy, adaptive false positive index (AFI), to eliminate spurious states without prior knowledge while maximizing sensitivity (</a>STAR Methods<a href="#bookmark102" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Third, EcoTyper deter- mines co-association patterns between cell states that form multicellular communities. In this work, a multicellular commu- nity is broadly defined as a collection of cell states that co-occur across independent tissue samples, agnostic to spatial neigh- borhoods. Such states may participate in diverse modes of communication, including contact-dependent interactions and indirect interactions mediated through diffusible secreted mole- cules (</a>Armingol et al., 2021<span style=" color: #000;">). EcoTyper can then query cell states and communities across datasets and platforms, allowing for large-scale assessment of tissue composition. Applications of EcoTyper include phenotyping and biomarker discovery from</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="300" height="94" alt="image" src="1-s2.0-S0092867421010618-main/Image_015.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="293" height="391" alt="image" src="1-s2.0-S0092867421010618-main/Image_016.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="10" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_017.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_018.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="44" height="103" alt="image" src="1-s2.0-S0092867421010618-main/Image_019.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="64" height="48" alt="image" src="1-s2.0-S0092867421010618-main/Image_020.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="34" alt="image" src="1-s2.0-S0092867421010618-main/Image_021.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="218" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_022.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="298" height="79" alt="image" src="1-s2.0-S0092867421010618-main/Image_023.png"/></span></p><ol id="l1"><li style="padding-top: 5pt;padding-left: 312pt;text-indent: -247pt;text-align: left;"><h4 style="display: inline;"><a name="bookmark93">C</a></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="253" height="257" alt="image" src="1-s2.0-S0092867421010618-main/Image_024.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="29" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_025.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="34" alt="image" src="1-s2.0-S0092867421010618-main/Image_026.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="125" height="38" alt="image" src="1-s2.0-S0092867421010618-main/Image_027.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="115" height="13" alt="image" src="1-s2.0-S0092867421010618-main/Image_028.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="77" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_029.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="254" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_030.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="165" height="122" alt="image" src="1-s2.0-S0092867421010618-main/Image_031.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="209" height="40" alt="image" src="1-s2.0-S0092867421010618-main/Image_032.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_033.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_034.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_035.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_036.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_037.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_038.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_039.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_040.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_041.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="298" height="79" alt="image" src="1-s2.0-S0092867421010618-main/Image_042.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="305" height="78" alt="image" src="1-s2.0-S0092867421010618-main/Image_043.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="43" alt="image" src="1-s2.0-S0092867421010618-main/Image_044.png"/></span></p></li><li style="padding-top: 5pt;padding-left: 293pt;text-indent: -229pt;text-align: left;"><h4 style="display: inline;">D E</h4></li></ol><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="33" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_045.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="123" height="239" alt="image" src="1-s2.0-S0092867421010618-main/Image_046.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="179" height="169" alt="image" src="1-s2.0-S0092867421010618-main/Image_047.png"/></span></p><p class="s27" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Figure 2. The cell-state landscape across 16 carcinomas</p><ol id="l2"><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;"><p class="s28" style="display: inline;">Heatmaps showing digitally purified expression profiles of 12 cell types decoded from 16 bulk epithelial tumor types, with genes as rows and tumor/adjacent normal tissue samples as columns. Heatmaps are organized by the most abundant cell state per sample.</p></li><li style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;"><p class="s28" style="display: inline;">UMAP projection of cell-state heterogeneity across tumor and adjacent normal specimens in the discovery cohort. Points are colored by the most abundant cell state per sample, with states colored identically to (A) (gray denotes S9).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">(legend continued on next page)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a name="bookmark4">fresh, frozen, or fixed biospecimens; investigation of intercellular signaling networks through integration of known ligand-receptor pairs; and exploration of multicellular communities in spatial transcriptomics data.</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">Atlas of transcriptionally defined cell states in 16 carcinomas</p><p class="s24" style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark172" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To demonstrate the capabilities of EcoTyper, we used it to gain insights into human carcinoma, the leading cause of cancer deaths worldwide (</a>Siegel et al., 2020<a href="#bookmark157" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and a class of malig- nancies for which extensive genomic and clinical data are pub- licly available. As carcinomas originate from epithelial cells, we started by selecting 12 cell types that together span the majority of immunological and structural cells found in human epithelial tumors: B cells, plasma cells, CD8 T cells, CD4 T cells, natural killer (NK) cells, monocytes/macrophages, dendritic cells, mast cells, neutrophils, fibroblasts, endothelial cells, and epithelial cells. We then assembled a collection of cell-type-specific gene expression signatures to discriminate each cell type using CIBERSORTx. For this purpose, we took advantage of previ- ously published gene expression signatures (</a><a href="#bookmark157" class="s10">Newman et al., </a>2015<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, </a>2019<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), each with extensive validation data supporting their analytical performance for deconvolving solid tumors, including carcinomas (</a>Figures S1<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and S1C; </a>STAR Methods<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We next explored the impact of key parameters on EcoTyper deconvolution, including cell-state abundance and collinearity of cell-state fractions (</a>Figure S1<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D; </a>STAR Methods<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). We applied EcoTyper to 500 simulated tumors constructed of admixtures of single-cell RNA sequencing (scRNA-seq) data from individ- uals with non-small cell lung cancer (NSCLC) (</a>Figure S1<span style=" color: #000;">E). Eco- Typer showed strong deconvolution performance on 26 known cell states across a range of input fractions (mean of 0.5%– 10%), both for cell-state identification (sensitivity = 0.92, positive predictive value = 0.86) and enumeration (median Pearson </span><span class="s7">r </span><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">= 0.93 between known and predicted proportions; </a><a href="#bookmark203" class="s10">Figures </a>S1<span style=" color: #000;">F–S1H). Moreover, it achieved a limit of detection of</span></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;">~0.5% mean fractional abundance while also demonstrating the capability to resolve states with correlated abundance pat- terns (<i>r </i><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">% 0.6; </a><span style=" color: #0097CF;">Figures S1</span>I and S1J).</p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark179" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Next, we compiled a discovery cohort consisting of 16 types of human carcinoma spanning 5,946 tumor and 529 adjacent normal transcriptomes profiled by the Cancer Genome Atlas (TCGA) (</a>Tatlow and Piccolo, 2016<a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Figure 1<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Tables S1 <a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S2<span style=" color: #000;">). These datasets were selected to maximize the consistency of specimen handling and processing, the accuracy of imputed cell fractions against orthogonal measures, the uniformity of expression levels across housekeeping genes, and the availabil-</span></p><p class="s24" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">ity of both genomic data and clinical follow-up for each bio- specimen (</a>Figures S1<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C, S1K, and S1L; </a>Table S2<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Applied to these data, which were uniformly processed and standardized, EcoTyper produced a matrix of 77,700 digitally purified expres- sion profiles, one for each evaluated cell type and patient sample (i.e., 12 cell types 3 6,475 samples) (</a>STAR Methods<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">The size and scope of this expression matrix provided an op- portunity to identify and validate tumor-associated cell states that are shared across cancers. First, we confirmed that all profiles showed strong evidence of cell-type specificity by com- parison to reference profiles derived from scRNA-seq data (</a><a href="#bookmark204" class="s10">Fig- </a>ure S2<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A). Next, we applied EcoTyper to model each digitally purified sample as a linear combination of discrete transcrip- tional programs (</a>Figure S2<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B). In this way,  purified samples were treated as bulk-sorted populations, allowing multiple tran- scriptional states per cell type to coexist per sample (e.g., </a><a href="#bookmark203" class="s10">Fig- </a>ures S1<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">G and S1H; </a>STAR Methods<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">After initial quality-control filtering (</a>STAR Methods<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), EcoTyper yielded 71 discrete cell states, ranging from 3 to 9 states per cell type (</a>Figures 2<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A, 2B, and </a>S2<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B–S2E). Most states were ubiqui- tous across carcinomas and significantly enriched in malignant tissue, highlighting key commonalities independent of tumor site (</a>Figure S2<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F; </a>Table S3<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Nevertheless, many states also var- ied in their histological or clinical distribution. For example, mul- tiple transcriptional programs distinguished neoplastic from adjacent normal tissues or adenocarcinomas from squamous cell carcinomas (</a>Figure S2<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F; </a>Table S3<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). We also observed funda- mental differences with respect to cell lineage and tumor type: epithelial states showed the strongest specificity for particular tumor types, followed by fibroblasts, endothelial cells, myeloid cells, and lymphocytes (</a>Figures S2<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">G and S2H; </a>Table S3<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">EcoTyper implements a supervised framework for reference- guided annotation, in which cell states learned in one dataset can be identified and statistically evaluated in another (</a><a href="#bookmark197" class="s10">STAR </a>Methods<a href="#bookmark103" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To assess the fidelity of the 71 cell states defined by EcoTyper, we queried the presence of each state in ~200,000 single-cell transcriptomes covering four types of human carci- noma: breast cancer (BRCA) (</a>Azizi et al., 2018<a href="#bookmark141" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), colorectal cancer (CRC) (</a>Lee et al., 2020<a href="#bookmark161" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), head and neck squamous cell carcinoma (HNSCC) (</a>Puram et al., 2017<a href="#bookmark129" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), and NSCLC (</a><a href="#bookmark129" class="s10">Guo </a>et al., 2018<a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Lambrechts et al., 2018<a href="#bookmark140" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Laughney et al., 2020<a href="#bookmark196" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark196" class="s10">Zilio- </a>nis et al., 2019<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Table S1<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). In all, 94% of cell states (67 of 71) were significantly recoverable in scRNA-seq data using reference- guided annotation coupled with permutation testing (</a>Table S3<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>STAR Methods<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). The recovery rate remained high regardless of platform, cell type, or dataset, underscoring the robustness of our results (</a>Figures S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A–S3C; </a>Table S3<span style=" color: #000;">). Moreover, we observed</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="657" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_048.png"/></span></p></li><li style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Expression of cell-state-specific marker genes (rows) across seven scRNA-seq datasets (columns) spanning four types of carcinoma (</a>Tables S1 <a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">and </a>S4<span style=" color: #000;">). Asterisks indicate cell states omitted from further analysis that were not distinguishable from potential doublets in scRNA-seq data.</span></p></li><li style="padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Enrichment of EcoTyper states in adjacent normal tissue, comparing the discovery cohort to an scRNA-seq tumor atlas (</a><a href="#bookmark138" class="a">Lambrechts et al., </a>2018<span style=" color: #000;">). In both cases, tumor and adjacent normal tissues from NSCLC were analyzed. Concordance was determined as the fraction of states with significant normal enrichment in both datasets, with significance determined by Fisher’s exact test.</span></p></li><li style="padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Top: H&amp;E staining of colorectal cancer (CRC) specimens with high (arrows, left) versus low (right) levels of foam cell macrophages. Bottom: Analysis of monocyte/macrophage marker genes (EcoTyper) in bulk RNA-seq profiles of laser micro-dissected stroma from CRC 393 and 380 (above) as well as another foam cell-depleted CRC tumor (CRC 406). Enrichment was calculated by pre-ranked gene set enrichment analysis applied to the log2 fold change of foam cell- high (n = 3) versus foam cell-low (n = 3) RNA-seq profiles (</a><a href="#bookmark98" class="a">Table </a>S1<span style=" color: #000;">). The scale bar (100 mm) is identical for both images.</span></p></li></ol><p class="s38" style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: justify;"><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">See also </a>Figures S2 <a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">and </a>S3<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" name="bookmark5">strikingly reproducible marker gene expression across all seven scRNA-seq tumor atlases, with a leave-one-out cross-validation rate of 90% (</a>Figure 2<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C; </a>Table S3<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Based on these assessments, we selected 69 of 71 states for further analysis, omitting 2 that mapped to potential doublets in scRNA-seq data (endothelial cells state 3, fibroblasts state 7; </a>Figure S2<span style=" color: #000;">B).</span></p><p class="s24" style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">As an alternative validation approach, we also tested whether states enriched in particular biological groupings (e.g., normal tissues) were recapitulated at the single-cell level. Indeed, after mapping single-cell transcriptomes to EcoTyper states, we observed significant concordance for states enriched in adja- cent normal tissues, adenocarcinomas, or squamous cell carci- nomas (</a>Figures 2<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D and </a>S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D; </a>Table S3<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Moreover, compared to other reference-guided annotation tools for supervised scRNA- seq classification, EcoTyper exhibited superior performance (</a>Figures S3<span style=" color: #000;">E–S3G).</span></p><p class="s24" style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We next annotated each state by comparison to known tran- scriptional programs, prominently expressed marker genes, and states defined by previous scRNA-seq studies (</a>Table S4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Approximately two-thirds of EcoTyper states were attributable to genes or phenotypes established in prior literature (</a><a href="#bookmark98" class="s10">Table </a>S4<span style=" color: #000;">). For example, without prior knowledge, EcoTyper identified </span><span class="s7">ANGPTL2</span><span style=" color: #000;">+/</span><span class="s7">NID2</span><a href="#bookmark132" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">+ tip-like endothelial cells implicated in tumor neovascularization (</a>Kadomatsu et al., 2014<a href="#bookmark194" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Zhao et al., 2018<a href="#bookmark161" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">); two fibroblast states previously described in HNSCC (</a><a href="#bookmark161" class="s10">Puram </a>et al., 2017<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (CAF1 and CAF2; </a>Figure 2<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A); an epithelial cell subset (state 3) with hallmarks of partial EMT (</a>Figures S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">H and S3I; </a>Table S4<span style=" color: #000;">); and canonical T cell subsets associated with pre- effector, exhaustion, and resting phenotypes (</span><span class="s7">CCR7</span><span style=" color: #000;">+, </span><span class="s7">LAG3</span><span style=" color: #000;">+, </span><span class="s7">KLF2</span><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">+, respectively; </a>Figure 2<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C; </a>Table S4<a href="#bookmark118" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). EcoTyper also re- vealed insights into cell types with poorly understood plasticity in cancer. For example, among cells of the monocyte/macro- phage lineage, which have emerging roles in cancer immuno- therapy (</a>Feng et al., 2019<span style=" color: #000;">), EcoTyper reconstructed nine </span><span class="s7">in vivo </span><span style=" color: #000;">phenotypes with broad representation, including states consis- tent with pro-inflammatory monocytes (</span><span class="s7">CCR2</span><span style=" color: #000;">+), classical M0 macrophages (</span><span class="s7">FABP4</span><span style=" color: #000;">+), and M1 macrophages (</span><span class="s7">CXCL9+</span><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a><a href="#bookmark93" class="s10">Fig- </a>ures 2<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C and </a>S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">J; </a>Table S4<span style=" color: #000;">). Four candidate subtypes of M2- like macrophages were also detectable (states 4–7), including states expressing known M2 marker genes such as </span><span class="s7">CD209 </span><span style=" color: #000;">and </span><span class="s7">CD163 </span><span style=" color: #000;">(state 4), </span><span class="s7">S1PR1 </span><span style=" color: #000;">(state 5), and </span><span class="s7">CHI3L2 </span><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(state 7) (</a><a href="#bookmark93" class="s10">Fig- </a>ure 2<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C; </a>Table S4<a href="#bookmark155" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Murray and Wynn, 2011<a href="#bookmark186" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Tong et al., 2020<a href="#bookmark189" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Weichand et al., 2017<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Importantly, nearly one-third of EcoTyper states appeared to be novel or not previously identified by scRNA-seq surveys of human carcinomas (</a>Table S4<span style=" color: #000;">). For example, among M2-like macrophages, we identified an </span><span class="s7">AEBP1</span><a href="#bookmark147" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">+ population (state 6) with marked similarity to foamy macrophages (</a><a href="#bookmark147" class="s10">Majdalawieh </a>et al., 2006<a href="#bookmark153" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), a lipid-laden phenotype frequently associated with atherosclerotic plaques (</a>Moore et al., 2013<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) but whose rele- vance across carcinomas is unclear (</a>Figure S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">J; </a>Table S4<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To corroborate this state, we performed bulk RNA-seq of stromal cells isolated from formalin-fixed paraffin-embedded human CRC tumor biopsies with high and low foamy macrophage con- tent based on H&amp;E staining (</a>Figure 2<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E, top; </a>STAR Methods<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Indeed, of nine monocyte/macrophage states identified by Eco- Typer, state 6 was uniquely enriched in foamy macrophage-rich stroma, supporting our result (</a>Figure 2<span style=" color: #000;">E, bottom).</span></p><p class="s24" style="padding-top: 3pt;padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Collectively, these analyses demonstrate the performance of EcoTyper and underscore its value for defining cell-type-specific transcriptional programs at scales that currently exceed the practical limitations of other technologies. Detailed descriptions of all identified states, including transcription factors, candidate surface markers, and scRNA-seq recovery statistics are pro- vided in the supplement (</a>Table S4<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">Global view of cell-state prognostic associations</p><p class="s24" style="padding-top: 1pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We and others have previously shown that cell-type-specific reference profiles derived from external sources, including bulk-sorted populations and scRNA-seq data, can predict can- cer clinical outcomes (</a>Gentles et al., 2015<a href="#bookmark142" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Li et al., 2016<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark158" class="s10">New- </a>man et al., 2019<a href="#bookmark168" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Rooney et al., 2015<span style=" color: #000;">). However, the prognostic impact of context-dependent cell states in human carcinoma is largely unknown. We therefore leveraged the unique output of EcoTyper to chart the prognostic landscape of 69 cell states across 15,008 tumors.</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the 16 epithelial cancer types surveyed in our discovery cohort, the majority of cell states (39 of 69) were significantly associated with overall survival (</a>Figure 3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A), and 49% (n = 34) were significant in multivariable analyses incorporating stage, age, and sex (</a>Table S5<a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Global survival associations dichoto- mized nearly all evaluated cell types into favorable and adverse states, highlighting their biological and clinical heterogeneity (</a>Figure 3<a href="#bookmark151" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A). For example, macrophage subsets annotated as M1 (state 3) and M2 (states 4–7) were associated with longer and shorter survival time, respectively, as found in prior studies (</a>Mehla and Singh, 2019<a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Figure 3<span style=" color: #000;">A). Surprisingly, among M2- like states, </span><span class="s7">AEBP1</span><a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">+ foamy macrophages were among the top five determinants of adverse survival, suggesting that foam cells could have widespread relevance as an immunotherapeutic target in cancer (</a>Figure 3<span style=" color: #000;">A). Other notable states associated with adverse risk included </span><span class="s7">CA9</span><span style=" color: #000;">+ fibroblasts (state 8) and </span><span class="s7">POSTN</span><a href="#bookmark119" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">+ fibroblasts (state 3), both of which have been implicated in tumor invasiveness (</a>Fiaschi et al., 2013<a href="#bookmark128" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark128" class="s10">Gonza´ lez-Gonza´ lez </a>and Alonso, 2018<a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), and pro-angiogenic tip-like endothelial cells (state 2) (</a>Figures 3<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A and </a>S4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S5<span style=" color: #000;">). Specific leukocyte pop- ulations dominated favorable outcomes across carcinomas, with leading states including </span><span class="s7">CCR7</span><span style=" color: #000;">+ naive/central memory CD4 T cells, </span><span class="s7">CD247</span><span style=" color: #000;">+ NK cells, </span><span class="s7">CD27</span><span style=" color: #000;">+ plasma cells, and </span><span class="s7">XCR1+ </span><a href="#bookmark170" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">cDC1-like dendritic cells, which are associated with CD8 T cell priming (</a>Sa´ nchez-Paulete et al., 2017<a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Figures 3<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A and </a>S4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S5<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To determine the generalizability of these results, we applied EcoTyper to quantitate all 69 cell states in an independent cohort of 9,062 epithelial tumor transcriptomes profiled by microarray (PRECOG), for which overall survival data are available (</a><a href="#bookmark125" class="s10">Gentles </a>et al., 2015<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Figure S4<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B). First, we confirmed that EcoTyper is robust to platform-specific variation between bulk RNA-seq and microarrays (</a>Figure S4<span style=" color: #000;">C). Next, we calculated state-specific survival associations in the PRECOG database, as measured by weighted </span><span class="s7">Z </span><span style=" color: #000;">scores. Remarkably, survival associations were highly concordant between TCGA and PRECOG (Pearson </span><span class="s7">r </span><a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">= 0.73, p = 1.2 3 10—12; </a>Figures 3<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and </a>S4<span style=" color: #000;">D), corroborating our findings and emphasizing the extensibility of EcoTyper to new datasets. We also observed high concordance for individual tu- mor types, such as colon, ovarian, and gastric cancers, for which</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="502" height="76" alt="image" src="1-s2.0-S0092867421010618-main/Image_049.png"/></span></p><h4 style="padding-top: 7pt;padding-left: 58pt;text-indent: 0pt;text-align: left;"><a name="bookmark6">A</a><a name="bookmark94">&zwnj;</a></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 116pt;text-indent: 0pt;text-align: left;"><span><img width="29" height="28" alt="image" src="1-s2.0-S0092867421010618-main/Image_050.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 171pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><span><img width="44" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_051.png"/></span> <span><img width="24" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_052.png"/></span> <span><img width="26" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_053.png"/></span> <span><img width="25" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_054.png"/></span> <span><img width="25" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_055.png"/></span> <span><img width="26" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_056.png"/></span> <span><img width="26" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_057.png"/></span> <span><img width="26" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_058.png"/></span> <span><img width="26" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_059.png"/></span> <span><img width="26" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_060.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="379" height="140" alt="image" src="1-s2.0-S0092867421010618-main/Image_061.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="54" height="238" alt="image" src="1-s2.0-S0092867421010618-main/Image_062.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="70" alt="image" src="1-s2.0-S0092867421010618-main/Image_063.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="54" height="238" alt="image" src="1-s2.0-S0092867421010618-main/Image_064.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="81" height="24" alt="image" src="1-s2.0-S0092867421010618-main/Image_065.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="207" height="27" alt="image" src="1-s2.0-S0092867421010618-main/Image_066.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="22" height="103" alt="image" src="1-s2.0-S0092867421010618-main/Image_067.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="608" height="145" alt="image" src="1-s2.0-S0092867421010618-main/Image_068.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="35" height="34" alt="image" src="1-s2.0-S0092867421010618-main/Image_069.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_070.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="355" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_071.png"/></span></p><h4 style="padding-left: 58pt;text-indent: 0pt;text-align: left;">B C</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="380" height="132" alt="image" src="1-s2.0-S0092867421010618-main/Image_072.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_073.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="46" height="67" alt="image" src="1-s2.0-S0092867421010618-main/Image_074.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="44" height="11" alt="image" src="1-s2.0-S0092867421010618-main/Image_075.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="17" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_076.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_077.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_078.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="28" height="13" alt="image" src="1-s2.0-S0092867421010618-main/Image_079.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="47" height="23" alt="image" src="1-s2.0-S0092867421010618-main/Image_080.png"/></span></p><p class="s27" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Figure 3. Cell-state-specific survival associations across 15,008 tumors</p><ol id="l3"><li style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Survival associations of 69 cell states in 5,946 tumors (discovery cohort), stratified by cell type and aggregated across malignancies. Marker genes for the most significant adverse and favorable states are indicated. See also </a>Figure S4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">A and </a><a href="#bookmark98" class="a">Table </a>S5<span style=" color: #000;">.</span></p></li><li style="padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">State-specific survival associations in the discovery cohort (TCGA) and an independent cohort of 9,062 epithelial tumor transcriptomes (PRECOG). Concordance and statistical significance were assessed by Pearson correlation (see also </a><a href="#bookmark206" class="a">Figure </a>S4<span style=" color: #000;">D).</span></p></li><li style="padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s28" style="display: inline;">Kaplan-Meier plots showing differences in overall survival between patients with high levels of M1-like macrophages (state 3) or M2 foamy-like macrophages (state 6) in three types of carcinoma. TCGA tumors were stratified by the median difference between M1 and M2 foamy-like macrophages; thresholds determined in TCGA were applied to PRECOG. Statistical significance was calculated by a two-sided log-rank test. HR, hazard ratio. 95% HR confidence intervals are shown in brackets.</p><p class="s38" style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: justify;"><a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">See also </a>Figure S4<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">M1 and M2 foamy-like macrophages predicted longer and shorter survival time, respectively (</a>Figure 3<span style=" color: #000;">C).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">Large-scale reconstruction of multicellular communities <i>in vivo</i></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">Tumors are complex ecosystems comprised of spatially and temporally linked cell states. To determine whether EcoTyper</p><p class="s24" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">can reconstruct multicellular ecosystems, we devised a data- driven approach for clustering cell states based on patterns of co-occurrence and mutual avoidance (</a>STAR Methods<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). By applying this approach to tumor samples in the discovery cohort (69 states 3 5,946 tumors), we identified ten strikingly cohesive cellular communities, which we termed ‘‘carcinoma ecotypes’’ (CEs) (</a>Figures 4<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A, 4B, </a>S5<span style=" color: #000;">A, and S5B). CEs ranged from 3 to 9</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="234" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_081.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="194" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_082.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_083.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_084.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_085.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_086.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="25" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_087.png"/></span></p><h4 style="padding-top: 7pt;padding-bottom: 3pt;padding-left: 65pt;text-indent: 0pt;text-align: left;"><a name="bookmark95">A B</a></h4><p class="s32" style="padding-left: 111pt;text-indent: 0pt;line-height: 10pt;text-align: left;"><span><img width="181" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_088.png"/></span><span class="s28">	</span><span><img width="18" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_089.png"/></span>	<span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_090.png"/></span>	<span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_091.png"/></span>	<span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_092.png"/></span>	<span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_093.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 64pt;text-indent: 0pt;text-align: left;"><span class="s33">	</span><span><img width="52" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_094.png"/></span> <span><img width="108" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_095.png"/></span> <span><img width="51" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_096.png"/></span> <span><img width="51" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_097.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="109" height="112" alt="image" src="1-s2.0-S0092867421010618-main/Image_098.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="374" height="216" alt="image" src="1-s2.0-S0092867421010618-main/Image_099.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="193" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_100.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="88" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_101.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="145" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_102.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="171" height="181" alt="image" src="1-s2.0-S0092867421010618-main/Image_103.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="41" height="50" alt="image" src="1-s2.0-S0092867421010618-main/Image_104.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="34" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_105.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_106.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_107.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_108.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_109.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_110.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_111.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_112.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_113.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_114.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="332" height="164" alt="image" src="1-s2.0-S0092867421010618-main/Image_115.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="33" alt="image" src="1-s2.0-S0092867421010618-main/Image_116.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_117.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_118.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_119.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_120.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_121.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="108" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_122.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_123.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="52" alt="image" src="1-s2.0-S0092867421010618-main/Image_124.png"/></span></p><h4 style="padding-top: 5pt;padding-left: 65pt;text-indent: 0pt;text-align: left;">C D E</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s27" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Figure 4. Large-scale reconstruction of multicellular communities <i>in vivo</i></p><ol id="l4"><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;"><p class="s38" style="display: inline;"><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Cell-state abundance profiles across 16 carcinomas, organized into ten carcinoma ecotypes (CEs). Only cell states and tumor samples assigned to CEs are shown (related to </a><a href="#bookmark207" class="a">Figures </a>S5<span style=" color: #000;">A and S5B). Tumor samples are ordered by the most abundant CE class per specimen.</span></p></li><li style="padding-left: 70pt;text-indent: -10pt;line-height: 8pt;text-align: left;"><p class="s38" style="display: inline;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">CE composition depicted as network diagrams. The width of each edge represents the Jaccard index across tumor samples (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">).</span></p></li></ol></li></ol><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">(C–E) Validation of CEs in scRNA-seq profiles. (C) Overview of the approach. (D) Heatmaps portraying co-occurrence relationships among cell-state abundance profiles, both in the discovery cohort (left) and in six scRNA-seq atlases spanning BRCA, CRC, HNSCC, and NSCLC (right; </a>Table S1<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Only tumor types matching those analyzed by scRNA-seq are shown. Cell-state fractions were analyzed to assess co-occurrence relationships. All states are grouped into predefined CEs (B), and only states assigned to CEs are shown (n = 58). Significantly recoverable CEs are indicated above the heatmaps (*p &lt; 0.05). ‘‘Co-occurrence index’’ is a measure of covariance that accounts for noise (</a>STAR Methods<span style=" color: #000;">). (E) Composition of selected CEs in a subset of samples profiled by scRNA-seq for which each CE is highest. Cell types within each CE are distinguished by color; cell states can be distinguished by matching each CE and cell type with the corresponding node in (B).</span></p><p class="s38" style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: justify;"><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">See also </a>Figure S5<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">distinct cell states per community (</a>Figures 4<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A and 4B), were robustly recovered independent of clustering approach (</a>Figure S5<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C), largely ubiquitous across human carcinomas (</a><a href="#bookmark95" class="s10">Fig- </a>ure 4<a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A), and highly distinct from recently described immunolog- ical subtypes in TCGA (</a>Thorsson et al., 2018<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Figure S5<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D). More- over, by aggregating across cell-state abundance profiles, CE composition could be assessed in a continuous manner. While nearly every tumor sample had a dominant CE (</a>Figure 4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A), most tumors were comprised of multiple CEs, highlighting modularity in neoplastic tissue composition (</a>Table S6<span style=" color: #000;">).</span></p><p style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;">To authenticate these results, we performed three technical experiments. First, we tested whether CEs are reproducible across platforms and independent datasets. Indeed, in 395 human carcinoma specimens profiled by both RNA-seq and microarrays, estimated CE levels were significantly correlated</p><p class="s24" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">between platforms (</a>Figure S5<a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E). Moreover, by performing dimensionality reduction with UMAP on cell-state abundance profiles, we observed nearly identical community structure in &gt; 6,000 held-out epithelial tumors (</a>Gentles et al., 2015<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a><a href="#bookmark207" class="s10">Fig- </a>ure S5<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F). Second, we tested whether CEs are enriched for cell states with potential for interaction, both within and across eco- types. When compared to background expectations, 60% of CEs were significantly enriched in ligand-receptor pairs, with multiple CE pairs also showing evidence of inter-ecotype inter- action (</a>Figure S5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">G; </a>Table S6<span style=" color: #000;">).</span></p><p style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;">Given these results, we next asked whether the ten CEs are detectable in single-cell data. Using the scRNA-seq compen- dium described above, which includes ~200,000 single-cell transcriptomes encompassing 76 tumor and 21 adjacent normal specimens from four carcinomas, we assigned individual cells to</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" name="bookmark7">EcoTyper states (</a>Figure 4<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C). We then determined the fractional abundance of each state within each tumor/normal sample and subsequently grouped cell states into the same CE classes defined by EcoTyper (</a>Figures 4<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C–4E). Finally, we determined whether states assigned to the same CE are more strongly co- associated than expected by random chance (</a>Figure 4<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C). In all, 80% of CEs were significantly detectable in scRNA-seq data at p &lt; 0.05. Moreover, 90% were detectable at p &lt; 0.06 (</a><a href="#bookmark95" class="s10">Fig- </a>ure 4<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D). This result was striking given potential confounding fac- tors in scRNA-seq data that could obscure CE detection, including modest sample sizes, low cell numbers per sample, sparsity in gene expression, and dissociation-induced distor- tions (</a>Figure S5<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">H). As an alternative approach, we determined the joint probability of obtaining ten CEs with equally strong co-associations by random chance. Relative to background expectations, the probability of obtaining our original result by random chance was less than 1 in 1,000,000 (p &lt; 10—6; </a>Figure S5<span style=" color: #000;">I).</span><a name="bookmark8">&zwnj;</a><a name="bookmark9">&zwnj;</a></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">Taken together, these data validate our approach, identify distinct multicellular communities in bulk and single-cell expres- sion data, and nominate CEs as fundamental units of cellular organization across human carcinomas.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">Carcinoma ecotype characteristics in 6,000 normal and neoplastic tissue specimens</p><p class="s24" style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Having identified ten dominant multicellular ecosystems in carci- noma, we next explored their cellular, genomic, and clinical char- acteristics (</a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S6<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Across the discovery cohort, eight CEs were significantly prognostic in univariable models, and five remained significant after multivariable adjustments for stage, age, and sex (</a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S6<span style=" color: #000;">). CE1- and CE2- high tumors were lymphocyte deficient, strongly linked to higher risk of death, and broadly distinguished by elevated levels of </span><span class="s7">POSTN</span><a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">+ fibroblasts and basal-like epithelial cells, respectively (</a>Figures 4<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and </a>5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A). CE3-high tumors, predictive of worse sur- vival outcome, were myeloid enriched, microsatellite instability (MSI) high (</a>Table S6<a href="#bookmark111" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), and associated with COSMIC mutational process 17, a signature found in multiple tumor types including esophageal and gastric cancers, where it has been linked, at least in part, to gastric reflux (</a>Christensen et al., 2019<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). CE4- high tumors were associated with myogenesis and males over 60 years of age (in part owing to their higher prevalence in HNSCC and prostate cancer), whereas CE5- through CE8-high tumors were enriched for smoking-related mutations, normal tissue, age-related mutations, and moderately favorable out- comes, respectively. Finally, CE9- and CE10-high tumors were proinflammatory (i.e., leukocyte rich), strongly associated with longer overall survival, and characterized by higher immunore- activity, including IFN-g signaling, and higher B cell content, respectively. Notably, two CEs were present at similar fre- quencies in tumor and adjacent normal tissues but depleted in healthy tissues (CE4, CE10), reflecting a potential field effect. Others, with the exception of CE6, were largely specific to neoplastic tissue (</a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B). Additional CE-specific features, including expressed ligand/receptor pairs and signaling path- ways, are provided in the supplemental information and can be interactively explored online (</a>Table S6<a href="https://ecotyper.stanford.edu/" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" target="_blank">; </a><a href="https://ecotyper.stanford.edu/" class="s10" target="_blank">https://ecotyper. </a>stanford.edu<span style=" color: #000;">).</span></p><p class="s22" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: left;">Multicellular prediction of overall survival and immunotherapy response</p><p class="s24" style="padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Since each carcinoma ecotype integrates contributions from multiple cell states, we reasoned that CE profiling might have the potential to improve clinical outcome prediction. To test this possibility, we first compared CEs against two molecular subtyping schemes for predicting overall survival in the discov- ery cohort: bulk NMF clustering and a collection of six pan-can- cer immune subtypes defined by TCGA (</a>Thorsson et al., 2018<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). The former was selected to assess the merit of EcoTyper decon- volution while the latter was selected as a comparator for TME classification. While bulk NMF subtypes were largely limited to heterogeneity in epithelial cells (</a>Figure S5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">J), pan-immune sub- types showed relatively modest prognostic associations (</a><a href="#bookmark98" class="s10">Table </a>S6<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). In contrast, CEs showed distinct advantages in all compar- ative analyses that we performed, both in terms of CE-specific survival associations and in multivariable survival models adjusted for bulk NMF and pan-immune subtypes (</a>Figures S5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">K and S5L; </a>Table S6<span style=" color: #000;">).</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark145" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Next, we asked whether CEs can predict immunotherapy response. To this end, we compiled tumor expression data from 571 individuals with advanced metastatic disease prior to receiving immune checkpoint blockade with anti-PDL1 (urothe- lial carcinoma), anti-PD1 (melanoma), or anti-CTLA4 (melanoma) monotherapy (</a>Liu et al., 2019<a href="#bookmark148" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Mariathasan et al., 2018<a href="#bookmark156" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark156" class="s10">Nathan- </a>son et al., 2017<a href="#bookmark164" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Riaz et al., 2017<a href="#bookmark187" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Van Allen et al., 2015<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). We included metastatic melanoma in this analysis as most non- epithelial cell states reliably generalized to this disease (</a><a href="#bookmark98" class="s10">Table </a>S6<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To quantify performance, we evaluated continuous associ- ations with overall survival and binary associations with immuno- therapy response. CE9, which is characterized by IFN-g signaling, outperformed other CEs for predicting superior out- comes across therapy types and outcome measures (</a>Figure 5<a href="#bookmark113" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C). We also compared CE profiling to 112 candidate biomarkers, including 69 cell states quantitated by EcoTyper, 25 parental populations enumerated by CIBERSORTx, tumor mutational burden (TMB), and 2 published bulk signatures of ICI response (</a>Cristescu et al., 2018<a href="#bookmark131" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Jerby-Arnon et al., 2018<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Surprisingly, CE9 abundance surpassed all other measures including those trained to predict ICI response (</a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C; </a>Table S6<span style=" color: #000;">). Together these data suggest that multicellular communities, even in the absence of optimization, can capture biological signal with supe- rior predictive value.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: left;">Spatiotemporal dynamics of proinflammatory communities</p><p class="s24" style="padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We next sought to determine whether carcinoma ecotypes show distinct patterns of spatial organization. To do so, we largely focused on CE9 and CE10, two proinflammatory communities with canonical T cell states and favorable overall survival but otherwise disparate genomic and cellular features (</a>Figure 5<span style=" color: #000;">). CE9-T cell states express activation and immunoregulatory genes, including markers of exhaustion, consistent with the as- sociation of CE9 with ICI response (e.g., </span><span class="s7">LAG3 </span><span style=" color: #000;">in CD8 T cell S3 and </span><span class="s7">CTLA4 </span><span style=" color: #000;">in CD4 T cell S1). In contrast, CE10-T cells express markers of naive and central memory cells (e.g., </span><span class="s7">CCR7</span><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a><a href="#bookmark97" class="s10">Fig- </a>ure 6<a href="#bookmark129" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A). Although such differences are well documented in tu- mor-associated T cells (</a>Guo et al., 2018<a href="#bookmark159" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Oh et al., 2020<a href="#bookmark195" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark195" class="s10">Zheng</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="212" height="23" alt="image" src="1-s2.0-S0092867421010618-main/Image_125.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="260" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_126.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="19" alt="image" src="1-s2.0-S0092867421010618-main/Image_127.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="284" height="500" alt="image" src="1-s2.0-S0092867421010618-main/Image_128.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="160" height="14" alt="image" src="1-s2.0-S0092867421010618-main/Image_129.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="20" height="79" alt="image" src="1-s2.0-S0092867421010618-main/Image_130.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="41" alt="image" src="1-s2.0-S0092867421010618-main/Image_131.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="85" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_132.png"/></span></p><h4 style="padding-top: 7pt;padding-left: 96pt;text-indent: 0pt;text-align: left;"><a name="bookmark96">A B</a></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 333pt;text-indent: 0pt;text-align: left;"><span><img width="10" height="95" alt="image" src="1-s2.0-S0092867421010618-main/Image_133.png"/></span> <span><img width="224" height="194" alt="image" src="1-s2.0-S0092867421010618-main/Image_134.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="158" height="23" alt="image" src="1-s2.0-S0092867421010618-main/Image_135.png"/></span></p><h4 style="padding-left: 70pt;text-indent: 0pt;text-align: center;">C</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 328pt;text-indent: 0pt;text-align: left;"><span><img width="246" height="227" alt="image" src="1-s2.0-S0092867421010618-main/Image_136.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 329pt;text-indent: 0pt;text-align: left;"><span><img width="248" height="31" alt="image" src="1-s2.0-S0092867421010618-main/Image_137.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s27" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Figure 5. Carcinoma ecotype characteristics and association with immunotherapy response</p><ol id="l5"><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Characteristics of carcinoma ecotypes in the discovery cohort. Top: CE-specific survival associations across 16 carcinomas, colored by favorable (blue) or adverse (red) survival (color scale identical to </a><a href="#bookmark206" class="a">Figure </a>S4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">A). Center: CIBERSORTx-inferred proportions of 12 major cell types (averaged and scaled), grouped by the most abundant CE per tumor. Bottom: Key features of each CE. Enrichment statistics were calculated by dividing tumors into classes for which the indicated CE is highest (</a><a href="#bookmark98" class="a">Table </a>S6<span style=" color: #000;">).</span></p></li><li style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s28" style="display: inline;">CE composition in normal tissues (GTEx), adjacent normal samples (discovery cohort), and primary tumor specimens (discovery cohort). Pan-carcinoma survival associations are also indicated. ns, not significant.</p></li><li style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Association of 122 features with overall survival and ICI response in 571 individuals with advanced melanoma (Mel.) or bladder cancer (BLCA). Results are ordered top to bottom by performance across therapies and outcome measures (</a><a href="#bookmark98" class="a">Table </a>S6<span style=" color: #000;">).</span></p></li></ol><p class="s38" style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: justify;"><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">See also </a>Figure S5<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">et al., 2017<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), their precise cellular communities have not been previously established. With EcoTyper, we found that CE9-T cells strongly co-occur with six cellular states, including ones resembling M1 macrophages, mature immunogenic dendritic cells, and activated B cells. Conversely, CE10-T cells co-occur with five cellular states, including those consistent with pro-in- flammatory monocytes, cDC1 dendritic cells, and naive/resting B cells (</a>Figures 4<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and </a>6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S4<span style=" color: #000;">). These results were</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">confirmed across seven scRNA-seq datasets via reference- guided annotation, reinforcing the notion that specific pheno- types preferentially co-occur as multicellular assemblies in the tumor microenvironment (</a>Figure 6<span style=" color: #000;">A).</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To check whether CE-specific phenotypes are spatially distinct, we first performed multicolor immunofluorescence (IF) staining for GZMB and GZMK (</a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and </a>S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S6<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), which respectively mark CE9 and CE10-T cells (</a>Figure 6<span style=" color: #000;">A). In</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="403" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_138.png"/></span></p><h4 style="padding-top: 7pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark97">A</a></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s5" style="padding-left: 64pt;text-indent: 0pt;text-align: left;"><span><img width="140" height="116" alt="image" src="1-s2.0-S0092867421010618-main/Image_139.png"/></span>	<span><img width="155" height="116" alt="image" src="1-s2.0-S0092867421010618-main/Image_140.png"/></span>	<span><img width="144" height="116" alt="image" src="1-s2.0-S0092867421010618-main/Image_141.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s5" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><span><img width="147" height="116" alt="image" src="1-s2.0-S0092867421010618-main/Image_142.png"/></span>	<span><img width="150" height="116" alt="image" src="1-s2.0-S0092867421010618-main/Image_143.png"/></span>	<span><img width="147" height="116" alt="image" src="1-s2.0-S0092867421010618-main/Image_144.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="273" height="256" alt="image" src="1-s2.0-S0092867421010618-main/Image_145.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="164" height="23" alt="image" src="1-s2.0-S0092867421010618-main/Image_146.png"/></span></p><ol id="l6"><li style="padding-left: 273pt;text-indent: -213pt;text-align: left;"><h4 style="display: inline;">D E</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 273pt;text-indent: 0pt;text-align: left;"><span><img width="214" height="205" alt="image" src="1-s2.0-S0092867421010618-main/Image_147.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="168" height="106" alt="image" src="1-s2.0-S0092867421010618-main/Image_148.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_149.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="168" height="106" alt="image" src="1-s2.0-S0092867421010618-main/Image_150.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="283" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_151.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="79" height="217" alt="image" src="1-s2.0-S0092867421010618-main/Image_152.png"/></span></p></li><li style="padding-left: 305pt;text-indent: -245pt;text-align: left;"><h4 style="display: inline;">F</h4></li></ol><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 2pt;text-indent: 0pt;text-align: left;"><span><img width="114" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_153.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 3pt;text-indent: 0pt;text-align: left;"><span><img width="103" height="179" alt="image" src="1-s2.0-S0092867421010618-main/Image_154.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 11pt;text-indent: 0pt;text-align: left;"><span><img width="103" height="32" alt="image" src="1-s2.0-S0092867421010618-main/Image_155.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="313" height="234" alt="image" src="1-s2.0-S0092867421010618-main/Image_156.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="105" height="26" alt="image" src="1-s2.0-S0092867421010618-main/Image_157.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="84" height="33" alt="image" src="1-s2.0-S0092867421010618-main/Image_158.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="136" height="89" alt="image" src="1-s2.0-S0092867421010618-main/Image_159.png"/></span></p><p class="s35" style="padding-left: 29pt;text-indent: 0pt;text-align: left;">(legend on next page)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: right;"><a href="#bookmark139" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" name="bookmark10">cancer, GZMB and GZMK have been observed to distinguish activated effector and transitional effector memory T cells, respectively (</a>Larkin et al., 2019<a href="#bookmark159" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Oh et al., 2020<a href="#bookmark195" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark195" class="s10">Zheng et al., </a>2017<a href="#bookmark126" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). We applied EcoTyper to 23 bulk tumor transcriptomes from patients with NSCLC (</a>Gentles et al., 2020<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and selected four specimens with divergent CE9 and CE10 composition. Mul- tiplexed staining of these specimens verified EcoTyper predic- tions (</a>Figure S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B; </a>Table S6<a href="#bookmark190" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Additionally, while GZMB+ T cells were localized to the tumor core, consistent with a link between chronic antigen stimulation and T cell exhaustion (</a><a href="#bookmark190" class="s10">Wherry and </a>Kurachi, 2015<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), GZMK+ T cells were largely excluded, instead localizing to the periphery (</a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B, top, and </a>S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A; </a>Table S6<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To extend our analysis beyond T cells, we further applied IF imaging to APOE+/CD68+ and CCR2+/CD68+ cells, which respectively mark CE9- and CE10-enriched states within the macrophage/monocyte lineage (</a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A and </a>S6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C). Using tumor specimens from the above analysis, we confirmed that relative cell-state abundances, as determined by IF, were concordant with those determined by EcoTyper (</a>Figure S6<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B). As observed for T cells, APOE+/CD68+ cells localized to the tu- mor core, whereas CCR2+/CD68+ cells localized to the periphery (</a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B, bottom, and </a>S6<span style=" color: #000;">C). These findings were consistent with EcoTyper deconvolution of a human breast carcinoma specimen profiled by </span><span class="s7">in situ </span><a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">spatially barcoded microarray data (10x Visium), regardless of whether we analyzed individual cell states (</a>Figure S6<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D) or the entire cellular community (</a>Figure 6<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C, top). Moreover, spatial differences between ecotypes were high- ly significant with regard to distance from tumor cells (</a>Figure 6<span style=" color: #000;">C, right), a finding that was extensible to a melanoma specimen</span></p><p class="s24" style="padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(</a>Figure 6<span style=" color: #000;">C, bottom).</span></p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To determine whether additional cell types and cancers show evidence of CE-specific colocalization patterns, we next explored cell-state co-associations in breast cancer, colorectal cancer, ovarian cancer, and melanoma samples profiled by spatial transcriptomics (</a>Figure 6<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D). We found that cell states generally colocalize in a CE-specific manner regardless of devel- opmental lineage or cancer type (</a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E and </a>S6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E). These patterns were also evident by visual inspection (</a>Figure S6<a href="#bookmark154" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D, left). To quantify CE spatial organization, we applied Moran’s I, a statistical measure of spatial autocorrelation (</a>Moran, 1950<span style=" color: #000;">). We found that some CEs were highly spatially aggregated (CE1, CE9, CE10) whereas others were more dispersed (CE4,</span></p><p class="s24" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">CE5, CE3) (</a>Figures S6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F and S6G). Nevertheless, nearly two- thirds of CEs were spatially enriched in CE-specific ligand-re- ceptor pairs identified in the discovery cohort (</a>Figure S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">H; </a><a href="#bookmark98" class="s10">Table </a>S6<span style=" color: #000;">), implying that most CEs, including CE9 and CE10, occur in spatially distinct cellular neighborhoods with enhanced interac- tion potential.</span></p><p class="s24" style="padding-left: 9pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Given these results, coupled with the observation that CE10 is generally further from the tumor core than CE9 and also present in adjacent normal tissue (</a>Figure 5<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B), we hypothesized that CE10 precedes CE9 during early tumor development. Consistent with this, we found that CE10 was more prevalent than CE9 during the earliest stages of squamous cell lung carcinogenesis, whereas in malignant tissue, CE9 was more prevalent than CE10 (</a><a href="#bookmark208" class="s10">Fig- </a>ure S6<a href="#bookmark180" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">I). Moreover, in precancerous lesions of lung squamous cell carcinoma collected from 33 subjects with known outcomes (</a>Teixeira et al., 2019<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), higher relative levels of CE10 were signifi- cantly associated with spontaneous regression, whereas higher relative levels of CE9 predicted progression to invasive cancer (area under the curve = 0.82; </a>Figure 6<span style=" color: #000;">F). Together these data further validate our approach, link CE dynamics to early lung cancer development, and provide a platform to systematically interrogate the diagnostic and therapeutic potential of tumor cellular ecosystems.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">DISCUSSION</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 9pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark175" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In this study, we describe EcoTyper, an integrated system for de- coding cell states and multicellular communities from gene expression data. EcoTyper is distinguished from related technol- ogies in several important ways: First, by imputing cellular het- erogeneity directly from RNA profiles of intact tissue biopsies, EcoTyper avoids distortions induced by physical cell isolation; does not require antibodies or preselection of phenotypic markers; and is applicable to fresh, frozen, and fixed specimens. Second, unlike previous deconvolution approaches, EcoTyper can accurately resolve transcriptional states from multiple cell types (&gt;10), assemble them into multicellular communities, quantify their relative composition, and query them across diverse expression datasets and platforms. Although EcoTyper was applied across 16 carcinomas in this work, it is generalizable to any tissue type and disease state for which suitable expres- sion data are available (</a>Steen et al., 2021<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="650" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_160.png"/></span></p><p class="s27" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Figure 6. Proinflammatory communities are spatially distinct and predictive of early lung cancer development</p><ol id="l7"><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;"><p class="s38" style="display: inline;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Heatmaps displaying differentially expressed genes between CE9 and CE10 in seven scRNA-seq tumor datasets (</a><a href="#bookmark98" class="a">Table </a>S1<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">), shown for cell types present in both CEs. For each dataset and cell state, mean scaled expression is shown (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">).</span></p></li><li style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Immunofluorescence imaging of CE9- and CE10-specific T cell states (DAPI, CD3, and GZMB or GZMK) and monocyte/macrophage states (DAPI, CD68, and APOE or CCR2) in NSCLC specimens (T29, T36) with paired bulk RNA-seq data (</a><a href="#bookmark208" class="a">Figures </a>S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">A and S6C; </a><a href="#bookmark98" class="a">Table </a>S6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). CE9 and CE10-specific marker genes are highlighted in (A). Images correspond to boxed regions in </a><a href="#bookmark208" class="a">Figures </a>S6<span style=" color: #000;">A and S6C. Scale bar of 20 mm is identical for all images. ‘‘Center’’ refers to the tumor core; ‘‘Edge’’ refers to the periphery of the tumor mass.</span></p></li><li style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;"><p class="s28" style="display: inline;">Left: Distribution of CE9 and CE10 in breast tumor and melanoma sections profiled by spatial transcriptomics. Tumor regions are demarcated by a dashed line. Right: Relative distance of CE9- and CE10-positive spots from tumor regions.</p></li><li style="padding-left: 70pt;text-indent: -10pt;line-height: 8pt;text-align: left;"><p class="s28" style="display: inline;">Schema for quantifying spatial colocalization of CE-specific cell states.</p></li><li style="padding-top: 1pt;padding-left: 69pt;text-indent: -9pt;text-align: left;"><p class="s38" style="display: inline;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Significance of cell-state colocalization within individual CEs, as measured across four tumor types (</a><a href="#bookmark98" class="a">Table </a>S1<span style=" color: #000;">).</span></p></li><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;"><p class="s38" style="display: inline;"><a href="#bookmark180" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Left: Schema illustrating clinical outcomes of 33 subjects for whom premalignant lung lesions were profiled by microarray (</a><a href="#bookmark180" class="a">Teixeira et al., </a>2019<span style=" color: #000;">) and assessed for CE9 and CE10 by EcoTyper. Right: Relative abundance of CE9 versus CE10 in premalignant lung lesions, stratified by clinical outcome. Group comparisons in</span></p></li></ol><p class="s28" style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: left;">(C) and (F) were performed using a two-sided unpaired Wilcoxon rank sum test. Data in (C) right, (E), and (F) are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.5 3 IQR of the box limits; IQR, interquartile range).</p><p class="s38" style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">See also </a>Figure S6<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 4pt;padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark130" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" name="bookmark11">While recent studies have revealed critical insights into tumor cellular communities using multiplexed imaging, these studies focused on single tumor types using a limited number of prede- fined phenotypic markers (</a>Jackson et al., 2020<a href="#bookmark134" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark134" class="s10">Keren et al., </a>2018<a href="#bookmark171" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Schu¨ rch et al., 2020<a href="#bookmark173" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Smith and Hodges, 2019<span style=" color: #000;">). By deploy- ing EcoTyper to analyze 16 types of human carcinoma spanning nearly 6,000 bulk tumor transcriptomes, we uncovered 69 tran- scriptionally defined cell states and ten previously unknown multicellular communities in a marker-agnostic manner. In doing so, we characterized multicellular communities at the transcrip- tional level across thousands of solid tumors, corroborated them in scRNA-seq data, and assessed their associations with ICI response and early cancer development. These data and asso- ciated analytical tools provide new opportunities for the develop- ment of diagnostic and therapeutic strategies that rely upon knowledge of tumor-associated cell states and their patterns of multicellular interaction.</span><a name="bookmark12">&zwnj;</a><a name="bookmark13">&zwnj;</a><a name="bookmark14">&zwnj;</a></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a name="bookmark98">In summary, we demonstrate how cell states and multicellular communities can be profiled from bulk tissue transcriptomes, recovered in expression datasets independent of platform, related to immunotherapy response, and tracked across space and developmental time. Our approach is accurate, is comple- mentary to existing single-cell assays, and has significant poten- tial for generating experimentally testable hypotheses. Given its unique capabilities, we anticipate that EcoTyper will prove useful for reconstructing cellular community structure at high resolution and massive scale in health and disease.</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Limitations of the study</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark169" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Despite the promise of EcoTyper, several challenges remain. For example, EcoTyper requires expression profiles that distinguish major cell types within a tissue type of interest, regardless of whether such profiles are used for expression deconvolution or directly for cell-state discovery. Given the rapid pace of single- cell sequencing efforts (e.g., Human Tumor Atlas Network [</a><a href="#bookmark169" class="s10">Rozen- </a>blatt-Rosen et al., 2020<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">]), this requirement is unlikely to be a major hurdle for most applications. Second, not all cell states are resolv- able by EcoTyper, either because they fall beneath the lower limit of detection, are not definable from the genes detected in cell- type-specific expression profiles, or exhibit nearly perfect covari- ance with other cell states (</a>Figures S1<span style=" color: #000;">D–S1J). Approaches to over- come these issues, such as hierarchical variants of NMF that can resolve nested cell states, are currently being investigated.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">STAR+METHODS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">Detailed methods are provided in the online version of this paper and include the following:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 65pt;text-indent: 0pt;text-align: left;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt;">d </a><a href="#bookmark197" class="s10">KEY RESOURCES TABLE</a></p><p style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;"><a href="#bookmark198" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt;">d </a><a href="#bookmark198" class="s10">RESOURCE AVAILABILITY</a></p><p class="s28" style="padding-top: 1pt;padding-left: 76pt;text-indent: 0pt;text-align: left;">B <span class="p">Lead contact</span></p><p class="s28" style="padding-top: 1pt;padding-left: 76pt;text-indent: 0pt;text-align: left;">B <span class="p">Materials availability</span></p><p class="s28" style="padding-top: 1pt;padding-left: 76pt;text-indent: 0pt;text-align: left;">B <span class="p">Data and code availability</span></p><p style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;"><a href="#bookmark199" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt;">d </a><a href="#bookmark199" class="s10">EXPERIMENTAL MODEL AND SUBJECT DETAILS</a></p><p class="s28" style="padding-top: 1pt;padding-left: 76pt;text-indent: 0pt;text-align: left;">B <span class="p">Human subjects</span></p><p style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;"><a href="#bookmark200" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt;">d </a><a href="#bookmark200" class="s10">METHOD DETAILS</a></p><p class="s28" style="padding-top: 1pt;padding-left: 76pt;text-indent: 0pt;text-align: left;">B <span class="p">EcoTyper framework</span></p><p class="s28" style="padding-top: 3pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Signature matrix validation</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Validation of expression purification</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Assessment of EcoTyper using simulated tumors</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">External datasets</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">UMAP visualization of cell states</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;line-height: 112%;text-align: left;">B <span class="p">Enrichment of cell states in known phenotypes </span>B <span class="p">Comparison of scRNA-seq annotation methods </span>B <span class="p">State-specific marker genes in scRNA-seq data </span>B <span class="p">Validation of scRNA-seq markers</span></p><p class="s28" style="padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Cell state representation by tumor type</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Enrichment of EMT genes in epithelial states</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Annotation of monocyte/macrophage states</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Laser capture microdissection and bulk RNA-seq</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Immunohistochemistry</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Survival analyses and bulk tumor subtyping</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Prediction of immunotherapy response</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">CE network visualization</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Ligand-receptor enrichment analysis</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">CE recovery in scRNA-seq data</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Feature analysis of carcinoma ecotypes</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">State-specific expression in CE9 and CE10</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Multicolor immunofluorescence imaging</span></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Analysis of spatial transcriptomics data</span></p><p style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;"><a href="#bookmark201" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt;">d  </a><a href="#bookmark201" class="s10">QUANTIFICATION AND STATISTICAL ANALYSIS</a></p><p style="padding-top: 1pt;padding-left: 21pt;text-indent: 0pt;text-align: left;"><a href="#bookmark202" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt;">d </a><a href="#bookmark202" class="s10">ADDITIONAL RESOURCES</a></p><p class="s28" style="padding-top: 1pt;padding-left: 33pt;text-indent: 0pt;text-align: left;">B <span class="p">Website</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s37" style="padding-left: 9pt;text-indent: 0pt;text-align: justify;">SUPPLEMENTAL INFORMATION</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s38" style="padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="https://doi.org/10.1016/j.cell.2021.09.014" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank">Supplemental information can be found online at </a><a href="https://doi.org/10.1016/j.cell.2021.09.014" class="a" target="_blank">https://doi.org/10.1016/j.cell. </a>2021.09.014<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s37" style="padding-left: 9pt;text-indent: 0pt;text-align: justify;">ACKNOWLEDGMENTS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s28" style="padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;">The authors would like to acknowledge A. Chaudhuri, E. Brown, G. Gulati, A. Usmani, and S. Sikandar for their critical feedback. This work was supported by the Fund for Cancer Informatics (A.J.G., M.D.), the American Association for Cancer Research (C.B.S., 19-40-12-STEE), the National Cancer Institute (A.M.N., R01CA255450 and R00CA187192; A.J.G., U54CA209971 and U24CA224309; A.A.A. and M.D., R01CA233975; M.V.D.R., R01CA229529),</p><p class="s28" style="padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;">the Bakewell Foundation (M.D., A.A.A.), the SDW/DT and Shanahan Family Foundations (A.A.A.), the Stinehart-Reed Foundation (A.A.A., A.M.N.), the Vir- ginia and D.K. Ludwig Fund for Cancer Research (A.A.A., A.M.N.)., the Stan- ford Bio-X Interdisciplinary Initiatives Seed Grants Program (IIP) (A.M.N.), and the Donald E. and Delia B. Baxter Foundation (A.M.N.).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s37" style="padding-left: 9pt;text-indent: 0pt;text-align: justify;">AUTHOR CONTRIBUTIONS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s28" style="padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;">B.A.L., C.B.S., A.J.G., and A.M.N. conceived of the study, developed strate- gies for related experiments, and wrote the paper. B.A.L. and C.B.S. devel- oped and implemented EcoTyper and analyzed the data with assistance from A.J.G. and A.M.N. M.M. designed and performed immunohistochem- istry experiments and related data analysis with assistance from J.P. and</p><p class="s28" style="padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;">M.V.D.R. M.M. designed and performed laser capture microdissection, li- brary preparation, and sequencing experiments with assistance from C.Z.</p><p class="s28" style="padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;">B.A.L. and A.A. implemented web infrastructure. A.E.P. assisted with data analysis. M.D. assisted in the collection and expression profiling of patient specimens. A.A.A. contributed to the conceptual development of EcoTyper and assisted with data interpretation. M.V.D.R. procured tissue specimens and assisted in data interpretation. All authors commented on the manuscript at all stages.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s37" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;"><a name="bookmark15">DECLARATION OF INTERESTS</a><a name="bookmark16">&zwnj;</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s28" style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;">M.D. reports research funding from Varian Medical Systems and Illumina; ownership interest in CiberMed and Foresight Diagnostics; patent filings related to cancer biomarkers; and paid consultancy from Roche, AstraZeneca, RefleXion and BioNTech. A.A.A. is a member of the <i>Cell </i>advisory board and reports research support from Bristol Meyers Squibb; ownership in- terest in CiberMed, FortySeven Inc., and Foresight Diagnostics; patent filings related to cancer biomarkers; and paid consultancy from Genentech, Roche, Chugai, Gilead, and Celgene. A.M.N. reports ownership interest in CiberMed and patent filings related to cancer biomarkers. B.A.L., C.B.S., A.A.A., A.J.G., and A.M.N. have filed patent application PCT/US2020/059196. The re- maining authors declare no potential conflicts of interest.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s28" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Received: November 19, 2020</p><p class="s28" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;">Revised: June 21, 2021</p><p class="s28" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;">Accepted: September 8, 2021</p><p class="s28" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;">Published: September 30, 2021</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s37" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">REFERENCES</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s38" style="padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref1" class="a" target="_blank" name="bookmark99">Abdelaal, T., Michielsen, L., Cats, D., Hoogduin, D., Mei, H., Reinders, M.J.T., and Mahfouz, A. (2019). A comparison of automatic cell identification methods for single-cell RNA sequencing data. Genome Biol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref1" class="s39" target="_blank">20</a>, 194<span style=" color: #000;">.</span><a name="bookmark100">&zwnj;</a></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref2" class="a" target="_blank" name="bookmark101">Aiello, N.M., Maddipati, R., Norgard, R.J., Balli, D., Li, J., Yuan, S., Yamazoe, T., Black, T., Sahmoud, A., Furth, E.E., et al. (2018). EMT subtype influences epithelial plasticity and mode of cell migration. Dev Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref2" class="s39" target="_blank">45</a>, 681–695.e4<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref3" class="a" target="_blank" name="bookmark102">Aran, D., Camarda, R., Odegaard, J., Paik, H., Oskotsky, B., Krings, G., Goga, A., Sirota, M., and Butte, A.J. (2017). Comprehensive analysis of normal adja- cent to tumor transcriptomes. Nat. Commun. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref3" class="s39" target="_blank">8</a>, 1077<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref4" class="a" target="_blank" name="bookmark103">Armingol, E., Officer, A., Harismendy, O., and Lewis, N.E. (2021). Deciphering cell-cell interactions and communication from gene expression. Nat. Rev. Genet. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref4" class="s39" target="_blank">22</a>, 71–88<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref5" class="a" target="_blank" name="bookmark104">Azizi, E., Carr, A.J., Plitas, G., Cornish, A.E., Konopacki, C., Prabhakaran, S., Nainys, J., Wu, K., Kiseliovas, V., Setty, M., et al. (2018). Single-Cell Map of Diverse Immune Phenotypes in the Breast Tumor Microenvironment. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref5" class="s39" target="_blank">174</a>, 1293–1308.36<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref6" class="a" target="_blank" name="bookmark105">Bifulco, C., Capone, M., Feng, Z., Madonna, G., Simeone, E., Curvietto, M., Mozzillo, N., Ciliberto, G., Botti, G., Fox, B.A., et al. (2014). MISIPI study: Mel- anoma ImmunoScore evaluation in patients treated with IPIlimumab. J. Transl. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref6" class="s39" target="_blank">12</a>, 11<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref7" class="a" target="_blank" name="bookmark106">Binnewies, M., Roberts, E.W., Kersten, K., Chan, V., Fearon, D.F., Merad, M., Coussens, L.M., Gabrilovich, D.I., Ostrand-Rosenberg, S., Hedrick, C.C., et al. (2018). Understanding the tumor immune microenvironment (TIME) for effec- tive therapy. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref7" class="s39" target="_blank">24</a>, 541–550<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref8" class="a" target="_blank" name="bookmark107">Blondel, V.D., Guillaume, J.-L., Lambiotte, R., and Lefebvre, E. (2008). Fast un- folding of communities in large networks. J. Stat. Mech. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref8" class="s39" target="_blank">2008</a>, P10008<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref9" class="a" target="_blank" name="bookmark108">Brunet, J.P., Tamayo, P., Golub, T.R., and Mesirov, J.P. (2004). Metagenes and molecular pattern discovery using matrix factorization. Proc. Natl. Acad. Sci. USA </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref9" class="s39" target="_blank">101</a>, 4164–4169<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref10" class="a" target="_blank" name="bookmark109">Butler, A., Hoffman, P., Smibert, P., Papalexi, E., and Satija, R. (2018). Inte- grating single-cell transcriptomic data across different conditions, technolo- gies, and species. Nat. Biotechnol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref10" class="s39" target="_blank">36</a>, 411–420<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref11" class="a" target="_blank" name="bookmark110">Carter, S.L., Cibulskis, K., Helman, E., McKenna, A., Shen, H., Zack, T., Laird, P.W., Onofrio, R.C., Winckler, W., Weir, B.A., et al. (2012). Absolute quantifica- tion of somatic DNA alterations in human cancer. Nat. Biotechnol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref11" class="s39" target="_blank">30</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref11" class="a" target="_blank">, </a>413–421<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref12" class="a" target="_blank" name="bookmark111">Christensen, A.P. (2018). NetworkToolbox: Methods and Measures for Brain, Cognitive, and Psychometric Network Analysis in R. R J. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref12" class="s39" target="_blank">10</a>, 422–439<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref13" class="a" target="_blank">Christensen, S., Van der Roest, B., Besselink, N., Janssen, R., Boymans, S., Martens, J.W.M., Yaspo, M.-L., Priestley, P., Kuijk, E., Cuppen, E., and Van Hoeck, A. (2019). 5-Fluorouracil treatment induces characteristic T&gt;G muta- tions in human cancer. Nat. Commun. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref13" class="s39" target="_blank">10</a>, 4571<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 4pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref14" class="a" target="_blank" name="bookmark112">Cortes, C., and Vapnik, V. (1995). Support-vector networks. Mach. Learn. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref14" class="s39" target="_blank">20</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref14" class="a" target="_blank">, </a>273–297<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref15" class="a" target="_blank" name="bookmark113">Cristescu, R., Mogg, R., Ayers, M., Albright, A., Murphy, E., Yearley, J., Sher, X., Liu, X.Q., Lu, H., Nebozhyn, M., et al. (2018). Pan-tumor genomic bio- markers for PD-1 checkpoint blockade-based immunotherapy. Science </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref15" class="s39" target="_blank">362</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref15" class="a" target="_blank">, </a>eaar3593<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref16" class="a" target="_blank" name="bookmark114">Csardi, G., and Nepusz, T. (2006). The igraph software package for complex </a>network research. InterJournal, 1695<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref17" class="a" target="_blank" name="bookmark115">Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref17" class="s39" target="_blank">29</a>, 15–21<span style=" color: #000;">.</span><a name="bookmark116">&zwnj;</a></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref18" class="a" target="_blank" name="bookmark117">Durinck, S., Spellman, P.T., Birney, E., and Huber, W. (2009). Mapping identi- fiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nat. Protoc. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref18" class="s39" target="_blank">4</a>, 1184–1191<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref19" class="a" target="_blank" name="bookmark118">Eisenberg, E., and Levanon, E.Y. (2013). Human housekeeping genes, revis- ited. Trends Genet. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref19" class="s39" target="_blank">29</a>, 569–574<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref20" class="a" target="_blank" name="bookmark119">Feng, M., Jiang, W., Kim, B.Y.S., Zhang, C.C., Fu, Y.-X., and Weissman, I.L. (2019). Phagocytosis checkpoints as new targets for cancer immunotherapy. Nat. Rev. Cancer </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref20" class="s39" target="_blank">19</a>, 568–586<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref21" class="a" target="_blank" name="bookmark120">Fiaschi, T., Giannoni, E., Taddei, M.L., Cirri, P., Marini, A., Pintus, G., Nativi, C., Richichi, B., Scozzafava, A., Carta, F., et al. (2013). Carbonic anhydrase IX from cancer-associated fibroblasts drives epithelial-mesenchymal transition in prostate carcinoma cells. Cell Cycle </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref21" class="s39" target="_blank">12</a>, 1791–1801<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref22" class="a" target="_blank" name="bookmark121">Foley, J.W., Zhu, C., Jolivet, P., Zhu, S.X., Lu, P., Meaney, M.J., and West, R.B. (2019). Gene expression profiling of single cells from archival tissue with laser- capture microdissection and Smart-3SEQ. Genome Res. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref22" class="s39" target="_blank">29</a>, 1816–1825<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref23" class="a" target="_blank" name="bookmark122">Fruchterman, T.M.J., and Reingold, E.M. (1991). Graph drawing by force- directed placement. Softw. Pract. Exper. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref23" class="s39" target="_blank">21</a>, 1129–1164<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref24" class="a" target="_blank" name="bookmark123">Galon, J., and Bruni, D. (2019). Approaches to treat immune hot, altered and cold tumours with combination immunotherapies. Nat. Rev. Drug Discov. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref24" class="s39" target="_blank">18</a>, 197–218<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref25" class="a" target="_blank" name="bookmark124">Galon, J., Costes, A., Sanchez-Cabo, F., Kirilovsky, A., Mlecnik, B., Lagorce- Page` s, C., Tosolini, M., Camus, M., Berger, A., Wind, P., et al. (2006). Type, density, and location of immune cells within human colorectal tumors predict clinical outcome. Science </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref25" class="s39" target="_blank">313</a>, 1960–1964<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref26" class="a" target="_blank" name="bookmark125">Gaujoux, R., and Seoighe, C. (2010). A flexible R package for nonnegative ma- trix factorization. BMC Bioinformatics </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref26" class="s39" target="_blank">11</a>, 367<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref27" class="a" target="_blank" name="bookmark126">Gentles, A.J., Newman, A.M., Liu, C.L., Bratman, S.V., Feng, W., Kim, D., Nair, V.S., Xu, Y., Khuong, A., Hoang, C.D., et al. (2015). The prognostic landscape of genes and infiltrating immune cells across human cancers. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref27" class="s39" target="_blank">21</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref27" class="a" target="_blank">, </a>938–945<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref28" class="a" target="_blank" name="bookmark127">Gentles, A.J., Hui, A.B.-Y., Feng, W., Azizi, A., Nair, R.V., Bouchard, G., Knowles, D.A., Yu, A., Jeong, Y., Bejnood, A., et al. (2020). A human lung tumor microenvironment interactome identifies clinically relevant cell-type cross- talk. Genome Biol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref28" class="s39" target="_blank">21</a>, 107<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref29" class="a" target="_blank" name="bookmark128">Gittleman, J.L., and Kot, M. (1990). Adaptation: statistics and a null model for estimating phylogenetic effects. Syst. Zool. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref29" class="s39" target="_blank">39</a>, 227–241<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref30" class="a" target="_blank" name="bookmark129">Gonza´ lez-Gonza´ lez, L., and Alonso, J. (2018). Periostin: A Matricellular Protein With Multiple Functions in Cancer Development and Progression. Front. On- col. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref30" class="s39" target="_blank">8</a>, 225<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref31" class="a" target="_blank" name="bookmark130">Guo, X., Zhang, Y., Zheng, L., Zheng, C., Song, J., Zhang, Q., Kang, B., Liu, Z., Jin, L., Xing, R., et al. (2018). Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref31" class="s39" target="_blank">24</a>, 978–985<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref32" class="a" target="_blank" name="bookmark131">Jackson, H.W., Fischer, J.R., Zanotelli, V.R.T., Ali, H.R., Mechera, R., Soysal, S.D., Moch, H., Muenst, S., Varga, Z., Weber, W.P., and Bodenmiller, B. (2020). The single-cell pathology landscape of breast cancer. Nature </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref32" class="s39" target="_blank">578</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref32" class="a" target="_blank">, </a>615–620<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref33" class="a" target="_blank">Jerby-Arnon, L., Shah, P., Cuoco, M.S., Rodman, C., Su, M.-J., Melms, J.C., Leeson, R., Kanodia, A., Mei, S., Lin, J.-R., et al. (2018). A Cancer Cell Program Promotes T Cell Exclusion and Resistance to Checkpoint Blockade. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref33" class="s39" target="_blank">175</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref33" class="a" target="_blank">, </a>984–997.e24<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s38" style="padding-top: 4pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref34" class="a" target="_blank" name="bookmark132">Kadomatsu, T., Endo, M., Miyata, K., and Oike, Y. (2014). Diverse roles of ANGPTL2 in physiology and pathophysiology. Trends Endocrinol. Metab. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref34" class="s39" target="_blank">25</a>, 245–254<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref35" class="a" target="_blank" name="bookmark133">Kaufman, L., and Rousseeuw, P.J. (1990). Partitioning Around Medoids (Pro- gram PAM). In Finding Groups in Data: An Introduction to Cluster Analysis </a>(John Wiley &amp; Sons), pp. 68–125<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref36" class="a" target="_blank" name="bookmark134">Keren, L., Bosse, M., Marquez, D., Angoshtari, R., Jain, S., Varma, S., Yang, S.R., Kurian, A., Van Valen, D., West, R., et al. (2018). A Structured Tumor-Im- mune Microenvironment in Triple Negative Breast Cancer Revealed by Multi- plexed Ion Beam Imaging. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref36" class="s39" target="_blank">174</a>, 1373–1387.19<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref37" class="a" target="_blank" name="bookmark135">Keren, L., Bosse, M., Thompson, S., Risom, T., Vijayaragavan, K., McCaffrey, E., Marquez, D., Angoshtari, R., Greenwald, N.F., Fienberg, H., et al. (2019). MIBI-TOF: A multiplexed imaging platform relates cellular phenotypes and tis- sue structure. Sci Adv </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref37" class="s39" target="_blank">5</a>, eaax5851<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref38" class="a" target="_blank" name="bookmark136">Kiselev, V.Y., Yiu, A., and Hemberg, M. (2018). scmap: projection of single-cell RNA-seq data across data sets. Nat. Methods </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref38" class="s39" target="_blank">15</a>, 359–362<span style=" color: #000;">.</span><a name="bookmark137">&zwnj;</a></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="https://doi.org/10.1101/060012" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank" name="bookmark138">Korotkevich, G., Sukhov, V., Budin, N., Shpak, B., Artyomov, M.N., and Ser- gushichev, A. (2021). Fast gene set enrichment analysis. bioRxiv. </a><a href="https://doi.org/10.1101/060012" class="a" target="_blank">https://doi. </a>org/10.1101/060012<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref40" class="a" target="_blank" name="bookmark139">Lambrechts, D., Wauters, E., Boeckx, B., Aibar, S., Nittner, D., Burton, O., Bas- sez, A., Decaluwe´ , H., Pircher, A., Van den Eynde, K., et al. (2018). Phenotype molding of stromal cells in the lung tumor microenvironment. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref40" class="s39" target="_blank">24</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref40" class="a" target="_blank">, </a>1277–1289<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref41" class="a" target="_blank" name="bookmark140">Larkin, J., Chiarion-Sileni, V., Gonzalez, R., Grob, J.-J., Rutkowski, P., Lao, C.D., Cowey, C.L., Schadendorf, D., Wagstaff, J., Dummer, R., et al. (2019). Five-Year Survival with Combined Nivolumab and Ipilimumab in Advanced Melanoma. N. Engl. J. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref41" class="s39" target="_blank">381</a>, 1535–1546<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref42" class="a" target="_blank" name="bookmark141">Laughney, A.M., Hu, J., Campbell, N.R., Bakhoum, S.F., Setty, M., Lavalle´ e, V.P., Xie, Y., Masilionis, I., Carr, A.J., Kottapalli, S., et al. (2020). Regenerative lineages and immune-mediated pruning in lung cancer metastasis. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref42" class="s39" target="_blank">26</a>, 259–269<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref43" class="a" target="_blank" name="bookmark142">Lee, H.-O., Hong, Y., Etlioglu, H.E., Cho, Y.B., Pomella, V., Van den Bosch, B., Vanhecke, J., Verbandt, S., Hong, H., Min, J.-W., et al. (2020). Lineage-depen- dent gene expression programs influence the immune landscape of colorectal cancer. Nat. Genet. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref43" class="s39" target="_blank">52</a>, 594–603<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref44" class="a" target="_blank" name="bookmark143">Li, B., Severson, E., Pignon, J.-C., Zhao, H., Li, T., Novak, J., Jiang, P., Shen, H., Aster, J.C., Rodig, S., et al. (2016). Comprehensive analyses of tumor im- munity: implications for cancer immunotherapy. Genome Biol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref44" class="s39" target="_blank">17</a>, 174<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref45" class="a" target="_blank" name="bookmark144">Lim, N., Tesar, S., Belmadani, M., Poirier-Morency, G., Mancarci, B.O., Sicher- man, J., Jacobson, M., Leong, J., Tan, P., and Pavlidis, P. (2021). Curation of over 10 000 transcriptomic studies to enable data reuse. Database </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref45" class="s39" target="_blank">2021</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref45" class="a" target="_blank">, </a>baab006<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref46" class="a" target="_blank" name="bookmark145">Lipta´ k, T. (1958). On the combination of independent tests. Magyar Tud. Akad. Mat. Kutato Int. Ko¨ zl. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref46" class="s39" target="_blank">3</a>, 171–196<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref47" class="a" target="_blank" name="bookmark146">Liu, D., Schilling, B., Liu, D., Sucker, A., Livingstone, E., Jerby-Arnon, L., Zim- mer, L., Gutzmer, R., Satzger, I., Loquai, C., et al. (2019). Integrative molecular and clinical modeling of clinical outcomes to PD1 blockade in patients with metastatic melanoma. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref47" class="s39" target="_blank">25</a>, 1916–1927<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref48" class="a" target="_blank" name="bookmark147">Maechler, M., Rousseeuw, P., Struyf, A., Hubert, M., and Hornik, K. (2012). </a>Cluster: cluster analysis basics and extensions (R package version 1), p. 56<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref49" class="a" target="_blank" name="bookmark148">Majdalawieh, A., Zhang, L., Fuki, I.V., Rader, D.J., and Ro, H.-S. (2006). Adipo- cyte enhancer-binding protein 1 is a potential novel atherogenic factor involved in macrophage cholesterol homeostasis and inflammation. Proc. Natl. Acad. Sci. USA </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref49" class="s39" target="_blank">103</a>, 2346–2351<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref50" class="a" target="_blank" name="bookmark149">Mariathasan, S., Turley, S.J., Nickles, D., Castiglioni, A., Yuen, K., Wang, Y., Kadel, E.E., III, Koeppen, H., Astarita, J.L., Cubas, R., et al. (2018). TGFb atten- uates tumour response to PD-L1 blockade by contributing to exclusion of T cells. Nature </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref50" class="s39" target="_blank">554</a>, 544–548<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref51" class="a" target="_blank">Mascaux, C., Angelova, M., Vasaturo, A., Beane, J., Hijazi, K., Anthoine, G., Buttard, B., Rothe, F., Willard-Gallo, K., Haller, A., et al. (2019). Immune evasion before tumour invasion in early lung squamous carcinogenesis. Nature </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref51" class="s39" target="_blank">571</a>, 570–575<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 4pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="https://arxiv.org/abs/1802.03426" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank" name="bookmark150">McInnes, L., Healy, J., and Melville, J. (2018). Umap: Uniform manifold approx- imation and projection for dimension reduction. arXiv, 1802.03426. </a><a href="https://arxiv.org/abs/1802.03426" class="a" target="_blank">https:// </a>arxiv.org/abs/1802.03426<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref53" class="a" target="_blank" name="bookmark151">Mehla, K., and Singh, P.K. (2019). Metabolic Regulation of Macrophage Polar- ization in Cancer. Trends Cancer </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref53" class="s39" target="_blank">5</a>, 822–834<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref54" class="a" target="_blank" name="bookmark152">Monaco, G., Lee, B., Xu, W., Mustafah, S., Hwang, Y.Y., Carre´ , C., Burdin, N., Visan, L., Ceccarelli, M., Poidinger, M., et al. (2019). RNA-Seq Signatures Normalized by mRNA Abundance Allow Absolute Deconvolution of Human Im- mune Cell Types. Cell Rep. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref54" class="s39" target="_blank">26</a>, 1627–1640.e7<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref55" class="a" target="_blank" name="bookmark153">Moore, K.J., Sheedy, F.J., and Fisher, E.A. (2013). Macrophages in atheroscle- rosis: a dynamic balance. Nat. Rev. Immunol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref55" class="s39" target="_blank">13</a>, 709–721<span style=" color: #000;">.</span></p><p style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref56" name="bookmark154">Moran, P.A.P. (1950). Notes on continuous stochastic phenomena. Biometrika</a></p><p class="s38" style="padding-top: 1pt;padding-left: 9pt;text-indent: 0pt;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref56" class="s39" target="_blank">37</a>, 17–23<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref57" class="a" target="_blank" name="bookmark155">Murray, P.J., and Wynn, T.A. (2011). Protective and pathogenic functions of macrophage subsets. Nat. Rev. Immunol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref57" class="s39" target="_blank">11</a>, 723–737<span style=" color: #000;">.</span><a name="bookmark156">&zwnj;</a></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref58" class="a" target="_blank" name="bookmark157">Nathanson, T., Ahuja, A., Rubinsteyn, A., Aksoy, B.A., Hellmann, M.D., Miao, D., Van Allen, E., Merghoub, T., Wolchok, J.D., Snyder, A., and Hammer- bacher, J. (2017). Somatic Mutations and Neoepitope Homology in Mela- nomas Treated with CTLA-4 Blockade. Cancer Immunol. Res. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref58" class="s39" target="_blank">5</a>, 84–91<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref59" class="a" target="_blank" name="bookmark158">Newman, A.M., Liu, C.L., Green, M.R., Gentles, A.J., Feng, W., Xu, Y., Hoang, C.D., Diehn, M., and Alizadeh, A.A. (2015). Robust enumeration of cell subsets from tissue expression profiles. Nat. Methods </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref59" class="s39" target="_blank">12</a>, 453–457<span style=" color: #000;">.</span></p><p style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref60">Newman, A.M., Steen, C.B., Liu, C.L., Gentles, A.J., Chaudhuri, A.A., Scherer,</a></p><p class="s38" style="padding-top: 1pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref60" class="a" target="_blank" name="bookmark159">F., Khodadoust, M.S., Esfahani, M.S., Luca, B.A., Steiner, D., et al. (2019). Determining cell type abundance and expression from bulk tissues with digital cytometry. Nat. Biotechnol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref60" class="s39" target="_blank">37</a>, 773–782<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref61" class="a" target="_blank" name="bookmark160">Oh, D.Y., Kwek, S.S., Raju, S.S., Li, T., McCarthy, E., Chow, E., Aran, D., Ilano, A., Pai, C.S., Rancan, C., et al. (2020). Intratumoral CD4+ T Cells Mediate Anti- tumor Cytotoxicity in Human Bladder Cancer. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref61" class="s39" target="_blank">181</a>, 1612–1625.e13<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref62" class="a" target="_blank" name="bookmark161">Paradis, E., and Schliep, K. (2019). ape 5.0: an environment for modern phylo- genetics and evolutionary analyses in R. Bioinformatics </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref62" class="s39" target="_blank">35</a>, 526–528<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref63" class="a" target="_blank" name="bookmark162">Puram, S.V., Tirosh, I., Parikh, A.S., Patel, A.P., Yizhak, K., Gillespie, S., Rodman, C., Luo, C.L., Mroz, E.A., Emerick, K.S., et al. (2017). Single-cell tran- scriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref63" class="s39" target="_blank">171</a>, 1611–1624.e24<span style=" color: #000;">.</span></p><p style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref64">Rahman, M., Jackson, L.K., Johnson, W.E., Li, D.Y., Bild, A.H., and Piccolo,</a></p><ol id="l8"><ol id="l9"><li style="padding-top: 1pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref64" class="a" target="_blank" name="bookmark163">(2015). Alternative preprocessing of RNA-Sequencing data in The Cancer Genome Atlas leads to improved analysis results. Bioinformatics </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref64" class="s39" target="_blank">31</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref64" class="a" target="_blank">,</a> 3666–3672<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref65" class="a" target="_blank" name="bookmark164">Ramilowski, J.A., Goldberg, T., Harshbarger, J., Kloppmann, E., Lizio, M., Sa- tagopam, V.P., Itoh, M., Kawaji, H., Carninci, P., Rost, B., and Forrest, A.R. (2015). A draft network of ligand-receptor-mediated multicellular signalling in human. Nat. Commun. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref65" class="s39" target="_blank">6</a>, 7866<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref66" class="a" target="_blank" name="bookmark165">Riaz, N., Havel, J.J., Makarov, V., Desrichard, A., Urba, W.J., Sims, J.S., Hodi, F.S., Martı´n-Algarra, S., Mandal, R., Sharfman, W.H., et al. (2017). Tumor and Microenvironment Evolution during Immunotherapy with Nivolumab. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref66" class="s39" target="_blank">171</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref66" class="a" target="_blank">, </a>934–949.e16<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref67" class="a" target="_blank" name="bookmark166">Ribas, A., and Wolchok, J.D. (2018). Cancer immunotherapy using checkpoint blockade. Science </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref67" class="s39" target="_blank">359</a>, 1350–1355<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref68" class="a" target="_blank" name="bookmark167">Risso, D., Schwartz, K., Sherlock, G., and Dudoit, S. (2011). GC-content normalization for RNA-Seq data. BMC Bioinformatics </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref68" class="s39" target="_blank">12</a>, 480<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref69" class="a" target="_blank" name="bookmark168">Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth, G.K. (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref69" class="s39" target="_blank">43</a>, e47<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref70" class="a" target="_blank" name="bookmark169">Rooney, M.S., Shukla, S.A., Wu, C.J., Getz, G., and Hacohen, N. (2015). Mo- lecular and genetic properties of tumors associated with local immune cyto- lytic activity. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref70" class="s39" target="_blank">160</a>, 48–61<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref71" class="a" target="_blank">Rozenblatt-Rosen, O., Regev, A., Oberdoerffer, P., Nawy, T., Hupalowska, A., Rood, J.E., Ashenberg, O., Cerami, E., Coffey, R.J., Demir, E., et al.; Human Tumor Atlas Network (2020). The Human Tumor Atlas Network: Charting Tu- mor Transitions across Space and Time at Single-Cell Resolution. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref71" class="s39" target="_blank">181</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref71" class="a" target="_blank">, </a>236–249<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s38" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref72" class="a" target="_blank" name="bookmark170">Sa´ nchez-Paulete, A.R., Teijeira, A., Cueto, F.J., Garasa, S., Pe´ rez-Gracia, J.L., Sa´ nchez-Arra´ ez, A., Sancho, D., and Melero, I. (2017). Antigen cross-presen- tation and T-cell cross-priming in cancer immunology and immunotherapy. Ann. Oncol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref72" class="s39" target="_blank">28 </a>(suppl_12), xii44–xii55<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref73" class="a" target="_blank" name="bookmark171">Schu¨ rch, C.M., Bhate, S.S., Barlow, G.L., Phillips, D.J., Noti, L., Zlobec, I., Chu, P., Black, S., Demeter, J., McIlwain, D.R., et al. (2020). Coordinated Cellular Neighborhoods Orchestrate Antitumoral Immunity at the Colorectal Cancer Invasive Front. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref73" class="s39" target="_blank">182</a>, 1341–1359.e19<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref74" class="a" target="_blank" name="bookmark172">Siegel, R.L., Miller, K.D., and Jemal, A. (2020). Cancer statistics, 2020. CA Cancer J. Clin. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref74" class="s39" target="_blank">70</a>, 7–30<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref75" class="a" target="_blank" name="bookmark173">Smith, E.A., and Hodges, H.C. (2019). The Spatial and Genomic Hierarchy of Tumor Ecosystems Revealed by Single-Cell Technologies. Trends Cancer </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref75" class="s39" target="_blank">5</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref75" class="a" target="_blank">, </a>411–425<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref76" class="a" target="_blank" name="bookmark174">Steen, C.B., Liu, C.L., Alizadeh, A.A., and Newman, A.M. (2020). Profiling Cell Type Abundance and Expression in Bulk Tissues with CIBERSORTx. In Stem Cell Transcriptional Networks: Methods and Protocols, B.L. Kidder, ed., </a>pp. 135–157, Springer<span style=" color: #000;">.</span></p><p class="s28" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a name="bookmark175">Steen, C.B., Luca, B.A., Esfahani, M.S., Azizi, A., Sworder, B.J., Nabet, B.Y., Kurtz, D.M., Liu, C.L., Khameneh, F., Advani, R.H., et al. (2021). The landscape of tumor cell states and ecosystems in diffuse large B cell lymphoma. Cancer Cell </a><i>39</i><a href="https://doi.org/10.1016/j.ccell.2021.08.011" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank">. Published online September 30, 2021. </a><a href="https://doi.org/10.1016/j.ccell.2021.08.011" target="_blank">https://doi.org/10.1016/j.ccell.</a></p><p class="s38" style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: justify;">2021.08.011<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref77" class="a" target="_blank" name="bookmark176">Stouffer, S.A., Suchman, E.A., Devinney, L.C., Star, S.A., and Williams, R.M., Jr. (1949). The American soldier: Adjustment during army life. (Studies in social psychology in World War II)</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref77" class="s39" target="_blank">Volume 1 </a>(Princeton University Press)<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref78" class="a" target="_blank" name="bookmark177">Stuart, T., Butler, A., Hoffman, P., Hafemeister, C., Papalexi, E., Mauck, W.M., 3rd, Hao, Y., Stoeckius, M., Smibert, P., and Satija, R. (2019). Comprehensive integration of single-cell data. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref78" class="s39" target="_blank">177</a>, 1888–1902.e21<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref79" class="a" target="_blank" name="bookmark178">Tan, Y., and Cahan, P. (2019). SingleCellNet: a computational tool to classify single cell RNA-Seq data across platforms and across species. Cell Systems </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref79" class="s39" target="_blank">9</a>, 207–213.e2<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref80" class="a" target="_blank" name="bookmark179">Tatlow, P.J., and Piccolo, S.R. (2016). A cloud-based workflow to quantify transcript-expression levels in public cancer compendia. Sci. Rep. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref80" class="s39" target="_blank">6</a>, 39259<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref81" class="a" target="_blank" name="bookmark180">Teixeira, V.H., Pipinikas, C.P., Pennycuick, A., Lee-Six, H., Chandrasekharan, D., Beane, J., Morris, T.J., Karpathakis, A., Feber, A., Breeze, C.E., et al. (2019). Deciphering the genomic, epigenomic, and transcriptomic landscapes of pre-invasive lung cancer lesions. Nat. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref81" class="s39" target="_blank">25</a>, 517–525<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref82" class="a" target="_blank" name="bookmark181">Therneau, T.M., and Grambsch, P.M. (2000). Modeling survival data: extend- </a>ing the Cox model (Springer), pp. 39–77<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref83" class="a" target="_blank" name="bookmark182">Thomas, A.C., Eijgelaar, W.J., Daemen, M.J.A.P., and Newby, A.C. (2015). Foam Cell Formation In Vivo Converts Macrophages to a Pro-Fibrotic Pheno- type. PLoS ONE </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref83" class="s39" target="_blank">10</a>, e0128163<span style=" color: #000;">.</span></p><p style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref84" name="bookmark183">Thorsson, V., Gibbs, D.L., Brown, S.D., Wolf, D., Bortone, D.S., Ou Yang,</a></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref84" class="a" target="_blank">T.-H., Porta-Pardo, E., Gao, G.F., Plaisier, C.L., Eddy, J.A., et al.; Cancer Genome Atlas Research Network (2018). The Immune Landscape of Cancer. Immunity </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref84" class="s39" target="_blank">48</a>, 812–830.e14<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 4pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref85" class="a" target="_blank" name="bookmark184">Thrane, K., Eriksson, H., Maaskola, J., Hansson, J., and Lundeberg, J. (2018). Spatially Resolved Transcriptomics Enables Dissection of Genetic Heteroge- neity in Stage III Cutaneous Malignant Melanoma. Cancer Res. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref85" class="s39" target="_blank">78</a>, 5970–5979<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref86" class="a" target="_blank" name="bookmark185">Tirosh, I., Izar, B., Prakadan, S.M., Wadsworth, M.H., 2nd, Treacy, D., Trom- betta, J.J., Rotem, A., Rodman, C., Lian, C., Murphy, G., et al. (2016). Dissect- ing the multicellular ecosystem of metastatic melanoma by single-cell RNA- seq. Science </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref86" class="s39" target="_blank">352</a>, 189–196<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref87" class="a" target="_blank" name="bookmark186">Tong, L., Li, J., Choi, J., Pant, A., Xia, Y., Jackson, C., Liu, P., Yi, L., Boussouf, E., Lim, M., and Yang, X. (2020). CLEC5A expressed on myeloid cells as a M2 biomarker relates to immunosuppression and decreased survival in patients with glioma. Cancer Gene Ther. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref87" class="s39" target="_blank">27</a>, 669–679<span style=" color: #000;">.</span><a name="bookmark187">&zwnj;</a></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref88" class="a" target="_blank" name="bookmark188">Van Allen, E.M., Miao, D., Schilling, B., Shukla, S.A., Blank, C., Zimmer, L., Sucker, A., Hillen, U., Foppen, M.H.G., Goldinger, S.M., et al. (2015). Genomic correlates of response to CTLA-4 blockade in metastatic melanoma. Science </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref88" class="s39" target="_blank">350</a>, 207–211<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref89" class="a" target="_blank" name="bookmark189">Venables, W., and Ripley, B. (2002). Modern applied statistics with S, Fourth </a>Edition (Springer)<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref90" class="a" target="_blank" name="bookmark190">Weichand, B., Popp, R., Dziumbla, S., Mora, J., Strack, E., Elwakeel, E., Frank, A.-C., Scholich, K., Pierre, S., Syed, S.N., et al. (2017). S1PR1 on tumor- associated macrophages promotes lymphangiogenesis and metastasis via NLRP3/IL-1b. J. Exp. Med. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref90" class="s39" target="_blank">214</a>, 2695–2713<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref91" class="a" target="_blank" name="bookmark191">Wherry, E.J., and Kurachi, M. (2015). Molecular and cellular insights into T cell exhaustion. Nat. Rev. Immunol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref91" class="s39" target="_blank">15</a>, 486–499<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref92" class="a" target="_blank" name="bookmark192">Whitlock, M.C. (2005). Combining probability from independent tests: the weighted Z-method is superior to Fisher’s approach. J. Evol. Biol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref92" class="s39" target="_blank">18</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref92" class="a" target="_blank">, </a>1368–1373<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref93" class="a" target="_blank" name="bookmark193">Yoshihara, K., Shahmoradgoli, M., Mart´ınez, E., Vegesna, R., Kim, H., Torres- Garcia, W., Trevin˜ o, V., Shen, H., Laird, P.W., Levine, D.A., et al. (2013). Infer- ring tumour purity and stromal and immune cell admixture from expression data. Nat. Commun. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref93" class="s39" target="_blank">4</a>, 2612<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref94" class="a" target="_blank" name="bookmark194">Zaykin, D.V. (2011). Optimally weighted Z-test is a powerful method for combining probabilities in meta-analysis. J. Evol. Biol. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref94" class="s39" target="_blank">24</a>, 1836–1841<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref95" class="a" target="_blank" name="bookmark195">Zhao, Q., Eichten, A., Parveen, A., Adler, C., Huang, Y., Wang, W., Ding, Y., Adler, A., Nevins, T., Ni, M., et al. (2018). Single-Cell Transcriptome Analyses Reveal Endothelial Cell Heterogeneity in Tumors and Changes following Anti- angiogenic Treatment. Cancer Res. </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref95" class="s39" target="_blank">78</a>, 2370–2382<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref96" class="a" target="_blank" name="bookmark196">Zheng, C., Zheng, L., Yoo, J.-K., Guo, H., Zhang, Y., Guo, X., Kang, B., Hu, R., Huang, J.Y., Zhang, Q., et al. (2017). Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing. Cell </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref96" class="s39" target="_blank">169</a>, 1342–1356.e16<span style=" color: #000;">.</span></p><p class="s38" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref97" class="a" target="_blank">Zilionis, R., Engblom, C., Pfirschke, C., Savova, V., Zemmour, D., Saatcioglu, H.D., Krishnan, I., Maroni, G., Meyerovitz, C.V., Kerwin, C.M., et al. (2019). Sin- gle-cell transcriptomics of human and mouse lung cancers reveals conserved myeloid populations across individuals and species. Immunity </a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref97" class="s39" target="_blank">50</a><a href="http://refhub.elsevier.com/S0092-8674(21)01061-8/sref97" class="a" target="_blank">, 1317– </a>1334.e10<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s40" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark17">STAR+METHODS</a><a name="bookmark18">&zwnj;</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark197">KEY RESOURCES TABLE</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="650" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_161.png"/></span></p><p class="s27" style="padding-top: 3pt;padding-bottom: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">REAGENT or RESOURCE SOURCE IDENTIFIER</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="650" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_162.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="18" alt="image" src="1-s2.0-S0092867421010618-main/Image_163.png"/></span></p><p class="s27" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Antibodies</p><p class="s41" style="padding-top: 4pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Mouse anti-CD3 Agilent Cat# M7254; RRID: AB_2631163</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="17" alt="image" src="1-s2.0-S0092867421010618-main/Image_164.png"/></span></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Rabbit anti-GZMK Millipore Sigma Cat# HPA063181; RRID: AB_2684955</p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Rabbit anti-GZMB Cell Signaling Technology Cat# 46890; RRID: AB_2799313</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="17" alt="image" src="1-s2.0-S0092867421010618-main/Image_165.png"/></span></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Mouse anti-APOE Abcam Cat# ab1906; RRID: AB_302668</p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Mouse anti-CCR2 Novus Cat# MAB150; RRID: AB_2247178</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="17" alt="image" src="1-s2.0-S0092867421010618-main/Image_166.png"/></span></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Rabbit anti-CD68 Cell Signaling Technology Cat# 76437; RRID: AB_2799882</p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Secondary anti-mouse AF647 Thermo Fisher Scientific Cat# A-21235; RRID: AB_2535804</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_167.png"/></span></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 156%;text-align: left;">Secondary anti-rabbit AF555 Thermo Fisher Scientific Cat# A-21428; RRID: AB_2535849 <span style=" color: #AB4D4C;">Biological samples</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="61" alt="image" src="1-s2.0-S0092867421010618-main/Image_168.png"/></span></p><p class="s41" style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: left;">4 mm and 7 mm full tissue sections of CRC tumor</p><p class="s41" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 115%;text-align: left;">biopsies (imaging and laser capture microdissection, respectively)</p><p class="s41" style="padding-left: 8pt;text-indent: 0pt;line-height: 8pt;text-align: left;">This work N/A</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="18" alt="image" src="1-s2.0-S0092867421010618-main/Image_169.png"/></span></p><p class="s41" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 156%;text-align: left;">4 mm full tissue sections of NSCLC tumor biopsies This work N/A<span style=" color: #AB4D4C;"> Critical commercial assays</span></p><p class="s41" style="padding-left: 53pt;text-indent: 0pt;line-height: 9pt;text-align: left;">Arcturus XT LCM System Thermo Fisher Scientific N/A</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="19" alt="image" src="1-s2.0-S0092867421010618-main/Image_170.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_171.png"/></span></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 156%;text-align: left;">NextSeq 500 High Output Kit v2.5 Illumina N/A <span style=" color: #AB4D4C;">Deposited data</span></p><p class="s41" style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: left;">CRC bulk RNA-seq This work GEO: GSE181817</p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="373" alt="image" src="1-s2.0-S0092867421010618-main/Image_172.png"/></span></p><p class="s27" style="padding-top: 4pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Datasets</p><p class="s43" style="padding-top: 4pt;padding-left: 53pt;text-indent: 0pt;line-height: 139%;text-align: left;"><a href="#bookmark126" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NSCLC bulk RNA-seq </a>Gentles et al. (2020) <a href="#bookmark179" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">GEO: GSE111907 Pre-processed bulk RNA-seq samples from TCGA </a><a href="https://osf.io/gqrz9/" class="s42" target="_blank">Tatlow and Piccolo (2016) https://osf.io/gqrz9/</a></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">PRECOG database </a><a href="https://precog.stanford.edu/" class="s42" target="_blank">Gentles et al. (2015) https://precog.stanford.edu/</a></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Additional PRECOG datasets This work </a>Table S1<span style=" color: #000;">B</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark103" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">BRCA scRNA-seq </a>Azizi et al. (2018) <span style=" color: #000;">GEO: GSE114727</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark141" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">CRC scRNA-seq </a>Lee et al. (2020) <span style=" color: #000;">GEO: GSE132465</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark161" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">HNSCC scRNA-seq </a>Puram et al. (2017) <span style=" color: #000;">GEO: GSE103322</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NSCLC1 scRNA-seq </a>Lambrechts et al. (2018) <span style=" color: #000;">ArrayExpress: E-MTAB-6149; E-MTAB-6653</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark196" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NSCLC2 scRNA-seq </a>Zilionis et al. (2019) <span style=" color: #000;">GEO: GSE127465</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark140" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NSCLC3 scRNA-seq </a>Laughney et al. (2020) <span style=" color: #000;">GEO: GSE123904</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark129" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NSCLC4 scRNA-seq </a>Guo et al. (2018) <span style=" color: #000;">GEO: GSE99254</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 139%;text-align: left;"><a href="#bookmark185" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Melanoma scRNA-seq </a>Tirosh et al. (2016) <a href="#bookmark101" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">GEO: GSE72056 GTEx samples for seven tissue types </a>Aran et al. (2017) <span style=" color: #000;">GEO: GSE86354</span></p><p class="s41" style="padding-left: 53pt;text-indent: 0pt;line-height: 115%;text-align: left;">TCGA count matrices used to guide the pre-processing of GTEx samples</p><p class="s41" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 115%;text-align: left;">Bulk RNA-seq of metastatic urothelial carcinoma (IMvigor210 phase II atezolizumab trial)</p><p class="s43" style="padding-left: 21pt;text-indent: 0pt;line-height: 9pt;text-align: left;">Rahman et al. (2015) <span style=" color: #000;">GEO: GSE62944</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s43" style="padding-top: 5pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">Mariathasan et al. (2018) <span style=" color: #000;">Imvigor210CoreBiologies R package</span></p><p class="s43" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark145" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Bulk RNA-seq of metastatic melanoma </a>Liu et al. (2019) <span style=" color: #000;">N/A</span></p><p class="s43" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 139%;text-align: left;"><a href="#bookmark156" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Bulk RNA-seq of metastatic melanoma </a><a href="https://github.com/hammerlab/melanoma-reanalysis" class="s42" target="_blank">Nathanson et al. (2017) </a>https://github.com/hammerlab/melanoma-reanalysis <a href="#bookmark164" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Bulk RNA-seq of metastatic melanoma </a><a href="https://github.com/riazn/bms038_analysis" class="s42" target="_blank">Riaz et al. (2017) https://github.com/riazn/bms038_analysis</a></p><p class="s43" style="padding-left: 53pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><a href="#bookmark187" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Bulk RNA-seq of metastatic melanoma </a>Van Allen et al. (2015) <span style=" color: #000;">N/A</span></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 115%;text-align: left;">Breast cancer Visium dataset (section 1, Space Ranger 1.1.0)</p><p style="padding-top: 3pt;padding-left: 142pt;text-indent: -116pt;line-height: 114%;text-align: left;"><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.1.0/V1_Breast_Cancer_Block_A_Section_1" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">10x Genomics </a><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.1.0/V1_Breast_Cancer_Block_A_Section_1" class="s42" target="_blank">https://support.10xgenomics.com/spatial-gene- expression/datasets/1.1.0/V1_Breast_Cancer_ Block_A_Section_1</a></p><p class="s41" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">(<i>Continued on next page</i>)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="647" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_173.png"/></span></p><p class="s34" style="padding-top: 1pt;padding-bottom: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Continued</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="650" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_174.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="360" alt="image" src="1-s2.0-S0092867421010618-main/Image_175.png"/></span></p><p class="s27" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">REAGENT or RESOURCE SOURCE IDENTIFIER</p><p class="s41" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;line-height: 115%;text-align: left;">Breast cancer Visium dataset (section 2, Space Ranger 1.1.0)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s41" style="padding-left: 59pt;text-indent: 0pt;line-height: 115%;text-align: left;">Colorectal cancer Visium dataset (Space Ranger 1.2.0)</p><p style="padding-top: 4pt;padding-left: 142pt;text-indent: -116pt;line-height: 115%;text-align: left;"><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.1.0/V1_Breast_Cancer_Block_A_Section_2" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">10x Genomics </a><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.1.0/V1_Breast_Cancer_Block_A_Section_2" class="s42" target="_blank">https://support.10xgenomics.com/spatial-gene- expression/datasets/1.1.0/V1_Breast_Cancer_ Block_A_Section_2</a></p><p style="padding-top: 1pt;padding-left: 142pt;text-indent: -116pt;line-height: 115%;text-align: left;"><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.2.0/Parent_Visium_Human_ColorectalCancer" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">10x Genomics </a><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.2.0/Parent_Visium_Human_ColorectalCancer" class="s42" target="_blank">https://support.10xgenomics.com/spatial-gene- expression/datasets/1.2.0/Parent_Visium_ Human_ColorectalCancer</a></p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.2.0/Parent_Visium_Human_OvarianCancer" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">Ovarian cancer Visium dataset (Space Ranger 1.2.0) 10x Genomics </a><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.2.0/Parent_Visium_Human_OvarianCancer" class="s42" target="_blank">https://support.10xgenomics.com/spatial-gene-</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: 0pt;line-height: 114%;text-align: left;"><a href="https://support.10xgenomics.com/spatial-gene-expression/datasets/1.2.0/Parent_Visium_Human_OvarianCancer" class="s42" target="_blank">expression/datasets/1.2.0/Parent_Visium_ Human_OvarianCancer</a></p><p style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark184" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Melanoma spatial transcriptomics </a><a href="https://www.spatialresearch.org/resources-published-datasets/doi-10-1158-0008-5472-can-18-0747/" class="s42" target="_blank">Thrane et al. (2018) https://www.spatialresearch.org/resources-</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: 0pt;line-height: 115%;text-align: left;"><a href="https://www.spatialresearch.org/resources-published-datasets/doi-10-1158-0008-5472-can-18-0747/" class="s42" target="_blank">published-datasets/doi-10-1158-0008-5472- can-18-0747/</a></p><p class="s43" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark180" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Microarray profiles of premalignant lung lesions </a>Teixeira et al. (2019) <span style=" color: #000;">GEO: GSE94611</span></p><p class="s41" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 115%;text-align: left;">Microarray profiles of lung squamous cell carcinogenesis</p><p class="s43" style="padding-top: 3pt;padding-left: 46pt;text-indent: 0pt;text-align: left;">Mascaux et al. (2019) <span style=" color: #000;">GEO: GSE33479</span></p><p class="s43" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 139%;text-align: left;"><a href="#bookmark182" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Foam cell macrophage gene set </a>Thomas et al. (2015) <a href="#bookmark99" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">N/A Epithelial, mesenchymal, and collagen gene sets </a>Aiello et al. (2018) <span style=" color: #000;">N/A</span></p><p class="s41" style="padding-left: 59pt;text-indent: 0pt;line-height: 114%;text-align: left;">Microarray profiles of lung squamous cell carcinoma (TCGA)</p><p class="s41" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 115%;text-align: left;">Microarray profiles of ovarian serous cystadenocarcinoma (TCGA)</p><p class="s41" style="padding-left: 9pt;text-indent: 0pt;line-height: 114%;text-align: left;">The Cancer Genome Atlas Network</p><p class="s41" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 115%;text-align: left;">The Cancer Genome Atlas Network</p><p style="padding-left: 44pt;text-indent: 0pt;line-height: 254%;text-align: left;"><a href="https://portal.gdc.cancer.gov/" class="s42" target="_blank">https://portal.gdc.cancer.gov/ https://portal.gdc.cancer.gov/</a></p><p class="s43" style="padding-left: 59pt;text-indent: 0pt;line-height: 139%;text-align: justify;"><a href="#bookmark157" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">LM22 signature matrix (CIBERSORTx) </a>Newman et al. (2015) <a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">N/A TR4 signature matrix (CIBERSORTx) </a>Newman et al. (2019) <a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">N/A Melanoma signature matrix (CIBERSORTx) </a>Newman et al. (2019) <span style=" color: #000;">N/A</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="416" alt="image" src="1-s2.0-S0092867421010618-main/Image_176.png"/></span></p><p class="s27" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Software and packages</p><p class="s41" style="padding-top: 4pt;padding-left: 362pt;text-indent: -302pt;line-height: 115%;text-align: left;">EcoTyper (v1.0) This work <a href="https://doi.org/10.25936/rssb-t744" class="s42" target="_blank">https://ecotyper.stanford.edu;https://doi.org/ 10.25936/rssb-t744</a></p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">CIBERSORTx (v1.0) </a><a href="https://cibersortx.stanford.edu/" class="s42" target="_blank">Newman et al. (2019) https://cibersortx.stanford.edu/</a></p><p class="s41" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 115%;text-align: left;">Workflow for benchmarking scRNA-seq annotation tools</p><p style="padding-top: 3pt;padding-left: 14pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/tabdelaal/scRNAseq_Benchmark/tree/snakemake_and_docker" class="s42" target="_blank">Abdelaal et al. (2019) https://github.com/tabdelaal/scRNAseq_</a></p><p style="padding-top: 1pt;padding-left: 130pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/tabdelaal/scRNAseq_Benchmark/tree/snakemake_and_docker" class="s42">Benchmark/tree/snakemake_and_docker</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">bcl2fastq (v2.20.0.422) Illumina </a><a href="https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html" class="s42" target="_blank">https://support.illumina.com/downloads/ bcl2fastq-conversion-software-v2-20.html</a></p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark120" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">umi_homopolymer.py </a><a href="https://github.com/jwfoley/3SEQtools" class="s42" target="_blank">Foley et al. (2019) https://github.com/jwfoley/3SEQtools</a></p><p style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark115" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">STAR (v2.7.3a) </a><a href="https://github.com/alexdobin/STAR" class="s42" target="_blank">Dobin et al. (2013) https://github.com/alexdobin/STAR</a></p><p style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark131" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">ImmRes_OE.R </a><a href="https://github.com/livnatje/ImmuneResistance" class="s42" target="_blank">Jerby-Arnon et al. (2018) https://github.com/livnatje/ImmuneResistance</a></p><p style="padding-top: 3pt;padding-left: 362pt;text-indent: -302pt;line-height: 114%;text-align: left;"><a href="#bookmark124" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NMF R package (v0.20.0) </a><a href="https://cran.r-project.org/web/packages/NMF/index.html" class="s42" target="_blank">Gaujoux and Seoighe (2010) https://cran.r-project.org/web/packages/ NMF/index.html</a></p><p style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark110" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">NetworkToolbox R package (v1.4.0) </a><a href="https://cran.r-project.org/web/packages/NetworkToolbox/index.html" class="s42" target="_blank">Christensen (2018) https://cran.r-project.org/web/packages/</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: 0pt;text-align: left;"><a href="https://cran.r-project.org/web/packages/NetworkToolbox/index.html" class="s42">NetworkToolbox/index.html</a></p><p style="padding-top: 3pt;padding-left: 362pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark146" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">cluster R package (v2.0.7) </a><a href="https://cran.r-project.org/web/packages/cluster/index.html" class="s42" target="_blank">Maechler et al. (2012) https://cran.r-project.org/web/packages/ cluster/index.html</a></p><p class="s43" style="padding-top: 1pt;padding-left: 246pt;text-indent: -186pt;line-height: 114%;text-align: right;"><a href="#bookmark108" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Seurat R package (v2.3.4 and v3.1.3) </a>Butler et al. (2018)<a href="#bookmark177" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">; </a><a href="#bookmark177" class="s42">Stuart et al. (2019)</a></p><p style="padding-top: 1pt;padding-left: 51pt;text-indent: 0pt;line-height: 114%;text-align: left;"><a href="https://cran.r-project.org/web/packages/Seurat/index.html" class="s42" target="_blank">https://cran.r-project.org/web/packages/ Seurat/index.html</a></p><p style="padding-top: 2pt;padding-left: 362pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark166" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">EDASeq R package (v2.14.1) </a><a href="https://bioconductor.org/packages/release/bioc/html/EDASeq.html" class="s42" target="_blank">Risso et al. (2011) https://bioconductor.org/packages/release/ bioc/html/EDASeq.html</a></p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark148" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Imvigor210CoreBiologies R package (v1.0.0) </a><a href="http://research-pub.gene.com/IMvigor210CoreBiologies/" class="s42" target="_blank">Mariathasan et al. (2018) http://research-pub.gene.com/</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: 0pt;text-align: left;"><a href="http://research-pub.gene.com/IMvigor210CoreBiologies/" class="s42">IMvigor210CoreBiologies/</a></p><p style="padding-top: 3pt;padding-left: 362pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark167" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">limma R package (v3.36.2) </a><a href="https://bioconductor.org/packages/release/bioc/html/limma.html" class="s42" target="_blank">Ritchie et al. (2015) https://bioconductor.org/packages/release/ bioc/html/limma.html</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark150" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">umap R package (v.0.2.0.0) </a><a href="https://cran.r-project.org/web/packages/umap/index.html" class="s42" target="_blank">McInnes et al. (2018) https://cran.r-project.org/web/packages/ umap/index.html</a></p><p style="padding-top: 1pt;padding-left: 362pt;text-indent: -302pt;line-height: 114%;text-align: left;"><a href="#bookmark116" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">BioMart R package (v2.38) </a><a href="https://bioconductor.org/packages/release/bioc/html/biomaRt.html" class="s42" target="_blank">Durinck et al. (2009) https://bioconductor.org/packages/release/ bioc/html/biomaRt.html</a></p><p class="s41" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">(<i>Continued on next page</i>)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="647" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_177.png"/></span></p><p class="s34" style="padding-top: 1pt;padding-bottom: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark19">Continued</a><a name="bookmark20">&zwnj;</a><a name="bookmark21">&zwnj;</a><a name="bookmark22">&zwnj;</a><a name="bookmark23">&zwnj;</a><a name="bookmark24">&zwnj;</a><a name="bookmark25">&zwnj;</a><a name="bookmark26">&zwnj;</a></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="650" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_178.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="654" height="214" alt="image" src="1-s2.0-S0092867421010618-main/Image_179.png"/></span></p><p class="s27" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">REAGENT or RESOURCE SOURCE IDENTIFIER</p><p style="padding-top: 4pt;padding-left: 355pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark136" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">fgsea R package (v1.14.0) </a><a href="https://bioconductor.org/packages/release/bioc/html/fgsea.html" class="s42" target="_blank">Korotkevich et al. (2021) https://bioconductor.org/packages/release/ bioc/html/fgsea.html</a></p><p style="padding-top: 1pt;padding-left: 355pt;text-indent: -302pt;line-height: 114%;text-align: left;"><a href="#bookmark181" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">survival R package (v2.42.3) </a><a href="https://cran.r-project.org/web/packages/survival/index.html" class="s42" target="_blank">Therneau and Grambsch (2000) https://cran.r-project.org/web/packages/ survival/index.html</a></p><p style="padding-top: 2pt;padding-left: 355pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark188" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">MASS R package (v7.3.50) </a><a href="https://cran.r-project.org/web/packages/MASS/index.html" class="s42" target="_blank">Venables and Ripley (2002) https://cran.r-project.org/web/packages/ MASS/index.html</a></p><p style="padding-top: 1pt;padding-left: 355pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark114" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">igraph R package (v1.2.2) </a><a href="https://cran.r-project.org/web/packages/igraph/index.html" class="s42" target="_blank">Csardi and Nepusz (2006) https://cran.r-project.org/web/packages/ igraph/index.html</a></p><p style="padding-top: 1pt;padding-left: 355pt;text-indent: -302pt;line-height: 114%;text-align: left;"><a href="#bookmark160" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">ape R package (v5.4.1) </a><a href="https://cran.r-project.org/web/packages/ape/index.html" class="s42" target="_blank">Paradis and Schliep (2019) https://cran.r-project.org/web/packages/ ape/index.html</a></p><p style="padding-top: 2pt;padding-left: 355pt;text-indent: -302pt;line-height: 115%;text-align: left;"><a href="#bookmark143" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Gemma </a><a href="http://gemma.msl.ubc.ca/%20arrays/showArrayDesign.html?id=420" class="s42" target="_blank">Lim et al. (2021) http://gemma.msl.ubc.ca/ arrays/ showArrayDesign.html?id=420</a></p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="https://cran.r-project.org/" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">R (v3.5.1) R CRAN </a><a href="https://cran.r-project.org/" class="s42" target="_blank">https://cran.r-project.org/</a></p><p class="s41" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Prism (v7) GraphPad Software, Inc. N/A</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">RESOURCE AVAILABILITY</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Lead contact</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="mailto:amnewman@stanford.edu" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" target="_blank">Further information and requests for resources should be directed to and will be fulfilled by the lead contact, Aaron M. Newman (</a>amnewman@stanford.edu<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Materials availability</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">This study did not generate new unique reagents.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark198">Data and code availability</a></p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;">Bulk RNA-seq data have been deposited at GEO and are publicly available as of the date of publication. The accession number is listed in the key resources table. This paper also analyzes existing, publicly available data. These accession numbers for the datasets are listed in the key resources table. Microscopy data reported in this paper will be shared by the lead contact upon request.</p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: left;"><a href="https://ecotyper.stanford.edu/" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" target="_blank">The original code for EcoTyper is publicly available as of the date of the publication for non-profit academic use. The DOI is listed in the key resources table. Updates to the code will be available from </a>https://ecotyper.stanford.edu<span style=" color: #000;">.</span></p><p style="padding-left: 61pt;text-indent: 0pt;line-height: 10pt;text-align: left;">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark199">EXPERIMENTAL MODEL AND SUBJECT DETAILS</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Human subjects</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">All clinical specimens in this study were collected with informed consent for research use and were approved by the Stanford Uni- versity Institutional Review Boards in accordance with the Declaration of Helsinki. Characteristics of each subject, including age and sex, are available in </a>Table S1<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark200">METHOD DETAILS</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">EcoTyper framework</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">EcoTyper performs the following modular steps, each graphically depicted in </a>Figures 1 <a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S1<span style=" color: #000;">A, with algorithmic details provided in the sections below.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s7" style="padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><span class="s36">d </span>In silico <span class="p">purification: This step enables imputation of cell type-specific gene expression profiles from bulk tissue transcriptomes. It consists of the following sections: overview of </span>in silico <span class="p">purification, signature matrix design, cell type fraction estimation, and expression purification. Of note, if cell type-specific expression profiles are already available, whether from scRNA-seq, bulk- sorted transcriptomes, or another source, this step can be skipped.</span></p><p class="s36" style="padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: left;">d <span class="p">Cell state discovery: This step enables identification and quantitation of cell type-specific transcriptional states. It consists of the following sections: cell state discovery, rank selection, and cell state quality control.</span></p><p class="s36" style="padding-left: 65pt;text-indent: 0pt;line-height: 10pt;text-align: left;">d <span class="p">Ecotype discovery: This step enables co-assignment of cell states into multicellular communities (ecotypes).</span></p><p class="s36" style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;">d <span class="p">Cell state and ecotype recovery: This step enables recovery of cell states and ecotypes in external expression datasets.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s31" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a name="bookmark27">Overview of </a><span class="s22">in silico </span>purification<a name="bookmark28">&zwnj;</a><a name="bookmark29">&zwnj;</a><a name="bookmark30">&zwnj;</a><a name="bookmark31">&zwnj;</a></p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">The first step of EcoTyper is to impute cell type-specific gene expression profiles from bulk tissue transcriptomes. By default, Eco- Typer employs CIBERSORTx (</a>Newman et al., 2019<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) for this purpose, though other deconvolution methods with comparable capa- bilities could be used. CIBERSORTx minimizes technical variation across platforms and can simultaneously purify expression profiles from multiple cell types (&gt; 10) at single-sample resolution. As input, CIBERSORTx requires a collection of optimized expression pro- files that discriminate each cell type of interest, commonly termed a ‘signature matrix’. Signature matrices can be derived from single- cell or bulk-sorted transcriptomes and should be designed to cover major lineages within a particular tissue type. Once a signature matrix has been generated and validated (</a>Newman et al., 2019<a href="#bookmark174" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Steen et al., 2020<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), CIBERSORTx is applied to a dataset of uniformly processed bulk tissue transcriptomes to enumerate the frequencies of each cell type in the signature matrix. These estimates are then used to impute cell type-specific gene expression profiles for each input sample. Importantly, only genes with sufficient signal are imputed for each cell type, thereby minimizing the influence of spurious expression estimates on downstream results (</a><a href="#bookmark158" class="s10">Newman </a>et al., 2019<a href="#bookmark174" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Steen et al., 2020<span style=" color: #000;">). The following equations and goals summarize the key CIBERSORTx steps used by EcoTyper:</span></p><ul id="l10"><li style="padding-left: 2pt;text-indent: -2pt;line-height: 6pt;text-align: left;"><p class="s44" style="display: inline;">;<span class="s45">j</span></p></li></ul><p style="text-indent: 0pt;text-align: left;"/><p style="padding-top: 5pt;padding-left: 262pt;text-indent: 0pt;line-height: 8pt;text-align: left;">B 3 F   = M0  <span class="s9">;  </span>1%<i>j</i>%<i>n                                                                                  </i>(1)</p><ul id="l11"><li style="padding-left: 310pt;text-indent: -2pt;line-height: 4pt;text-align: left;"><p class="s44" style="display: inline;">;<span class="s45">j</span></p></li></ul><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 12pt;padding-left: 245pt;text-indent: 0pt;text-align: left;">diagðG<i>i</i><span class="s9">;</span>•<span class="s9">;</span>• 3  FÞ = M<i>i</i><span class="s9">;</span>•<span class="s9">;  </span>1%<i>i</i>%<i>g                                                                           </i>(2)</p><p style="padding-top: 6pt;padding-left: 59pt;text-indent: 8pt;line-height: 111%;text-align: justify;">Given B, an <i>m </i>3 <i>c </i>signature matrix consisting of <i>m </i>marker genes by <i>c </i>distinct cell types (<i>Signature matrix design </i>below), and M0, an <i>m </i>3 <i>n </i>matrix of bulk tissue gene expression profiles consisting of the same <i>m </i>genes by <i>n </i>samples, the goal of <span style=" color: #0097CF;">Equation 1 </span>is to impute F,a <i>c </i>3 <i>n </i>matrix consisting of the fractional abundances of <i>c </i>cell types for each sample in M0. M is the full <i>g </i>3 <i>n </i>matrix of bulk gene expression profiles (<i>g </i>denotes the number of genes), of which M0 is a subset (Note that M<i>i</i><span class="s9">;</span>• and M•<span class="s9">;</span><i>j </i>denote row <i>i </i>and column <i>j </i>of matrix M, respectively). Once F is imputed (<i>Cell type fraction estimation </i>below), the goal of <span style=" color: #0097CF;">Equation 2</span>, which summarizes the high-resolution expression purification step of CIBERSORTx, is to impute G, a <i>g </i>3 <i>n </i>3 <i>c </i>matrix consisting of <i>g </i>genes, <i>n </i>samples, and <i>c </i>cell types, given F and M (<i>Expression purification </i><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">below). For further details, see </a><a href="#bookmark158" class="s10">Newman et al. </a><span style=" color: #0097CF;">(2019)</span>.</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Signature matrix design</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To deconvolve 12 major cell types in human carcinomas (</a>Figure 1<a href="#bookmark157" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), we employed a hierarchical strategy in which two signature matrices, each previously validated in solid tumors (</a>Newman et al., 2015<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Newman et al., 2019<a href="#bookmark126" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), were serially applied. First, major cellular compartments in epithelial tumors were deconvolved using TR4, a signature matrix consisting of epithelial (EPCAM+), endo- thelial (CD31+), fibroblast (CD10+), and bulk immune cell (CD45+) populations that were sorted from freshly resected surgical tumor samples from patients with NSCLC (</a>Gentles et al., 2020<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Newman et al., 2019<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Through a series of benchmarking experiments, both from prior literature and the current work, we confirmed the high accuracy and generalizability of this matrix across multiple epithelial tumor types (</a>Figures S1<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and S1C; </a>Table S2<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Newman et al., 2019<a href="#bookmark157" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To resolve leukocyte phenotypes, we employed LM22, a widely validated signature matrix consisting of 22 functionally-defined human hematopoietic cell subsets (</a>Newman et al., 2015<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). We aggre- gated LM22 subsets into B cells, plasma cells, CD8 T cells, CD4 T cells, natural killer (NK) cells, monocytes/macrophages, dendritic cells, mast cells, and neutrophils according to the mapping scheme provided in </a>Table S2<span style=" color: #000;">. Because eosinophils were largely unde- tectable, they were excluded from further analysis.</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Cell type fraction estimation</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To impute cell type proportions, CIBERSORTx was applied independently to each tumor type in the TCGA discovery cohort (</a>Figure 1<span style=" color: #000;">, see also </span><span class="s7">External datasets</span><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) as previously described (</a>Newman et al., 2019<span style=" color: #000;">), using B-mode batch correction for LM22, no batch correc- tion for TR4, no quantile normalization, and otherwise default parameters. To unify the signature matrices, leukocyte fractions from LM22 were rescaled to sum to 1 within each sample, then multiplied by total immune content imputed by TR4, yielding matrix F (</span>Equation 1<span style=" color: #000;">).</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Expression purification</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">To impute cell type-specific gene expression profiles, we provided two inputs to the high-resolution module of CIBERSORTx: the imputed fractions of all 12 cell types in the discovery cohort and a bulk expression matrix containing all tumor and adjacent normal samples (<i>External datasets</i>). For this step, we restricted our analysis to protein coding genes, as annotated in GENCODE v24. High- resolution expression purification was run with default parameters, yielding matrix G (<span style=" color: #0097CF;">Equation 2</span><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Notably G can also be derived from alternative sources (e.g., scRNA-seq) or deconvolution methods, as noted above (see also </a><span style=" color: #0097CF;">Figure S1</span>A).</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Cell state discovery</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">EcoTyper leverages variants of nonnegative matrix factorization (NMF) coupled with dedicated heuristics to identify, recover, and validate cell states. Collectively, these approaches (i) identify cell states <i>de novo </i>from matrix G while maximizing the positive predic- tive value (PPV) of state discovery; (ii) estimate the relative abundance of each identified cell state within each sample; and (iii) enable the recovery of cell states in external expression datasets across platforms.</p><p style="padding-left: 59pt;text-indent: 8pt;line-height: 110%;text-align: justify;">Given <i>c </i>cell types, let V<i>i</i>)G•<span class="s9">;</span>•<span class="s9">;</span><i>i </i>be a <i>g</i>3<i>n </i>cell type-specific expression matrix for cell type <i>i </i>consisting of <i>g </i>rows (the number of genes) and <i>n </i>columns (the number of samples). The primary objective of NMF is to factorize V<i>i </i>into two non-negative matrices: a <i>g</i>3<i>k </i>matrix, W, and a <i>k</i>3<i>n </i>matrix, H, where <i>k </i>is the matrix rank (i.e., number of clusters):</p><p style="padding-top: 4pt;padding-left: 283pt;text-indent: 0pt;text-align: left;">V<i>i </i>= W 3 H (3)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In EcoTyper, we employ gene-level standardization to improve the sensitivity of data clustering with NMF while enabling general- ization of EcoTyper across platforms. The benefit of standardization can be seen, for example, in simulated tumors where sensitivity improved from 69% to 92% (</a><span style=" color: #0097CF;">Figure S1</span>F). More specifically, as input, each gene <i>j </i>in V<i>i </i>is individually adjusted to log2 expression and standardized to mean zero and unit variance (mj = 0, sj = 1) within each dataset (in this work, each tumor type). To satisfy the non- negativity requirement of NMF, each cell type-specific expression matrix V<i>i </i>is then ‘posneg transformed’. This converts an input</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 6pt;text-align: left;">expression matrix V<i>i </i>into two input matrices, V + and V<span class="s46">--</span>, each containing positive values ðV + Þ or negative values with the sign in-</p><p class="s48" style="padding-top: 1pt;padding-left: 88pt;text-indent: 0pt;line-height: 26%;text-align: left;"><span class="s47">-- </span>i i <span class="s49">* </span>i</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 9pt;text-align: left;">verted ðV<span class="s50">i </span>Þ (<span style=" color: #0097CF;">Equation 4</span>). The two matrices are concatenated to produce a <i>d</i>3<i>n </i>matrix V<span class="s50">i </span>where <i>d </i>= 2<i>g </i>(<span style=" color: #0097CF;">Equation 4</span>).</p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;line-height: 9pt;text-align: left;">0<span class="s9">; </span><i>otherwise</i></p><p style="text-indent: 0pt;text-align: left;"/><p class="s7" style="padding-left: 259pt;text-indent: 0pt;text-align: left;"><span class="s47">V</span><span class="s51">+</span><span class="s32"> </span><span class="s47">)</span><span class="s52">(</span><span class="s53"> </span><span class="p">V</span>i<span class="s9">;  </span>if <span class="p">V</span>i &gt; <span class="p">0</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 12pt;text-indent: 0pt;text-align: center;">(4)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 5pt;padding-left: 12pt;text-indent: 0pt;line-height: 45%;text-align: center;"><span class="s47">V</span><span class="s54">-</span>-<span class="s47">)</span><span class="s52">(</span><span class="s53"> </span>—V<i>i</i><span class="s9">;  </span><i>if </i>V<i>i &lt; </i>0</p><p class="s55" style="padding-left: 12pt;text-indent: 0pt;line-height: 9pt;text-align: center;">i               <span class="p">0</span><span class="s9">; </span><span class="s7">otherwise</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;line-height: 12pt;text-align: left;">V*<span class="s56">i</span><span class="s45"> </span>)</p><p style="text-indent: 0pt;text-align: left;"/><p class="s45" style="text-indent: 0pt;line-height: 4pt;text-align: left;">i</p><p style="text-indent: 0pt;line-height: 8pt;text-align: left;">--</p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s53" style="padding-bottom: 4pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">  <span class="s57">V</span><span class="s58">+ </span>!</p><p style="text-indent: 0pt;line-height: 9pt;text-align: left;">V</p><p style="padding-left: 302pt;text-indent: 0pt;line-height: 9pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 93%;text-align: left;">To identify and quantitate cell states within V*<span class="s50">i</span><span class="s48"> </span><a href="#bookmark107" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, EcoTyper applies NMF via Kullback-Leibler (KL) divergence minimization (</a><a href="#bookmark107" class="s10">Brunet et al., </a><span style=" color: #0097CF;">2004</span>), which starts with random initializations of the W and H matrices (<i>NMF </i><a href="#bookmark124" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">R package version 0.20.0) (</a><a href="#bookmark124" class="s10">Gaujoux and Seoighe,</a></p><p style="text-indent: 0pt;text-align: left;"><span class="s59">P</span><span class="s56">n</span><span class="s45"> </span><span class="s60">［</span>H<i>kn</i>V*<span class="s56">dn</span><span class="s45"> </span><span class="s60">I</span>ðW 3 HÞ<i>dn</i><span class="s60">］</span></p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><span style=" color: #0097CF;">2010</span>). For clarity, let V*)V*<span class="s50">i</span><span class="s48"> </span>. This approach iteratively updates the following two equations until KL divergence is minimized:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="115" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_180.png"/></span></p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">n</p><p style="text-indent: 0pt;text-align: left;"/><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">kn</p><p style="text-indent: 0pt;text-align: left;"/><p class="s61" style="padding-top: 2pt;padding-left: 235pt;text-indent: 0pt;text-align: left;">W<span class="s45">dk </span>)<span class="p">W</span><span class="s45">dk                     </span><span class="s62">P</span><span class="s53"> </span><span class="s63">H</span></p><p style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">(5)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="118" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_181.png"/></span></p><p class="s53" style="text-indent: 0pt;line-height: 76%;text-align: left;">P <span class="s64">W</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s61" style="padding-top: 6pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">H<span class="s45">kn </span>)<span class="p">H</span><span class="s45">kn</span></p><p style="padding-top: 2pt;text-indent: 0pt;text-align: left;"><span class="s59">P</span><span class="s56">d</span><span class="s45"> </span><span class="s60">［</span>W<i>dk</i>V*<span class="s56">dn</span><span class="s45"> </span><span class="s60">I</span>ðW 3 HÞ<i>dn</i><span class="s60">］</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">d</p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">dk</p><p style="padding-left: 38pt;text-indent: 0pt;line-height: 7pt;text-align: left;"><span style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;"> </span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 6pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">(6)</p><p style="padding-top: 5pt;padding-left: 53pt;text-indent: 8pt;line-height: 110%;text-align: justify;">Here, the matrix rank <i>k </i>specifies the number of states or clusters (see <i>Rank selection </i>below); the basis matrix W encodes a refer- ence profile for each cell state in cell type <i>i</i>; and the mixture coefficients matrix H encodes the representation (relative abundance) of each cell state in each sample. For each rank and cell type, the model with the lowest root mean squared error (RMSE) from 50 re- starts (by default) is selected. The mixture coefficients matrix H is then rescaled such that each column sums to 1 (i.e., each sample is represented as a mixture of cell state proportions that sum to 1 within each cell type), yielding H<span class="s65">^</span>.</p><p style="padding-left: 61pt;text-indent: 0pt;line-height: 9pt;text-align: left;">Although posneg transformation is required to satisfy the non-negativity constraint of NMF following standardization, it can lead to</p><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">the identification of spurious cell states driven by features with more negative values than positive ones (</a><span style=" color: #0097CF;">Figure S1</span>F). To combat this, we devised an adaptive false positive index (AFI), an index defined as the ratio between the sum of weights from the W matrix cor- responding to the negative and positive features (<span style=" color: #0097CF;">Equation 7</span>). Specifically, the W matrix produced by NMF with posneg transforma- tion is of size <i>d</i>3<i>k</i>, with the top <i>g </i>values per column (state) corresponding to the weights of the features in V + and the bottom <i>g </i>values</p><p style="padding-left: 181pt;text-indent: 0pt;line-height: 1pt;text-align: left;">-- <span class="s66">*</span></p><p class="s53" style="text-indent: 0pt;line-height: 9pt;text-align: left;">P</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;line-height: 85%;text-align: left;">corresponding to the weights of V<span class="s50">i </span>. Within V<span class="s50">i </span>, we calculate AFI for each state <i>s </i>as follows:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s53" style="text-indent: 0pt;line-height: 9pt;text-align: left;">P</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 59pt;text-indent: 0pt;line-height: 5pt;text-align: right;">AFI )</p><p class="s45" style="padding-top: 4pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">d r <span class="s32">= </span>g <span class="s32">+ 1</span></p><p class="s61" style="padding-top: 5pt;text-indent: 0pt;text-align: left;">W<span class="s45">rs</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 5pt;text-align: right;">(7)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="55" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_182.png"/></span></p><p class="s45" style="padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">s g</p><p class="s45" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">r <span class="s32">= 1</span></p><p class="s61" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">W<span class="s45">rs</span></p><p class="s24" style="padding-top: 5pt;padding-left: 53pt;text-indent: 8pt;line-height: 110%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In simulated tumors, we found that 84% of false positives were eliminated by requiring AFI &lt; 1 without impacting sensitivity (</a><a href="#bookmark203" class="s10">Fig- </a>ure S1<a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F). In real tumors from TCGA (</a>Figure 1<a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), 85% of states with AFI &lt; 1 were significantly recoverable in scRNA-seq data via refer- ence-guided annotation, whereas only 3.7% of states were significantly recoverable with AFI R 1 (</a>Figure S2<span style=" color: #000;">C; see </span><span class="s7">Cell state and ecotype recovery </span><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">below for the recovery procedure). Thus, standardization and AFI filtering can improve sensitivity and PPV (</a><a href="#bookmark203" class="s10">Fig- </a>ure S1<span style=" color: #000;">F). Following AFI filtering and other quality control steps (</span><span class="s7">Cell state quality control </span><span style=" color: #000;">below), filtered states are removed from H</span><span class="s65">^</span><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, yielding H* (</a>Figure S1<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A). We interchangeably refer to the values in H* as cell state abundances or fractions. For analyses in which samples are assigned to specific cell states, each sample was assigned to the state with highest relative abundance among all states of a given cell type (e.g., </a>Figures 2<span style=" color: #000;">A and 2B).</span></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">In practice, each digitally-purified expression matrix V<i>i </i>is analyzed independently by EcoTyper. Based on considerations of runtime efficiency and precedent in scRNA-seq workflows, EcoTyper analyzes the top 1,000 genes with highest relative dispersion per cell type ðV<i>i</i>Þ by default. Notably, in fibroblasts and epithelial cells, two cell types with the highest number of variably expressed genes in our data, clustering results were similar regardless of whether the top 1,000 or 2,000 most variable genes were used (data not shown). To calculate dispersion, genes in log2 space were averaged across samples and binned into 20 groups by 5 percentile increments.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a name="bookmark32">The relative dispersion of each gene was then calculated as the difference between its dispersion and the median dispersion of genes within the same expression bin, divided by the median absolute deviation of the dispersion of genes within the same expression bin. </a><span class="s31">Rank selection</span><a name="bookmark33">&zwnj;</a><a name="bookmark34">&zwnj;</a><a name="bookmark35">&zwnj;</a></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">To determine <i>k </i><a href="#bookmark107" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(the number of clusters or states) for each cell type, we evaluated several approaches, including ones that rely on minimizing error measures (e.g., RMSE, KL divergence), optimizing information-theoretic metrics, or calculating the cophenetic co- efficient (CC), which quantifies classification stability and ranges from 0 to 1, with 1 being maximally stable (</a><span style=" color: #0097CF;">Brunet et al., 2004</span>). With the exception of CC, such approaches either failed to converge or were dependent on the number of genes imputed. Therefore, we selected CC for EcoTyper. While the rank at which CC starts decreasing is typically selected, this approach is challenging to apply in situations where the CC exhibits a multi-modal shape across ranks, as we found for some cell types. Therefore, we developed a heuristic approach more suitable for such settings. For each cell type, <i>k </i>was automatically selected from a range of 2–20 (by default). Specifically, we determined the first occurrence in the interval 2–20 for which CC dropped below 0.95 (by default), having been above this level for at least two consecutive ranks. We then selected the rank immediately adjacent to this crossing point that was closest to</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">0.95 (by default). We applied this approach for all cell types except two. First, for epithelial cells there was a steep drop in CC at <i>k </i>= 8, after which it stabilized just below 0.95. In this case, we chose <i>k </i>= 8. Second, since the CC never decreased below 0.95 for neutro- phils, we selected <i>k </i><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">= 5, the rank at which the CC stabilized. The stability of these results is illustrated in </a><span style=" color: #0097CF;">Figure S2</span>E.</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Cell state quality control</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We applied three quality control filters to eliminate non-robust states (</a><span style=" color: #0097CF;">Figure S2</span>B). First, we determined the number of ‘marker’ genes <i>n </i>in each state <i>s </i>with (i) nonzero expression in at least 50% of samples and (ii) the highest log2 fold change in state <i>s </i>relative to other states. States with <i>n </i>% 10 marker genes were omitted. Second, we removed all states with AFI R 1 (<span style=" color: #0097CF;">Equation 7</span>), with the exception of one epithelial state (state S6) with a borderline AFI ratio (1.12) and &gt; 50 marker genes. Third, we identified poor-quality cell states using a dropout score, which flags states whose marker genes exhibit anomalously low variance and high expression across the dis- covery cohort. To calculate the dropout score for each marker gene, we determined the maximum fraction of samples for which the gene had the same value. We also calculated the average log2 expression of the gene across samples. We averaged each quantity, scaled to unit variance across states, converted them to <i>z </i>scores, and removed states with a mean <i>Z </i><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">&gt; 1.96 (p &lt; 0.05) (</a><span style=" color: #0097CF;">Figure S2</span>D). In all analyses involving discrete assignments of samples to cell states, samples that were assigned to discarded states were consid- ered unassigned.</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Ecotype discovery</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">To identify multicellular communities (i.e., ecotypes), we devised an approach in which pairwise co-associations between cell states were maximized while mutual avoidance within a cluster was minimized. First, we used the Jaccard index to quantify the degree of overlap between each pair of cell states in H* (see <i>Cell state discovery</i><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) across tumor samples in the discovery cohort (</a><a href="#bookmark203" class="s10">Figure </a><span style=" color: #0097CF;">S1</span>A).</p><p style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Toward this end, we discretized each cell state <i>s </i>from H*<span class="s50">i</span><span class="s48">   </span>ð1 %<i>i </i>% <i>c</i>Þ into a binary vector a of length <i>l</i>, where <i>l </i>= the number of tumor samples in the discovery cohort and <i>c </i>is the number of cell types. These vectors constituted a binary matrix A, with 69 rows (states) 3 <i>l </i>columns (samples). Given sample <i>w</i>, if state <i>s </i>was the most abundant state among all states in cell type <i>i</i>, we set A<i>s</i><span class="s9">;</span><i>w </i>to 1; otherwise</p><p class="s7" style="padding-left: 59pt;text-indent: 0pt;line-height: 110%;text-align: justify;"><span class="p">A</span>s<span class="s9">;</span>w <span class="p">) 0. We then computed all pairwise Jaccard indices on the rows (states) in matrix A, yielding matrix J with 69 rows 3 69 col- umns. Using the hypergeometric test, we evaluated the null hypothesis that any given pair of cell states </span>s <span class="p">and </span>q <span class="p">has no overlap. In cases where the hypergeometric p value was &gt; 0.01, the Jaccard index for J</span>s<span class="s9">;</span>q <span class="p">was set to 0 (i.e., no overlap). To identify communities while accommodating outliers, the updated Jaccard matrix J0 was hierarchically clustered using average linkage with Euclidean distance (</span>hclust <span class="p">in the R </span>stats <a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">package). The optimal number of clusters was then determined via silhouette width maximization (</a><a href="#bookmark207" class="s10">Fig- ure </a><span class="s24">S5</span><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A). Clusters with % 2 cell states were eliminated from further analysis, leaving 10 clusters, which we termed carcinoma eco- types (CEs) (</a><a href="#bookmark207" class="s10">Figure </a><span class="s24">S5</span><span class="p">B).</span></p><p class="s24" style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark106" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To evaluate the robustness of CE definitions, we applied two alternative methods to J0: Louvain community detection (</a><a href="#bookmark106" class="s10">Blondel et al., </a>2008<a href="#bookmark133" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and k-medoids (</a><a href="#bookmark133" class="s10">Kaufman and Rousseeuw, </a>1990<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a><a href="#bookmark207" class="s10">Figure </a>S5<span style=" color: #000;">C). To evaluate the Louvain algorithm (</span><span class="s7">NetworkToolbox v1.4.0 </span><a href="#bookmark110" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">R package (</a><a href="#bookmark110" class="s10">Christensen, </a>2018<span style=" color: #000;">)), we determined the set of parameters, </span><span class="s7">gamma</span><span style=" color: #000;">, that produced each number of clusters be- tween 2 and 30 and selected the value of gamma that produced the number of clusters with the highest mean silhouette width. To evaluate k-medoids (</span><span class="s7">pam </span><span style=" color: #000;">function in the R package, </span><span class="s7">cluster v2.0.</span><a href="#bookmark146" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">7 (</a><a href="#bookmark146" class="s10">Maechler et al., </a>2012<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">)), we varied the number of centroids be- tween 2 and 30 and selected the number that maximized the mean silhouette width. To promote a fair comparison, we filtered out communities with less than three states (as above), then rendered the comparisons as river plots (</a><a href="#bookmark207" class="s10">Figure </a>S5<span style=" color: #000;">C).</span></p><p style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;">To estimate CE abundance, cell state abundances (which were previously scaled to sum to 1 within each cell type in each sample) from each CE were averaged. The resulting values were normalized to sum to 1 across all CEs in each sample. To assign samples to CEs, we applied a two-sided t test with unequal variance to evaluate the difference in estimated abundance between the cell states in each CE relative to the abundances of all cell states in other CEs. The resulting p values were corrected for multiple hypothesis testing using the Benjamini-Hochberg method. Each sample was assigned to the CE with the highest CE abundance if: (i) its corresponding q-value was less than 0.25 and (ii) the sample was assigned to at least one cell state contributing to CEs. Otherwise, the sample re- mained unassigned. For melanoma datasets, epithelial cell states were ignored when determining CE assignments.</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Cell state and ecotype recovery</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">We leveraged the internal structure of the NMF model to devise a reference-based strategy for recovering cell states in new samples.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 53pt;text-indent: 8pt;text-align: justify;"><a name="bookmark36">In classical NMF, matrices W and H are iteratively updated according to </a><span style=" color: #0097CF;">Equations 5 </span>and <span style=" color: #0097CF;">6 </span>until convergence. In a new expression dataset M*, preprocessed as described for V*<span class="s50">i</span><span class="s48">   </span>above, one can reuse the previously fit cell type-specific basis matrix W (<span style=" color: #0097CF;">Equations 5 </span>and <span style=" color: #0097CF;">6</span><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) in order to determine the mixture coefficients matrix H0 (</a><a href="#bookmark203" class="s10">Figure </a><span style=" color: #0097CF;">S1</span>A):<a name="bookmark37">&zwnj;</a><a name="bookmark38">&zwnj;</a></p><p style="padding-top: 6pt;padding-left: 274pt;text-indent: 0pt;text-align: left;">M* = W 3 H0                                                                                          (8)</p><p style="padding-top: 6pt;padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">This procedure consists of iteratively updating H0 until convergence of <span style=" color: #0097CF;">Equation 8</span>. Following this, states previously identified as non-robust by AFI filtering and other quality control steps (<i>Cell state quality control</i>) are removed, yielding mixture coefficients matrix H00. This approach has three advantages over alternative methods for supervised classification. First, the mathematical structure of the original model is maintained when classifying new samples. This eliminates the need to train another classifier and avoids the introduction of new assumptions or biases that lead to information loss. Second, this approach mirrors the output of the original model, facilitating consistent interpretation. Third, unlike methods that perform supervised classification independently for each sample, the matrix H0 is jointly updated across all samples, increasing the robustness of cell state recovery.</p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We implemented this framework within EcoTyper and applied it to external expression datasets analyzed in this work (</a>Figure 1<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Table S1<span style=" color: #000;">). In each case, EcoTyper solved </span>Equation 8 <a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">using the cell type-specific weight matrices defined in the discovery cohort. Prior to analysis, each gene was log2-transformed and scaled to unit variance within each tumor type (TCGA/PRECOG), healthy tis- sue type (GTEx), spatial transcriptomics array, or individual dataset (scRNA-seq data, immunotherapy datasets, and early tumor development datasets), as appropriate (</a>Table S1<span style=" color: #000;">). Once cell states were quantitated, CE abundance could be determined as described in </span><span class="s7">Ecotype discovery</span><a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">. To assess the accuracy of cell state recovery, we applied this approach in multiple scenarios, including cross-platform analyses and scRNA-seq labeling (e.g., </a>Figures S3<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A–S3G, </a>S4<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B–S4D, </a>S5<span style=" color: #000;">E, S5F, S5H, and S5I).</span></p><p class="s31" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Significance of cell state recovery</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 107%;text-align: justify;">To determine the statistical significance of reference-guided cell state recovery from scRNA-seq data, we devised a permutation- based procedure. First, we assigned single-cell transcriptomes to cell states by solving <span style=" color: #0097CF;">Equation 8</span>. Each cell of a given cell type was assigned to the state with maximum weight. Next, for each state <i>s </i>and its corresponding list of predefined marker genes g<span class="s50">s</span><span class="s48"> </span>(same as those defined in <i>Cell state quality control </i>step 1, but without the percent expression filter), we calculated, for each gene</p><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">s<span class="s67">;</span>i</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;text-align: justify;"><i>j </i>in g<span class="s50">s</span>, the average fold change between the cells assigned to state <i>s </i>and the cells assigned to other states, obtained after log2 trans- forming the normalized counts and scaling to zero mean and unit variance across cells. The average log2 fold change, <i>FCs</i>, of marker genes g<span class="s50">s</span><span class="s48"> </span>in state <i>s </i>was compared with the corresponding 100 values, <i>FCshuf </i><span class="s9">; </span>1%<i>i</i>%100, obtained by independently shuffling the expression values of each gene in g<span class="s50">s</span><span class="s48"> </span>across all cells and solving <span style=" color: #0097CF;">Equation 8 </span>one hundred times. We then calculated a z-score to quan-</p><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">s<span class="s67">;</span>i</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;line-height: 9pt;text-align: justify;">tify the probability that <i>FCs </i>is significantly higher than <i>FCshuf </i>, using the formula:</p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">s<span class="s44">;</span>i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s7" style="text-indent: 0pt;line-height: 8pt;text-align: left;">sd</p><p style="text-indent: 0pt;text-align: left;"/><p class="s45" style="text-indent: 0pt;line-height: 10pt;text-align: left;"><span class="s68">FC</span>s<span class="s44">;</span>i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s7" style="padding-bottom: 4pt;padding-left: 12pt;text-indent: 0pt;line-height: 19pt;text-align: center;">FCs <span class="p">— </span>mean<span class="s52">（</span>FCshuf <span class="s52">）</span></p><p style="padding-left: 268pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="103" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_183.png"/></span></p><p class="s59" style="padding-left: 12pt;text-indent: 0pt;line-height: 82%;text-align: center;"><span class="s69">z</span><span class="s70">s  </span><span class="s71">=             </span>（   <span class="s45">shuf </span>）</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark176" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For scRNA-seq datasets where more than 2,500 cells from a particular cell type were available, the procedure was applied on a set of 2,500 randomly selected cells. This was done to mitigate imbalances in the number of cells per cell type and for the sake of compu- tational efficiency. However, even without this step, results were comparable (data not shown). The resulting z-scores were com- bined across scRNA-seq datasets using Stouffer’s method (</a>Stouffer et al., 1949<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a>Tables S3 <a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S6<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Signature matrix validation</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;">To validate the hierarchical signature matrix strategy presented above (<i>Signature matrix design</i><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), we created artificial tumor profiles using single-cell transcriptomes obtained from five scRNA-seq tumor atlases spanning three epithelial tumor types: CRC, HNSCC, and NSCLC (</a><a href="#bookmark203" class="s10">Figure </a><span style=" color: #0097CF;">S1</span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B; </a><a href="#bookmark98" class="s10">Table </a><span style=" color: #0097CF;">S1</span><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). From each scRNA-seq dataset, we simulated mixtures of defined fractions for the 12 cell types analyzed in this work (</a><a href="#bookmark92" class="s10">Figure </a><span style=" color: #0097CF;">1</span>). First, we calculated means m<span class="s72">1</span><span class="s9">; </span>.<span class="s9">;  </span>m<span class="s72">12</span><span class="s66"> </span>and standard deviations s1<span class="s9">; </span>.<span class="s9">;  </span>s12 for each cell type <i>i</i>1<span class="s9">; </span>.<span class="s9">;  </span><i>i</i>12</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: left;">using fractions imputed by CIBERSORTx for each of the same three tumor types in the discovery cohort. Next, we sampled cell frac-</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 107%;text-align: justify;">tions from a Gaussian distribution in which Nðm  = m<span class="s50">i</span><span class="s48"> </span><span class="s9">;</span>s  = maxð0<span class="s9">:</span>25<span class="s9">;</span>3s<i>i</i>ÞÞ, for each cell type <i>i</i>. We then set negative fractions to 0 and re-normalized fractions to sum to 1 across all 12 cell types. Using the resulting fractions, we sampled 1,000 cells per dataset with replacement, aggregated their transcriptomes in non-log linear space into a pseudo-bulk mixture, then scaled the resulting mixture to TPM. Overall, 100 pseudo-bulk mixtures were created per dataset. CIBERSORTx deconvolution was applied to these mixtures as</p><p class="s24" style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">described above, but without batch correction. Performance is shown for 55 cell type/dataset pairs (</a>Figure S1<span style=" color: #000;">B), excluding two pairs with high gene dropout within CIBERSORTx signature matrices (&gt; 40% of genes with zero expression).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Validation of expression purification</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To evaluate the cell type-specificity of purified expression profiles within G (</a><span style=" color: #0097CF;">Figure S2</span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A), we reanalyzed seven published scRNA-seq atlases of human carcinomas (</a><span style=" color: #0097CF;">Table S1</span>). First, we restricted scRNA-seq data to protein-coding genes (GENCODE v24). Next, we scaled each log2-adjusted gene <i>j </i>to unit variance within each dataset. We then calculated, for each gene <i>j</i>, the average log2 fold change (FC) between each cell and the remaining cell types combined. Next, we averaged FCs for each cell type across the seven datasets and defined cell type-specific genes that satisfy the following three requirements: (i) FC of gene <i>j </i>is &gt; 0.1 in cell type <i>i</i>; (ii) FC</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a name="bookmark39">of gene </a><i>j </i>is maximized in cell type <i>i</i>; (iii) 2nd highest FC of gene <i>j </i><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">is at least 0.1 lower than its maximum FC. We calculated pairwise Jaccard indices between detectably expressed genes imputed by CIBERSORTx and cell type-specific genes identified from scRNA- seq data. This process was repeated for each cell type, yielding the 12 3 12 Jaccard similarity matrix shown in </a><span style=" color: #0097CF;">Figure S2</span>A.<a name="bookmark40">&zwnj;</a><a name="bookmark41">&zwnj;</a><a name="bookmark42">&zwnj;</a><a name="bookmark43">&zwnj;</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Assessment of EcoTyper using simulated tumors</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To investigate the technical performance of EcoTyper, we devised a simulation framework (</a>Figures S1<span style=" color: #000;">D–S1J) for creating pseudo- bulk gene expression profiles (GEPs) in which pre-defined cell states from scRNA-seq data are aggregated from individual cells, normalized to account for dropout, and mixed in defined proportions.</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;line-height: 10pt;text-align: justify;">Cell states for simulated tumors</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">As input, we selected a large scRNA-seq atlas of NSCLC tumors and adjacent normal tissues generated by Lambrechts and col- leagues (</a><span style=" color: #0097CF;">Lambrechts et al., 2018</span>). All author-supplied states and annotations were used (<i>scRNA-seq tumor atlases </i><a href="#bookmark108" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">below) with the following exceptions. First, cell types with &lt; 2 states were excluded. Next, to avoid imbalances in state representation, states with more than 250 cells were down-sampled without replacement to 250 cells. Following this step, single-cell transcriptomes of lymphoid cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells were each loaded into a separate Seurat object (Seurat v3.1.3 (</a><span style=" color: #0097CF;">Butler et al., 2018</span><a href="#bookmark177" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><span style=" color: #0097CF;">Stuart et al., 2019</span>)). To focus on well-resolved states within each Seurat object, we calculated the average log2 GEP of variable features within each state (<i>FindVariableFeatures </i>with default parameters), applied hierarchical clustering to the resulting GEPs (<i>hclust </i>in R <i>stats </i>with Pearson correlation), and determined the number of clusters via silhouette width maximization. States within a given cluster were assessed for marker genes using <i>FindMarkers </i>(<i>only.pos </i>= TRUE, <i>min.pct </i>= 0.1, <i>logfc.threshold </i>= 0.05) and were merged if (i) their median pairwise correlation was &gt; 0.9 and (ii) their top 10 marker genes ranked by log2 fold change (FC) were not uniformly significant (<i>Q </i>&lt; 0.05). Next, we applied <i>FindMarkers </i><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">to each Seurat object (as above) in order to define the top 10 marker genes per state by FC, then averaged the FC among all 10 genes, yielding FC10. States with FC10 &lt; 1 or FC10 at least two- fold lower than the state with highest FC10 were eliminated. In all, 26 states spanning seven cell types with at least two states each satisfied these criteria (</a><span style=" color: #0097CF;">Figure S1</span>E).</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Adjustment for cell type-specific dropout</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark152" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We observed considerable cell type-specific variation in the fold change between marker genes (FC10 above), consistent with differences in RNA content per cell type (</a><span style=" color: #0097CF;">Monaco et al., 2019</span>) and technical dropout. To address such biases, we determined for each gene <i>j </i>in state <i>s</i>, the fraction of values with zero expression; the average fraction across all genes was denoted <i>fs</i>. We then sorted each log2-adjusted gene <i>j </i>in state <i>s </i>by decreasing expression and calculated the mean m<span class="s50">js </span>and standard deviation s<i>js </i>of</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: justify;">expression values within the top 1 — <i>fs </i>fraction of cells. Next, for each gene <i>j </i>in state <i>s</i>, we replaced zeros by sampling from</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;">maxð0<span class="s9">; </span>Nðm  =  m<span class="s50">is</span><span class="s48"> </span><span class="s9">; </span>s  =  s<i>is</i>ÞÞ. Finally, to create a pseudo-bulk GEP P<i>s </i>for each state <i>s</i>, we averaged the log2 expression of all cells assigned to  <i>s</i>, anti-logged the result, and normalized the resulting profile to sum to TPM.</p><p class="s31" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: justify;">Assembly of pseudo-bulk mixtures</p><p class="s66" style="text-indent: 0pt;line-height: 6pt;text-align: left;">*</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 107%;text-align: justify;">To create biologically plausible mixtures from each state profile P<i>s</i>, we obtained the means m1, ., m7 and standard deviations s1, ., s7 of each cell type <i>i </i>from the discovery cohort (<i>Cell type fraction estimation </i>above). We then sampled fractions F* for each cell type <i>i </i>from a Gaussian distribution with Nðm  = m<span class="s50">i</span><span class="s48"> </span><span class="s9">;</span>s  = 3s<i>i</i>Þ. Fractions were resampled until higher than 0.001. We also sampled state frac- tions S  for each cell type <i>i </i>from a Gaussian distribution with mean and standard deviation equal to 1 divided by number of states in <i>i</i>.</p><p style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Negative values were set to 0, and the resulting state fractions were normalized to sum to 1 for each cell type.</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 8pt;line-height: 110%;text-align: justify;">Pseudo-bulk GEPs were assembled from cell type fractions F*, cell state fractions S*, and cell state GEPs P in two stages. In the first stage, we created cell type-specific GEPs for each sample <i>w</i>. To make such profiles more realistic, for each state <i>s </i>in cell type <i>i </i>and for each sample <i>w</i>, we first simulated a transcriptome-wide noise vector by sampling from Nðm  = 0<span class="s9">;</span>s  = s<i>i</i>Þ. We then added the resulting vector to S*<span class="s50">sw</span><span class="s48"> </span>, the fraction of state <i>s </i>in sample <i>w</i>, yielding v<i>sw</i>. After setting all negative values in v<i>sw </i>to 0, we multiplied P<i>s </i>by</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 77%;text-align: justify;">v<i>sw</i>, yielding GEP P*<span class="s50">s</span><span class="s48">w</span>. Next, for each sample <i>w</i>, we combined all P*<span class="s50">sw</span><span class="s48"> </span>GEPs within a given cell type <i>i </i>by summation and normalized the</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">resulting cell type-specific GEP to TPM, yielding C<i>iw</i>. In the second stage, we repeated this process at the cell type level. Namely, we created a noise vector for each cell type <i>i </i>and sample <i>w </i>using the same process described above. We then added the noise vector to F*<span class="s50">i</span><span class="s48">w</span>, which denotes the fraction of cell type <i>i </i>in sample <i>w</i>. After setting negative values in the resulting vector v<i>iw </i>to 0, we multiplied C<i>iw</i></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: justify;">by v<i>iw</i>. The resulting GEPs were combined by summation for each sample and then normalized to TPM.</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Baseline performance and evaluation</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For </a>Figures S1<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F–S1H, we generated 500 pseudo-bulk mixtures as described above. We then ran EcoTyper in three ways: (i) default (</a>Figures S1<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F–S1H); (ii) without gene-level standardization and AFI filtering (</a>Figure S1<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F); and (iii) with gene-level standardization but without AFI filtering (</a>Figure S1<span style=" color: #000;">F).</span></p><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">s</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To quantify the performance of EcoTyper in </a><span style=" color: #0097CF;">Figures S1</span>F–S1J, we used reference-guided annotation (<i>Cell state and ecotype recov- ery</i><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) to label the scRNA-seq atlas shown in </a><span style=" color: #0097CF;">Figure S1</span>E. Next, we calculated pairwise Jaccard indices between known states (denoted s<i>K </i>) and predicted states (denoted s<i>P</i>) for all cells assigned to EcoTyper states. Jaccard indices were set to 0 for pairs of states in which the overlap was not significant (p &gt; 0.05, hypergeometric test). Separately, we calculated pairwise Pearson correlations be- tween the fractions determined by EcoTyper for each identified state and the fractions known for each ground truth state in pseudo- bulk mixtures. For each cell type <i>i</i>, we considered state <i>s </i>in s<i>K </i>(denoted s<i>K </i>) to be detected if (i) its Jaccard index was positive and</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: justify;">mutually highest with state <i>q </i>in s<i>P </i>(denoted s<i>P</i>) for all states in cell type <i>i</i>; and (ii) s<i>K </i>had the highest Pearson correlation with s<i>P</i>, for all</p><p class="s48" style="padding-left: 226pt;text-indent: 0pt;line-height: 6pt;text-align: left;">q s q</p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 9pt;text-align: left;">states s<i>K </i>in cell type <i>i</i>.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s31" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark44">Limit of detection and collinearity analyses</a><a name="bookmark45">&zwnj;</a><a name="bookmark46">&zwnj;</a><a name="bookmark47">&zwnj;</a></p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For </a>Figures S1<span style=" color: #000;">I and S1J, we quantified EcoTyper performance as a function of (i) the mean abundance of spike-in subpopulations and</span></p><p style="text-indent: 0pt;line-height: 8pt;text-align: left;">ð Þ</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(ii) collinearity in cell state abundances. We selected fibroblasts and dendritic cells for these analyses, as they are contrasted by their relative disparity in abundance and function. For the limit of detection analysis in </a><span style=" color: #0097CF;">Figure S1</span>I, we implemented the procedure described in <i>Assembly of pseudo-bulk mixtures </i>with the following exceptions. First, for the parent cell type <i>i </i>of the spike-in state <i>s</i>, we sampled cell type fractions from N m = 0<span class="s9">:</span>05<span class="s9">; </span>s = 0<span class="s9">:</span>15 . Then, given a mean spike-in fraction q, we sampled state fractions</p><p class="s66" style="text-indent: 0pt;line-height: 6pt;text-align: left;">0<span class="s67">:</span>05</p><p style="text-indent: 0pt;text-align: left;"/><p class="s66" style="text-indent: 0pt;line-height: 6pt;text-align: left;">0<span class="s67">:</span>05</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;text-align: justify;">from min<span class="s73">(</span>1<span class="s9">;</span>max<span class="s73">(</span>0<span class="s9">;</span>N<span class="s73">(</span>m  = <span class="s74">  </span><u>q  </u><span class="s9">;</span>s  = <span class="s74">  </span><u>3q </u><span class="s73">111</span>. To adjust for any deviation from  q, the fractions of state <i>s </i>were multiplied by a constant</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 9pt;text-align: left;">scaling factor to ensure that their mean across pseudo-bulk mixtures was exactly q. The remaining state fractions from cell type <i>i </i>were</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">sampled as described in <i>Assembly of pseudo-bulk mixtures </i>and scaled to sum to one minus the fraction of <i>s </i>in sample <i>w</i>. We then repeated the process described in <i>Assembly of pseudo-bulk mixtures </i>at the cell type level for all cell types except cell type <i>i</i>.</p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the collinearity analysis in </a><a href="#bookmark203" class="s10">Figure </a><span style=" color: #0097CF;">S1</span>J, we assessed fibroblasts (C2 versus C6) and dendritic cells (C5 versus C12). To achieve a predefined Pearson correlation level <span class="s76">l </span>between the fractions of cell states <i>s</i>1 and <i>s</i>2 (e.g., fibroblasts C2 versus C6) within cell type <i>i</i>, we  performed  the  following  three-step  procedure.  In  the  first  step,  we  sampled  cell  type  fractions  F*<span class="s50">i</span><span class="s48">    </span>for  cell  type  <i>i  </i>from</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: left;">Nðm = 0<span class="s9">:</span>05<span class="s9">; </span>s = 0<span class="s9">:</span>15Þ for all tumors (in this case, n = 500). The remaining cell types and states were sampled as described for the limit</p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;">of detection analysis. In the second step, for states <i>s</i>1 and <i>s</i>2, we sampled state fractions from a bivariate normal distribution with mean and covariance matrix:</p><p style="text-indent: 0pt;line-height: 6pt;text-align: right;">1</p><p class="s9" style="text-indent: 0pt;line-height: 8pt;text-align: right;">;</p><p style="text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;line-height: 8pt;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s53" style="padding-top: 8pt;padding-bottom: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">(<u> </u>1</p><p style="text-indent: 0pt;line-height: 9pt;text-align: left;">m =</p><p style="padding-left: 223pt;text-indent: 0pt;line-height: 9pt;text-align: left;"/><p class="s78" style="padding-bottom: 3pt;padding-left: 17pt;text-indent: 0pt;line-height: 14pt;text-align: left;">0<u> </u><span class="s75">1 </span><span class="p">    </span><span class="s74">    </span></p><p class="s7" style="text-indent: 0pt;line-height: 8pt;text-align: left;">n</p><p style="padding-left: 28pt;text-indent: 0pt;line-height: 8pt;text-align: left;"/><p style="padding-left: 6pt;text-indent: 0pt;text-align: left;"><i>r </i>3<u> 1</u> <span class="s78">1</span></p><p class="s7" style="text-indent: 0pt;line-height: 8pt;text-align: left;">n</p><p style="text-indent: 0pt;text-align: left;"/><p class="s53" style="text-indent: 0pt;line-height: 9pt;text-align: left;">@</p><p style="text-indent: 0pt;text-align: left;"/><p class="s9" style="text-indent: 0pt;line-height: 9pt;text-align: left;">; <span class="p">s =</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s32" style="padding-left: 15pt;text-indent: 0pt;line-height: 5pt;text-align: left;">2                  <span class="s80">      </span></p><p class="s53" style="text-indent: 0pt;line-height: 9pt;text-align: left;">B</p><p style="text-indent: 0pt;text-align: left;"/><p class="s32" style="text-indent: 0pt;line-height: 4pt;text-align: left;">2</p><p class="s53" style="padding-left: 4pt;text-indent: 0pt;line-height: 9pt;text-align: left;">C</p><p style="text-indent: 0pt;text-align: left;"/><p class="s53" style="text-indent: 0pt;line-height: 9pt;text-align: left;">A</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 12pt;text-indent: 0pt;line-height: 41%;text-align: center;"><i>n   n              </i><i>r  </i>3  <span class="s54">1   </span> 1</p><p class="s82" style="padding-left: 196pt;text-indent: 0pt;text-align: center;">n<span class="s32">2     </span>n<span class="s32">2</span></p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 8pt;line-height: 107%;text-align: justify;">Here, <i>r </i>was initially set to the target correlation between states <i>s</i>1 and <i>s</i>2 (i.e., <span class="s76">l</span>) and <i>n </i>was set to the number of states in <i>i</i>. Next, for each pair of state fractions in sample <i>w</i>, <i>fs</i>1<span class="s9">;</span><i>w </i>and <i>fs</i>2<span class="s9">;</span><i>w</i>, we set negative values to 0 and rescaled <i>fs</i>1<span class="s9">;</span><i>w </i>and <i>fs</i>2<span class="s9">;</span><i>w </i>to sum to 1 if their sum was &gt; 1. We then scaled the remaining cell states of cell type <i>i </i>by the quantity 1 — ð<i>fs</i>1<span class="s9">;</span><i>w </i>+  <i>fs</i>2<span class="s9">;</span><i>w</i>Þ. In the third step, we repeated step 2 by performing a grid search in the space ½ —1<span class="s9">;  </span>1] using increments of 0.01 to identify the value of <i>r </i>that produced the closest corre- lation with <span class="s76">l</span>, i.e., <i>cor</i>ð<i>fs</i>1 3F*<span class="s50">i</span><span class="s48"> </span><span class="s9">;  </span><i>fs</i>2 3F*<span class="s50">i</span><span class="s48"> </span>Þz<span class="s76">l</span>.</p><p class="s22" style="padding-top: 9pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">External datasets</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Full details of each dataset used, including data type, sample type, source and normalization approach are available in </a>Table S1<span style=" color: #000;">.</span></p><p class="s31" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;">Discovery cohort</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark179" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Pre-processed bulk RNA-seq profiles of tumor and adjacent normal tissue specimens from The Cancer Genome Atlas (TCGA) were downloaded (</a>Tatlow and Piccolo, 2016<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and scaled to TPM. These data were then curated to assemble a discovery cohort based on the following three criteria. First, to focus on tumor samples of epithelial origin, we excluded brain cancers (GBM, LGG), blood can- cers (DLBC, LAML, LCML), sarcomas (SARC, UCS), and melanomas (SKCM, UVM) (</a>Table S2<span style=" color: #000;">). Second, we tested whether house- keeping genes were uniformly expressed across tumor types. By performing hierarchical clustering (</span><span class="s7">hclust </span><span style=" color: #000;">in the R </span><span class="s7">stats </span><a href="#bookmark117" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">package with complete linkage and Euclidean distance) on the log2 expression matrix of 11 previously defined housekeeping genes (</a><a href="#bookmark117" class="s10">Eisen- </a>berg and Levanon, 2013<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), we identified two robust clusters using the silhouette method (</a>Figure S1<span style=" color: #000;">K). The minority cluster, which con- sisted of five tumor types (ACC, KICH, KIRC, KIRP and PCPG), was excluded. Next, we tested whether CIBERSORTx coupled with TR4 (see </span><span class="s7">Signature matrix design and Cell type fraction estimation</span><a href="#bookmark192" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) could reliably impute epithelial composition across tumor types via comparison to ESTIMATE (</a>Yoshihara et al., 2013<span style=" color: #000;">). Although Pearson correlation coefficients between the two methods were gener- ally high (</span><span class="s7">r </span><a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">&gt; 0.8 for 90% of tumor types), mean squared errors (MSEs) were variable (</a>Figure S1<a href="#bookmark203" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">L). Using K-means and silhouette maxi- mization to jointly cluster Pearson correlation coefficients and MSEs, we identified a single outgroup (THYM, LIHC, MESO, TCGT) with high MSE which we omitted from further analysis (</a>Figure S1<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">L; </a>Table S2<a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Finally, to mitigate the influence of technical variation on deconvolution results, we removed samples if they were (i) flagged as poor quality by prior studies (</a>Newman et al., 2019<a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><a href="#bookmark183" class="s10">Thorsson </a>et al., 2018<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) or (ii) generated on an Illumina platform other than HiSeq2000 v2, which encompassed ~85% of the remaining evaluable tumors. The final discovery cohort, which was uniformly processed and standardized, consisted of 16 carcinomas spanning 5,946 tumor and 529 adjacent normal samples (</a>Tables S1 <a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S2<span style=" color: #000;">).</span></p><p class="s31" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">scRNA-seq tumor atlases</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We compiled and curated scRNA-seq tumor atlases from seven datasets covering breast cancer, colorectal cancer, head and neck squamous cell carcinoma, non-small cell lung cancer (NSCLC), and melanoma (</a>Table S1<span style=" color: #000;">). All datasets were pre-processed and scaled to TPM or counts per million (CPM), as appropriate. Author-supplied cell type assignments were used with the following exceptions:</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s36" style="padding-left: 65pt;text-indent: 0pt;text-align: left;">d <a href="#bookmark103" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the breast cancer dataset of </a><span class="s24">Azizi et al. (2018)</span><span class="p">, cells labeled as regulatory T cells were grouped with total CD4 T cells.</span></p><p class="s24" style="padding-top: 1pt;padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: left;"><span class="s36">d </span><a href="#bookmark141" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the colorectal cancer dataset of </a>Lee et al. (2020)<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, the authors’ clusters were mapped to cell types using the schema in </a>Table S1<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><a name="bookmark48"><span class="s36">d </span></a><a href="#bookmark161" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the head and neck squamous cell carcinoma dataset of </a><span style=" color: #0097CF;">Puram et al. (2017) </span><a href="#bookmark185" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and the melanoma dataset of </a><span style=" color: #0097CF;">Tirosh et al. (2016)</span>, T cells were not divided into CD8 and CD4 T cells by the authors. Thus, we annotated them using the <i>FindClusters </i>function in <i>Seurat </i>v2.3.4, applied on the first 20 principal components of each dataset, with the resolution parameter set to 0.1, and other parameters set to default. In both datasets, CD8 and CD4 T cell clusters distinguished by high expression of <i>CD8A/CD8B </i>and <i>CD4/IL7R</i>, respectively, were resolved.<a name="bookmark49">&zwnj;</a><a name="bookmark50">&zwnj;</a><a name="bookmark51">&zwnj;</a><a name="bookmark52">&zwnj;</a></p><p class="s36" style="padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;">d <a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the NSCLC dataset of </a><span class="s24">Lambrechts et al. (2018)</span><span class="p">, cell clusters identified by the authors were mapped to phenotypic labels as follows: For clusters defined in Figure 4B Lambrechts et al., clusters 1, 2, 5, and 7 were assigned to B cells, clusters 3 and 6 to plasma cells, and cluster 4 to mast cells. For clusters defined in Figure 4F of Lambrechts et. al., clusters 1, 2, 3, 4, 6, 8, 10, and 11 were assigned to monocytes/macrophages, clusters 5, 9, and 12 to dendritic cells, and cluster 7 to neutrophils. For clusters defined in Figure 5B of Lambrechts et. al., clusters 2, 4, 5, and 8 were assigned to CD8 T cells, clusters 1, 3, and 9 to CD4 T cells, and cluster 6 to NK cells.</span></p><p class="s24" style="padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><span class="s36">d </span><a href="#bookmark140" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the NSCLC dataset of </a>Laughney et al. (2020)<span style=" color: #000;">, cells annotated as ‘‘Breg’’ were assigned to B cells; ‘‘IG’’ to plasma cells; ‘‘NK’’ and ‘‘NKT’’ to NK cells; and ‘‘Th,’’ ‘‘Tm’’ and ‘‘Treg’’ to T cells. T cells were subdivided into CD4 and CD8 T cells using the </span><span class="s7">FindClusters </span><span style=" color: #000;">function in </span><span class="s7">Seurat </span><a href="#bookmark108" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">v2.3.4 (</a>Butler et al., 2018<a href="#bookmark177" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Stuart et al., 2019<span style=" color: #000;">), applied on the first 20 principal components, with the resolution parameter set to 0.1, and other parameters set to default.</span></p><p class="s36" style="padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;">d <a href="#bookmark196" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">In the NSCLC dataset of </a><span class="s24">Zilionis et al. (2019)</span><span class="p">, CD4 T cell subsets, dendritic cell subsets, and monocyte/macrophage subsets were merged into their corresponding parental lineages. Only cells collected from tumors were analyzed.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Clinically-annotated tumor transcriptomes</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We analyzed 9,062 pre-normalized carcinoma transcriptomes from the Prediction of Cancer Outcomes using Genomic Profiles (PRECOG) database, along with additional datasets listed in </a>Table S1<a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, all of which were processed according to the PRECOG work- flow (</a>Gentles et al., 2015<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). All datasets (n = 35) were filtered to only include malignancies with matching tumor types in the discovery cohort and with at least 100 samples and available overall survival data (</a>Table S1<span style=" color: #000;">). For Figures </span>S4<a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C and </a>S5<a href="https://portal.gdc.cancer.gov/" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" target="_blank">E, we downloaded pre- processed bulk RNA-seq and Affymetrix HT Human Genome U133A microarray expression profiles of squamous cell lung cancer (LUSC) and ovarian cancer (OV) patients from the GDC portal (</a>https://portal.gdc.cancer.gov/<span style=" color: #000;">).</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Healthy tissue transcriptomes</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Raw feature counts for GTEx samples (GSE86354) were downloaded and filtered to retain the seven normal tissue types that had a corresponding tumor type in the discovery cohort (</a>Table S1<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To address differences in normalization between TCGA and GTEx (</a><a href="#bookmark96" class="s10">Fig- </a>ure 5<a href="#bookmark101" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B), we integrated and co-normalized the discovery cohort and GTEx using the following procedure (</a>Aran et al., 2017<a href="#bookmark162" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). First, we merged count matrices from TCGA (GSE62944) (</a>Rahman et al., 2015<a href="#bookmark166" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and GTEx, applied upper quartile normalization using the EDA- Seq package in R (</a>Risso et al., 2011<span style=" color: #000;">), calculated CPM, then log2-transformed the data. We then determined the unit variance scaling parameters specific for each gene in each TCGA tumor type necessary to bring the corresponding GTEx tissue type into the same</span></p><p style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">space. Specifically, for a given gene <i>j</i>, we calculated its mean m<span class="s50">jt </span>and variance s<i>jt </i>across tumor samples from tumor type <i>t</i>. Then, the log2 expression level <i>ejw </i>of gene <i>j </i>in the GTEx sample <i>w</i>, from the tissue matching the tissue-of-origin for tumor type <i>t</i>, was normalized using the formula:</p><p class="s85" style="padding-top: 3pt;padding-left: 12pt;text-indent: 0pt;line-height: 12pt;text-align: center;"><span class="s82">e</span><span class="s32">new</span><span class="s83">)</span><span class="s84">e</span>j<span class="s86">w  </span><span class="s87">—</span><span class="s75"> m</span>jt</p><p class="s45" style="padding-left: 12pt;text-indent: 0pt;line-height: 55%;text-align: center;">jw               <span class="s83">s</span><span class="s88">jt</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 68pt;text-indent: 0pt;text-align: left;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">The resulting set of 1,423 normalized GTEx samples (</a>Table S1<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) was used for the analysis described in </a>Figure 5<span style=" color: #000;">B.</span></p><p class="s31" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">Immunotherapy</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For analyses related to ICI response (</a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C), we downloaded clinically-annotated bulk tumor transcriptomes from patients with metastatic urothelial carcinoma (bladder cancer) and metastatic melanoma (</a>Table S1<span style=" color: #000;">). The former was generated by the IMvigor210 phase II atezolizumab trial and obtained via the R library </span><span class="s7">Imvigor210CoreBiologies </span><a href="#bookmark148" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">version 1.0.0 (</a>Mariathasan et al., 2018<a href="#bookmark156" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Raw read counts were converted to TPM. For the latter, normalized count data and clinical annotations were downloaded and converted to TPM (</a>Nathanson et al., 2017<a href="#bookmark145" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Liu et al., 2019<a href="#bookmark164" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Riaz et al., 2017<a href="#bookmark187" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Van Allen et al., 2015<span style=" color: #000;">).</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Spatial transcriptomics</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="https://www.10xgenomics.com/spatial-transcriptomics/" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" target="_blank">Pre-processed spatial transcriptomics datasets of breast, colorectal, and ovarian carcinoma specimens were downloaded from 10x Genomics (</a>https://www.10xgenomics.com/spatial-transcriptomics/<a href="#bookmark184" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Melanoma spatial transcriptomics data were obtained from </a>Thrane et al. (2018) <a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(</a>Table S1<span style=" color: #000;">).</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Early carcinoma development</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark180" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Pre-processed expression profiles and clinical data of premalignant lesions that either progressed to lung squamous cell carcinoma or spontaneously regressed (</a>Teixeira et al., 2019<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) were downloaded from the Gene Expression Omnibus (GSE94611) (</a>Figure 6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F; </a>Table S1<a href="#bookmark149" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Raw expression profiles of lung tissue along different precursor stages of lung squamous cell carcinogenesis (</a><a href="#bookmark149" class="s10">Mascaux </a>et al., 2019<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) were downloaded from the Gene Expression Omnibus (GSE33479) (</a>Figure S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">I; </a>Table S1<span style=" color: #000;">). Raw Agilent array files were loaded using function </span><span class="s7">read.maimages </span><span style=" color: #000;">from </span><span class="s7">limma v3.36.2 </span><a href="#bookmark167" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(</a>Ritchie et al., 2015<span style=" color: #000;">), with parameters </span><span class="s7">source = agilent.median, green.only = T, columns = list(G = rMedianSignal, Gb = rBGMedianSignal), annotation = c(Row, Col, ProbeName, SystematicName), </span><span style=" color: #000;">background- corrected using function </span><span class="s7">backgroundCorrect </span><span style=" color: #000;">from </span><span class="s7">limma</span><span style=" color: #000;">, with parameters </span><span class="s7">method = normexp, offset = </span><a href="#bookmark143" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">1, quantile normalized and annotated using Gemma (</a>Lim et al., 2021<span style=" color: #000;">). Probes that mapped to the same gene were averaged.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark53">UMAP visualization of cell states</a><a name="bookmark54">&zwnj;</a><a name="bookmark55">&zwnj;</a><a name="bookmark56">&zwnj;</a></p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To create the UMAP projections in </a><span style=" color: #0097CF;">Figures 2</span><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and </a><span style=" color: #0097CF;">S2</span>F, we first calculated for each cell type, a Euclidean distance matrix between cell state abundances in H* (<i>Cell state discovery</i>). This was done for all TCGA samples assigned to cell states after quality control filtering (<i>Cell state quality control</i>). A UMAP embedding was then created for each cell type using the resulting distance matrix and otherwise default parameters (<i>umap </i><a href="#bookmark150" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">package v0.2.0.0 in R (</a><span style=" color: #0097CF;">McInnes et al., 2018</span>)).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;line-height: 10pt;text-align: left;">Enrichment of cell states in known phenotypes</p><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">s<span class="s67">;</span>i</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;text-align: justify;">To determine whether cell states are over-represented in a known phenotype <span class="s89">P </span>(e.g., adjacent normal tissue), we performed the following procedure. First, we counted the number <i>Ns </i>of samples/cells of phenotype <span class="s89">P </span>assigned to state <i>s </i>by Ecotyper. Then, for 1,000 iterations, we calculated the number <i>Nshuf </i>of samples/cells of phenotype <span class="s89">P </span>assigned to state <i>s </i>after shuffling the cell state assignment labels at iteration <i>i</i>, thus generating a null distribution. Based on this distribution, we calculated a z-score:</p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">s<span class="s44">;</span>i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s53" style="text-indent: 0pt;line-height: 76%;text-align: left;">）<span class="s90">N</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s7" style="padding-bottom: 4pt;padding-left: 12pt;text-indent: 0pt;line-height: 19pt;text-align: center;">Ns <span class="p">— </span>mean<span class="s52">（</span>Nshuf <span class="s52">）</span></p><p style="padding-left: 273pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="90" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_184.png"/></span></p><p class="s7" style="padding-left: 59pt;text-indent: 0pt;line-height: 3pt;text-align: right;">zs <span class="p">=</span></p><p class="s7" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">sd<span class="s52">（</span></p><p class="s45" style="padding-top: 2pt;padding-left: 4pt;text-indent: 0pt;line-height: 82%;text-align: left;">shuf s<span class="s44">;</span>i</p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 108%;text-align: justify;"><span style=" color: #000;">Z-scores &gt; 1.96 (p &lt; 0.05) were considered significantly enriched in phenotype </span><span class="s89">P </span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">(</a>Table S3<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). This procedure was applied to calcu- late cell state enrichment in adjacent normal tissues and adenocarcinomas versus squamous cell carcinomas (</a>Figures 2<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D and </a>S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D– S3G; </a>Table S3<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Comparison of scRNA-seq annotation methods</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the analysis in </a>Figures S3<a href="#bookmark100" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E–S3G, we benchmarked EcoTyper against the top three classifiers from a previous comparative anal- ysis of scRNA-seq annotation tools (Figure 8 of (</a>Abdelaal et al., 2019<a href="#bookmark112" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">)): SVM (</a>Cortes and Vapnik, 1995<a href="#bookmark178" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), SingleCellNet (</a><a href="#bookmark178" class="s10">Tan and Ca- </a>han, 2019<a href="#bookmark137" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and scmapcell (</a>Kiselev et al., 2018<a href="#bookmark100" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Using code supplied by the authors (</a>Abdelaal et al., 2019<a href="#bookmark100" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), we first verified the output of each classifier on selected datasets with known performance (</a>Abdelaal et al., 2019<a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (data not shown). We then trained each clas- sifier on 69 EcoTyper states using digitally-purified expression profiles from the TCGA discovery cohort. To evaluate performance, we employed a previously published scRNA-seq dataset of non-small cell lung cancer (</a>Lambrechts et al., 2018<span style=" color: #000;">) for which unambiguous ground truth labels were available: (i) cells obtained from tumor versus adjacent normal tissues, and (ii) cells obtained from adenocarcinoma versus squamous cell carcinoma specimens. Tissue-of-origin enrichments were determined as described above (</span><span class="s7">Enrichment of cell states in known phenotypes</span><span style=" color: #000;">). When applied with default parameters, SingleCellNet and scmapcell each assigned cells to only 16 and 11 EcoTyper states, respectively. Therefore, to minimize the fraction of unassigned states, we made the following modifications: For SingleCellNet, we set </span><span class="s7">nRand </span><span style=" color: #000;">= 0 (</span><span class="s7">sc_makeClassifier </span><span style=" color: #000;">function); for scmapcell, we set </span><span class="s7">threshold </span><span style=" color: #000;">=</span></p><ol id="l12"><ol id="l13"><li style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><p class="s24" style="display: inline;"><span style=" color: #000;">(</span><span class="s7">scmapCell2Cluster </span><a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">function). While these modifications had no appreciable impact on concordance, they uniformly improved the number of assignable states. To evaluate performance, we calculated three metrics (</a><a href="#bookmark205" class="s10">Figure </a>S3<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">G): (i) concordance, defined as the fraction of cell states for which the enrichments between the discovery cohort and scRNA-seq agree; (ii) the p value quantifying the significance of concordance (calculated as described in the captions of </a><a href="#bookmark93" class="s10">Figures </a>2<a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D and </a>S3<span style=" color: #000;">D for adjacent normal tissues and adeno- carcinoma versus squamous cell carcinoma, respectively); and (iii) the number of states recovered.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">State-specific marker genes in scRNA-seq data</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">The number of genes imputed by CIBERSORTx is adaptively determined for each cell type (</a><span style=" color: #0097CF;">Newman et al., 2019</span>). To both extend the number of marker genes assigned to each state and assess robustness, we mapped single-cell transcriptomes to EcoTyper states as described above (<i>Cell state and ecotype recovery</i><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). This was done for every scRNA-seq atlas utilized in this work (</a><span style=" color: #0097CF;">Table S1</span>). For each gene <i>j </i>and state <i>s</i>, we then considered the following metrics for prioritizing marker genes from scRNA-seq data:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 65pt;text-indent: 0pt;text-align: left;"><span class="s36">d </span>The number of scRNA-seq datasets <i>n</i>1 in which gene <i>j </i>is expressed (i.e., mean TPM/CPM &gt; 0)</p><p style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;"><span class="s36">d </span>The number of scRNA-seq datasets <i>n</i>2 for which gene <i>j </i>is assigned to state <i>s</i></p><p style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;"><span class="s36">d  </span>The ratio <i>n</i>3)<i>n</i>2<span class="s9">=</span><i>n</i>1</p><p style="padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: left;"><span class="s36">d </span>The number of distinct tumor types <i>n</i>4 for which gene <i>j </i>is assigned to state <i>s </i>for at least one scRNA-seq dataset from that tu- mor type</p><p style="padding-left: 65pt;text-indent: 0pt;text-align: left;"><span class="s36">d </span>The number of scRNA-seq datasets <i>n</i>5 for which gene <i>j </i>is significantly differentially expressed using Seurat (<i>Q </i>&lt; 0.05)</p><p class="s7" style="padding-top: 1pt;padding-left: 74pt;text-indent: -9pt;line-height: 107%;text-align: left;"><span class="s36">d  </span><span class="p">The quantity </span>n<span class="p">6) </span>MetaZj<span class="s9">;</span>s<span class="p">, defined as an aggregate z-score for differential expression of gene </span>j <span class="p">in state </span>s <span class="p">across all evaluable scRNA-seq datasets, derived as detailed below</span></p><p class="s7" style="padding-left: 74pt;text-indent: -9pt;line-height: 107%;text-align: left;"><span class="s36">d  </span><span class="p">The quantity </span>n<span class="p">7)</span>FCj<span class="s9">;</span>s<span class="p">, where </span>FCj<span class="s9">;</span>s <span class="p">is defined as the mean log2 fold change of gene </span>j <span class="p">in state </span>s <span class="p">within each evaluable scRNA-seq dataset, aggregated by mean across all evaluable datasets</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s7" style="padding-left: 53pt;text-indent: 8pt;text-align: justify;"><span class="p">For each state </span>s<span class="p">, the above quantities f</span>n<span class="p">1</span><span class="s9">; </span>n<span class="p">2</span><span class="s9">; </span><span class="p">/</span><span class="s9">; </span>n<span class="p">7g were converted to rank space and averaged across measures, yielding a composite  score  for  each  gene  </span>j  <span class="p">denoted  </span>Sj<span class="s9">;</span>s<span class="p">.  We  combined  manually  curated  genes  with  the  top  marker  genes  selected  by decreasing </span>Sj<span class="s9">;</span>s <a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">in </a><a href="#bookmark93" class="s10">Figure </a><span class="s24">2</span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C and </a><a href="#bookmark98" class="s10">Table </a><span class="s24">S4</span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; all manually curated genes are noted in </a><a href="#bookmark98" class="s10">Table </a><span class="s24">S4</span><span class="p">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s7" style="padding-top: 3pt;padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a name="bookmark57"><span class="p">Prior to calculating the seven quantities above, for each scRNA-seq dataset </span></a>d <span class="p">and cell type </span>i<span class="p">, we excluded cell states with &lt; 5 as- signed cells along with the cells mapping to them. Genes were assigned to cell states based on the state with the maximum log2 fold change relative to other states, across scRNA-seq datasets. Ties were broken by the state for which the gene had the highest average log2 fold change. Genes were excluded if the assigned state differed from the state identified by EcoTyper. To calculate </span>n<span class="p">5, we used </span>FindMarkers <a href="#bookmark108" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">in Seurat v3.1.3 (</a><span class="s24">Butler et al., 2018</span><a href="#bookmark177" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><span class="s24">Stuart et al., 2019</span><span class="p">), with </span>min.pct <span class="p">= 0.1 and </span>logfc.threshold <span class="p">= 0.05. To calculate </span>n<span class="p">6, we converted the nominal (unadjusted) p values calculated by Seurat into two-sided </span>z<span class="p">-scores, with the direction deter- mined by the orientation of the fold change of gene </span>j <span class="p">in state </span>s<span class="p">. We then aggregated </span>z<a href="#bookmark176" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">-scores across datasets by Stouffer’s method (</a><span class="s24">Stouffer et al., 1949</span><span class="p">), converted the resulting meta-z scores to two-sided p values, adjusted the resulting p values for multiple hy- pothesis testing via the Benjamini-Hochberg procedure, and converted the resulting adjusted p values back to two-sided z-scores (with direction determined as above), yielding </span>MetaZj<span class="s9">;</span>s<span class="p">.</span><a name="bookmark58">&zwnj;</a><a name="bookmark59">&zwnj;</a><a name="bookmark60">&zwnj;</a><a name="bookmark61">&zwnj;</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Validation of scRNA-seq markers</p><p class="s7" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 110%;text-align: justify;"><span class="p">To assess the robustness of the top markers selected as described above, we employed the following leave-one-out cross-validation (LOOCV) procedure. Briefly, we applied the above-mentioned marker selection strategy (</span>State-specific marker genes in scRNA-seq data<span class="p">) to all scRNA-seq datasets except one, and for each cell type </span>i <span class="p">and state </span>s<span class="p">, we assessed the top 10 marker genes, as defined by decreasing score </span>Sj<span class="s9">;</span>s<span class="p">, in the held-out dataset. To do this, we first scaled each gene in the held-out dataset to log2 expression and unit variance across all cells mapping to cell type </span>i<span class="p">. For each state </span>s <span class="p">in </span>i<span class="p">, we calculated the mean expression of each gene and averaged</span></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: justify;">these values across the 10 marker genes, yielding <i>AvgSs</i>. We then determined the state <i>s</i>0 in which <i>s</i>0 =  maxð<i>AvgSs</i>Þ. We tallied all</p><p style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">states for which <i>s</i>0 =</p><p class="s48" style="padding-left: 59pt;text-indent: 0pt;line-height: 5pt;text-align: right;">s<span class="s67">˛</span>i</p><p class="s7" style="padding-left: 1pt;text-indent: 0pt;line-height: 8pt;text-align: left;">s<span class="p">, then repeated this process for all held-out datasets, yielding a LOOCV rate for each state </span>s<span class="p">. Across all states,</span></p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">the mean LOOCV was 90%.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Cell state representation by tumor type</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark204" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For </a><span style=" color: #0097CF;">Figures S2</span>G and S2H, we quantified the enrichment of each EcoTyper state <i>s </i>in each tumor type <i>t </i>from the discovery cohort. To do this, we first classified each tumor sample by the most abundant cell state per cell type. We then applied Fisher’s exact test to determine whether the number of samples of tumor type <i>t </i>assigned to state <i>s </i><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">was higher than expected by random chance. The resulting p values were adjusted for multiple hypothesis testing across cell types using the Benjamini-Hochberg approach (</a><a href="#bookmark98" class="s10">Ta- </a><span style=" color: #0097CF;">ble S3</span>).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Enrichment of EMT genes in epithelial states</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For </a>Figures S3<a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">H and S3I, we classified the following scRNA-seq datasets using EcoTyper: </a>Lambrechts et al. (2018)<a href="#bookmark196" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, </a><a href="#bookmark196" class="s10">Zilionis et al. </a>(2019)<a href="#bookmark161" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, </a>Puram et al. (2017) <a href="#bookmark141" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>Lee et al. (2020)<a href="#bookmark99" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">. Epithelial, mesenchymal, and collagen gene sets from </a>Aiello et al. (2018) <a href="#bookmark116" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">were con- verted to human gene symbols using biomaRt v2.38 (</a>Durinck et al., 2009<span style=" color: #000;">). Metagenes were calculated by the geometric mean of each signature. For clarity of display, metagenes were scaled such that values greater than the 99th percentile of expression per dataset were set to 1.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Annotation of monocyte/macrophage states</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To perform the analysis shown in </a>Figure S3<a href="#bookmark157" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">J, we assembled previously normalized whole transcriptome data of human monocyte and macrophage subsets, including classical M0 macrophages and polarized M1/M2 macrophages (</a>Newman et al., 2015<a href="#bookmark182" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). For each cell subset, we rank-ordered each gene in the transcriptome by calculating the average log2 fold change of each cell type rela- tive to the others. To incorporate foam cell macrophages into this analysis, we used a previously published differential expression analysis of foamy versus non-foamy macrophages isolated from ApoE null mice (</a>Thomas et al., 2015<a href="#bookmark116" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Mouse gene symbols (GRCm38.p6) were converted to homologous human gene symbols (GRCh38.p13) using biomaRt v2.38 (</a>Durinck et al., 2009<span style=" color: #000;">). We evaluated the top 50 marker genes (defined in </span><span class="s7">State-specific marker genes in scRNA-seq data</span><span style=" color: #000;">) of each monocyte/macrophage state in mean log2 fold change-ordered transcriptomes using pre-ranked Gene Set Enrichment Analysis (GSEA) (</span><span class="s7">fgsea </span><a href="#bookmark136" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">version 1.14.0 (</a><a href="#bookmark136" class="s10">Kor- </a>otkevich et al., 2021<span style=" color: #000;">)), with 10,000 permutations.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Laser capture microdissection and bulk RNA-seq</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Related to </a>Figure 2<a href="#bookmark120" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E, 7 mm full tissue sections of formalin-fixed paraffin-embedded (FFPE) CRC tumors (patients 380, 393, and 406) were prepared as previously described (</a>Foley et al., 2019<a href="#bookmark120" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and areas of approximately 500 stromal cells were dissected using an Arcturus XT LCM System. Sequencing libraries were prepared as described in ‘‘Smart-3SEQ starting from FFPE tissue on Arcturus LCM’’ protocol for HS caps as previously described (</a>Foley et al., 2019<a href="#bookmark115" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) and amplified for 22 PCR cycles. Library size distribution and concentration were assessed using Agilent 2200 TapeStation and qPCR with a dual-labeled probe. Libraries were sequenced using an Illumina NextSeq 500 instrument with High Output Kit v2.5, with 76 bases for read 1 and 8 bases for the P5 index read. Base calls from the NextSeq 500 were demultiplexed and converted to FASTQ format with bcl2fastq (Illumina). The five-base unique molecular identifier (UMI) sequence and G-overhang were extracted from FASTQ data and A-tails were removed with umi_homopolymer.py. Reads were aligned and further processed to remove duplicates using STAR (</a>Dobin et al., 2013<span style=" color: #000;">). Bulk gene expression profiles were normalized to TPM.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark62">Immunohistochemistry</a><a name="bookmark63">&zwnj;</a><a name="bookmark64">&zwnj;</a></p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark205" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To confirm foamy macrophages by staining (</a>Figure S3<span style=" color: #000;">K), 4 mm tissue sections of CRC patients 380 and 393 were deparaffinized, rehydrated, stained with H&amp;E, and imaged at 20 3 magnification. Subsequently, slide coverslips were removed, and antigen retrieval was performed in EDTA pH 9 buffer for 5 min in 95oC in a pressure cooker. Slides were next stained with CD68 XP monoclonal anti- body (1/200, rabbit, D4B9C, Cell Signaling Technology Cat# 76437, RRID: AB_2799882) and imaged at 20 3 magnification.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Survival analyses and bulk tumor subtyping</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the analyses in </a>Figures 3<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, </a>5 <a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, and </a>Table S5<span style=" color: #000;">, we applied univariable Cox proportional hazards regression to link the relative abundance of each cell state (or CE) to overall survival (</span><span class="s7">survival </span><a href="#bookmark181" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">R package v2.42.3 (</a>Therneau and Grambsch, 2000<a href="#bookmark144" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">)). This was done separately for each tumor type and dataset. The resulting z-scores were integrated across datasets of the same tumor type using Lip- tak’s method (</a>Lipta´ k, 1958<a href="#bookmark191" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) with weights set to the inverse of the Cox model coefficient standard errors (</a>Whitlock, 2005<a href="#bookmark176" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Meta-z scores were further combined across tumor types using Stouffer’s method (</a>Stouffer et al., 1949<a href="#bookmark193" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Zaykin, 2011<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). To assess the association of each cell state and CE with overall survival after multivariable adjustment for age, sex, and pathologic stage, Cox regression was applied to (i) the relative abundance of each cell state (or CE), (ii) age as a continuous variable, (iii) sex as a binary variable, and (iv) stage as a categorical variable. Multivariable models were fit for each tumor type separately and global meta-z-scores were calculated using Stouffer’s method (</a>Table S5<span style=" color: #000;">). All survival z-scores were converted to two-sided –log10 p values for clarity.</span></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To obtain the Kaplan Meier plots shown in </a><span style=" color: #0097CF;">Figure 3</span>C, we started by calculating the difference vector (denoted d) between the imputed abundances of monocyte/macrophage states 6 and 3. To identify a threshold in d that maximally stratifies overall survival, we divided d into 20 possible cut-points at even 5 percentile intervals within tumor types in the discovery cohort. We then determined the hazard ratio and log-rank p value for each potential cut-point. Next, we converted the hazard ratios and –log10 p values to rank space and determined the value <i>b </i>with highest mean rank. For each tumor type, we optimized <i>b </i>in the discovery cohort (TCGA) and used it to stratify survival curves in the discovery (TCGA) and validation (PRECOG) cohorts.</p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the analyses shown in </a>Figures S5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">K and S5L and </a>Table S6<a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, we benchmarked the prognostic significance of CEs against two alternative subtyping approaches applied to the discovery cohort (</a>Figure 1<a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">): (i) NMF applied to bulk tumors, which we assessed in order to directly evaluate the merit of cell type deconvolution, and (ii) a collection of six pan-cancer immune subtypes defined by weighted gene correlation network analysis, finite normal mixture modeling, and ensemble voting (</a>Thorsson et al., 2018<span style=" color: #000;">). For the former, we applied bulk NMF clustering as described in </span><span class="s7">Cell state discovery </span><span style=" color: #000;">and </span><span class="s7">Rank selection </span><span style=" color: #000;">but without the expression decon- volution and AFI filtering steps. Overall survival (OS) models and meta-z statistics were calculated as described above. |</span><span class="s7">Z</span><span style=" color: #000;">| &gt; 1.96 (two-tailed p value &lt; 0.05) was considered significant.</span></p><p class="s24" style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><span style=" color: #000;">We also applied multivariable stepwise Cox regression to assess the combined value of all three classification schemes for pre- dicting OS across 16 carcinomas. The multivariable Cox model contained four groups of covariates: (i) CE abundances, (ii) bulk NMF subtype abundances, (iii) pan-immune subtypes, and (iv) tumor types. To mitigate collinearity issues, for each group of covariates, we removed the class with the least significant association with OS. For covariate groups (i) – (iii) above, this was done after bivariable adjustment for cancer type. Samples with no pan-immune subtype classification (n = 272) were not considered. The backward step- wise selection procedure implemented in </span><span class="s7">stepAIC </span><a href="#bookmark188" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">from the R package MASS v7.3.50 (</a>Venables and Ripley, 2002<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) was applied with default parameters. Among significant covariates, CEs were strongly preferred in stepwise composite models (</a>Table S6<span style=" color: #000;">), further highlighting their unique prognostic value.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Prediction of immunotherapy response</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the analysis in </a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C, we evaluated the following candidate correlates of ICI response (</a>Table S6<span style=" color: #000;">):</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 65pt;text-indent: 0pt;text-align: left;">Cell state and CE abundance vectors predicted by EcoTyper (n = 69 and 10, respectively).</p><p class="s24" style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">CIBERSORTx proportions of epithelial cells (bladder cancer only), melanoma cells (melanoma datasets only), fibroblasts, endo- thelial cells, the nine immune cell types in </a>Figure 1<a href="#bookmark92" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, and LM22 subsets not covered in </a>Figure 1<span style=" color: #000;">. Immune subsets were evaluated scaled to total immune content and scaled relative to all non-redundant cell types.</span></p><p style="padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><span class="s36">d </span>IMvigor210 dataset: CIBERSORTx was run with LM22 and TR4 signature matrices, as described above (see <i>Signature matrix design </i>and <i>Cell type fraction estimation</i>).</p><p class="s36" style="padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: justify;">d <a href="#bookmark158" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Melanoma datasets: CIBERSORTx was run with LM22 (B-mode batch correction) and a previously described scRNA-seq- based signature matrix covering melanoma cells, fibroblasts, endothelial cells, and immune subsets from melanoma tumor bi- opsies (B-mode batch correction) (</a><span class="s24">Newman et al., 2019</span><span class="p">). Immune cell fractions in the latter were replaced with LM22 in order to scale LM22 fractions into absolute space.</span></p><p class="s24" style="padding-left: 65pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark164" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Tumor mutational burden (TMB) estimates were used as supplied by the authors: the number of nonsynonymous mutations per sample (</a>Riaz et al., 2017<a href="#bookmark187" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a>Van Allen et al., 2015<a href="#bookmark156" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), the number of point mutations per sample (</a>Nathanson et al., 2017<a href="#bookmark145" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), the number of neoantigens per sample (</a>Liu et al., 2019<a href="#bookmark148" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), and neoantigen burden per megabase (</a>Mariathasan et al., 2018<span style=" color: #000;">).</span></p><p style="padding-left: 65pt;text-indent: 0pt;text-align: left;">Previous signatures of ICI response and/or T cell cytotoxicity:</p><p style="padding-top: 1pt;padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><span class="s36">d </span><a href="#bookmark131" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Immune resistance program score (</a><span style=" color: #0097CF;">Jerby-Arnon et al., 2018</span>), calculated using code supplied by the authors (ImmRes_OE.R script, run using the TPM matrix of each dataset as input and parameter <i>sig </i>set to <i>res.sig </i>object from the resistance.program. RData environment).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s36" style="padding-top: 3pt;padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><a name="bookmark65">d </a><a href="#bookmark113" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">18-gene T cell-inflamed score (</a><span class="s24">Cristescu et al., 2018</span><span class="p">). The log2 expression values of each gene were scaled to unit variance across pretreatment samples, and the resulting scaled values were averaged within each sample.</span><a name="bookmark66">&zwnj;</a><a name="bookmark67">&zwnj;</a></p><p style="padding-left: 71pt;text-indent: 0pt;line-height: 10pt;text-align: left;"><span class="s36">d </span><a href="#bookmark168" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Cytolytic score (</a><span style=" color: #0097CF;">Rooney et al., 2015</span>), calculated as the geometric mean of <i>GZMA </i>and <i>PRF1 </i>expression<i>.</i></p><p class="s36" style="padding-top: 1pt;padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;">d <a href="#bookmark123" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Estimated fractions of CD8 and CD4 memory T cells (CIBERSORTx) used as a proxy for the original Immunoscore (</a><a href="#bookmark123" class="s10">Galon </a><span class="s24">et al., 2006</span><span class="p">).</span></p><p style="padding-left: 81pt;text-indent: -9pt;line-height: 112%;text-align: justify;"><span class="s36">d </span>Immune-related markers previously used to explore the utility of Immunoscore in melanoma (<i>CD8A</i>, <i>CD3D, MS4A1</i>, <i>FOXP3</i>, <i>CD274</i>, <i>CD163</i><a href="#bookmark104" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">) (</a><span style=" color: #0097CF;">Bifulco et al., 2014</span>) or associated with exhaustion/activation (<i>PDCD1</i>, <i>CTLA4</i>, and <i>IFNG</i>), each log2-adjusted and scaled to unit variance within the pretreatment samples of each dataset.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 59pt;text-indent: 8pt;line-height: 112%;text-align: justify;"><a href="#bookmark144" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">All ICI expression datasets were TPM normalized prior to analysis. Only RNA-seq profiles of pretreatment tumors were analyzed. Each of the above measures was estimated independently in each dataset to avoid possible batch effects. We applied univariable Cox proportional hazards regression to each measure and extracted the z-score capturing its association with overall survival. We also assessed each measure’s binary association with response to therapy using a two-sided Wilcoxon test, from which we calcu- lated a z-score from the Wilcoxon p value. Z-scores were integrated across datasets by therapy type (aPD1, aPD1, or aCTLA4) using Liptak’s method (</a>Lipta´ k, 1958<a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), with the number of samples as weights. The ranks of the resulting z-scores were calculated for each combination of outcome association and therapy type and then averaged to yield a final rank for each measure (</a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C; </a>Table S6<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">CE network visualization</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the network schematic depicted in </a><span style=" color: #0097CF;">Figure 4</span>B, weighted undirected networks, representing the cell states from each CE were constructed using the <i>igraph </i><a href="#bookmark114" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">package, version 1.2.2 (</a><span style=" color: #0097CF;">Csardi and Nepusz, 2006</span>). The edge weights were proportional to the Jaccard index between cell states (<i>Ecotype discovery </i><a href="#bookmark121" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">above), and the layout of each network was generated using the forced directed layout algorithm by Fruchterman and Reingold (</a><span style=" color: #0097CF;">Fruchterman and Reingold, 1991</span>), implemented in <i>the layout_with_fr </i>function.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">Ligand-receptor enrichment analysis</p><p class="s7" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 111%;text-align: justify;"><a href="#bookmark163" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To determine whether CEs enrich for potential heterotypic interactions, we used a list of ligand-receptor pairs (</a><a href="#bookmark163" class="s10">Ramilowski et al., </a><span class="s24">2015</span><span class="p">) and assessed their statistical enrichment both within each CE and between CEs. We started by determining CE-specific dif- ferential expression. For each cell state of a given CE </span>i<span class="p">, we assessed whether each gene, scaled to unit variance within each tumor type, was overexpressed in samples assigned to CE </span>i <span class="p">relative to samples assigned to the remaining CEs. This was done using the digitally-purified expression profiles produced by CIBERSORTx. Statistical significance was determined using a two-sided Welch’s t test with unequal variances, corrected for multiple hypothesis testing (Benjamini-Hochberg). Genes with a </span>q-<span class="p">value &lt; 0.05 and log2 fold-change &gt; 0.1 were considered significant. Next, for each unique pair of distinct states </span>s<span class="s9">; </span>q <span class="p">from CE </span>i <span class="p">and </span>j<span class="p">, respectively, we calcu- lated the number of putative ligand-receptor pairs, </span>lrref <span class="p">, for which the ligand was overexpressed in cell state </span>s <span class="p">and the receptor over- expressed in cell state </span>q<span class="p">. We compared the number of putative ligand-receptor pairs in states </span>s<span class="s9">; </span>q <span class="p">against a null distribution of 100 samples, </span>lrnull<span class="p">, obtained by randomly drawing </span>g<span class="p">1 and </span>g<span class="p">2 genes from list </span>l<span class="p">, where </span>g<span class="p">1 is the number of genes overexpressed in state </span>s<span class="p">; </span>g<span class="p">2 is the number of genes overexpressed in state </span>q<span class="p">; and </span>l <span class="p">is the non-redundant set of genes among experimentally-determined ligand/ receptor pairs that overlap genes imputed by CIBERSORTx in states </span>s <span class="p">and </span>q<span class="p">. We obtained a two-sided z-score for each pair of cell states </span>s<span class="s9">; </span>q <span class="p">using the following formula:</span></p><p class="s91" style="padding-top: 5pt;padding-left: 12pt;text-indent: 0pt;line-height: 34%;text-align: center;"><span class="s79">z     </span><span class="p">= </span>lrref <span class="s75">— </span>mean<span class="s75">ð</span>lrnull<span class="s75">Þ</span></p><p class="s45" style="padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">s<span class="s44">;</span>q</p><p class="s7" style="padding-left: 26pt;text-indent: 0pt;text-align: left;">sd<span class="p">ð</span>lrnull<span class="p">Þ</span></p><p class="s7" style="padding-top: 6pt;padding-left: 59pt;text-indent: 8pt;line-height: 110%;text-align: justify;"><span class="p">To obtain a measure of enrichment of ligands expressed in CE </span>i <span class="p">and their receptors in CE </span>j<span class="p">, we integrated individual z-scores for pairwise state comparisons, </span>s<span class="s9">; </span>q<span class="p">, where </span>s <span class="p">is a state in CE </span>i <span class="p">and </span>q <span class="p">is a state in CE </span>j<a href="#bookmark176" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, using Stouffer’s method (</a><span class="s24">Stouffer et al., 1949</span><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). Meta z-scores were converted to –log10 p values for ease of interpretation. The resulting p values are plotted in </a><span class="s24">Figure S5</span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">G and ligand- receptor pairs for each CE and CE pair are provided in </a><span class="s24">Table S6</span><span class="p">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: left;">CE recovery in scRNA-seq data</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To determine whether CEs are detectable at the single-cell level (</a><span style=" color: #0097CF;">Figure 4</span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D), we analyzed six scRNA-seq tumor atlases that collec- tively cover 97 tumor and adjacent normal tissue samples and all major cell types analyzed in this work (</a><span style=" color: #0097CF;">Table S1</span>). We calculated significance at the level of individual CEs and across all CEs simultaneously (<i>Probability of CE detection </i>below). In both cases, we assigned single-cell transcriptomes to EcoTyper states blinded to CE identity (see <i>Cell state and ecotype recovery</i>). We also calcu- lated, for each tumor or adjacent normal sample <i>i</i>, the fraction of each cell state <i>j </i>within each parental cell type, yielding matrix F, with 58 rows (i.e., the number of cell states within CEs) and 97 columns (i.e., the number of samples). To mitigate the impact of distortions in state representation due to tissue dissociation, noise, dropout, and under-sampling, we devised a <i>co-occurrence index </i>that inte- grates four alternative approaches for correlating cell-state abundance profiles via ensemble averaging. First, we calculated two ver- sions of F with different denominators for calculating cell-state abundance: one version limited the denominator to the set of cells that could be assigned to EcoTyper states ðF1Þ; the other did not ðF2Þ. Next, we calculated four spearman correlation matrices: Two matrices were calculated directly from the rows of F1 and F2, yielding matrices C1 and C2; the others were calculated after</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a name="bookmark68">replacement of zeros in F1 and F2 with NA, yielding C3 and C4. The latter provides robustness to under-sampling of cell states within individual tumor or normal tissue samples. We averaged the four matrices into matrix C with 58 rows 3 58 columns.</a><a name="bookmark69">&zwnj;</a><a name="bookmark70">&zwnj;</a><a name="bookmark71">&zwnj;</a></p><p class="s31" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Probability of CE detection</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">We implemented a permutation-based approach to determine whether cell states within a given CE co-associate more strongly than expected by random chance (</a><span style=" color: #0097CF;">Figure 4</span>D). First, we set all diagonal entries in C to NA. Next, for each CE <i>i </i>and cell state <i>s</i>, we calculated</p><p style="padding-left: 53pt;text-indent: 0pt;text-align: justify;">the mean co-occurrence index between state <i>s </i>and the other states in CE <i>i</i>, denoted m<span class="s72">1</span>, and the mean co-occurrence index between state <i>s </i>and all other states in the remaining CEs, denoted m<span class="s72">2</span>. We then calculated D<i>s </i>as m<span class="s72">1 </span>— m<span class="s72">2 </span>and repeated this process for all states in CE <i>i</i>. The test statistic for CE <i>i </i>is <i>mean</i>ðDÞ, denoted <i>AvgDifref </i>. To derive a null distribution, we permuted each row of C 1,000 times,</p><p class="s7" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 109%;text-align: justify;"><span class="p">each time repeating the above procedure. For each row in the permuted matrix, we swapped NA (diagonal entry in the original matrix) with the new diagonal entry prior to calculating </span>mean<span class="p">ðDÞ. This yielded the null distribution, </span>AvgDifshuf<span class="s9">;</span>i<span class="p">. We calculated the significance of CE </span>i <span class="p">according the following formula:</span></p><p class="s45" style="text-indent: 0pt;line-height: 6pt;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s91" style="padding-top: 5pt;padding-left: 12pt;text-indent: 0pt;line-height: 62%;text-align: center;"><span class="s79">z  </span><span class="p">= </span>AvgDifref <span class="s75">— </span>mean<span class="s75">ð</span>AvgDifshuf<span class="s92">;</span>i<span class="s75">Þ</span></p><p class="s7" style="padding-left: 59pt;text-indent: 0pt;line-height: 9pt;text-align: right;">sd<span class="p">ð</span>AvgDif</p><p class="s45" style="text-indent: 0pt;line-height: 10pt;text-align: left;">shuf<span class="s44">;</span>i <span class="s61">Þ</span></p><p style="padding-top: 6pt;padding-left: 61pt;text-indent: 0pt;text-align: left;">Z-scores were converted to <i>P </i>values for ease of interpretation.</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">To obtain a global statistic for the joint probability of CE detection in scRNA-seq data, we applied the above approach with mod- ifications. Specifically, for each CE <i>i </i>and cell state <i>j</i>, we calculated the mean co-occurrence index between state <i>s </i>and the other</p><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s48" style="text-indent: 0pt;line-height: 6pt;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 53pt;text-indent: 0pt;text-align: justify;">states in CE <i>i</i>, denoted m<span class="s50">i</span><span class="s48"> </span>, and repeated this process after permuting the rows (as above) to calculate m<i>shuf </i>. We then counted the num- ber of CEs (out of 10) with m<span class="s50">i</span><span class="s48"> </span>Rm<i>shuf </i>for all <i>i</i><a href="#bookmark207" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">. Results from running this procedure one million times are shown in </a><a href="#bookmark207" class="s10">Figure </a><span style=" color: #0097CF;">S5</span>I.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Feature analysis of carcinoma ecotypes</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark96" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the analysis shown in </a>Figure 5<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A, we compiled and curated pre-computed data covering genomic characteristics and mutational signatures in TCGA tumors (</a>Table S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">). We also analyzed Hallmark gene sets from MSigDB and physiological variables. Continuous and discrete features were analyzed separately. For continuous features, we analyzed bulk tumors based on their CE assignment (i.e., the most abundant CE class per sample; </a>Table S6<span style=" color: #000;">). To incorporate Hallmark gene sets, we averaged all component genes in log2 space after scaling each gene to unit variance expression across samples. Enrichment/depletion of each feature across CEs was calculated by performing a two-sided Wilcoxon test to compare the sample-level values of each feature in a CE relative to other CEs. The resulting p values were adjusted for multiple hypothesis testing across all evaluated features using the Benjamini-Hochberg method. Features with a q-value &lt; 0.05 were considered significantly enriched/depleted. The magnitude of enrichment/depletion was calculated as the difference in the average value of each feature across samples within a given CE relative to other CEs. For discrete features (i.e., sex; age binarized as R 60 or &lt; 60 years), CE-specific associations were determined by applying a two-sided Wilcoxon test to compare the relative abundance of each CE between groups (e.g., male versus female). P values were adjusted for multiple hypothesis testing as described above. The magnitude of the enrichment/depletion was calculated as the average CE abun- dance within each group versus the other group (e.g., R 60 versus &lt; 60 years).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">State-specific expression in CE9 and CE10</p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Related to </a><span style=" color: #0097CF;">Figure 6</span><a href="#bookmark108" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A, for each scRNA-seq dataset, differential expression analysis was performed between CE9 and CE10-specific cell states, for each cell type with states in both ecotypes, using Seurat v3.1.3 (</a><span style=" color: #0097CF;">Butler et al., 2018</span><a href="#bookmark177" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">; </a><span style=" color: #0097CF;">Stuart et al., 2019</span>). Count data were log2-adjusted using <i>NormalizeData </i><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">with default parameters. For each cell type in </a><span style=" color: #0097CF;">Figure 6</span>A, differentially expressed genes between CE9- and CE10-specific states were identified using <i>FindMarkers </i>with <i>min.pct </i>= 0.1 and <i>logfc.threshold </i><a href="#bookmark176" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">= 0.05. To integrate across datasets, for each gene in a cell type, nominal p values were converted to z-scores and z-scores were combined across scRNA-seq datasets using Stouffer’s method (</a><span style=" color: #0097CF;">Stouffer et al., 1949</span>). Meta z-scores were then converted to p values, which were corrected by the Benjamini-Hochberg method. For each cell type, genes with a q-value &lt; 0.25 and expressed in at least 5% of evaluable cells were considered differentially expressed. If &lt; 10 genes passed this filter, we selected marker genes from the table described in <i>State-spe- cific marker genes in scRNA-seq data</i>. To admit genes from this table, we required a q-value &lt; 0.25, average log2 fold change &gt; 0.1, and average non-zero expression in at least 5% of cells in CE9 or CE10, across scRNA-seq datasets.</p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">Once significantly differentially expressed genes were identified, we selected the top 500 genes by average log2 fold change for each cell state, or the minimum between the number of marker genes in CE9 and CE10 states if less than 500 were available. Within each scRNA-seq dataset, the final list of genes was log2 adjusted and unit variance-normalized across cells, then averaged by ecotype. Prior to plotting, we applied unit variance normalization across genes to mitigate dataset-specific variation in the magnitude of expression.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Multicolor immunofluorescence imaging</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For the analyses in </a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">B and </a>S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">A–S6C, and </a>Table S6<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">,4 mm full tissue sections of NSCLC tumor biopsies (</a>Tables S1 <a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">and </a>S6<span style=" color: #000;">) were deparaffinized and rehydrated. Antigen retrieval was performed using EDTA pH 9 buffer at 95oC for 10 min. Sections were blocked for 20 min with horse serum and stained for 1 h with primary antibodies: CD3 (1/50, mouse, F7.2.38, (Agilent Cat# M7254, RRID: AB_2631163), GZMK (1/800, rabbit, Millipore Sigma Cat# HPA063181, RRID: AB_2684955), GZMB (1/100, rabbit,</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-top: 3pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" name="bookmark72">D6E9W, Cell Signaling Technology Cat# 46890, RRID: AB_2799313), APOE (1/500, mouse, Abcam Cat# ab1906, RRID: AB_302668), CCR2 (1/200, mouse, 48607, R and D Systems Cat# MAB150, RRID: AB_2247178), CD68 (rabbit, D4B9C, Cell Signaling Technology Cat# 76437, RRID: AB_2799882). Sections were subsequently stained with secondary anti-mouse AF647 (1/100, Thermo Fisher Scientific Cat# A-21235, RRID: AB_2535804) and secondary anti-rabbit AF555 (1/100, Thermo Fisher Scientific Cat# A-21428, RRID: AB_2535849) antibodies for 30 min. Sections were then mounted in ProLong Gold Antifade reagent with DAPI and coverslip- ped. Stained sections were imaged with a Keyence BZ-X800 microscope at 20 3 magnification. Exposure times were as follows: GZMB 1/6 s, GZMK 1/150 s, DAPI 1/2.5 s, APOE 1/85 s, CCR2 1/30 s, CD68 1/6 s. With exposure time 1/12 s, we observed two types of GZMK positive CD3 cells: dim and bright expressors. We selected exposure time 1/150 s to visualize the bright expressors. Raw images were adjusted in Adobe Photoshop by setting pixels to zero with values lower than: 112 in the DAPI channel, 60 in the CD3 channel, 17 in the GZMB channel, 30 in the GZMK channel, 45 in the APOE channel, 40 in the CCR2 channel, and 30 in the CD68 channel. Antibody aggregates in the CCR2 channel were removed by detecting connected components, computing their convex hulls, and removing those with an area of the convex hull below 40 pixels. Staining artifacts in the APOE channel were removed by detecting connected components and removing ones with an area below 40 or above 3000 pixels. Brightness and contrast were additionally adjusted in the GZMB channel by setting brightness to —96 and contrast to 100, in the APOE channel by setting brightness to 50, in the CCR2 channel by setting brightness to 150, and in the CD68 channel by setting brightness to 150 and contrast to –50. To quantify CD3 GZMB/K double-positive pixels (</a>Table S6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">), adjusted single-channel images of size 1150 3 1540 mm were binarized to 0 and 1 using the following thresholds: (i) for GZMB staining, we used GZMB signal between the 90th and 95th percentile and CD3 signal between the 40th and 100th percentile. (ii) For GZMK staining, the full signal intensity was used. Next, binarized CD3 and GZMB/GZMK signals were multiplied to obtain double-positive pixels. The mean double-positive pixel value across all pixels per image was computed. Finally, the mean CD3 GZMB/K double-positive signal per case was computed from images of (i) the close edge and center of the tumor, and (ii) the far edge of the tumor (</a>Figure S6<span style=" color: #000;">A).</span><a name="bookmark73">&zwnj;</a><a name="bookmark74">&zwnj;</a><a name="bookmark75">&zwnj;</a><a name="bookmark76">&zwnj;</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Analysis of spatial transcriptomics data</p><p style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;">Using the approach described in <i>Cell state and ecotype recovery</i><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, we applied EcoTyper to quantitate 69 cell states in spatial tran- scriptomics (ST) arrays of breast cancer (2 sections, 1 patient), colorectal cancer (1 section), ovarian cancer (1 section) and melanoma (8 sections, 4 patients) (</a><span style=" color: #0097CF;">Table S1</span>). Although multiple cell states from the same cell type could theoretically be present in a given spatially-barcoded spot, for the sake of stringency given the low number of cells per spot, we restricted our analysis to the most abun- dant cell state inferred per cell type. Accordingly, for each spot, we set the most abundant cell state per cell type to 1 and the rest to 0. To normalize each cell state by its parental cell type, we applied CIBERSORTx to breast, colorectal and ovarian cancer specimens as described in <i>Signature matrix design </i>and <i>Cell type fraction estimation</i>. We repeated the same approach for melanoma arrays but replaced TR4 with a melanoma-specific signature matrix as described in <i>Survival analyses and bulk tumor subtyping</i>. We then multi- plied cell state abundances by the imputed proportions of their corresponding parental cell types. This was done independently for each spot. Finally, the resulting abundances were scaled such that the 99th percentile within the cell type was set to 1. We also enumerated CEs in each spot by averaging the relative fractions of their constituent cell states. The resulting CE abundances were also scaled such that the 99th percentile value across all 10 CEs was set to 1.</p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">ST visualization</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To create the plots in </a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C (left) and </a>S6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">D, we linearly scaled the proportions of epithelial cells and melanoma cells estimated by CIBERSORTx such that the 1st percentile became 0 and the 99th percentile became 1. To display the fractional representation of two cell types in the same spot (e.g., CE9 and epithelial cells), we projected them as colors with opacity (alpha) levels proportional to their abundance. For epithelial cells and melanoma cells, fractional abundances in the range of 0.2 to 1 were linearly mapped into the alpha range of 0.1 to 1. Otherwise, fractional abundances in the range of 0 to 0.9 were linearly mapped into the alpha range of 0 to 1. The same procedure was used to display CE9/CE10 T cell and monocyte/macrophages states in </a>Figure S6<span style=" color: #000;">D (left).</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">ST distance quantification</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">Regions containing tumor cells were demarcated using annotations published by the authors (melanoma arrays) or obtained from 10x Genomics (breast cancer arrays). This was done by co-registering each array with its corresponding histopathological image. We then identified spots in which CE9 abundance was positive and calculated the average Euclidean distance from each spot to all tumor spots. This process was repeated for CE10, with the results shown in </a>Figure 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">C (right). The same procedure was applied in </a>Figure S6<span style=" color: #000;">D (right) for individual cell states.</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">ST co-localization analysis</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For </a>Figures 6<a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">E and </a>S6<span style=" color: #000;">E, we analyzed the ST arrays described above to determine whether CE-specific cell states are more colocal- ized with each other than expected by random chance. First, we calculated Spearman cross-correlation matrices of cell states from each CE across barcoded spots, pooled across all arrays of a given cancer type. Next, we applied the framework described in </span><span class="s7">Prob- ability of CE detection </span><span style=" color: #000;">to assess the statistical significance of CE-specific co-localization patterns.</span></p><p class="s31" style="padding-left: 59pt;text-indent: 0pt;text-align: justify;">Spatial aggregation patterns</p><p class="s24" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">For </a>Figure S6<a href="#bookmark127" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">F, we calculated Moran’s I (</a>Gittleman and Kot, 1990<span style=" color: #000;">) using the R package </span><span class="s7">ape </span><a href="#bookmark160" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">v5.4.1 (</a>Paradis and Schliep, 2019<span style=" color: #000;">). As input, we supplied for each spot, the relative CE abundance (estimated as described above) and the immediately adjacent neigh- boring spots.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s31" style="padding-top: 3pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark77">Enrichment of CE ligand-receptor genes</a><a name="bookmark78">&zwnj;</a><a name="bookmark79">&zwnj;</a><a name="bookmark80">&zwnj;</a></p><p style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;"><a href="#bookmark208" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">To create the plot in </a><span style=" color: #0097CF;">Figure S6</span>H, for each CE <i>j</i>, and for each ST array, we first calculated the log2 fold-change across the transcrip- tome between the spots in which CE <i>j </i>was detected versus those in which it was not. Using the resulting ranked transcriptome, we applied pre-ranked GSEA (<i>fgsea </i><a href="#bookmark136" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">version 1.14.0 (</a><span style=" color: #0097CF;">Korotkevich et al., 2021</span>)) with 10,000 permutations to calculate the enrichment of candidate ligands and receptors identified for state <i>s </i>in CE <i>j </i>that participate in candidate ligand-receptor interactions with other cell types in CE <i>j </i>(<i>Ligand-receptor enrichment analysis</i><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;">, </a><span style=" color: #0097CF;">Table S6</span>).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark201">QUANTIFICATION AND STATISTICAL ANALYSIS</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: justify;">Linear relationships were modeled by linear regression (<i>R2</i>), and a t test was used to assess whether the result was significantly nonzero. When data were normally distributed, group comparisons were determined using a two-sided t test with unequal variance; otherwise, an unpaired or paired two-sided Wilcoxon test was applied as appropriate. Results with p &lt; 0.05 were considered signif- icant unless stated otherwise. Data analyses were performed with R and Prism v7+ (GraphPad Software, Inc.). The investigators were not blinded to allocation during experiments and outcome assessment. No sample-size estimates were performed to ensure adequate power to detect a pre-specified effect size.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;"><a name="bookmark202">ADDITIONAL RESOURCES</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s22" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">Website</p><p class="s24" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="https://ecotyper.stanford.edu/carcinoma" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt;" target="_blank">An interactive web application implemented in R/Shiny is hosted at </a>https://ecotyper.stanford.edu/carcinoma<span style=" color: #000;">. The website runs on a Nginx server on a virtual machine and allows users to perform the following analyses:</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s36" style="padding-left: 74pt;text-indent: -9pt;line-height: 112%;text-align: justify;">d <span class="p">Explore and interact with (i) a heatmap of expression signatures associated with all cell states identified in each tumor type; (ii) downloadable signatures associated with each cell state; (iii) heatmaps of cell state and CE associations with survival in different cancer types; (iv) cell state and CE distributions in different cancer types; and (v) predicted ligand-receptor pairs within each CE.</span></p><p class="s36" style="padding-left: 65pt;text-indent: 0pt;text-align: left;">d <span class="p">Quantify carcinoma cell states and CEs in user-provided bulk RNA-seq or microarray data.</span></p><p class="s36" style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;">d <span class="p">Assign carcinoma cell states to single cells from user-provided scRNA-seq data.</span></p><p class="s36" style="padding-top: 1pt;padding-left: 65pt;text-indent: 0pt;text-align: left;">d <span class="p">Visualize and download results of carcinoma cell state and CE assignments.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 8pt;line-height: 112%;text-align: justify;">Users can also download all data and results associated with this work, as well as plots, gene sets, and other data derived from the interactive tools available on the site. The website additionally includes a tutorial providing information on how to navigate the site and run analyses.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_185.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="25" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_186.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="49" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_187.png"/></span></p><p class="s8" style="padding-top: 7pt;padding-left: 53pt;text-indent: 0pt;line-height: 18pt;text-align: left;"><span><img width="36" height="18" alt="image" src="1-s2.0-S0092867421010618-main/Image_188.png"/></span><span class="s93">                                                                                                                                                        </span> <span><img width="20" height="20" alt="image" src="1-s2.0-S0092867421010618-main/Image_189.png"/></span>             <span class="h2">ll</span></p><p class="s12" style="padding-left: 53pt;text-indent: 0pt;line-height: 15pt;text-align: left;">Resource</p><p style="text-indent: 0pt;text-align: left;"><span><img width="142" height="103" alt="image" src="1-s2.0-S0092867421010618-main/Image_190.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="239" height="107" alt="image" src="1-s2.0-S0092867421010618-main/Image_191.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="186" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_192.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="110" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_193.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="66" height="81" alt="image" src="1-s2.0-S0092867421010618-main/Image_194.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="14" height="62" alt="image" src="1-s2.0-S0092867421010618-main/Image_195.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="245" height="134" alt="image" src="1-s2.0-S0092867421010618-main/Image_196.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="91" height="14" alt="image" src="1-s2.0-S0092867421010618-main/Image_197.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="133" height="91" alt="image" src="1-s2.0-S0092867421010618-main/Image_198.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="63" height="90" alt="image" src="1-s2.0-S0092867421010618-main/Image_199.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="109" height="134" alt="image" src="1-s2.0-S0092867421010618-main/Image_200.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="167" height="114" alt="image" src="1-s2.0-S0092867421010618-main/Image_201.png"/></span></p><p class="s94" style="padding-top: 11pt;padding-left: 52pt;text-indent: 0pt;text-align: left;"><a name="bookmark203">Supplemental figures</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 152pt;text-indent: 0pt;text-align: left;"><span><img width="239" height="150" alt="image" src="1-s2.0-S0092867421010618-main/Image_202.png"/></span> <span><img width="145" height="138" alt="image" src="1-s2.0-S0092867421010618-main/Image_203.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s5" style="padding-left: 162pt;text-indent: 0pt;text-align: left;"><span><img width="120" height="90" alt="image" src="1-s2.0-S0092867421010618-main/Image_204.png"/></span>	<span><img width="229" height="89" alt="image" src="1-s2.0-S0092867421010618-main/Image_205.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 152pt;text-indent: 0pt;text-align: left;"><span><img width="189" height="118" alt="image" src="1-s2.0-S0092867421010618-main/Image_206.png"/></span> <span><img width="107" height="25" alt="image" src="1-s2.0-S0092867421010618-main/Image_207.png"/></span> <span><img width="73" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_208.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="657" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_209.png"/></span></p><p style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark92" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Figure S1. Implementation and performance assessment of EcoTyper, related to </a><a href="#bookmark92" class="s42">Figure 1</a></p><ol id="l14"><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><span style=" color: #000;">The discovery and recovery phases of a typical EcoTyper workflow. For a detailed description of each step, including matrix definitions and algorithms, see</span><a href="#bookmark197" class="a"> STAR </a>Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">. GEP, gene expression profile. AFI, adaptive false positive index. (B) Performance of cell type-specific reference signatures for enumerating 12 epithelial, immune, and stromal cell lineages with CIBERSORTx. Each point represents the Pearson correlation coefficient between CIBERSORTx and ground truth proportions in 100 tumors simulated from the indicated scRNA-seq dataset (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). All points are statistically significant (p &lt; 7.6 3 10—5, Pearson correlation). For each cell type, means and standard errors are expressed as bars and error bars, respectively. (C) Boxplots showing Pearson correlation co- efficients between cell fractions estimated by CIBERSORTx (epithelial, endothelial + fibroblasts, immune cells) and orthogonal measures in the discovery cohort: ESTIMATE (tumor purity and stromal score), leukocyte fractions inferred by methylation signatures (</a><a href="#bookmark183" class="a">Thorsson et al., </a>2018<a href="#bookmark109" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">), and tumor purity determined by ABSOLUTE (</a><a href="#bookmark109" class="a">Carter et al., </a>2012<span style=" color: #000;">). (D-J) </span><span class="s35">In silico </span><a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">framework for evaluating EcoTyper performance. (D) Workflow. (E) scRNA-seq atlas of 26 previously-defined NSCLC states (</a><a href="#bookmark138" class="a">Lambrechts et al., </a>2018<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) used to create simulated tumors. (F) Impact of gene-level standardization and AFI filtering on the recovery of cell states in 500 simulated tumors (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Sn, sensitivity; PPV, positive predictive value. (G) Pearson correlation matrix comparing estimated and known fractions of fibroblast states in 500 simulated tumors. (H) Boxplot showing the distribution of Pearson correlation coefficients between state fractions imputed by EcoTyper and ground truth proportions. Only states significantly recovered in scRNA-seq data are shown (n = 24). (I) Assessment of the limit of detection for cell state recovery, as determined using simulated tumors (n = 250 or 500) with cell states (fibroblasts C2 or dendritic cells C12) added in defined proportions (x axis). Detection was determined as described in </a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">. States with an insignificant Jaccard index (y axis) were set to zero. (J) Analysis of collinearity between state fractions. The heatmap shows whether pairs of cell states with defined collinearity between spike-in fractions were resolvable by EcoTyper in 500 simulated tumors (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). Three replicates were performed per cell type. (K) </span><span class="s35">Top: </span><a href="#bookmark117" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Dendrogram showing TCGA tumor types clustered by housekeeping gene expression (n = 11 genes (</a><a href="#bookmark117" class="a">Eisenberg and Levanon, </a>2013<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). </span><span class="s35">Bottom: </span><a href="#bookmark192" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Silhouette widths for a range of cluster numbers, with the optimal cluster number (maximum silhouette width) indicated by a dashed line. (L) Comparison of CIBERSORTx and ESTIMATE (</a><a href="#bookmark192" class="a">Yoshihara et al., </a>2013<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) for inferring epithelial content and tumor purity, respectively, in 20 tumor types (</a><a href="#bookmark98" class="a">Table </a>S2<span style=" color: #000;">). Results are expressed as Pearson correlation coefficient (y axis) and mean squared error (MSE) (x axis). Two clusters are indicated, including an outgroup (red) identified by K-means clustering coupled with silhouette analysis (bottom). Data in (C), (H), and (I) are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.53IQR of the box limits; IQR, interquartile range).</span></p></li></ol></li></ol></ol><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_210.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="14" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_211.png"/></span></p><p class="s97" style="padding-top: 7pt;padding-left: 52pt;text-indent: 0pt;line-height: 18pt;text-align: left;"><span class="s95">Cell                                                                                    </span><span><img width="20" height="20" alt="image" src="1-s2.0-S0092867421010618-main/Image_212.png"/></span><span class="s96">   </span>e<b>ll</b>=&gt;ress</p><p class="s12" style="padding-left: 53pt;text-indent: 0pt;line-height: 15pt;text-align: left;"><a name="bookmark204">Resource</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s98" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s100" style="padding-top: 4pt;padding-left: 98pt;text-indent: -11pt;line-height: 107%;text-align: left;"><span class="s99">A </span>Validation <span style=" color: #080808;">of cell-type specificity of CIBERSORTx genes in </span>scRNA-seq</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_213.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="135" height="125" alt="image" src="1-s2.0-S0092867421010618-main/Image_214.png"/></span></p><p class="s103" style="padding-left: 20pt;text-indent: 0pt;line-height: 120%;text-align: center;"><span class="s102">(a</span>O<span class="s104">」</span><span class="s105">  </span>D<span class="s106">S-</span><span class="s107">N)</span></p><p class="s109" style="padding-left: 20pt;text-indent: 0pt;line-height: 120%;text-align: center;"><span class="s108">芒</span>a&gt;033<span class="s104">」</span><span class="s107">JO</span>aoueo!1<span class="s110">』</span>uB</p><p style="text-indent: 0pt;text-align: left;"/><p class="s100" style="padding-left: 112pt;text-indent: -9pt;line-height: 108%;text-align: left;">ene specificity <span style=" color: #080808;">index</span></p><p class="s111" style="padding-top: 3pt;padding-left: 99pt;text-indent: 0pt;line-height: 29pt;text-align: left;">,} 1<span style=" color: #2D2D2D;">:Q ,1</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 230pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="6" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_215.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 144pt;text-indent: 0pt;line-height: 3pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_216.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_217.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_218.png"/></span></p><p class="s102" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s102" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s109" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0      0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s109" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">今sua<span class="s113">0</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">002051 001</p><p style="text-indent: 0pt;text-align: left;"/><p class="s115" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">250</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 11pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="3" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_219.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s101" style="text-indent: 0pt;line-height: 134%;text-align: center;">Significance of <span style=" color: #1A1A1A;">cell state </span>recovery in <span style=" color: #1A1A1A;">scRNA-seq </span>according to adaptive false positive index <span style=" color: #1A1A1A;">(AFI)</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="19" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_220.png"/></span></p><p class="s116" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o<span class="s117">l</span></p><p style="text-indent: 0pt;text-align: left;"/><ol id="l15"><ol id="l16"><li style="padding-top: 2pt;padding-left: 25pt;text-indent: -12pt;line-height: 7pt;text-align: left;"><p class="s118" style="display: inline;">x 1Q-6 <span class="s119">今。</span></p><p class="s120" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">c-</p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_221.png"/></span></p><p class="s121" style="padding-left: 12pt;text-indent: 0pt;line-height: 6pt;text-align: center;">．     <span style=" color: #1A1A1A;">＄ 导 </span><span class="s123">1 </span><span class="s115">00</span></p><p style="padding-left: 28pt;text-indent: 0pt;line-height: 2pt;text-align: left;"><span><img width="2" height="2" alt="image" src="1-s2.0-S0092867421010618-main/Image_222.png"/></span></p><p class="s124" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">芒 <span class="s120">o</span></p><p class="s125" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">仁  <span class="s126">Q)</span></p><p class="s127" style="padding-left: 12pt;text-indent: 0pt;line-height: 4pt;text-align: center;">&#39;2&#39;&#39;{&#39;</p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_223.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_224.png"/></span></p><p class="s128" style="padding-left: 12pt;text-indent: 0pt;line-height: 5pt;text-align: center;">&quot;&#39;-,,:  0 75</p><p class="s129" style="padding-left: 12pt;text-indent: 0pt;line-height: 4pt;text-align: center;">&lt;t&gt;Z</p><p class="s130" style="padding-left: 12pt;text-indent: 0pt;line-height: 4pt;text-align: center;">.!ill&#39;.&#39;.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="21" height="33" alt="image" src="1-s2.0-S0092867421010618-main/Image_225.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="15" alt="image" src="1-s2.0-S0092867421010618-main/Image_226.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="2" alt="image" src="1-s2.0-S0092867421010618-main/Image_227.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_228.png"/></span></p><p class="s131" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．<span class="s132">护</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s106" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">号 <span class="s100">i;l </span><span class="s128">O</span><span class="s115">50</span></p><p style="padding-left: 77pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="4" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_229.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s133" style="text-indent: 0pt;line-height: 14pt;text-align: left;">。</p><p class="s5" style="padding-left: 50pt;text-indent: 0pt;text-align: left;">	<span><img width="5" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_230.png"/></span>	<span><img width="59" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_231.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s101" style="padding-left: 15pt;text-indent: 0pt;text-align: left;"><span><img width="6" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_232.png"/></span><span class="s8"> </span>ropout <span style=" color: #1A1A1A;">score </span>across <span style=" color: #1A1A1A;">cell </span>states</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="97" height="92" alt="image" src="1-s2.0-S0092867421010618-main/Image_233.png"/></span></p><p class="s134" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">D</p><p style="text-indent: 0pt;text-align: left;"/><p class="s100" style="padding-left: 23pt;text-indent: 0pt;text-align: left;">0.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 49pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="8" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_234.png"/></span></p><p class="s135" style="padding-left: 59pt;text-indent: 0pt;line-height: 21pt;text-align: right;">I <span style=" color: #B6B6B6;">.. </span>I</p><p class="s137" style="text-indent: 0pt;line-height: 6pt;text-align: right;">之 <span class="s100">1 </span><span class="s115">&lt; </span><span class="s101">1</span></p><p class="s101" style="padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">AFI</p><p class="s109" style="padding-left: 14pt;text-indent: 0pt;line-height: 38pt;text-align: left;">它巳</p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_235.png"/></span></p><p class="s138" style="padding-left: 15pt;text-indent: 0pt;line-height: 29pt;text-align: left;">g <span class="s139">苞 </span><span class="s127">0 </span><span class="s115">25</span></p><p class="s122" style="padding-left: 14pt;text-indent: 0pt;line-height: 23pt;text-align: left;">芒罗</p><p class="s140" style="padding-left: 14pt;text-indent: 0pt;text-align: left;">0<span class="s141">&quot;- </span>0<span class="s128">u </span><span class="s115">00</span></p><p class="s142" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">Q)</p><p class="s143" style="padding-left: 4pt;text-indent: 0pt;line-height: 25pt;text-align: left;">＿</p><p class="s100" style="padding-left: 4pt;text-indent: 0pt;line-height: 21pt;text-align: left;">004</p><p class="s100" style="padding-left: 7pt;text-indent: 0pt;line-height: 5pt;text-align: left;">;,1 <span style=" color: #2D2D2D;">&lt; </span><span style=" color: #080808;">1</span></p><p class="s134" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">F</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 12pt;text-indent: 0pt;line-height: 6pt;text-align: center;">AFI</p><p class="s144" style="padding-left: 32pt;text-indent: 0pt;line-height: 30pt;text-align: left;">O.O  <span style=" color: #4D4B4B;">j </span><span style=" color: #9C9C9C;">• .....   -   </span><span style=" color: #FD595B;">:    </span><span style=" color: #D4362D;">11,   </span><span class="s149">0</span></p><p class="s130" style="padding-left: 12pt;text-indent: 0pt;line-height: 21pt;text-align: center;">-1        0        1        2       3</p><p class="s100" style="padding-left: 12pt;text-indent: 0pt;line-height: 12pt;text-align: center;">opout score</p><p class="s100" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">Cell state   <span style=" color: #9C9C9C;">•   </span>Non- outlier  <span style=" color: #D4362D;">• </span>Outlier</p><p style="text-indent: 0pt;text-align: left;"><span><img width="127" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_236.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_237.png"/></span></p><p class="s134" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">B</p><p style="text-indent: 0pt;text-align: left;"/><p class="s152" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">      </p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 125pt;text-indent: 0pt;line-height: 69pt;text-align: left;">Eco Typer <span style=" color: #1A1A1A;">workflow</span></p><p class="s101" style="padding-left: 200pt;text-indent: 0pt;line-height: 69pt;text-align: center;">NMF     <span style=" color: #1A1A1A;">consensus </span>index</p><p style="text-indent: 0pt;text-align: left;"><span><img width="336" height="270" alt="image" src="1-s2.0-S0092867421010618-main/Image_238.png"/></span></p><p class="s134" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 196pt;text-indent: 0pt;line-height: 60pt;text-align: center;">across samples</p><p class="s101" style="padding-left: 53pt;text-indent: 0pt;line-height: 62pt;text-align: left;">UMAP of <span style=" color: #1A1A1A;">cell </span>state fractions</p><p style="text-indent: 0pt;text-align: left;"><span><img width="126" height="24" alt="image" src="1-s2.0-S0092867421010618-main/Image_239.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="127" height="161" alt="image" src="1-s2.0-S0092867421010618-main/Image_240.png"/></span></p><p class="s153" style="text-indent: 0pt;line-height: 23pt;text-align: left;">尸曰</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 189pt;text-indent: 0pt;line-height: 174pt;text-align: center;">Discovery <span style=" color: #1A1A1A;">cohort</span></p><p class="s101" style="padding-left: 190pt;text-indent: 0pt;line-height: 167pt;text-align: center;">16 tumo<span class="s124">「 </span>types</p><p class="s100" style="padding-left: 189pt;text-indent: 0pt;line-height: 159pt;text-align: center;">5,946 <span style=" color: #080808;">tumor </span>samples</p><p class="s100" style="padding-left: 189pt;text-indent: 0pt;line-height: 151pt;text-align: center;">529 adjacent normal samples</p><p class="s101" style="padding-left: 105pt;text-indent: 0pt;line-height: 134pt;text-align: left;">i     c   <span class="s154">霍芯霖</span>ion   deconvolution <span style=" color: #1A1A1A;">with</span></p><p class="s101" style="padding-left: 190pt;text-indent: 0pt;line-height: 115pt;text-align: center;">Purified <span style=" color: #1A1A1A;">gene expression</span></p><p class="s100" style="padding-left: 191pt;text-indent: 0pt;line-height: 107pt;text-align: center;">profiles of <span style=" color: #080808;">12 </span>cell <span style=" color: #080808;">t</span><span style=" color: #2D2D2D;">ypes</span></p><p class="s115" style="padding-left: 168pt;text-indent: 0pt;line-height: 178pt;text-align: center;"><span class="s127">B </span><span style=" color: #1A1A1A;">cells      P </span>asma <span style=" color: #1A1A1A;">cells  CD8 </span><span style=" color: #080808;">T </span>ce<span style=" color: #080808;">ll</span>s</p><p class="s100" style="padding-left: 159pt;text-indent: 0pt;line-height: 132pt;text-align: center;">CD4 <span style=" color: #080808;">T </span>cells      NK <span style=" color: #2D2D2D;">ce</span><span style=" color: #080808;">ll</span><span style=" color: #2D2D2D;">s      </span>Mono/Mac</p><p class="s155" style="padding-left: 158pt;text-indent: 0pt;line-height: 178pt;text-align: center;">B <span class="s100">cells</span></p><p class="s156" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s</p><p style="text-indent: 0pt;text-align: left;"/><p class="s157" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e <span style=" color: #2D2D2D;">c</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s156" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s157" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">t</p><p style="text-indent: 0pt;text-align: left;"/><p class="s122" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">r</p><p style="text-indent: 0pt;text-align: left;"/><p class="s158" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: justify;"><span class="s102">d </span>n <span class="s157">e </span>D</p><p style="text-indent: 0pt;text-align: left;"/><p class="s122" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">NdVwn</p><p style="text-indent: 0pt;text-align: left;"/><p class="s100" style="padding-left: 158pt;text-indent: 0pt;line-height: 128pt;text-align: center;">CD4 <span style=" color: #080808;">T </span>cells</p><p class="s101" style="padding-left: 147pt;text-indent: 0pt;line-height: 178pt;text-align: center;">Pl<span style=" color: #2D2D2D;">asma ce</span>ll<span style=" color: #2D2D2D;">s</span></p><p class="s101" style="padding-left: 147pt;text-indent: 0pt;line-height: 128pt;text-align: center;">NK <span style=" color: #1A1A1A;">cells</span></p><p class="s100" style="padding-left: 11pt;text-indent: 0pt;line-height: 178pt;text-align: left;">CDS <span style=" color: #080808;">T </span>cells</p><p class="s100" style="padding-left: 13pt;text-indent: 0pt;line-height: 128pt;text-align: left;">Mono/Mac</p><p style="text-indent: 0pt;text-align: left;"><span><img width="115" height="137" alt="image" src="1-s2.0-S0092867421010618-main/Image_241.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_242.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_243.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_244.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="22" height="14" alt="image" src="1-s2.0-S0092867421010618-main/Image_245.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_246.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="160" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_247.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_248.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_249.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_250.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_251.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_252.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_253.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_254.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="2" alt="image" src="1-s2.0-S0092867421010618-main/Image_255.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_256.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="2" alt="image" src="1-s2.0-S0092867421010618-main/Image_257.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="2" alt="image" src="1-s2.0-S0092867421010618-main/Image_258.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="180" height="25" alt="image" src="1-s2.0-S0092867421010618-main/Image_259.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_260.png"/></span></p><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s159" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">.</p><p style="text-indent: 0pt;text-align: left;"/><p class="s160" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">b</p><p style="text-indent: 0pt;text-align: left;"/><p class="s161" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">T<span style=" color: #3A9CB5;">.</span><span class="s163">气</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s164" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">.<span style=" color: #775D44;">t</span><span class="s166">复</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s167" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．<span class="s168">户</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s169" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．<span class="s170">勹</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s171" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">？</p><p style="text-indent: 0pt;text-align: left;"/><p class="s172" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">.<span class="s173">?</span><span class="s174">9</span><span class="s175">_</span><span class="s176">e</span><span class="s177">·</span><span class="s178">· </span><span class="s179">·</span><span class="s180">2</span><span class="s181">-</span></p><p class="s182" style="padding-top: 4pt;text-indent: 0pt;line-height: 120%;text-align: center;">叫</p><p style="text-indent: 0pt;text-align: left;"/><p class="s183" style="text-indent: 0pt;line-height: 120%;text-align: right;">P</p><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5        0</p><p class="s184" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1        1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">,     <span style=" color: #1A1A1A;">°</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s109" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">(an1e<span style=" color: #2D2D2D;">&gt;</span>b 601- <span style=" color: #080808;">iua</span><span class="s185">E</span> ouuaa6eJa&gt;<span class="s185">V</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s109" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">&#39;   )</p><p style="text-indent: 0pt;text-align: left;"/><p class="s152" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">  <span class="s134">G</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s186" style="text-indent: 0pt;line-height: 24pt;text-align: left;">c,&lt;&gt;<span class="s187">:</span><span class="s188">&lt;f</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s189" style="text-indent: 0pt;line-height: 26pt;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s190" style="text-indent: 0pt;line-height: 26pt;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 106pt;text-indent: 0pt;line-height: 1097pt;text-align: left;">i         <span class="s133">它</span>:    <span class="s133">骂</span><span style=" color: #1A1A1A;">oni       of cell states</span>b<span style=" color: #2D2D2D;">y    </span>NMF</p><p class="s101" style="padding-left: 29pt;text-indent: 0pt;line-height: 1093pt;text-align: left;">Dend<span style=" color: #2D2D2D;">r</span>iti<span style=" color: #2D2D2D;">c </span><span style=" color: #1A1A1A;">cells Mast cells</span></p><p class="s100" style="padding-left: 5pt;text-indent: 0pt;line-height: 1093pt;text-align: left;">Neutrophils</p><p class="s100" style="padding-left: 59pt;text-indent: 0pt;line-height: 1086pt;text-align: right;">Mast cells</p><p class="s100" style="padding-left: 20pt;text-indent: 0pt;line-height: 1086pt;text-align: left;">eutrophils</p><p class="s100" style="padding-left: 100pt;text-indent: 0pt;line-height: 2176pt;text-align: left;">30 <span style=" color: #080808;">transcriptional </span>cell states</p><p class="s100" style="padding-left: 96pt;text-indent: 0pt;line-height: 2168pt;text-align: left;">(clusters) across <span style=" color: #080808;">12 </span>cell types</p><p class="s100" style="padding-left: 132pt;text-indent: 0pt;line-height: 6498pt;text-align: left;"><span style=" color: #080808;">Filter </span>out cell <span style=" color: #2D2D2D;">s</span><span style=" color: #080808;">tates </span>with AFI <span class="s137">之</span>1</p><p class="s191" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">亨</p><p style="text-indent: 0pt;text-align: left;"/><p class="s191" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">灾</p><p style="text-indent: 0pt;text-align: left;"/><p class="s109" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">L 令</p><p style="text-indent: 0pt;text-align: left;"/><p class="s192" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">｀</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 110pt;text-indent: 0pt;line-height: 6490pt;text-align: left;">---. <span style=" color: #1A1A1A;">(Figure </span>S2C) <span style=" color: #1A1A1A;">or </span><span style=" color: #2D2D2D;">&lt;</span>10 marker <span style=" color: #1A1A1A;">genes</span></p><p class="s100" style="padding-left: 132pt;text-indent: 0pt;line-height: 6482pt;text-align: left;">(n <span style=" color: #080808;">= </span>55)</p><p class="s101" style="padding-left: 210pt;text-indent: 0pt;line-height: 6467pt;text-align: center;">ter <span style=" color: #1A1A1A;">out cell states </span><span style=" color: #2D2D2D;">w</span>ith <span style=" color: #1A1A1A;">outlier</span></p><p class="s100" style="padding-left: 16pt;text-indent: 0pt;line-height: 6487pt;text-align: left;">Fibroblasts <span style=" color: #080808;">Endothelial</span></p><p class="s101" style="padding-left: 7pt;text-indent: 0pt;line-height: 6487pt;text-align: left;">Epithel<span style=" color: #2D2D2D;">a</span>i l</p><p class="s101" style="padding-left: 28pt;text-indent: 0pt;line-height: 6475pt;text-align: left;">Frbrobl<span style=" color: #2D2D2D;">asts</span></p><p class="s101" style="padding-left: 8pt;text-indent: 0pt;line-height: 6475pt;text-align: left;">Endoth<span style=" color: #2D2D2D;">e</span>lial <span style=" color: #2D2D2D;">ce</span>ll<span style=" color: #2D2D2D;">s </span>Epithelial <span style=" color: #1A1A1A;">cells</span></p><p class="s193" style="padding-left: 59pt;text-indent: 0pt;line-height: 32386pt;text-align: right;">_y_</p><p class="s100" style="padding-left: 15pt;text-indent: 0pt;line-height: 32395pt;text-align: left;">dropout score (n <span style=" color: #080808;">= </span>4) (Figure <span style=" color: #080808;">S20)</span></p><p class="s100" style="padding-left: 11pt;text-indent: 0pt;line-height: 32376pt;text-align: left;">cell states</p><p class="s100" style="padding-left: 43pt;text-indent: 0pt;line-height: 32376pt;text-align: left;">C nsensus <span style=" color: #080808;">inde</span><span style=" color: #2D2D2D;">x</span></p><p class="s119" style="padding-left: 210pt;text-indent: 0pt;line-height: 97131pt;text-align: center;">-.   曰 <span class="s101">ter </span><span class="s100">out cell states with potential</span></p><p class="s100" style="padding-left: 210pt;text-indent: 0pt;line-height: 97124pt;text-align: center;">doublet profiles (n <span style=" color: #080808;">= </span>2)</p><p class="s100" style="padding-left: 119pt;text-indent: 0pt;line-height: 97105pt;text-align: left;">69 cell states</p><p class="s100" style="padding-left: 56pt;text-indent: 0pt;line-height: 97135pt;text-align: left;">2 <span style=" color: #2D2D2D;">0</span><span style=" color: #4D4B4B;">.</span>4 0<span style=" color: #625B5B;">.</span>6 <span style=" color: #2D2D2D;">0</span><span style=" color: #4D4B4B;">.</span><span style=" color: #2D2D2D;">8 </span>1</p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_261.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_262.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_263.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_264.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_265.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_266.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_267.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_268.png"/></span></p><p class="s196" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">i</p><p style="text-indent: 0pt;text-align: left;"/><p class="s197" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">f</p><p style="text-indent: 0pt;text-align: left;"/><p class="s198" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">、<span class="s199">·</span><span class="s200">·</span><span class="s201">3</span><span class="s202">•</span><span class="s203">•</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s184" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span style=" color: #1A1A1A;">(an1e</span><span style=" color: #2D2D2D;">&gt;</span>b<span class="s204">＇</span> <span class="s205">&#39;</span>6<span class="s204">°</span>01-<span class="s205">)</span>iua<span class="s185">E</span><span style=" color: #1A1A1A;"> ouuaa6eJa&gt;</span><span class="s185">＜</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s206" style="text-indent: 0pt;line-height: 26pt;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 168pt;text-indent: 0pt;line-height: 97125pt;text-align: center;">Ti<span style=" color: #2D2D2D;">ss</span>ue</p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_269.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_270.png"/></span></p><p class="s207" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s208" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">勹<span style=" color: #B1DD8A;">－</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s210" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">，</p><p style="text-indent: 0pt;text-align: left;"/><p class="s211" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">仑</p><p style="text-indent: 0pt;text-align: left;"/><p class="s210" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">，</p><p style="text-indent: 0pt;text-align: left;"/><p class="s212" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">山<span style=" color: #895D66;">冈</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s214" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">·<span class="s215">上</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s216" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">^<span style=" color: #8580B5;">谨</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s218" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">闸</p><p style="text-indent: 0pt;text-align: left;"/><ul id="l17"><li style="padding-left: 44pt;text-indent: -6pt;line-height: 97118pt;text-align: left;"><p class="s100" style="display: inline;">Tumor  <span style=" color: #366BAC;">e  </span><span style=" color: #2D2D2D;">Act1acen</span>n<span style=" color: #2D2D2D;">t </span>ormal</p></li></ul></li></ol></ol><p class="s100" style="padding-left: 58pt;text-indent: 0pt;line-height: 97108pt;text-align: left;">Cell state</p><p class="s115" style="padding-left: 33pt;text-indent: 0pt;line-height: 97102pt;text-align: left;"><span style=" color: #E21A70;">e </span>s1 <span style=" color: #59AA48;">e </span>s3 <span style=" color: #FD7501;">e </span>ss <span style=" color: #A55228;">e </span>s? <span style=" color: #9C9C9C;">e </span>sg</p><p class="s115" style="padding-left: 33pt;text-indent: 0pt;line-height: 97095pt;text-align: left;"><span style=" color: #3A9CB5;">e </span>s2 <span style=" color: #7B4DA1;">e </span>s4 <span style=" color: #FDC603;">e </span>ss <span style=" color: #F480E6;">e </span>sa</p><p class="s101" style="padding-left: 82pt;text-indent: 0pt;line-height: 97131pt;text-align: left;">UMAP1</p><p class="s228" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">g</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 87pt;text-indent: 0pt;line-height: 97111pt;text-align: left;">Tissue</p><ul id="l18"><li style="padding-left: 70pt;text-indent: -6pt;line-height: 97102pt;text-align: left;"><p class="s115" style="display: inline;"><span style=" color: #080808;">Tum</span>or <span style=" color: #366BAC;">e </span>Adjacen<span style=" color: #080808;">t </span>normal</p><p class="s101" style="padding-left: 210pt;text-indent: 0pt;line-height: 679757pt;text-align: center;">nchment of cell <span style=" color: #1A1A1A;">states </span>in individual tumor types</p><p class="s100" style="padding-left: 210pt;text-indent: 0pt;line-height: 679748pt;text-align: center;">(averaged <span style=" color: #080808;">across 16 </span>carcinomas)</p><p class="s100" style="padding-left: 59pt;text-indent: 0pt;line-height: 679735pt;text-align: right;">Cell state</p><ul id="l19"><li style="padding-left: 246pt;text-indent: -8pt;line-height: 679726pt;text-align: right;"><p class="s100" style="display: inline;">S01</p></li><li style="padding-left: 246pt;text-indent: -8pt;line-height: 679718pt;text-align: right;"><p class="s100" style="display: inline;">S02</p></li><li style="padding-left: 246pt;text-indent: -8pt;line-height: 679710pt;text-align: right;"><p class="s100" style="display: inline;">S03</p></li></ul></li></ul><p class="s101" style="padding-left: 20pt;text-indent: 0pt;line-height: 679756pt;text-align: left;"><span class="s229">H </span>Enrichment of <span style=" color: #1A1A1A;">cell </span>states in individual tumor types</p><p class="s100" style="padding-left: 81pt;text-indent: 0pt;line-height: 679748pt;text-align: left;">(averaged <span style=" color: #080808;">by cell type)</span></p><p class="s101" style="padding-left: 200pt;text-indent: 0pt;line-height: 679729pt;text-align: center;">II <span style=" color: #1A1A1A;">type</span></p><ul id="l20"><li style="padding-left: 194pt;text-indent: -9pt;line-height: 679719pt;text-align: left;"><p class="s144" style="display: inline;">B <span class="s100">cells</span></p></li></ul><ul id="l21"><li style="padding-left: 194pt;text-indent: -8pt;line-height: 679711pt;text-align: left;"><p class="s101" style="display: inline;">Plasma <span style=" color: #1A1A1A;">cells</span></p><ul id="l22"><li style="padding-left: 246pt;text-indent: -8pt;line-height: 4758096pt;text-align: right;"><p class="s100" style="display: inline;">S04</p></li><li style="padding-left: 246pt;text-indent: -8pt;line-height: 4758087pt;text-align: right;"><p class="s100" style="display: inline;">S05</p></li><li style="padding-left: 246pt;text-indent: -8pt;line-height: 4758080pt;text-align: right;"><p class="s100" style="display: inline;">S06</p></li></ul></li></ul><ul id="l23"><li style="padding-left: 246pt;text-indent: -8pt;line-height: 4758071pt;text-align: right;"><p class="s144" style="display: inline;">SO?</p><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5</p><p class="s114" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/></li><li style="padding-left: 246pt;text-indent: -8pt;line-height: 4758078pt;text-align: right;"><p class="s144" style="display: inline;">SOB</p></li></ul><p class="s230" style="padding-left: 110pt;text-indent: 0pt;line-height: 42%;text-align: left;">．<span class="s231">．</span><span class="s232">••</span></p><ul id="l24"><li style="padding-left: 38pt;text-indent: -8pt;line-height: 4758096pt;text-align: left;"><p class="s100" style="display: inline;">CDS <span style=" color: #080808;">T </span>cells</p></li><li style="padding-left: 38pt;text-indent: -8pt;line-height: 4758089pt;text-align: left;"><p class="s101" style="display: inline;"><span style=" color: #1A1A1A;">CD4 </span>T <span style=" color: #2D2D2D;">ce</span>lls</p></li><li style="padding-left: 38pt;text-indent: -8pt;line-height: 4758080pt;text-align: left;"><p class="s100" style="display: inline;">NK cells</p></li><li style="padding-left: 38pt;text-indent: -8pt;line-height: 4758072pt;text-align: left;"><p class="s100" style="display: inline;">Mono/Mac</p><p class="s101" style="padding-left: 38pt;text-indent: 0pt;line-height: 4758078pt;text-align: left;">Dendntic <span style=" color: #1A1A1A;">cells</span></p><p class="s233" style="padding-left: 44pt;text-indent: 0pt;line-height: 28548431pt;text-align: center;">。</p><p class="s235" style="padding-left: 110pt;text-indent: 0pt;line-height: 28548432pt;text-align: left;"><span style=" color: #BA2F28;">- </span>, I <span style=" color: #2D2D2D;">, , </span>I</p><p class="s237" style="padding-left: 26pt;text-indent: 0pt;line-height: 28548465pt;text-align: left;">．<span class="s150">•    </span><span class="s100">S09</span></p><p class="s100" style="padding-left: 12pt;text-indent: 0pt;line-height: 28548462pt;text-align: center;">Cell <span style=" color: #080808;">lineage</span></p><ul id="l25"><li style="padding-left: 63pt;text-indent: -10pt;line-height: 28548453pt;text-align: left;"><p class="s101" style="display: inline;">Ep1thel1al <span style=" color: #2D2D2D;">ce</span>ll<span style=" color: #2D2D2D;">s</span></p></li></ul></li></ul><p class="s238" style="padding-left: 53pt;text-indent: 0pt;line-height: 28548445pt;text-align: left;">D <span class="s101">Non-immune</span></p><p class="s100" style="padding-left: 12pt;text-indent: 0pt;line-height: 28548438pt;text-align: center;">stroma</p><p class="s238" style="padding-left: 12pt;text-indent: 0pt;line-height: 28548432pt;text-align: center;">D   <span class="s100">Myeloid</span></p><ul id="l26"><li style="padding-left: 145pt;text-indent: -8pt;line-height: 28548472pt;text-align: left;"><p class="s100" style="display: inline;">Mast <span style=" color: #2D2D2D;">ce</span><span style=" color: #080808;">lls</span></p><p class="s101" style="padding-left: 172pt;text-indent: 0pt;line-height: 28548464pt;text-align: center;">Neutrophil<span style=" color: #2D2D2D;">s</span></p></li><li style="padding-left: 145pt;text-indent: -8pt;line-height: 28548456pt;text-align: left;"><p class="s101" style="display: inline;">Fibroblasts</p></li><li style="padding-left: 145pt;text-indent: -8pt;line-height: 28548448pt;text-align: left;"><p class="s101" style="display: inline;">Endothelial <span style=" color: #1A1A1A;">cells</span></p></li><li style="padding-left: 145pt;text-indent: -8pt;line-height: 28548440pt;text-align: left;"><p class="s101" style="display: inline;">Epithelial <span style=" color: #1A1A1A;">cells</span></p></li></ul><p class="s239" style="padding-left: 112pt;text-indent: 0pt;line-height: 107374182pt;text-align: left;">真。 汽贮<span class="s240">t · </span>罗$ 约<span class="s115">Pc eft• </span>罗度。<span class="s241">rt·</span>。<span class="s241">i</span></p><p class="s242" style="padding-left: 99pt;text-indent: 0pt;line-height: 8pt;text-align: left;">＄心了心心<span class="s115">oo&lt;-            </span><span class="s102">。笭。.;,-,&lt;.- </span><span class="s243">.,&quot;</span></p><p class="s244" style="padding-left: 117pt;text-indent: 0pt;line-height: 4pt;text-align: left;">1:,0 <span class="s100">&lt;) &lt;-</span></p><p class="s101" style="padding-left: 12pt;text-indent: 0pt;line-height: 107374182pt;text-align: left;"><span class="s184">亡 </span>l L<span style=" color: #2D2D2D;">y</span>mphoid</p><p class="s243" style="padding-left: 111pt;text-indent: 0pt;line-height: 107374182pt;text-align: left;">笭</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="657" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_271.png"/></span></p><p style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark93" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Figure S2. Identification and characterization of cell states in 16 types of carcinoma, related to </a><a href="#bookmark93" class="s42">Figure 2</a></p><ol id="l27"><li style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Heatmap depicting the cell type specificity of digitally-purified expression profiles. Cell type specificity was evaluated using the Jaccard index, scaled to unit variance by rows and columns, between (i) detectably expressed genes imputed by CIBERSORTx for each cell type in the discovery cohort (rows) and (ii) cell type-specific genes identified in scRNA-seq tumor atlases (columns) (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (B) Schema showing cell state identification and quality control steps, resulting in 69 cell states (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). NMF, non-negative matrix factorization. (C) </span><span class="s35">Left: </span><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Association between AFI (adaptive false positive index) and the significance of cell-state recovery in a compendium of carcinomas profiled by scRNA-seq (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). Statistical significance between groups was determined by a two-sided Wilcoxon rank sum test. </span><span class="s35">Right: </span><span style=" color: #000;">Same as the left panel but showing the fraction of cell states recovered in scRNA-seq data as a function of AFI. (D) Identification of poor-quality cell states using a ‘‘dropout score’’ – analogous to a </span><span class="s35">Z</span><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">-score – which quantifies states whose marker genes exhibit low variance and high expression across the discovery cohort (</a>STAR Methods<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). States with a dropout score &gt; 1.96 (p &lt; 0.05) were omitted. (E) Heatmaps of NMF consensus matrices, obtained from 50 restarts with random initializations, shown for all cell states in </a><a href="#bookmark93" class="a">Figures </a>2<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">A–2C. Each row and column within the matrix represent a sample. The consensus index indicates cluster stability and represents the fraction of 50 NMF restarts in which a pair of samples co-clustered in the same cell state. Tumor and adjacent normal samples are indicated. (F) UMAP of digitally-purified expression profiles distinguishing tumor and adjacent normal specimens (related to </a><a href="#bookmark93" class="a">Figure </a>2<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">B). (G,H) Boxplots showing the enrichment of each cell state per tumor type, averaged across all 16 carcinomas and grouped according to cell type (G) or averaged across all cell states per cell type and grouped according to tumor type (H) (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Plots in G,H are ordered by decreasing median enrichment per cell type or tumor type, respectively. Of note, the specific numbering used for each state (i.e., S1, S2, etc.) is arbitrary and specific to a given cell type. Only cell states with positive enrichment values were considered (</a><a href="#bookmark98" class="a">Table </a>S3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Cell states in panel H are colored by parental cell type for clarity. Underlying data are provided in </a><a href="#bookmark98" class="a">Table </a>S3<span style=" color: #000;">. Data in (C) left, (G), and (H) are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.53IQR of the box limits; IQR, interquartile range).</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_272.png"/></span></p><p class="s245" style="padding-top: 7pt;padding-left: 52pt;text-indent: 0pt;line-height: 18pt;text-align: left;"><span class="s95">Cell </span>c&#39;Ce<span class="h2">ll</span>=&gt;ress</p><p class="s246" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s12" style="padding-left: 53pt;text-indent: 0pt;line-height: 15pt;text-align: left;"><a name="bookmark205">Resource</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-top: 4pt;padding-left: 88pt;text-indent: -19pt;text-align: left;"><span class="s247">A </span>Comparison of <span style=" color: #080808;">EcoTyper </span>results across scRNA-seq platforms</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s249" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">B</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-top: 4pt;padding-left: 60pt;text-indent: -22pt;line-height: 122%;text-align: left;">Number <span style=" color: #181818;">of expressed </span>genes <span style=" color: #181818;">across different scRNA-seq </span>profiling <span style=" color: #181818;">platforms</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s101" style="padding-top: 3pt;padding-left: 37pt;text-indent: -6pt;line-height: 125%;text-align: left;">Recovery <span style=" color: #181818;">of cell states across scRNA-seq platforms according </span>to <span style=" color: #181818;">number of cells</span></p><p style="padding-left: 29pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="175" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_273.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="129" height="41" alt="image" src="1-s2.0-S0092867421010618-main/Image_274.png"/></span></p><p class="s250" style="text-indent: 0pt;line-height: 68pt;text-align: left;">勹</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-left: 210pt;text-indent: 0pt;line-height: 125%;text-align: center;"><span style=" color: #080808;">T </span>cells <span style=" color: #080808;">from </span>NSCLC tumo<span class="s251">「</span>S profiled by distinct</p><p class="s248" style="padding-left: 210pt;text-indent: 0pt;line-height: 2pt;text-align: center;">scRNA-seq platforms</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s252" style="padding-left: 59pt;text-indent: 0pt;line-height: 5pt;text-align: right;">..</p><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">p</p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">assaJdxasaua5 <span class="s254">0 </span><span class="s255">」</span>a  <span style=" color: #080808;">nuueaJIV</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s256" style="padding-left: 59pt;text-indent: 0pt;line-height: 4pt;text-align: right;">Q)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="79" height="123" alt="image" src="1-s2.0-S0092867421010618-main/Image_275.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_276.png"/></span></p><p class="s257" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s251" style="padding-left: 202pt;text-indent: 0pt;text-align: center;">＊＊＊七</p><p class="s259" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">－</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-top: 1pt;padding-left: 210pt;text-indent: 0pt;line-height: 5pt;text-align: center;"><span style=" color: #181818;">••••          Cell </span>type                                       &#39;<span class="s260">&#39;</span><span class="s261">{</span>-&#39;<span class="s260">&#39;</span><span class="s261">  </span>2<span style=" color: #2A2A2A;">0</span></p><p class="s101" style="padding-top: 3pt;padding-left: 2pt;text-indent: 0pt;text-align: left;">Original <span style=" color: #2A2A2A;">ce</span>ll numb<span style=" color: #2A2A2A;">e</span>r<span style=" color: #2A2A2A;">s</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_277.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="85" height="125" alt="image" src="1-s2.0-S0092867421010618-main/Image_278.png"/></span></p><p class="s263" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">*</p><p style="text-indent: 0pt;text-align: left;"/><p class="s262" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">20</p><p class="s101" style="padding-left: 2pt;text-indent: 0pt;line-height: 111%;text-align: center;">D<span style=" color: #2A2A2A;">ownsa</span>mpled to the <span style=" color: #181818;">same </span>number <span style=" color: #181818;">of cells</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_279.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="85" height="126" alt="image" src="1-s2.0-S0092867421010618-main/Image_280.png"/></span></p><p class="s101" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">ns</p><p class="s101" style="padding-top: 1pt;padding-left: 38pt;text-indent: 0pt;line-height: 4pt;text-align: left;">ns</p><p style="text-indent: 0pt;text-align: left;"><span><img width="60" height="22" alt="image" src="1-s2.0-S0092867421010618-main/Image_281.png"/></span></p><p class="s264" style="padding-top: 6pt;padding-left: 99pt;text-indent: 0pt;line-height: 22pt;text-align: left;">;;t t</p><p class="s193" style="padding-left: 12pt;text-indent: 0pt;line-height: 9pt;text-align: center;">&#39;&#39;</p><p class="s259" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">－</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s257" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">3</p><p style="text-indent: 0pt;text-align: left;"/><ul id="l28"><li style="padding-left: 120pt;text-indent: -7pt;line-height: 6pt;text-align: right;"><p class="s248" style="display: inline;">CD4Tcells</p></li></ul></li></ol><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">q<span class="s265">E</span></p><p style="text-indent: 0pt;text-align: left;"/><ul id="l29"><li style="padding-left: 112pt;text-indent: -20pt;line-height: 63%;text-align: right;"><p class="s101" style="display: inline;">&quot; <span style=" color: #181818;">CD8 </span>T cell<span style=" color: #2A2A2A;">s</span></p></li></ul><p class="s266" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: right;">·<span class="s267">N</span><span style=" color: #080808;">-</span>·</p><p class="s268" style="text-indent: 0pt;line-height: 6pt;text-align: right;">芒<span class="s269">15</span></p><p class="s256" style="padding-left: 59pt;text-indent: 0pt;line-height: 2pt;text-align: right;">Q)</p><p class="s270" style="padding-left: 59pt;text-indent: 0pt;line-height: 8pt;text-align: right;">i:i</p><p class="s271" style="text-indent: 0pt;line-height: 13pt;text-align: left;">飞</p><p style="text-indent: 0pt;text-align: left;"/><p class="s272" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">—<span class="s273">中七</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s259" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">－</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s257" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s274" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">一</p><p style="text-indent: 0pt;text-align: left;"/><p class="s275" style="padding-bottom: 2pt;text-indent: 0pt;line-height: 0pt;text-align: right;">-</p><p class="s276" style="text-indent: 0pt;line-height: 6pt;text-align: left;">a,</p><p style="padding-left: 48pt;text-indent: 0pt;line-height: 6pt;text-align: left;"/><p class="s101" style="padding-left: 12pt;text-indent: 0pt;line-height: 9pt;text-align: center;">ns    <span class="s277">&#39;        </span><span class="s278">&#39;</span></p><p class="s279" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">点<span class="s280">＂</span><span class="s281">宁</span><span class="s282"> </span><span style=" color: #181818;">卢</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s284" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">;</p><p class="s263" style="padding-top: 5pt;padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">sn[</p><p style="text-indent: 0pt;text-align: left;"/><p class="s285" style="padding-left: 62pt;text-indent: 0pt;line-height: 84%;text-align: left;">15 <span class="s267">&#39;&#39;</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="129" height="40" alt="image" src="1-s2.0-S0092867421010618-main/Image_282.png"/></span></p><p class="s286" style="padding-top: 8pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">:;。</p><p class="s248" style="padding-left: 78pt;text-indent: 0pt;text-align: left;">scRNA-seq <span style=" color: #080808;">dataset </span><span class="s287">(.) </span>10 <span style=" color: #080808;">10</span></p><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5</p><p style="text-indent: 0pt;text-align: left;"/><p class="s288" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">孛</p><p style="text-indent: 0pt;text-align: left;"/><ul id="l30"><li style="padding-top: 4pt;padding-left: 88pt;text-indent: -7pt;line-height: 6pt;text-align: left;"><p class="s101" style="display: inline;">NSCLC1 <span style=" color: #181818;">(Lambrechts et al.) </span>o</p><p class="s289" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">士<span class="s290">L</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s259" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">－</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s257" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s258" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s256" style="padding-left: 171pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Q) <span class="s291">5</span></p><p class="s292" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s293" style="text-indent: 0pt;line-height: 20pt;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/></li><li style="padding-left: 88pt;text-indent: -7pt;line-height: 9pt;text-align: left;"><p class="s101" style="display: inline;">NSCLC2 <span style=" color: #181818;">(Zil1on1s et al ) </span><span class="s294">恃</span></p><p class="s295" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/></li><li style="padding-left: 88pt;text-indent: -7pt;line-height: 9pt;text-align: left;"><p class="s101" style="display: inline;">N SCLC<span style=" color: #2A2A2A;">3 </span><span style=" color: #181818;">(Laughney et al.) </span><span class="s257">色</span></p><p class="s296" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">8<span class="s297">勹或</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">尽</p><p style="text-indent: 0pt;text-align: left;"/><p class="s298" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">令<span class="s253">＄</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s299" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">＆</p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">,</p><p style="text-indent: 0pt;text-align: left;"/><p class="s298" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o<span class="s253">o</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s300" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">恣<span class="s301">｀</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s263" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">、令</p><p style="text-indent: 0pt;text-align: left;"/><p class="s251" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">4 <span class="s302">`</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s303" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">＄</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">求</p><p class="s303" style="padding-left: 5pt;text-indent: 0pt;line-height: 120%;text-align: left;">令<span class="s304">＄</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s299" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">＆</p><p style="text-indent: 0pt;text-align: left;"/><p class="s254" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0<span class="s263">允</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s257" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">令<span class="s305">L</span><span class="s268">`</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">今</p><p style="text-indent: 0pt;text-align: left;"/><p class="s268" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">＋</p><p style="text-indent: 0pt;text-align: left;"/><p class="s306" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0<span class="s307">\</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s299" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">恣</p><p style="text-indent: 0pt;text-align: left;"/><p class="s295" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s298" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0<span class="s253">$</span></p><p style="text-indent: 0pt;text-align: left;"/></li><li style="padding-left: 88pt;text-indent: -7pt;line-height: 15pt;text-align: left;"><p class="s101" style="display: inline;">NSCLC4 <span style=" color: #181818;">(Guo et al.) </span><span class="s308">易</span></p></li></ul><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-left: 85pt;text-indent: -16pt;line-height: 110%;text-align: left;"><span class="s309">D </span>Validation of adenocarcinoma/squamous <span style=" color: #080808;">cell </span>carcinoma enrichments <span style=" color: #080808;">in </span>scRNA-seq data</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="185" height="61" alt="image" src="1-s2.0-S0092867421010618-main/Image_283.png"/></span></p><p class="s257" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">凌</p><p style="text-indent: 0pt;text-align: left;"/><p class="s299" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">4 <span class="s308">E</span></p><p class="s254" style="padding-left: 8pt;text-indent: 0pt;line-height: 120%;text-align: left;">今</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-top: 4pt;padding-left: 50pt;text-indent: -13pt;line-height: 118%;text-align: left;">Comparison of cell state classification across scRNA-seq classifiers</p><p class="s310" style="padding-top: 4pt;padding-left: 36pt;text-indent: -21pt;text-align: left;">f <span class="s248">Validation of cell states enriched in adjacent </span><span class="s101">normal tissues</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s311" style="text-indent: 0pt;line-height: 29pt;text-align: left;">＿</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 80pt;text-indent: -7pt;line-height: 10pt;text-align: right;">Bulk RNA-seq <span style=" color: #181818;">scRNA-seq</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-left: 18pt;text-indent: -12pt;line-height: 111%;text-align: left;">Concordance p-value <span style=" color: #080808;">= </span>0.03 Cell states (n <span style=" color: #080808;">= </span>69)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="123" height="28" alt="image" src="1-s2.0-S0092867421010618-main/Image_284.png"/></span></p><p class="s312" style="padding-top: 1pt;padding-left: 47pt;text-indent: 0pt;line-height: 39%;text-align: left;">I <span class="s313">[</span></p><p class="s262" style="padding-top: 2pt;padding-left: 50pt;text-indent: 0pt;text-align: center;"><span style=" color: #181818;">Cell </span>sta<span style=" color: #080808;">tesidentified </span>wi<span style=" color: #080808;">thEcoT</span>yper<span style=" color: #181818;">,n </span><span style=" color: #080808;">TCGA </span>(d<span style=" color: #080808;">i</span>scovery)</p><p class="s262" style="padding-top: 1pt;padding-left: 19pt;text-indent: 0pt;text-align: center;"><span style=" color: #080808;">L</span>amb<span class="s314">「</span><span class="s291">echts </span><span style=" color: #181818;">et </span>al <span style=" color: #181818;">scRNA-seq dataset</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="150" alt="image" src="1-s2.0-S0092867421010618-main/Image_285.png"/></span></p><p class="s248" style="padding-left: 30pt;text-indent: 0pt;line-height: 8pt;text-align: left;">(validation) <span class="s299">。 七</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s113" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span style=" color: #2A2A2A;">(V93</span>L<span class="s316">) </span>     <span class="s251">9</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s317" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">+</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">Compare <span style=" color: #080808;">four classifiers</span></p><p class="s318" style="padding-top: 3pt;text-indent: 1pt;line-height: 74%;text-align: left;">求。<span class="s266">&lt;-I </span><span class="s319">6&#39;</span></p><p class="s320" style="padding-left: 12pt;text-indent: 0pt;line-height: 17pt;text-align: center;">式<span class="s321">,  0.,</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="48" height="150" alt="image" src="1-s2.0-S0092867421010618-main/Image_286.png"/></span></p><p class="s268" style="text-indent: 0pt;line-height: 6pt;text-align: left;">谷</p><p style="text-indent: 0pt;text-align: left;"/><p class="s322" style="text-indent: 0pt;line-height: 10pt;text-align: left;">0 <span class="s323">;p</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s324" style="padding-left: 12pt;text-indent: 0pt;line-height: 27%;text-align: center;">&lt;v<span class="s325">c,O    </span><span class="s326">._::.      </span><span class="s327">&lt;&#39;.)</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-left: 89pt;text-indent: 0pt;line-height: 2pt;text-align: left;">Adenocarc,noma</p><p class="s101" style="padding-top: 4pt;padding-left: 6pt;text-indent: 0pt;line-height: 5pt;text-align: left;">Enriched in: <span style=" color: #181818;">Squamous cell</span></p><p class="s328" style="padding-left: 6pt;text-indent: 0pt;line-height: 1pt;text-align: left;">, <span class="s248">, carcinoma</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 4pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="45" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_287.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 4pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="45" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_288.png"/></span></p><p class="s329" style="padding-left: 58pt;text-indent: 0pt;line-height: 11pt;text-align: left;">¢ <span class="s330">+ +</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 82pt;text-indent: 0pt;text-align: left;"><span><img width="106" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_289.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-top: 4pt;padding-left: 2pt;text-indent: 0pt;line-height: 2pt;text-align: left;">Classify</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Normal</p><p class="s331" style="padding-left: 83pt;text-indent: 0pt;line-height: 5pt;text-align: left;">Enrichment <span class="s248">-log</span><span class="s332">10 </span><span class="s101">P </span><span class="s333">■</span><span class="s334">二 </span><span class="s335">二</span><span class="s336">噩，</span></p><p class="s291" style="padding-left: 142pt;text-indent: 0pt;line-height: 1pt;text-align: left;">2 <span style=" color: #181818;">1 0 1 2</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s337" style="text-indent: 0pt;line-height: 15pt;text-align: left;">G</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 117pt;text-indent: -34pt;line-height: 11pt;text-align: left;">Comparison <span style=" color: #181818;">of </span>performance across <span style=" color: #181818;">classifiers </span><span style=" color: #2A2A2A;">Va</span>l1dat1on <span style=" color: #181818;">of cell                adiacent</span></p><p class="s262" style="padding-left: 112pt;text-indent: 0pt;text-align: left;">sta<span style=" color: #080808;">te</span>s <span style=" color: #181818;">enriched in: </span><span style=" color: #080808;">normal </span><span style=" color: #181818;">tissues</span></p><p style="padding-left: 163pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="94" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_290.png"/></span></p><p class="s338" style="padding-left: 59pt;text-indent: 0pt;line-height: 0pt;text-align: right;">¢ <span class="s307">令</span></p><p class="s248" style="padding-left: 54pt;text-indent: 0pt;line-height: 4pt;text-align: left;">SVM SingleCellNet scmapcell</p><p style="text-indent: 0pt;text-align: left;"><span><img width="48" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_291.png"/></span></p><p class="s339" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">=</p><p style="text-indent: 0pt;text-align: left;"/><p class="s340" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">sa1e1s</p><p style="text-indent: 0pt;text-align: left;"/><p class="s251" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0<span style=" color: #2A2A2A;">69</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">r</p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e</p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">p</p><p style="text-indent: 0pt;text-align: left;"/><p class="s317" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">|</p><p style="text-indent: 0pt;text-align: left;"/><p class="s317" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">yT</p><p style="text-indent: 0pt;text-align: left;"/><p class="s308" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s119" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s341" style="text-indent: 0pt;line-height: 23pt;text-align: left;">+</p><p style="text-indent: 0pt;text-align: left;"/><p class="s342" style="padding-left: 30pt;text-indent: 0pt;line-height: 16pt;text-align: left;">              I<span class="s343"> I</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="186" height="56" alt="image" src="1-s2.0-S0092867421010618-main/Image_292.png"/></span></p><p class="s248" style="padding-top: 9pt;padding-left: 27pt;text-indent: 0pt;text-align: left;">Compare performance across <span style=" color: #2A2A2A;">class</span><span style=" color: #080808;">ifiers</span></p><ol id="l31"><li style="padding-top: 1pt;padding-left: 24pt;text-indent: -6pt;line-height: 6pt;text-align: left;"><p class="s248" style="display: inline;">Concordance of cell states enriched in</p><p class="s248" style="padding-left: 24pt;text-indent: 0pt;line-height: 0pt;text-align: left;">ad<span class="s254">」</span>acent <span style=" color: #080808;">normal </span>tissues</p><p class="s262" style="padding-left: 5pt;text-indent: 0pt;line-height: 5pt;text-align: left;">s<span style=" color: #080808;">ingle</span></p><p class="s262" style="padding-left: 6pt;text-indent: 0pt;line-height: 2pt;text-align: left;">ce<span style=" color: #080808;">ll</span>s</p><p class="s262" style="padding-left: 3pt;text-indent: 0pt;line-height: 5pt;text-align: left;">enrichment</p><p class="s262" style="text-indent: 0pt;line-height: 1pt;text-align: left;">(-log<span style=" color: #545454;">,, </span>p-value)</p><p class="s345" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">0 <span class="s346">仁二</span><span class="s258">勹 </span><span style=" color: #181818;">2</span></p><p class="s265" style="padding-top: 2pt;padding-left: 7pt;text-indent: -7pt;line-height: 78%;text-align: left;">口<span class="s248">Unassigned </span><span class="s262">states</span></p><p class="s308" style="text-indent: 0pt;line-height: 14pt;text-align: left;">。 <span class="s322">&#39; </span>。<span class="s322">-i </span><span class="s348">0 </span>。<span class="s349">。 谷</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s350" style="text-indent: 0pt;line-height: 16pt;text-align: right;">人 感 <span class="s295">。</span></p><p class="s263" style="text-indent: 0pt;line-height: 5pt;text-align: left;">今</p><p style="text-indent: 0pt;text-align: left;"/><p class="s351" style="text-indent: 0pt;line-height: 11pt;text-align: right;">令丿<span class="s340">   </span><span class="s308">... </span><span class="s352">-&lt;:- 。$</span><span class="s353">Q</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="83" height="14" alt="image" src="1-s2.0-S0092867421010618-main/Image_293.png"/></span></p><p class="s248" style="padding-left: 83pt;text-indent: 0pt;line-height: 13pt;text-align: left;">Concordance <span style=" color: #3F3F3D;">wit</span>h discovery<span class="s355">(%) </span><span class="s356">昌</span><span class="s357">....</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="25" alt="image" src="1-s2.0-S0092867421010618-main/Image_294.png"/></span></p><table style="border-collapse:collapse" cellspacing="0"><tr style="height:11pt"><td style="width:26pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt"><p class="s358" style="text-indent: 0pt;line-height: 9pt;text-align: left;">暑匡 <span style=" color: #77544F;">了</span></p></td><td style="width:22pt;border-top-style:solid;border-top-width:1pt;border-bottom-style:solid;border-bottom-width:1pt"><p class="s359" style="text-indent: 0pt;line-height: 9pt;text-align: center;">了了<span style=" color: #AA7567;">可</span></p></td><td style="width:12pt;border-top-style:solid;border-top-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:2pt;border-right-color:#F4E4E2"><p class="s361" style="padding-left: 2pt;text-indent: 0pt;text-align: left;">2<span style=" color: #3F3F3D;">.</span><span style=" color: #080808;">1</span></p></td></tr><tr style="height:11pt"><td style="width:26pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt"><p class="s358" style="padding-top: 1pt;text-indent: 0pt;line-height: 8pt;text-align: left;">匿 <span style=" color: #AA4436;">一</span></p></td><td style="width:22pt;border-top-style:solid;border-top-width:1pt;border-bottom-style:solid;border-bottom-width:1pt"><p class="s365" style="padding-top: 1pt;padding-left: 3pt;text-indent: 0pt;line-height: 8pt;text-align: center;">袒</p></td><td style="width:12pt;border-top-style:solid;border-top-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:2pt;border-right-color:#F4E4E2"><p class="s361" style="padding-top: 5pt;padding-left: 1pt;text-indent: 0pt;line-height: 4pt;text-align: left;">2 1</p></td></tr></table><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-top: 4pt;padding-left: 68pt;text-indent: 0pt;text-align: left;">Concordance <span style=" color: #2A2A2A;">with discovery (-log,, P)</span></p><p class="s248" style="padding-top: 2pt;padding-left: 96pt;text-indent: 0pt;text-align: left;">Number <span style=" color: #2A2A2A;">of </span>assigned <span style=" color: #2A2A2A;">states </span><span class="s366">I</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p></li><li style="padding-left: 21pt;text-indent: -6pt;line-height: 87%;text-align: left;"><p class="s248" style="display: inline;">Concordance of <span style=" color: #2A2A2A;">ce</span><span style=" color: #080808;">ll </span>states enriched in adenocarcinoma or squamous cell carcinoma</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s367" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">m<span class="s368"> sel</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s158" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">senals</p><p style="text-indent: 0pt;text-align: left;"/><p class="s369" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">二</p><p style="text-indent: 0pt;text-align: left;"/><p class="s303" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">eglleal</p><p style="text-indent: 0pt;text-align: left;"/><p class="s368" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s367">1</span> <span class="s370">c</span>a</p><p style="text-indent: 0pt;text-align: left;"/><p class="s369" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">卢</p><p style="text-indent: 0pt;text-align: left;"/><p class="s371" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">.<span class="s368">alts</span><span style=" color: #080808;">d</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s294" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">ea<span class="s372">q</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s339" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">htilbes</p><p style="text-indent: 0pt;text-align: left;"/><p class="s339" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">p <span class="s373">o</span>r-</p><p style="text-indent: 0pt;text-align: left;"/><p class="s374" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e<span class="s294">bA</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s375" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">f <span class="s376">if</span>N</p><p style="text-indent: 0pt;text-align: left;"/><p class="s339" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">t</p><p style="text-indent: 0pt;text-align: left;"/><p class="s265" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">x<span class="s376">p</span><span class="s307">．</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s375" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s294" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e</p><p style="text-indent: 0pt;text-align: left;"/><p class="s368" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">_</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 33pt;text-indent: -27pt;text-align: left;"><span class="s337">H </span>Expression of collagen <span style=" color: #181818;">and </span>mesenchymal <span style=" color: #181818;">gene </span>sets in <span style=" color: #181818;">epithelial </span>cell <span style=" color: #181818;">states</span></p><p class="s303" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s</p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-top: 1pt;padding-left: 54pt;text-indent: 0pt;text-align: left;">in scRNA-seq data</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s378" style="text-indent: 0pt;line-height: 28pt;text-align: left;"> 冷守。。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s262" style="padding-left: 112pt;text-indent: 4pt;line-height: 108%;text-align: left;">Va<span style=" color: #080808;">hdat,on </span><span style=" color: #181818;">of cell         adenoca, rcn oma</span><span style=" color: #3F3F3D;">/ </span>sta<span style=" color: #080808;">t</span>es <span style=" color: #181818;">enriched in:  </span>sq<span style=" color: #080808;">uamou</span>s <span style=" color: #181818;">cell </span>ca<span style=" color: #080808;">rcinom </span>a</p><p class="s257" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 11pt;text-align: right;"> ;;.、</p><p class="s379" style="text-indent: 0pt;line-height: 16pt;text-align: right;">汾了 泸<span class="s380">r </span><span class="s381">eo&#39;-&#39; &#39;2&gt;&quot;</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="83" height="53" alt="image" src="1-s2.0-S0092867421010618-main/Image_295.png"/></span></p><p class="s382" style="padding-left: 83pt;text-indent: 0pt;line-height: 13pt;text-align: left;"><span class="s262">Concorda</span><span class="s101">n</span><span class="s262">ce with </span><span class="s248">discovery</span><span style=" color: #2A2A2A;">(%) </span><span style=" color: #8A0101;">昌！</span>昌<span style=" color: #8A0101;">，</span>！<span style=" color: #A13423;">昌</span>冒<span style=" color: #A13423;">-</span></p><p class="s262" style="text-indent: 0pt;line-height: 10pt;text-align: left;"><span class="s356">瞿瞿</span><span class="s355">。</span>24 0.4<span style=" color: #080808;">1 </span>0.22</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-top: 3pt;padding-left: 68pt;text-indent: 0pt;text-align: left;">Concordance <span style=" color: #2A2A2A;">with discovery </span>(- log<span style=" color: #545454;">,, </span>P)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s248" style="padding-left: 48pt;text-indent: 0pt;text-align: left;">Cell <span style=" color: #2A2A2A;">s</span><span style=" color: #080808;">tate </span><span class="s158">令 今 </span><span class="s384">0 &#39;), </span><span class="s253">今。今谷</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="47" alt="image" src="1-s2.0-S0092867421010618-main/Image_296.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="102" height="127" alt="image" src="1-s2.0-S0092867421010618-main/Image_297.png"/></span></p><p class="s385" style="padding-top: 4pt;text-indent: 0pt;text-align: left;">• <span style=" color: #9AB1D4;">• • • •</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s385" style="text-indent: 0pt;text-align: left;">• <span style=" color: #804FA3;">• </span><span style=" color: #8C85BC;">• </span><span style=" color: #BFD1E4;">• • </span><span style=" color: #9AB1D4;">•</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">t<span class="s390">e</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1 2 3 4 5 6 7 8</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">a</p><p style="text-indent: 0pt;text-align: left;"/><p class="s391" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s</p><p style="text-indent: 0pt;text-align: left;"/><p class="s391" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">sssssss</p><p style="text-indent: 0pt;text-align: left;"/><p class="s394" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s258">3 </span><span class="s392">咖</span><span class="s393">咖</span>申<span class="s395">咖</span>申申<span class="s396">咖</span>白</p><p style="text-indent: 0pt;text-align: left;"/><p class="s397" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">.</p><p style="text-indent: 0pt;text-align: left;"/><p class="s398" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">,  <span class="s399">:</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s400" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">｀</p><p style="text-indent: 0pt;text-align: left;"/><p class="s401" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">. t <span style=" color: #9C9C9C;">!</span><span class="s403">1</span><span style=" color: #2A2A2A;">| </span><span class="s405">＇</span><span class="s406"> </span><span class="s308">b</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s407" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">,</p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0      5      0       5      0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0      7       5      2      0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s158" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">uo1ssaJdxa<span class="s408">p</span>az<span class="s408">q</span>e<span class="s367">E</span><span class="s368">  </span><span class="s184">N</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-top: 4pt;padding-left: 12pt;text-indent: 12pt;line-height: 181%;text-align: left;">Collagen gene set Mesenchymal gene set</p><p class="s398" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s409" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">• ••</p><p style="text-indent: 0pt;text-align: left;"/><p class="s398" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s407" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">J</p><p style="text-indent: 0pt;text-align: left;"/><p class="s401" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">.:<span style=" color: #2A2A2A;">_</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-top: 5pt;padding-left: 86pt;text-indent: 0pt;line-height: 6pt;text-align: left;">Relative expression</p><p style="text-indent: 0pt;text-align: left;"><span><img width="33" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_298.png"/></span></p><p class="s410" style="text-indent: 0pt;line-height: 12pt;text-align: left;">. <span class="s261">2,</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s411" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s</p><p style="text-indent: 0pt;text-align: left;"/><p class="s399" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">g</p><p style="text-indent: 0pt;text-align: left;"/><p class="s368" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">ts</p><p style="text-indent: 0pt;text-align: left;"/><p class="s401" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">＇</p><p style="text-indent: 0pt;text-align: left;"/><h3 style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．<span class="s398">．</span><span class="s412">．</span></h3><p style="text-indent: 0pt;text-align: left;"/><p class="s401" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">，</p><p style="text-indent: 0pt;text-align: left;"/><p class="s401" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">&#39;</p><p style="text-indent: 0pt;text-align: left;"/><p class="s339" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">lF</p><p style="text-indent: 0pt;text-align: left;"/><p class="s253" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">pEee</p><p style="text-indent: 0pt;text-align: left;"/><p class="s377" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s413" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">」<span class="s205">O</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 70pt;text-indent: 12pt;line-height: 11pt;text-align: left;">Lo<span style=" color: #2A2A2A;">w </span><span class="s414">亡 </span>High Fraction <span style=" color: #181818;">concordant datasets</span></p><p class="s415" style="text-indent: 0pt;line-height: 12pt;text-align: left;">黜</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-left: 96pt;text-indent: 0pt;line-height: 5pt;text-align: left;">Numbe<span class="s416">「 </span>of <span style=" color: #2A2A2A;">assigned states</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s101" style="padding-left: 80pt;text-indent: -14pt;text-align: left;"><span class="s337">J </span>Enrichment <span style=" color: #181818;">of </span>monocyte <span style=" color: #181818;">and </span>macrophage <span style=" color: #181818;">state </span>markers in purified <span style=" color: #181818;">cell populations</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="197" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_299.png"/></span></p><p class="s413" style="padding-top: 2pt;padding-left: 131pt;text-indent: 0pt;line-height: 5pt;text-align: left;">亡 <span class="s417">N </span><span class="s418">&lt;&#39;&gt; </span><span class="s416">&lt;!)寸&quot;&#39;,-.. </span><span class="s419">&quot;&#39;O&gt;</span></p><p class="s420" style="padding-left: 12pt;text-indent: 0pt;line-height: 8pt;text-align: left;">I <span class="s421">0 </span><span class="s238">1</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="407" height="100" alt="image" src="1-s2.0-S0092867421010618-main/Image_300.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="197" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_301.png"/></span></p><p class="s422" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">k</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-top: 31pt;padding-left: 151pt;text-indent: 0pt;line-height: 160%;text-align: center;">Foamy macrophage-high <span style=" color: #181818;">(CRC 393) </span>H&amp;E                                <span style=" color: #181818;">CD68</span></p><p class="s101" style="padding-top: 39pt;padding-left: 43pt;text-indent: -13pt;line-height: 160%;text-align: left;">Foamy macrophage-low <span style=" color: #181818;">(CRC 380) </span>H&amp;E <span style=" color: #181818;">CD68</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="95" height="43" alt="image" src="1-s2.0-S0092867421010618-main/Image_302.png"/></span></p><p class="s355" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s423" style="padding-bottom: 2pt;padding-left: 129pt;text-indent: 0pt;line-height: 2pt;text-align: left;">■ ■</p><p class="s406" style="text-indent: 0pt;line-height: 12pt;text-align: left;">勺<span class="s424">«</span></p><p style="padding-left: 304pt;text-indent: 0pt;text-align: left;"/><p class="s425" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s425" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s248" style="padding-left: 59pt;text-indent: 0pt;line-height: 0pt;text-align: right;">Monoc ytes</p><p class="s101" style="padding-left: 68pt;text-indent: 10pt;line-height: 114%;text-align: right;"><span style=" color: #181818;">MO </span>macrophages <span style=" color: #181818;">M1 </span>mac<span class="s124">「</span>ophages Foam<span style=" color: #2A2A2A;">y </span><span style=" color: #181818;">macrophages</span></p><p class="s426" style="padding-left: 1pt;text-indent: 0pt;line-height: 11pt;text-align: left;">(/) (/) (/) (/) (/) (/) (/) (/) (/)</p><p class="s355" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s355" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s404" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">勺</p><p style="text-indent: 0pt;text-align: left;"/><p class="s427" style="padding-left: 8pt;text-indent: 0pt;line-height: 2pt;text-align: left;">_n</p><p style="text-indent: 0pt;text-align: left;"><span><img width="48" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_303.png"/></span></p><p class="s101" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">Depleted <span class="s428">◄</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="55" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_304.png"/></span></p><p class="s248" style="padding-left: 210pt;text-indent: 0pt;line-height: 10pt;text-align: center;">GSEA <span style=" color: #2A2A2A;">- </span><span style=" color: #080808;">10 9</span><span style=" color: #545454;">10 </span>Q  <span class="s429">c....._  </span><span class="s430">,</span></p><p class="s276" style="padding-top: 2pt;padding-left: 13pt;text-indent: 0pt;text-align: left;">► <span class="s101">Enriched</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s355" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s355" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s307" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">勺</p><p style="text-indent: 0pt;text-align: left;"/><p class="s404" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">«</p><p style="text-indent: 0pt;text-align: left;"/><p class="s119" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">了</p><p style="text-indent: 0pt;text-align: left;"/><p class="s113" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">嚷</p><p style="text-indent: 0pt;text-align: left;"/><p class="s431" style="text-indent: 0pt;line-height: 51pt;text-align: left;">—</p><p style="text-indent: 0pt;text-align: left;"/><p class="s101" style="padding-left: 53pt;text-indent: 0pt;line-height: 4pt;text-align: left;">100µm</p><p class="s269" style="padding-left: 144pt;text-indent: 0pt;line-height: 7pt;text-align: left;">4 2 0 2 4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="650" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_305.png"/></span></p><p style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><a href="#bookmark93" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Figure S3. Analysis of cell-type-specific phenotypic states, related to </a><a href="#bookmark93" class="s42">Figure 2</a></p><p class="s38" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">(A–C) Impact of scRNA-seq platform on reference-guided cell state recovery from single-cell transcriptomes. (A) Outline of the approach. To minimize con- founding factors while maximizing the number of evaluable platforms and independent studies, we focused on T cells within NSCLC, which were profiled by two droplet-based assays, inDrop and 10x Chromium 30, and one plate-based assay (SMART-seq2), within our scRNA-seq compendium (</a><a href="#bookmark98" class="a">Table S</a>1<span style=" color: #000;">). (B) Number of detectably expressed genes per cell (TPM/CPM &gt; 0), averaged by cell state and stratified by dataset and platform. Colors indicate source dataset and shapes indicate T cell type. (C) Significance of cell state recovery by platform. </span><span class="s35">Left: </span><span style=" color: #000;">All evaluable cells. </span><span class="s35">Right: </span><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">The number of single cells per dataset was equalized for each cell state by down-sampling without replacement. Cell state colors and shapes are identical to panel B. EcoTyper was applied for reference-guided recovery in all cases. Pairwise comparisons in B,C were determined using a two-sided Wilcoxon rank sum test (ns, not significant; *p &lt; 0.05; ***p &lt; 0.001; ****p &lt; 0.0001). (D) Enrichment of EcoTyper states in adenocarcinomas versus squamous cell carcinomas (</a><a href="#bookmark197" class="a">STAR Method</a>s<a href="#bookmark138" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">), comparing the discovery cohort, which was digitally purified from bulk RNA-seq data (TCGA), against EcoTyper states recovered from an scRNA-seq tumor atlas (</a><a href="#bookmark138" class="a">Lambrechts et al., </a>2018<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). In both cases, NSCLC tumors were analyzed (adenocarcinoma and squamous cell carcinoma from TCGA and Lambrechts et al.). Concordance was determined as the fraction of states shared in each of three classes: adenocarcinoma, squamous cell carcinoma, or neither; significance was assessed by Chi-square test. (E-G) Comparison of supervised methods for scRNA-seq classification (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (E) Schema of the approach. (F) Heatmap showing the enrichment of cell states in adjacent normal tissues in the TCGA discovery cohort (left) versus their enrichment in adjacent normal tissues after supervised classification of single-cell transcriptomes (right) (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Unassigned states are indicated in gray. (G) Summary of performance. Significance was calculated as described in </a><a href="#bookmark93" class="a">Figure </a>2<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">D (normal enrichment) and panel D (histological subtype enrichment) (see also </a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark99" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (H,I) Impact of partial EMT (epithelial–mesenchymal transition) on epithelial cell state detection. (H) Expression of collagen and mesenchymal gene sets (</a><a href="#bookmark99" class="a">Aiello et al., </a>2018<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) (</a><a href="#bookmark98" class="a">Table S</a>4<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) in epithelial cell states defined by EcoTyper and analyzed across three types of carcinoma profiled by scRNA-seq (</a><a href="#bookmark98" class="a">Table S</a>1<a href="#bookmark99" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (I) Boxplots depicting the relative expression of an epithelial-specific gene set (</a><a href="#bookmark99" class="a">Aiello et al., 20</a>18<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) in epithelial and fibroblast states defined by EcoTyper and analyzed in the same datasets as in panel H (</a><a href="#bookmark197" class="a">STAR Method</a>s<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). ‘Fraction concordant datasets’ denotes the proportion of scRNA-seq datasets in which differential expression of the indicated genes is the same direction. (J) Gene set enrichment analysis  of  state-specific  marker  genes  from  the  monocyte/macrophage  lineage  within  four  pre-ranked  gene  lists  corresponding  to  previously  published expression profiles of monocytes and three distinct macrophage subsets (</a><a href="#bookmark197" class="a">STAR Method</a>s<a href="#bookmark93" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (K) Paired H&amp;E and CD68 staining of foamy macrophage-enriched and depleted CRC tumor specimens (related to </a><a href="#bookmark93" class="a">Figure </a>2<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">E; </a><a href="#bookmark197" class="a">STAR Method</a>s<span style=" color: #000;">). Scale bar of 100 mm is identical for all images. Data in (B), (C), and (I) are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.53IQR of the box limits; IQR, inter- quartile range).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s5" style="padding-left: 88pt;text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_306.png"/></span>	<span><img width="172" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_307.png"/></span>	<span><img width="10" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_308.png"/></span>	<span><img width="137" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_309.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><table style="border-collapse:collapse;margin-left:138.857pt" cellspacing="0"><tr style="height:14pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:17pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:8pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:14pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:25pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#D5372E;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#D5372E;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:22pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-top-color:#D5372E;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-top-color:#D5372E;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:17pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:8pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:11pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#D5372E;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#D5372E;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:16pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:20pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:9pt;border-top-style:solid;border-top-width:1pt;border-top-color:#D5372E;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:dotted;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:25pt;border-top-style:solid;border-top-width:1pt;border-top-color:#D5372E;border-left-style:dotted;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr></table><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 366pt;text-indent: 0pt;line-height: 6pt;text-align: left;"><span><img width="165" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_310.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 366pt;text-indent: 0pt;text-align: left;"><span><img width="34" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_311.png"/></span> <span><img width="113" height="19" alt="image" src="1-s2.0-S0092867421010618-main/Image_312.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s432" style="padding-left: 104pt;text-indent: 0pt;line-height: 6pt;text-align: left;"><span><img width="35" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_313.png"/></span> <span><img width="213" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_314.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="239" height="366" alt="image" src="1-s2.0-S0092867421010618-main/Image_315.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="47" height="136" alt="image" src="1-s2.0-S0092867421010618-main/Image_316.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="106" alt="image" src="1-s2.0-S0092867421010618-main/Image_317.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="24" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_318.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="47" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_319.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="44" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_320.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="44" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_321.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="31" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_322.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="39" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_323.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="53" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_324.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="37" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_325.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="42" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_326.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="41" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_327.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_328.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="53" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_329.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="250" height="200" alt="image" src="1-s2.0-S0092867421010618-main/Image_330.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="147" alt="image" src="1-s2.0-S0092867421010618-main/Image_331.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="202" height="292" alt="image" src="1-s2.0-S0092867421010618-main/Image_332.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="154" height="25" alt="image" src="1-s2.0-S0092867421010618-main/Image_333.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="149" height="156" alt="image" src="1-s2.0-S0092867421010618-main/Image_334.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="113" height="32" alt="image" src="1-s2.0-S0092867421010618-main/Image_335.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="97" height="133" alt="image" src="1-s2.0-S0092867421010618-main/Image_336.png"/></span></p><p style="padding-top: 4pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark94" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" name="bookmark206">Figure S4. Analysis of cell-state-specific survival associations in the discovery cohort and PRECOG, related to </a><a href="#bookmark94" class="s42">Figure 3</a></p><ol id="l32"><li style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Heatmap showing overall survival associations of 69 cell states in 5,946 tumors and 16 tumor types (discovery cohort), stratified by cell type and tumor type, and aggregated across malignancies using Stouffer’s method (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">, related to </a><a href="#bookmark94" class="a">Figure </a>3<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">A). Columns were ordered by hierarchical clustering using complete linkage and Euclidean distance. For each cell state, an expressed marker gene is also shown (see also </a><a href="#bookmark98" class="a">Table </a>S4<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (B) Concordance between cell state fractions identified by EcoTyper in digitally-purified expression profiles (x axis) and bulk tissue RNA-seq profiles (y axis) from the discovery cohort (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). These data illustrate the ability of EcoTyper to deconvolve cell state representation from bulk expression profiles. (C) </span><span class="s35">Left: </span><span style=" color: #000;">Approach for evaluating the sensitivity of EcoTyper to the bulk expression profiling platform. LUSC, lung squamous cell carcinoma. OV, ovarian carcinoma. </span><span class="s35">Right: </span><a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Pearson correlation co- efficients between estimated levels of 69 cell states in paired tumor transcriptomes from TCGA profiled by bulk RNA-seq and microarray (n = 129 LUSC and 266 OV). Data are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.53IQR of the box limits; IQR, interquartile range). (D) Scatterplot depicting the correlation between pan-carcinoma state-specific survival associations in the discovery cohort (TCGA) and an independent cohort of 9,062 epithelial tumor transcriptomes (PRECOG) (</a><a href="#bookmark125" class="a">Gentles et al., </a>2015<a href="#bookmark94" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Overall survival associations for 69 cell states are expressed as –log10 p values, multiplied by 1 if the state is associated with adverse outcomes and —1 if associated with favorable outcomes. Concordance and statistical significance were assessed by Pearson correlation (related to </a>Figure 3<span style=" color: #000;">B). The 95% confidence band is also indicated.</span></p></li></ol></li></ol><p style="text-indent: 0pt;text-align: left;"><span><img width="60" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_337.jpg"/></span></p><p class="s433" style="padding-top: 7pt;padding-left: 60pt;text-indent: 0pt;text-align: left;">t&#39;Ce<span class="h2">ll</span>=&gt;ress</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 99pt;text-indent: 0pt;text-align: left;"><span><img width="36" height="18" alt="image" src="1-s2.0-S0092867421010618-main/Image_338.png"/></span></p><p class="s12" style="padding-top: 1pt;padding-left: 60pt;text-indent: 0pt;text-align: left;"><a name="bookmark207">Resource</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-top: 4pt;padding-left: 74pt;text-indent: 0pt;text-align: left;"><span class="s434">A </span>Hierarchical <span style=" color: #1F1F1F;">clustering </span>of significant</p><p class="s434" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">B<span class="s435">Carcinoma ecotypes (CEs) after</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s437" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 6pt;padding-left: 28pt;text-indent: 0pt;text-align: left;">Cell state overlap between alternative CE definitions</p><p class="s436" style="padding-top: 1pt;padding-left: 90pt;text-indent: 0pt;text-align: left;">cell-state co-occurrence <span style=" color: #0C0C0C;">patterns</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="88" height="105" alt="image" src="1-s2.0-S0092867421010618-main/Image_339.png"/></span></p><p class="s435" style="padding-top: 6pt;padding-left: 148pt;text-indent: 0pt;text-align: left;">Cluster <span style=" color: #1F1F1F;">(n </span><span class="s438">= </span>18)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="16" alt="image" src="1-s2.0-S0092867421010618-main/Image_340.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="48" alt="image" src="1-s2.0-S0092867421010618-main/Image_341.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_342.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_343.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_344.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_345.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_346.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_347.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_348.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_349.png"/></span></p><p class="s439" style="text-indent: 0pt;line-height: 22pt;text-align: left;">一<span style=" color: #0C0C0C;">。 </span><span style=" color: #3B905B;">:</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s441" style="padding-left: 59pt;text-indent: 0pt;line-height: 64pt;text-align: right;">I<span class="s442">。</span>:<span class="s443">三</span></p><p class="s435" style="padding-left: 149pt;text-indent: 0pt;line-height: 4pt;text-align: center;">J  <span style=" color: #1F1F1F;">ccard </span>index</p><p class="s444" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: left;">lo3</p><p class="s436" style="padding-top: 1pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span style=" color: #0C0C0C;">filtering </span>clusters with <span class="s445">&lt; </span>3 <span style=" color: #0C0C0C;">states</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="120" alt="image" src="1-s2.0-S0092867421010618-main/Image_350.png"/></span></p><p class="s446" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">·</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1234567891</p><p style="text-indent: 0pt;text-align: left;"/><p class="s448" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">C</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">EEEEEEEEE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s450" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">ccccccccc</p><p style="text-indent: 0pt;text-align: left;"/><p class="s451" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s452" style="padding-left: 14pt;text-indent: 0pt;text-align: left;">I I</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s453" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 7pt;padding-left: 32pt;text-indent: 0pt;text-align: left;">ccard index</p><p class="s437" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 1pt;padding-left: 66pt;text-indent: 0pt;text-align: left;">0.3</p><p class="s435" style="padding-top: 4pt;padding-left: 18pt;text-indent: -3pt;text-align: left;">Hierarchical <span style=" color: #1F1F1F;">clustering</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="79" height="106" alt="image" src="1-s2.0-S0092867421010618-main/Image_351.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_352.png"/></span></p><p class="s435" style="padding-left: 18pt;text-indent: 3pt;text-align: justify;">E1 <span style=" color: #1F1F1F;">CE2</span></p><p class="s436" style="padding-left: 18pt;text-indent: 0pt;line-height: 80%;text-align: justify;">CE3 <span style=" color: #0C0C0C;">CE4 </span>CE5</p><p class="s436" style="padding-left: 18pt;text-indent: 0pt;text-align: justify;">CE6 CE? CE8 CE9</p><p class="s436" style="padding-left: 14pt;text-indent: 0pt;text-align: justify;">CE10</p><p class="s435" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">Fill</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-top: 4pt;padding-left: 23pt;text-indent: 0pt;text-align: left;">K-medoids</p><p class="s436" style="padding-top: 1pt;padding-left: 40pt;text-indent: 0pt;text-align: left;">CE1</p><p class="s435" style="padding-top: 1pt;padding-left: 40pt;text-indent: 0pt;line-height: 88%;text-align: justify;">CE2 <span style=" color: #1F1F1F;">CE3 CE4 CE5</span></p><p class="s436" style="padding-left: 40pt;text-indent: 0pt;text-align: left;">CE6</p><p class="s436" style="padding-left: 40pt;text-indent: 0pt;line-height: 7pt;text-align: left;">CE7</p><p class="s454" style="padding-left: 24pt;text-indent: 0pt;line-height: 7pt;text-align: left;">,,,_ <span style=" color: #1F1F1F;">CEB</span></p><p class="s436" style="padding-left: 40pt;text-indent: 0pt;line-height: 112%;text-align: left;">CE9 CE10</p><p class="s455" style="padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">I <span class="s435">CE11</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_353.png"/></span></p><p class="s435" style="padding-left: 59pt;text-indent: 0pt;line-height: 7pt;text-align: right;">F1lt</p><p class="s435" style="padding-top: 4pt;padding-left: 13pt;text-indent: -3pt;text-align: left;">Hierarchical <span style=" color: #1F1F1F;">clustering</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="78" height="111" alt="image" src="1-s2.0-S0092867421010618-main/Image_354.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_355.png"/></span></p><p class="s435" style="padding-top: 1pt;padding-left: 12pt;text-indent: 4pt;line-height: 91%;text-align: justify;">E1 <span style=" color: #1F1F1F;">CE2 CE3 CE4 </span><span class="s456">CES</span></p><p class="s436" style="padding-top: 1pt;padding-left: 9pt;text-indent: 3pt;line-height: 111%;text-align: justify;">CE6 CE? CE8 CE9 CE10</p><p class="s435" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">Filt</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-top: 4pt;padding-left: 27pt;text-indent: 0pt;text-align: left;">Louva1n</p><p class="s435" style="padding-left: 39pt;text-indent: 0pt;text-align: justify;">CE1 CE2 <span style=" color: #1F1F1F;">CE3</span></p><ul id="l33"><li style="padding-left: 39pt;text-indent: -5pt;line-height: 7pt;text-align: left;"><p class="s435" style="display: inline;">CE4</p><p class="s436" style="padding-top: 1pt;padding-left: 39pt;text-indent: 0pt;text-align: left;">CE5</p><p class="s436" style="padding-top: 5pt;padding-left: 39pt;text-indent: 0pt;text-align: left;">CE6</p><p class="s436" style="padding-top: 5pt;padding-left: 39pt;text-indent: 0pt;line-height: 115%;text-align: left;">CE? <span style=" color: #0C0C0C;">CE8</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="129" height="111" alt="image" src="1-s2.0-S0092867421010618-main/Image_356.png"/></span></p><p class="s457" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s458" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">WN</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s459">i</span>no<span class="s461">b</span>te<span class="s462">R</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">t   d</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">a <span class="s463">s</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">n</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">le e a</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">rc<span class="s464">y</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">r n</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o  a</p><p style="text-indent: 0pt;text-align: left;"/><p class="s465" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c<span class="s466">d</span><span class="s467"> r</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">a <span class="s460">r</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">$ n a</p><p style="text-indent: 0pt;text-align: left;"/><p class="s462" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s<span class="s460"> uo</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s461" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o<span class="s468">b </span><span class="s462">r</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s469" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">r</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 8pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s470">c</span><span class="s471"> </span><span class="s472">rC </span><span class="s470">m</span><span class="s473">-  </span>uaa<span class="s467">各</span>a<span class="s463">q</span> uo11e1aJJO)!uosJe d</p><p class="s474" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1  2 3456 7</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E  E EEEE E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s450" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c cccccc c c <span class="s466">E</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s475" style="padding-left: 81pt;text-indent: -7pt;line-height: 109%;text-align: left;">D <span class="s435">Sample overlap between carcinoma ecotypes and immune subtypes defined by Thorsson et </span><span class="s436">al</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 298pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="2" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_357.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_358.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="16" height="21" alt="image" src="1-s2.0-S0092867421010618-main/Image_359.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="26" alt="image" src="1-s2.0-S0092867421010618-main/Image_360.png"/></span></p><p class="s476" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s476" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">f<span style=" color: #1F1F1F;">|</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s477" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">色</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0   5</p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0   7</p><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1   0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5</p><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">DJQS<span class="s463">1</span>f\N <span class="s463">p</span>ue<span style=" color: #1F1F1F;">&gt;e</span><span class="s478">」</span><span style=" color: #1F1F1F;">e</span><span class="s478">」</span><span class="s479">  </span><span style=" color: #1F1F1F;">OJ)!!</span><span class="s466">E</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;">3  46</p><p class="s450" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c cc</p><p style="text-indent: 0pt;text-align: left;"/><p class="s436" style="padding-top: 5pt;padding-left: 210pt;text-indent: 0pt;text-align: center;">C2</p><p style="padding-left: 298pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="2" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_361.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 323pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="4" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_362.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-left: 47pt;text-indent: 30pt;line-height: 145%;text-align: left;">UMAP of cell state fractions n <span class="s480">= </span>11,414 tumor <span style=" color: #1F1F1F;">samples</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="151" height="94" alt="image" src="1-s2.0-S0092867421010618-main/Image_363.png"/></span></p><p class="s474" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">F</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">NdVW</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">q</p><p style="text-indent: 0pt;text-align: left;"/><p class="s468" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">E</p><p style="text-indent: 0pt;text-align: left;"/><p class="s481" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">­</p><p style="text-indent: 0pt;text-align: left;"/><p class="s468" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">n</p><p style="text-indent: 0pt;text-align: left;"/><p class="s469" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s</p><p style="text-indent: 0pt;text-align: left;"/><p class="s481" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">ce</p><p style="text-indent: 0pt;text-align: left;"/><p class="s468" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">f   A</p><p style="text-indent: 0pt;text-align: left;"/><p class="s468" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 5pt;padding-left: 161pt;text-indent: 6pt;line-height: 109%;text-align: left;">TCGA (n = 4<span style=" color: #42423D;">,</span>729)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5</p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">2</p><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-left: 161pt;text-indent: 1pt;line-height: 109%;text-align: left;">PRECOG <span style=" color: #1F1F1F;">(n </span>= 6,685)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_364.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_365.png"/></span></p><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">SE gE<span class="s483">1</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s450" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s436" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">00,</p><p style="padding-left: 272pt;text-indent: 0pt;line-height: 3pt;text-align: left;"><span><img width="5" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_366.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s484" style="padding-top: 6pt;padding-left: 4pt;text-indent: 0pt;line-height: 4pt;text-align: left;">(;<u> </u><span class="s486">,</span></p><p class="s435" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">UMAP1</p><p style="text-indent: 0pt;text-align: left;"><span><img width="45" height="2" alt="image" src="1-s2.0-S0092867421010618-main/Image_367.png"/></span></p><p class="s487" style="text-indent: 0pt;line-height: 13pt;text-align: left;">■</p><p style="text-indent: 0pt;text-align: left;"/><p class="s436" style="padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">Carcinoma CE1</p><p class="s436" style="padding-left: 59pt;text-indent: 0pt;line-height: 0pt;text-align: right;">ecotype <span class="s488">CE2</span></p><p class="s436" style="padding-top: 6pt;padding-left: 36pt;text-indent: 0pt;line-height: 13pt;text-align: left;">CE? <span class="s489">■</span>CE10</p><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">456</p><p style="text-indent: 0pt;text-align: left;"/><p class="s490" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">•</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">EEE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s491" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">•</p><p style="text-indent: 0pt;text-align: left;"/><p class="s450" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s450" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">cc</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-left: 36pt;text-indent: 0pt;line-height: 0pt;text-align: left;">CE8</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_368.png"/></span></p><p class="s492" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">v<span class="s493">&quot;&#39;0</span></p><ul id="l34"><li style="padding-left: 113pt;text-indent: -8pt;line-height: 12pt;text-align: right;"><p class="s435" style="display: inline;">CE3</p></li></ul></li></ul><ul id="l35"><li style="padding-left: 36pt;text-indent: -8pt;line-height: 12pt;text-align: left;"><p class="s435" style="display: inline;">CE9</p></li></ul><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s494" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">G</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 5pt;padding-left: 210pt;text-indent: 0pt;line-height: 117%;text-align: center;"><span><img width="11" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_369.png"/></span>richment of ligand-receptor pairs within and between <span style=" color: #1F1F1F;">carcinoma </span>ecotypes</p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_370.png"/></span></p><p class="s435" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;line-height: 6pt;text-align: right;">ceptors</p><p class="s435" style="padding-top: 4pt;padding-left: 36pt;text-indent: 0pt;text-align: left;">CEs plotted according to sparsity and depletion indices</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 44pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="7" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_371.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_372.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_373.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_374.png"/></span></p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">f E</p><p class="s495" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">o<span class="s496">C</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">r<span class="s463">n</span> 3</p><p style="text-indent: 0pt;text-align: left;"/><p class="s495" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e<span class="s496">.</span>_</p><p style="text-indent: 0pt;text-align: left;"/><p class="s497" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">bmset <span class="s498">o</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s451" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">u</p><p style="text-indent: 0pt;text-align: left;"/><p class="s499" style="padding-left: 6pt;text-indent: -5pt;line-height: 120%;text-align: left;">N<span class="s467">t</span><span class="s497">a  s</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 7pt;text-indent: 0pt;line-height: 120%;text-align: left;">d</p><p style="text-indent: 0pt;line-height: 120%;text-align: left;"><br/></p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s497">eret </span>  01<span class="s498">e </span>  M P</p><p class="s449" style="padding-left: 7pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s500">ed (</span><span class="s501">二</span><span style=" color: #1F1F1F;">ueJ</span><span class="s502">)</span>xa<span class="s463">p</span>u1unr1a1da<span class="s463">o</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s474" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">H</p><p style="text-indent: 0pt;text-align: left;"/><p class="s493" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">CE8&quot;</p><p class="s435" style="padding-top: 5pt;padding-left: 29pt;text-indent: -2pt;line-height: 119%;text-align: left;">Global <span style=" color: #1F1F1F;">validation </span>of carcinoma ecotypes in scRNA-seq data</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="14" height="120" alt="image" src="1-s2.0-S0092867421010618-main/Image_375.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_376.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_377.png"/></span></p><p class="s503" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5</p><p class="s500" style="padding-left: 3pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s500" style="padding-left: 3pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s504" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">x</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">6</p><p style="text-indent: 0pt;text-align: left;"/><p class="s495" style="text-indent: 0pt;line-height: 120%;text-align: right;">+</p><p class="s449" style="text-indent: 0pt;line-height: 120%;text-align: right;">4  5  6  7</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">7</p><p style="text-indent: 0pt;text-align: left;"/><p class="s436" style="padding-top: 4pt;padding-left: 26pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Bx10<span class="s505">5</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="61" alt="image" src="1-s2.0-S0092867421010618-main/Image_378.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="14" alt="image" src="1-s2.0-S0092867421010618-main/Image_379.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="11" alt="image" src="1-s2.0-S0092867421010618-main/Image_380.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_381.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_382.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_383.png"/></span></p><p class="s506" style="text-indent: 0pt;line-height: 4pt;text-align: left;">17</p><p style="text-indent: 0pt;text-align: left;"/><p class="s507" style="text-indent: 0pt;line-height: 79pt;text-align: left;">恳 <span class="s508">i</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s495" style="text-indent: 0pt;line-height: 9pt;text-align: right;">石一<span style=" color: #4B1159;">．</span></p><p class="s436" style="padding-top: 3pt;padding-left: 14pt;text-indent: 0pt;text-align: left;">11 1<span style=" color: #595B4F;">. </span><span style=" color: #0C0C0C;">ICE1</span></p><p class="s493" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">CE<span style=" color: #800101;">f</span></p><p class="s436" style="padding-left: 45pt;text-indent: 0pt;line-height: 7pt;text-align: left;">.CE1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s512" style="padding-left: 114pt;text-indent: 0pt;line-height: 3pt;text-align: left;"><span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_384.png"/></span>	<span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_385.png"/></span>	<span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_386.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_387.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_388.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_389.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_390.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_391.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_392.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_393.png"/></span></p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">6  5</p><p style="text-indent: 0pt;text-align: left;"/><p class="s513" style="text-indent: 0pt;line-height: 14pt;text-align: left;">言。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="text-indent: 16pt;text-align: left;">CE2 <span style=" color: #42423D;">213.3</span>I CE3</p><p class="s436" style="padding-top: 1pt;padding-left: 16pt;text-indent: 0pt;line-height: 109%;text-align: center;">CE4 <span style=" color: #0C0C0C;">CE5</span></p><p class="s436" style="padding-left: 7pt;text-indent: 0pt;line-height: 7pt;text-align: center;">1 <span style=" color: #0C0C0C;">lcE6</span></p><p class="s436" style="padding-top: 1pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">CE7</p><p style="text-indent: 0pt;text-align: left;"/><p class="s514" style="padding-top: 9pt;padding-left: 111pt;text-indent: 0pt;line-height: 24pt;text-align: left;">, _ <span style=" color: #42423D;">3</span>::<span style=" color: #42423D;">5 </span><span style=" color: #0C0C0C;">1 </span><span class="s517">CE8</span></p><p class="s518" style="text-indent: 0pt;line-height: 11pt;text-align: right;">亡<span class="s519">w</span>｀<span class="s519">w</span>＂<span class="s519">w</span>寸<span class="s519">w</span>&quot;&#39;<span class="s519">w</span>&quot;&#39;....<span class="s519">w</span><span class="s520">0</span><span class="s521">w</span><span class="s520">,&quot;&#39;0</span></p><p class="s522" style="padding-left: 96pt;text-indent: 0pt;line-height: 4pt;text-align: left;">O O O O O O O O O U<span class="s523">」</span></p><p class="s524" style="text-indent: 0pt;line-height: 4pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-bottom: 3pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">CE9 <span class="s525">e</span></p><p style="padding-left: 50pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="3" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_394.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_395.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="149" height="30" alt="image" src="1-s2.0-S0092867421010618-main/Image_396.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_397.png"/></span></p><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">4  3</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">2  1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 7pt;text-indent: 0pt;line-height: 120%;text-align: center;">d</p><p class="s497" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: center;">s t<span class="s498">e</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s502" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s<span class="s447"> e</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">e<span class="s502">I</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s497" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">p</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">L</p><p style="text-indent: 0pt;text-align: left;"/><p class="s500" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">ed</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="text-indent: 0pt;line-height: 7pt;text-align: left;">CE5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-top: 5pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">CE3</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 7pt;padding-left: 63pt;text-indent: 0pt;text-align: left;">CE1</p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_398.png"/></span></p><p class="s456" style="padding-top: 4pt;text-indent: 0pt;line-height: 6pt;text-align: right;">5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 23pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><span><img width="12" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_399.png"/></span></p><p class="s503" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">5</p><p class="s500" style="padding-left: 3pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p class="s500" style="padding-left: 3pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s504" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">x</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">l<span class="s502">,</span>uanba</p><p style="text-indent: 0pt;text-align: left;"/><p class="s526" style="text-indent: 0pt;line-height: 67pt;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-left: 15pt;text-indent: 0pt;text-align: left;">CE6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_400.png"/></span></p><p class="s467" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">• • •</p><p style="text-indent: 0pt;text-align: left;"/><p class="s493" style="text-indent: 0pt;line-height: 8pt;text-align: left;">CE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="text-indent: 0pt;line-height: 7pt;text-align: left;">CE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="text-indent: 0pt;line-height: 7pt;text-align: left;">Recovery ,n</p><p style="padding-left: 18pt;text-indent: 0pt;line-height: 7pt;text-align: left;"/><p class="s436" style="padding-left: 18pt;text-indent: 0pt;text-align: left;">scRNA-seq</p><ul id="l36"><li style="padding-top: 3pt;padding-left: 25pt;text-indent: -10pt;text-align: left;"><p class="s435" style="display: inline;">P <span class="s527">&lt; </span>0.05</p></li><li style="padding-top: 3pt;padding-left: 25pt;text-indent: -10pt;text-align: left;"><p class="s435" style="display: inline;">P <span class="s527">&gt; </span>0.05</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 17pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="6" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_401.png"/></span></p><p class="s528" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">，</p><p style="text-indent: 0pt;text-align: left;"/><p class="s436" style="padding-top: 5pt;padding-left: 26pt;text-indent: 0pt;text-align: left;">2x1Q<span class="s505">5</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="90" height="13" alt="image" src="1-s2.0-S0092867421010618-main/Image_402.png"/></span></p><p class="s435" style="padding-left: 50pt;text-indent: 0pt;line-height: 7pt;text-align: left;">01</p><p style="text-indent: 0pt;text-align: left;"><span><img width="35" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_403.png"/></span></p><p class="s435" style="padding-left: 93pt;text-indent: 0pt;line-height: 11pt;text-align: left;">Enrichment sea<span class="s523">「</span>e <span class="s519">I         I</span></p><p class="s436" style="padding-left: 96pt;text-indent: 0pt;line-height: 7pt;text-align: left;">(- 109 <span style=" color: #42423D;">1 0 </span><span style=" color: #0C0C0C;">p-value) </span>o 0.5 <span style=" color: #0C0C0C;">1 1</span><span style=" color: #42423D;">.</span>5</p><p class="s436" style="padding-left: 90pt;text-indent: -35pt;line-height: 112%;text-align: left;"><span style=" color: #0C0C0C;">Less </span>sparse <span class="s501">重 </span>More sparse <span style=" color: #0C0C0C;">S </span>arsity <span style=" color: #0C0C0C;">index </span>(rank)</p><p class="s436" style="padding-top: 1pt;padding-left: 57pt;text-indent: -1pt;text-align: left;"><span style=" color: #0C0C0C;">No. of random </span>CEs with co-occu<span class="s504">「</span>rence index at <span style=" color: #0C0C0C;">least </span>as <span style=" color: #0C0C0C;">high </span>as <span style=" color: #0C0C0C;">the reference</span></p><p class="s529" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">,</p><p style="text-indent: 0pt;text-align: left;"/><p class="s481" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">J</p><p style="text-indent: 0pt;text-align: left;"/><p class="s435" style="padding-top: 7pt;padding-left: 105pt;text-indent: 0pt;text-align: left;">Comparison of bulk NMF subtypes with EcoTyper cell states <span class="s530">K </span>Epithelial states associated <span class="s530">L</span></p><p class="s436" style="padding-left: 59pt;text-indent: 0pt;line-height: 7pt;text-align: right;">with <span style=" color: #0C0C0C;">overall </span>survival</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s435" style="padding-top: 5pt;padding-left: 17pt;text-indent: 6pt;line-height: 119%;text-align: left;">CEs associated with overall survival</p><p style="text-indent: 0pt;text-align: left;"><span><img width="21" height="19" alt="image" src="1-s2.0-S0092867421010618-main/Image_404.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="268" height="70" alt="image" src="1-s2.0-S0092867421010618-main/Image_405.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_406.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="10" height="71" alt="image" src="1-s2.0-S0092867421010618-main/Image_407.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="21" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_408.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="123" height="165" alt="image" src="1-s2.0-S0092867421010618-main/Image_409.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="21" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_410.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="66" alt="image" src="1-s2.0-S0092867421010618-main/Image_411.jpg"/></span></p><p class="s531" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">邑</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 2pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s467">(客</span><span class="s466">)</span>sa1e1s<span class="s463">1</span>el!a411da</p><p class="s532" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">3-}<span class="s449">SOU</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s533" style="padding-left: 12pt;text-indent: 0pt;line-height: 15pt;text-align: center;">、<span class="s524">0</span></p><p class="s467" style="padding-bottom: 1pt;padding-left: 125pt;text-indent: 0pt;line-height: 3pt;text-align: left;">＼ 令 翁 <span class="s471">虔 冀 </span><span class="s534">i </span><span class="s523">今 </span><span class="s535">c,e, </span><span class="s536">、 今</span></p><p style="padding-left: 370pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="9" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_412.png"/></span></p><p class="s501" style="padding-left: 141pt;text-indent: 0pt;line-height: 6pt;text-align: left;">＼<span class="s524">0</span>令 <span class="s537">。</span></p><p class="s541" style="padding-left: 137pt;text-indent: 0pt;line-height: 2pt;text-align: left;"><span class="s436">cf) </span><span class="s538">,is- </span><span class="s539">、今 令 。 心 </span><span class="s540">R </span>-&lt;.. <span class="s535">&#39;&quot;&#39; c, </span><span class="s542">&amp; c, </span>,0e.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_413.png"/></span></p><p class="s543" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">化</p><p style="text-indent: 0pt;text-align: left;"/><p class="s544" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">飞<span class="s545">t</span><span class="s546">、</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">、</p><p style="text-indent: 0pt;text-align: left;"/><p class="s547" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">O</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s502">5</span> <span class="s531">空</span>3 BOcso<span class="s502">g</span>c<span class="s502">g</span> 5</p><p style="text-indent: 0pt;text-align: left;"/><p class="s449" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">s  <span class="s467">lA</span>nsl:J:\:/Nl1n</p><p style="text-indent: 0pt;text-align: left;"/><p class="s550" style="padding-left: 127pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><span class="s548">«. \ </span><span class="s549">&lt;(令) </span><span class="s524">0 </span>忒寻。<span class="s551">0</span>岁 。令 <span style=" color: #0C0C0C;">心。｀。谷令。今</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_414.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_415.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_416.png"/></span></p><p class="s552" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">圈</p><p style="text-indent: 0pt;text-align: left;"/><p class="s437" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s451" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s553" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">，</p><p style="text-indent: 0pt;text-align: left;"/><p class="s554" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">．</p><p style="text-indent: 0pt;text-align: left;"/><p class="s545" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"/><ul id="l37"><li style="padding-left: 152pt;text-indent: -17pt;line-height: 17pt;text-align: left;"><p class="s555" style="display: inline;">- 国<span style=" color: #938054;">卫</span><span class="s557">1</span><span class="s558">■</span><span class="s559">1</span><span class="s560">■</span><span class="s561">1 </span><span class="s562">-    </span><span class="s563">嘈</span><span class="s564">-  </span><span class="s565">11</span><span class="s566">可 </span><span class="s567">谓 </span><span class="s568">■</span><span class="s569">11</span><span class="s435">Cell state</span></p></li></ul></li></ul></li></ol></ol><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_417.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_418.png"/></span></p><p class="s527" style="text-indent: 0pt;text-align: left;"><span class="s570">■</span>S1      <span class="s571">■</span>S4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s436" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">Cell state</p><p class="s538" style="text-indent: 0pt;line-height: 15pt;text-align: left;">S3       <span class="s572">■</span>S6</p><p style="text-indent: 0pt;text-align: left;"/><p class="s527" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: right;"><span class="s573">■</span>S2      <span class="s574">■</span>S5</p><p class="s435" style="padding-top: 6pt;padding-left: 72pt;text-indent: -3pt;text-align: left;">Jaccard index</p><p style="text-indent: 0pt;text-align: left;"><span><img width="30" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_419.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="55" height="51" alt="image" src="1-s2.0-S0092867421010618-main/Image_420.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_421.png"/></span></p><h1 style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</h1><p style="text-indent: 0pt;text-align: left;"/><p class="s575" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">｀<span class="s576">叉求</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s577" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">。</p><p style="text-indent: 0pt;text-align: left;"/><p class="s467" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">入</p><p style="text-indent: 0pt;text-align: left;"/><p class="s467" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">今</p><p style="text-indent: 0pt;text-align: left;"/><p class="s467" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">心、</p><p style="text-indent: 0pt;text-align: left;"/><p class="s467" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">今</p><p style="text-indent: 0pt;text-align: left;"/><p class="s460" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0<span class="s578">，</span>L</p><p style="text-indent: 0pt;text-align: left;"/><p class="s579" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">•</p><p style="text-indent: 0pt;text-align: left;"/><p class="s580" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">•</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">789</p><p style="text-indent: 0pt;text-align: left;"/><p class="s447" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">sss</p><p style="text-indent: 0pt;text-align: left;"/><p class="s535" style="padding-left: 64pt;text-indent: 0pt;line-height: 11pt;text-align: left;">0<span style=" color: #4B1159;">1 </span><span class="s527">0 6</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s468" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">、令</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="657" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_422.png"/></span></p><p class="s43" style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;line-height: 110%;text-align: justify;"><a href="#bookmark95" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Figure S5. Identification, robustness, potential ligand-receptor interactions, and prognostic utility of carcinoma ecotypes, related to </a><a href="#bookmark95" class="s42">Figures </a>4 <a href="#bookmark96" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">and </a><a href="#bookmark96" class="s42">5</a></p><p class="s38" style="padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">(A,B) Identification of carcinoma ecotypes (CEs) from significantly co-associated cell states, defined as cell state pairs with higher Jaccard indices than expected by random chance (</a>STAR Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (A) Heatmap depicting Jaccard matrix of significant cell state pairs (left), organized into the number of clusters that maximize the silhouette width by hierarchical clustering (right; n = 18 clusters; </a>STAR Methods<a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (B) Same as panel A but after omission of clusters with &lt; 3 cell states (CEs analyzed in this work). (C) River plots depicting the overlap between CEs identified by hierarchical clustering (this work) and alternative clustering strategies. Filt = clusters with &lt; 3 cell states. (D) River plot showing sample overlap between CEs and TCGA pan-cancer immune subtypes (</a>Thorsson et al., 2018<a href="#bookmark206" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) evaluated within the discovery cohort. (E) Same as </a>Figure S4<a href="#bookmark125" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">C but showing CE quantitation by EcoTyper in LUSC and OV tumor transcriptomes profiled by bulk RNA-seq and microarray. Data are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.53IQR of the box limits; IQR, interquartile range). (F) UMAP representation of cell state fractions across samples from the discovery cohort and an external cohort of 6,685 tumors (PRECOG (</a>Gentles et al., 2015<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">)), colored by CE assignment. Only tumor samples assigned to CEs are shown (</a>STAR Methods<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). (G) Matrix showing the statistical enrichment of ligand/receptor pairs (rows and columns, respectively) between distinct cell states, both within and between CEs (</a>STAR Methods<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). All identified ligand-receptor pairs identified in the discovery cohort are provided in </a>Table S6<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">. (H) Plot showing sparsity index, depletion index, and the number of states per cell for each CE, related to </a>Figure 4<a href="#bookmark95" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">D. CEs that were not significantly detected in scRNA-seq data are highlighted in red. Sparsity index is defined as the fraction of scRNA-seq samples with no cells assigned to a given cell state, averaged across all states within each CE. Depletion index is defined as the log2 fold change between the mean fraction of each CE in bulk RNA-seq (TCGA discovery cohort) versus scRNA-seq. (I) Histogram showing the number of randomly generated CEs with a co-occurrence index as high, or higher than, non-randomized (i.e., reference) CEs (x axis), tallied per iteration and repeated 1M times (y axis). In each iteration, 10 random CEs were generated maintaining the size and cell type composition of the original CEs (related to </a>Figure 4<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">D; </a>STAR Methods<span style=" color: #000;">).</span></p><p class="s38" style="padding-left: 53pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">(J) Jaccard matrices illustrating sample overlap between EcoTyper states and pan-carcinoma subtypes defined by bulk NMF clustering (</a>STAR Methods<span style=" color: #000;">).</span></p><p class="s38" style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 117%;text-align: justify;"><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">For each cell type characterized by EcoTyper, samples were classified into discrete subtypes by the state with highest abundance. (K) Impact of bulk NMF subtypes on the association between each EcoTyper epithelial state and overall survival (OS). Survival models were determined with Cox regression (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark183" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). |Z| &gt; 1.96 was considered significant. (L) Same as K but for individual CEs adjusted for pan-immune subtypes (</a>Thorsson et al., 2018<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) or bulk NMF subtypes (see also </a>Table S6<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">; </a>STAR Methods<span style=" color: #000;">).</span></p><p class="s433" style="padding-left: 59pt;text-indent: 0pt;line-height: 29pt;text-align: right;"><span><img width="60" height="45" alt="image" src="1-s2.0-S0092867421010618-main/Image_423.jpg"/></span><span class="s8">      </span>t&#39;Ce<span class="h2">ll</span>=&gt;ress <span class="s95">Cell</span></p><p class="s12" style="padding-left: 59pt;text-indent: 0pt;line-height: 11pt;text-align: right;"><a name="bookmark208">Resource</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_424.png"/></span></p><p class="s582" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">A</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 171pt;text-indent: 32pt;line-height: 107%;text-align: left;">fluorescence <span style=" color: #414141;">imaging </span>of GZMB<span style=" color: #676969;">· </span>and GZMK<span style=" color: #525252;">· </span>T cells <span style=" color: #414141;">in </span>NSCLC <span style=" color: #414141;">20• magoifi </span><span class="s587">笠 </span><span style=" color: #414141;">,lion Region of interest</span></p><p class="s586" style="padding-left: 186pt;text-indent: 0pt;line-height: 5pt;text-align: left;">T29 <span style=" color: #414141;">T29 T36</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="298" height="455" alt="image" src="1-s2.0-S0092867421010618-main/Image_425.png"/></span></p><p class="s586" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">Ceo&lt;e,   Cose <span class="s588">的 </span>ge  Fa,<span class="s589">织</span>&lt;ge       <span style=" color: #414141;">Ceow   </span>c,<span class="s589">函织勾&#39;&#39;&quot;呤</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s583" style="padding-left: 14pt;text-indent: -8pt;line-height: 83%;text-align: left;"><span class="s590">8 </span><span style=" color: #414141;">d </span>CE10 T <span style=" color: #414141;">cell and </span>mono<span style=" color: #525252;">/m</span>ac states by EccType, an<span style=" color: #525252;">,</span>d mmunofluo,escence(IF) <span style=" color: #414141;">m</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="59" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_426.png"/></span></p><p class="s583" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">NSCLC samples <span style=" color: #525252;">(</span>T29 and T36)</p><p class="s583" style="padding-top: 1pt;padding-left: 76pt;text-indent: 2pt;line-height: 149%;text-align: left;">EccTypec T29 <span style=" color: #181818;">T</span><span style=" color: #414141;">36</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_427.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_428.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="13" height="6" alt="image" src="1-s2.0-S0092867421010618-main/Image_429.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_430.png"/></span></p><p class="s591" style="text-indent: 0pt;line-height: 72pt;text-align: left;">一</p><p style="text-indent: 0pt;text-align: left;"/><p class="s592" style="text-indent: 0pt;line-height: 22pt;text-align: left;">I</p><p style="text-indent: 0pt;text-align: left;"/><p class="s593" style="text-indent: 0pt;line-height: 25pt;text-align: left;">言忑 言<span style=" color: #5074A1;">七二</span><span class="s595">i </span><span class="s596">二</span></p><p class="s583" style="text-indent: 0pt;line-height: 5pt;text-align: right;">IF</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 21pt;text-indent: 0pt;line-height: 12pt;text-align: left;">T <span style=" color: #414141;">cells </span>IGG: ;  <span class="s597">骂</span></p><p class="s598" style="padding-left: 1pt;text-indent: 0pt;line-height: 3pt;text-align: left;">DA&gt;,</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s599" style="padding-bottom: 4pt;padding-left: 2pt;text-indent: 0pt;text-align: left;">CD3</p><p style="padding-left: 2pt;text-indent: 0pt;line-height: 2pt;text-align: left;"><span><img width="6" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_431.png"/></span></p><p class="s600" style="text-indent: 0pt;text-align: left;">G<span><img width="6" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_432.png"/></span><span class="s601"> </span><span style=" color: #F41313;">B</span></p><p class="s584" style="text-indent: 0pt;line-height: 5pt;text-align: left;">36</p><p class="s583" style="text-indent: 0pt;line-height: 12pt;text-align: left;">Mc, <span class="s603">心;</span>es</p><p style="padding-left: 129pt;text-indent: 0pt;text-align: left;"><span style=" color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;">	</span><span style=" color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;">	</span><span style=" color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;">	</span><span><img width="47" height="19" alt="image" src="1-s2.0-S0092867421010618-main/Image_433.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s604" style="padding-left: 54pt;text-indent: 0pt;line-height: 4pt;text-align: left;">U<span class="s605">七，严</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s584" style="text-indent: 0pt;line-height: 6pt;text-align: left;">T29 <span class="s606">T29 </span><span class="s607">T </span><span class="s608">邓</span></p><p class="s586" style="padding-left: 70pt;text-indent: 0pt;line-height: 5pt;text-align: left;">Coale, Ck&gt; <span class="s589">羽 织 </span>Igo <span style=" color: #414141;">Fa, </span><span class="s609">叩 </span>o Coale, Cl <span class="s589">函 织 扣 </span>Fa,-</p><p class="s418" style="padding-left: 15pt;text-indent: 0pt;line-height: 5pt;text-align: left;">T <span style=" color: #414141;">cells </span><span style=" color: #2F2F2F;">I</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s583" style="text-indent: 4pt;line-height: 115%;text-align: left;">Moaocytes/ Macrophages I</p><p class="s610" style="text-indent: 0pt;line-height: 9pt;text-align: left;">芦 :</p><p class="s611" style="text-indent: 0pt;line-height: 12pt;text-align: left;">怎<span style=" color: #AA4944;">严户</span></p><p style="padding-left: 186pt;text-indent: 0pt;text-align: left;"><span style=" color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;">	</span><span style=" color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;">	</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_434.png"/></span></p><p class="s613" style="padding-left: 59pt;text-indent: 0pt;line-height: 11pt;text-align: right;">.,,</p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_435.png"/></span></p><p class="s614" style="padding-left: 59pt;text-indent: 0pt;line-height: 9pt;text-align: right;">o,</p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_436.png"/></span></p><p class="s615" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">G <span class="s616">MK</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s586" style="padding-top: 4pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">T36</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="23" height="9" alt="image" src="1-s2.0-S0092867421010618-main/Image_437.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_438.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="47" height="19" alt="image" src="1-s2.0-S0092867421010618-main/Image_439.jpg"/></span></p><p class="s583" style="padding-left: 140pt;text-indent: -11pt;line-height: 83%;text-align: left;"><span class="s590">D </span><span class="s605">酶 </span>tdbutloa <span style=" color: #414141;">of CE9 </span>aad CE10 T <span class="s617">氓 </span>I aad mono<span style=" color: #525252;">/m</span>ac states la a bceast tumo, <span style=" color: #414141;">speclmea</span></p><p class="s618" style="text-indent: 0pt;line-height: 71pt;text-align: left;">厂；</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">profiled by spatial tcaascdptom<span style=" color: #525252;">l</span>cs</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="33" height="110" alt="image" src="1-s2.0-S0092867421010618-main/Image_440.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="78" height="38" alt="image" src="1-s2.0-S0092867421010618-main/Image_441.png"/></span></p><p class="s605" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">,....,,.,.』</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 417pt;text-indent: 0pt;line-height: 3pt;text-align: left;"><span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_442.png"/></span></p><p class="s619" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">c</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 210pt;text-indent: 0pt;line-height: 4pt;text-align: center;">Mulllcolo, lmmun fluo<span class="s620">「</span>escenceImaging <span style=" color: #414141;">of </span>APOE·and CCR2·mono/mac In NSC<span style=" color: #070707;">L</span>C</p><p class="s584" style="text-indent: 0pt;line-height: 5pt;text-align: left;">T29 T29 T36</p><p style="text-indent: 0pt;text-align: left;"/><p class="s584" style="padding-left: 210pt;text-indent: 0pt;text-align: center;">20, magomcatloo                                                                      R <span class="s622">的 </span>loooflote cest</p><p class="s586" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: right;"><span style=" color: #414141;">Co,10, </span>Cl<span class="s623">函钩&#39;&#39;&quot;钩 </span>Ceola, <span style=" color: #414141;">Clo</span><span class="s624">迈忒,., .,</span><span class="s625">忒,.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 40pt;text-indent: 0pt;line-height: 2pt;text-align: left;"><span><img width="2" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_443.png"/></span></p><p class="s626" style="text-indent: 0pt;line-height: 81pt;text-align: left;">□□</p><p style="text-indent: 0pt;text-align: left;"/><p class="s627" style="text-indent: 0pt;line-height: 28pt;text-align: left;">..</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 12pt;text-indent: 0pt;line-height: 4pt;text-align: center;">P  <span style=" color: #414141;">= </span>6.8•10 <span class="s628">书</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="78" height="38" alt="image" src="1-s2.0-S0092867421010618-main/Image_444.png"/></span></p><p class="s629" style="padding-left: 75pt;text-indent: 0pt;line-height: 16pt;text-align: left;">H</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s630" style="padding-left: 59pt;text-indent: 0pt;line-height: 1pt;text-align: right;">DAPI</p><p class="s631" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">DAPI</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_445.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_446.png"/></span></p><ul id="l38"><li style="padding-top: 6pt;padding-left: 135pt;text-indent: -4pt;line-height: 6pt;text-align: right;"><p class="s586" style="display: inline;">CE9 Moo<span class="s588">叭</span>Mac(A PO<span class="s632">的</span></p></li></ul><p class="s628" style="padding-left: 11pt;text-indent: 0pt;line-height: 3pt;text-align: left;">飞 <span class="s633">0</span></p><p class="s634" style="text-indent: 0pt;line-height: 6pt;text-align: left;">&gt;</p><p style="text-indent: 0pt;text-align: left;"/><p class="s635" style="padding-left: 10pt;text-indent: 0pt;line-height: 3pt;text-align: left;">0 C</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s636" style="padding-left: 11pt;text-indent: 0pt;line-height: 4pt;text-align: left;">丘 <span class="s637">E</span></p><p class="s638" style="padding-left: 11pt;text-indent: 0pt;line-height: 5pt;text-align: left;">岱呈</p><p class="s583" style="padding-left: 21pt;text-indent: 0pt;line-height: 3pt;text-align: left;">C E9 CE10</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s599" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">CD<span class="s639">邸</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_447.png"/></span></p><p class="s640" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">o,</p><p class="s584" style="padding-top: 4pt;padding-left: 45pt;text-indent: 0pt;text-align: left;">T36</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s606" style="padding-top: 3pt;padding-left: 45pt;text-indent: 0pt;text-align: left;">T29 <span class="s584">T29 T36</span></p><p class="s586" style="padding-left: 116pt;text-indent: 0pt;line-height: 3pt;text-align: left;">c&#39;&quot;&quot;&#39;c,,<span class="s588">跄的</span>g, Fa<span class="s588">『忒 </span>9, c&#39;&quot;&quot;&#39;c,,<span class="s636">过的</span><span style=" color: #414141;">g, </span><span class="s636">&quot; 的 </span><span style=" color: #414141;">g,</span></p><ul id="l39"><li style="padding-left: 24pt;text-indent: -4pt;line-height: 6pt;text-align: left;"><p class="s584" style="display: inline;">CE10Mo <span class="s622">心 </span>Ma cc <span style=" color: #525252;">2•1 </span>Mono/Mac</p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_448.png"/></span></p></li><li style="padding-left: 24pt;text-indent: -3pt;line-height: 5pt;text-align: left;"><p class="s586" style="display: inline;">Epi&lt;he,;a,c <span class="s641">,s</span></p></li></ul><p class="s583" style="padding-top: 1pt;padding-left: 2pt;text-indent: 0pt;line-height: 10pt;text-align: left;"><span class="s642">E </span><span class="s617">砬 </span>locall, atlon <span style=" color: #414141;">of </span>cell states across fou <span class="s620">「</span></p><p class="s583" style="padding-left: 17pt;text-indent: 0pt;line-height: 5pt;text-align: left;">tumo<span class="s620">「 </span>types p<span class="s620">「</span>ofiled by <span style=" color: #414141;">spatial </span>t<span class="s620">「</span>anscclptoml cs</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="105" height="57" alt="image" src="1-s2.0-S0092867421010618-main/Image_449.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_450.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="27" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_451.png"/></span></p><ul id="l40"><li style="padding-left: 62pt;text-indent: -10pt;text-align: left;"><p class="s643" style="display: inline;">I <span class="s644">■</span></p></li></ul><p class="s645" style="padding-top: 2pt;padding-left: 51pt;text-indent: 0pt;line-height: 4pt;text-align: left;">I <span class="s583">CE</span></p><ul id="l41"><li style="padding-left: 4pt;text-indent: -4pt;line-height: 73%;text-align: left;"><p class="s646" style="display: inline;">圆    <span class="s647">w m</span></p></li></ul><p style="text-indent: 0pt;text-align: left;"/><ul id="l42"><li style="padding-top: 1pt;padding-left: 90pt;text-indent: -4pt;line-height: 32%;text-align: left;"><p class="s586" style="display: inline;"><span class="s648">_</span><span class="s649"> </span>m<span class="s650">m</span><span class="s647">        </span>m<span class="s650">m</span></p></li></ul><p class="s651" style="padding-top: 7pt;padding-left: 210pt;text-indent: 0pt;text-align: center;">&quot;&quot;</p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_452.png"/></span></p><p class="s652" style="padding-top: 8pt;padding-left: 135pt;text-indent: 0pt;line-height: 10pt;text-align: left;">&quot;<span class="s653"> </span><span class="s654">T36</span><span class="s584"> </span><span class="s655">■</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_453.png"/></span></p><p class="s656" style="padding-left: 129pt;text-indent: 0pt;line-height: 7pt;text-align: left;">&#39;&quot;&quot;</p><p class="s657" style="padding-left: 132pt;text-indent: 0pt;line-height: 4pt;text-align: left;">CR2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s605" style="padding-left: 210pt;text-indent: 0pt;text-align: center;">『.-,让</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s583" style="text-indent: 0pt;text-align: left;">mane rrela <span class="s617">虹</span></p><p class="s658" style="text-indent: 0pt;line-height: 6pt;text-align: center;">■</p><p class="s659" style="text-indent: 0pt;line-height: 74%;text-align: left;">■<span class="s660">m</span></p><p style="text-indent: 0pt;text-align: left;"/><ul id="l43"><li style="padding-left: 19pt;text-indent: -4pt;line-height: 67%;text-align: left;"><p class="s660" style="display: inline;">m <span class="s661">■</span>m o</p></li></ul><p style="text-indent: 0pt;text-align: left;"><span><img width="87" height="72" alt="image" src="1-s2.0-S0092867421010618-main/Image_454.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="29" height="30" alt="image" src="1-s2.0-S0092867421010618-main/Image_455.png"/></span></p><p class="s662" style="padding-left: 19pt;text-indent: 0pt;text-align: left;">CE</p><p style="text-indent: 0pt;text-align: left;"><br/></p><ul id="l44"><li style="padding-top: 2pt;padding-left: 21pt;text-indent: -7pt;line-height: 6pt;text-align: left;"><p class="s663" style="display: inline;">忒飞，了&#39;:</p></li></ul><p class="s662" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">M  <span class="s589">呻  沁</span></p><p class="s664" style="padding-left: 12pt;text-indent: 0pt;line-height: 2pt;text-align: center;">O<span style=" color: #414141;">.. </span>J ..<span class="s588">比闭&quot;</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="16" height="11" alt="image" src="1-s2.0-S0092867421010618-main/Image_456.png"/></span></p><p class="s585" style="padding-left: 12pt;text-indent: 0pt;line-height: 4pt;text-align: center;">M<span style=" color: #414141;">a&quot;  c ,</span></p><p class="s668" style="text-indent: 0pt;line-height: 14pt;text-align: left;"><span class="s666">，</span><span class="s667">&#39;</span><span style=" color: #525252;">,</span><span class="s667">.</span><span style=" color: #525252;">.</span>心<span class="s669">&quot;</span>&quot;~<span style=" color: #414141;">呛</span><span class="s669">&quot;</span>&#39;&#39;&quot;&quot;</p><p style="text-indent: 0pt;text-align: left;"/><p class="s671" style="padding-left: 12pt;text-indent: 0pt;line-height: 3pt;text-align: center;">t <span class="s588">的 归</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="15" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_457.png"/></span></p><p class="s672" style="padding-top: 3pt;padding-left: 19pt;text-indent: 0pt;text-align: left;">p <span class="s673">价 &#39; &#39;</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s674" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">G</p><p style="text-indent: 0pt;text-align: left;"/><p class="s638" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">ac<span class="s675">M</span>  <span class="s676">／</span><span class="s677">门</span><span class="s678">己</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s638" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">珈<span class="s679">i</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s582" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">F</p><p style="text-indent: 0pt;text-align: left;"/><p class="s680" style="text-indent: 0pt;line-height: 81pt;text-align: left;">□□<span style=" color: #525252;">□□□</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 326pt;text-indent: -21pt;line-height: 107%;text-align: left;">Spat,al d,stnbut,oa of <span style=" color: #414141;">caccmoma </span>ecotypes m a b<span class="s620">「</span>east caccmoma spec,mea pmfiled by spat,al tcaascnptom,cs</p><p class="s681" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">g</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-top: 1pt;padding-left: 295pt;text-indent: 0pt;text-align: left;">CE1 CE2 CE3 <span style=" color: #414141;">CE4 </span>CE5</p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_458.png"/></span></p><p class="s679" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">de</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 155pt;text-indent: 0pt;line-height: 2pt;text-align: left;"><span><img width="3" height="3" alt="image" src="1-s2.0-S0092867421010618-main/Image_459.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_460.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="68" alt="image" src="1-s2.0-S0092867421010618-main/Image_461.jpg"/></span></p><p class="s679" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">3    2    1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s685" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span style=" color: #2F2F2F;">泗</span><span class="s683"> </span><span class="s684">＾</span>～<span class="s686">耍贷 </span><span class="s687">＇</span>芒<span class="s688">g</span><span class="s689">g w</span><span class="s690">-</span><span class="s691">se</span><span class="s692">哟</span><span class="s691">L</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 230pt;text-indent: 0pt;text-align: left;">Caoce,type</p><ul id="l45"><li style="padding-top: 2pt;padding-left: 235pt;text-indent: -4pt;text-align: left;"><p class="s583" style="display: inline;">Bceast</p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_462.png"/></span></p></li><li style="padding-top: 1pt;padding-left: 235pt;text-indent: -4pt;line-height: 125%;text-align: left;"><p class="s583" style="display: inline;">Colo<span class="s620">「</span>ectal <span style=" color: #414141;">Ovaclao</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_463.png"/></span></p></li><li style="padding-left: 235pt;text-indent: -4pt;text-align: right;"><p class="s584" style="display: inline;">Melaooma</p></li></ul><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s5" style="padding-left: 165pt;text-indent: 0pt;text-align: left;"><span><img width="2" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_464.png"/></span>	<span><img width="13" height="13" alt="image" src="1-s2.0-S0092867421010618-main/Image_465.png"/></span></p><p class="s679" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">de</p><p style="text-indent: 0pt;text-align: left;"/><p class="s680" style="padding-top: 8pt;padding-bottom: 2pt;padding-left: 20pt;text-indent: 0pt;line-height: 68pt;text-align: left;">□□□□□</p><p class="s694" style="padding-left: 145pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="9" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_466.png"/></span><span class="s693">	</span><span><img width="9" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_467.png"/></span> <span><img width="15" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_468.png"/></span> <span><img width="3" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_469.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="231" height="120" alt="image" src="1-s2.0-S0092867421010618-main/Image_470.png"/></span></p><p class="s695" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">I<span class="s696"> </span><span class="s697">I</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s699" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span style=" color: #A3B39C;">:</span>av<span style=" color: #ACD18E;">i</span>O<span class="s701">丿</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s638" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">』</p><p style="text-indent: 0pt;text-align: left;"/><p class="s702" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">:</p><p style="text-indent: 0pt;text-align: left;"/><p class="s703" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">:<span class="s704">erB</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 128pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span class="s590">H </span>Ana <span style=" color: #070707;">l</span>ysis o<span style=" color: #070707;">f l</span>igand-receptor <span style=" color: #181818;">pairs in </span>four <span style=" color: #181818;">tumor t</span><span style=" color: #414141;">ypes </span>profiled by spatia<span style=" color: #070707;">l </span><span style=" color: #181818;">transcriptom,cs</span></p><p class="s583" style="padding-left: 51pt;text-indent: 0pt;line-height: 1pt;text-align: left;">CE <span style=" color: #414141;">- CE </span><span style=" color: #181818;">1 </span>SC r er ss</p><p style="text-indent: 0pt;text-align: left;"><span><img width="128" height="134" alt="image" src="1-s2.0-S0092867421010618-main/Image_471.png"/></span></p><p class="s705" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">-</p><p style="text-indent: 0pt;text-align: left;"/><p class="s706" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">L</p><p style="text-indent: 0pt;text-align: left;"/><p class="s707" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">-r</p><p style="text-indent: 0pt;text-align: left;"/><p class="s711" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;"><span class="s708">f</span><span style=" color: #3F9C38;">f</span><span class="s710">:</span>-<span class="s712">l</span>e</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-top: 1pt;padding-left: 46pt;text-indent: 0pt;text-align: left;">lung cancer developmental stages</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s512" style="padding-left: 248pt;text-indent: 0pt;line-height: 3pt;text-align: left;"><span><img width="7" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_472.png"/></span>	<span><img width="4" height="4" alt="image" src="1-s2.0-S0092867421010618-main/Image_473.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="28" alt="image" src="1-s2.0-S0092867421010618-main/Image_474.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="14" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_475.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_476.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_477.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="96" height="10" alt="image" src="1-s2.0-S0092867421010618-main/Image_478.png"/></span></p><p class="s713" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">言</p><p style="text-indent: 0pt;text-align: left;"/><p class="s714" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">陌<span style=" color: #A79E9A;">玉且</span><span style=" color: #C38C7C;">咭呾</span><span style=" color: #A17C83;">匡</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-top: 2pt;padding-left: 134pt;text-indent: 0pt;line-height: 118%;text-align: center;"><span style=" color: #181818;">Ligand-recept  </span>r (<span style=" color: #070707;">L</span>R    airs identified fo<span style=" color: #070707;">r </span>eac<span style=" color: #070707;">h </span>state within each CE (TCGA)</p><p style="padding-left: 130pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="98" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_479.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="99" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_480.png"/></span></p><p class="s584" style="padding-top: 4pt;padding-left: 36pt;text-indent: 10pt;line-height: 260%;text-align: justify;"><span style=" color: #181818;">Normal </span><span style=" color: #525252;">- </span><span style=" color: #070707;">H </span>yperp <span style=" color: #181818;">lasia </span>- <span style=" color: #2F2F2F;">Metaplas,a </span>-</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s718" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">`</p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 37pt;text-indent: 0pt;text-align: left;">P = 5 x 1Q-<span style=" color: #525252;">3</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="2" height="5" alt="image" src="1-s2.0-S0092867421010618-main/Image_481.png"/></span></p><p class="s418" style="padding-left: 136pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Tumors <span class="s254">如 </span>17 <span class="s719">论 </span>ns <span style=" color: #2F2F2F;">prof ed by</span></p><p class="s583" style="padding-bottom: 3pt;padding-left: 145pt;text-indent: 0pt;line-height: 8pt;text-align: left;">spatial transcriptomics <span><img width="8" height="12" alt="image" src="1-s2.0-S0092867421010618-main/Image_482.png"/></span></p><p class="s720" style="padding-left: 130pt;text-indent: 0pt;line-height: 6pt;text-align: left;"><span><img width="98" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_483.png"/></span>	<span><img width="8" height="7" alt="image" src="1-s2.0-S0092867421010618-main/Image_484.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="98" height="8" alt="image" src="1-s2.0-S0092867421010618-main/Image_485.png"/></span></p><p class="s721" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">”</p><p style="text-indent: 0pt;text-align: left;"/><p class="s721" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s722" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">、<span class="s681">守</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s418" style="padding-bottom: 1pt;padding-left: 135pt;text-indent: 0pt;line-height: 115%;text-align: center;">Deterrrnne <span style=" color: #2F2F2F;">whelher st e­ specific LR genes are spatially­ ennched ,n parent CE</span></p><p style="padding-left: 131pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="96" height="0" alt="image" src="1-s2.0-S0092867421010618-main/Image_486.png"/></span></p><p class="s723" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">广 <span style=" color: #8C8C8C;">皿</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s583" style="padding-left: 130pt;text-indent: 0pt;text-align: left;">splas,a-</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s702" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">尸</p><p style="text-indent: 0pt;text-align: left;"/><p class="s713" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">，</p><p style="text-indent: 0pt;text-align: left;"/><p class="s725" style="padding-left: 59pt;text-indent: 0pt;text-align: right;">s ec, ,,</p><p class="s610" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">nem</p><p style="text-indent: 0pt;text-align: left;"/><p class="s258" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">al</p><p style="text-indent: 0pt;text-align: left;"/><p class="s683" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">处 ,&lt;;:,· <span class="s726">t </span><span class="s617">兮</span></p><p class="s418" style="padding-top: 1pt;padding-left: 59pt;text-indent: 0pt;text-align: right;">C - CE10</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s681" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">a5e1s1e1ua<span class="s727">E</span>oola&gt;a<span style=" color: #070707;">a</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s593" style="padding-left: 1pt;text-indent: 0pt;line-height: 120%;text-align: left;">勹</p><p style="text-indent: 0pt;text-align: left;"/><p class="s729" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">,<span style=" color: #414141;">,,.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 53pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="657" height="1" alt="image" src="1-s2.0-S0092867421010618-main/Image_487.png"/></span></p><p style="padding-top: 2pt;padding-left: 53pt;text-indent: 0pt;text-align: left;"><a href="#bookmark97" style=" color: #AB4D4C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Figure S6. Spatial analysis of carcinoma ecotypes and evolutionary dynamics of CE9 and CE10, related to </a><a href="#bookmark97" class="s42">Figure 6</a></p><ol id="l46"><li style="padding-top: 1pt;padding-left: 53pt;text-indent: 0pt;line-height: 118%;text-align: justify;"><p class="s38" style="display: inline;"><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Three-channel immunofluorescence (IF) imaging of DAPI, CD3, and either GZMB (top) or GZMK (bottom) in non-small cell lung cancer (NSCLC) specimens with paired RNA-seq data (related to </a><a href="#bookmark98" class="a">Table </a>S6<span style=" color: #000;">). </span><span class="s35">Left: </span><span style=" color: #000;">Geographically-distinct NSCLC regions at 20 3 magnification (scale bar of 200 mm is identical for all images). </span><span class="s35">Right: </span><a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Close-up images corresponding to boxed regions on the left. Scale bar of 20 mm is identical for all close-up images. ‘Center’ refers to the tumor core; ‘Close edge’ refers to the invasive margin and immediate periphery of the tumor mass; ‘Far edge’ refers to the periphery of the tumor mass. (B) Comparison of three- channel IF imaging of CE9 and CE10-specific T cell and monocyte/macrophage states (</a><a href="#bookmark97" class="a">Figure </a>6<span style=" color: #000;">B) against EcoTyper deconvolution of bulk RNA-seq profiles from the same tumors. </span><span class="s35">Top: </span><span style=" color: #000;">Composition of T cell and monocyte/macrophage states determined by EcoTyper. </span><span class="s35">Bottom: </span><a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Qualitative abundances determined by IF imaging. See also </a><a href="#bookmark98" class="a">Table </a>S6<span style=" color: #000;">. ND, not detected. (C) Same as panel A but for DAPI, CD68, and either APOE (top) or CCR2 (bottom) in tumor sections adjacent to those in panel A. (D) </span><span class="s35">Left: </span><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">EcoTyper deconvolution of CE9 and CE10-specific T cell and monocyte/macrophage states in a breast carcinoma specimen profiled by 10x Visium (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). </span><span class="s35">Right: </span><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">Relative distance of each state from tumor cells (demarcated by a dashed line; </a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark98" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). Group comparisons were performed using a two-sided unpaired Wilcoxon rank sum test. (E) Heatmaps depicting Spearman cross-correlation matrices of cell states organized by CE across barcoded spots in spatial transcriptomics arrays of breast cancer specimens (n = 2 sections, 1 patient), a colorectal cancer specimen, an ovarian cancer specimen, and melanoma specimens (n = 8 sections, 4 patients) (</a><a href="#bookmark98" class="a">Table </a>S1<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">), related to </a><a href="#bookmark97" class="a">Figures </a>6<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">D and 6E. The specificity of cross-correlation patterns for each CE was assessed as described in </a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark97" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">. Only TME cell states were evaluated in melanoma. Significance is plotted in </a><a href="#bookmark97" class="a">Figure </a>6<span style=" color: #000;">E. (F) CE-specific spatial aggregation patterns, as calculated by applying Moran’s I to spatial transcriptomics data from four cancer types. (G) Visualization of CE organization in a breast carcinoma specimen profiled by 10x Visium. The relative abundance of each CE is normalized such that each state contributes equally. (H) Spatial analysis of CE- specific ligand-receptor (LR) pairs in four tumor types profiled by spatial transcriptomics (same as E). For each set of LR pairs between state </span><span class="s35">i </span><span style=" color: #000;">and the other states in CE </span><span class="s35">j</span><span style=" color: #000;">, we determined whether LR genes overexpressed in </span><span class="s35">i </span><span style=" color: #000;">are enriched in spots where </span><span class="s35">j </span><a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">was predicted to be present (</a><a href="#bookmark197" class="a">STAR </a>Methods<a href="#bookmark149" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">). CEs and cell types are colored as in panel E. (I) Dynamics of CE9 and CE10 composition across key developmental stages in squamous cell lung carcinogenesis (</a><a href="#bookmark149" class="a">Mascaux et al., </a>2019<a href="#bookmark197" style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">) (</a><a href="#bookmark197" class="a">STAR </a>Methods<span style=" color: #000;">). Group comparisons (normal tissues versus others; squamous cell carcinoma (SCC) samples versus others) were determined using a two-sided Wilcoxon rank sum test. Data in (D) right, (F), and (I) are represented as box plots (center line, median; box limits, upper and lower quartiles; whiskers, largest and smallest values within 1.53IQR of the box limits; IQR, interquartile range).</span></p></li></ol></body></html>
