<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Single-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma</title><meta name="author" content="Ruiping Wang"/><meta name="description" content="Nature Medicine, doi:10.1038/s41591-020-1125-8"/><style type="text/css"> * {margin:0; padding:0; text-indent:0; }
 .s1 { color: #FFF; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 20pt; }
 .s2 { color: #FFF; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s3 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 24pt; }
 .s4 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s5 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 4pt; }
 .s6 { color: black; font-family:"Segoe UI Emoji", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; vertical-align: 3pt; }
 h1 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 9.5pt; }
 .s7 { color: #96171A; font-family:"Malgun Gothic Semilight", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 41.5pt; }
 .s8 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s10 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s12 { color: black; font-family:Cambria, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s13 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s14 { color: #F00; font-family:"Palatino Linotype", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 9pt; }
 .s16 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 9pt; }
 .s17 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 2pt; }
 .s18 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s19 { color: black; font-family:"Segoe UI Emoji", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 2pt; }
 .s20 { color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s21 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .h2, h2 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 9pt; }
 .s22 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s23 { color: black; font-family:"Palatino Linotype", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s24 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 h4 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 8pt; }
 .s25 { color: black; font-family:Calibri, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s26 { color: black; font-family:Cambria, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s27 { color: black; font-family:Cambria, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 2pt; }
 .s28 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s29 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s30 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s31 { color: #FFCA52; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s32 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; vertical-align: 1pt; }
 .s33 { color: #D8580A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 h3 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 8.5pt; }
 .s34 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s35 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; vertical-align: -2pt; }
 .s36 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; vertical-align: -1pt; }
 .s37 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 3pt; }
 .s38 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; vertical-align: -3pt; }
 .s39 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt; }
 .s40 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt; }
 .s41 { color: #848689; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s42 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -2pt; }
 .s43 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; vertical-align: 1pt; }
 .s44 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -3pt; }
 .s45 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -1pt; }
 .s46 { color: #EE68A4; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s48 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s49 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 8pt; vertical-align: 2pt; }
 .s50 { color: black; font-family:Cambria, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt; }
 .s51 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 1pt; }
 .s52 { color: #808285; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s53 { color: black; font-family:"Trebuchet MS", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s54 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s55 { color: black; font-family:"Trebuchet MS", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s56 { color: #949698; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s57 { color: black; font-family:"Sitka Text"; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s58 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -1pt; }
 .s59 { color: #949698; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s60 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s61 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 8pt; vertical-align: 3pt; }
 .s63 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 6.5pt; }
 .s64 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s65 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s66 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s67 { color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 8pt; vertical-align: 1pt; }
 .s68 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s69 { color: #FFF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s71 { color: #4BC676; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s72 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s73 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: -3pt; }
 .s74 { color: #5BADD8; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s75 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 2pt; }
 .s76 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 1pt; }
 .s77 { color: #4BC676; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s78 { color: #FDB53C; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s79 { color: #DF1186; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s80 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: -2pt; }
 .s81 { color: #5BADD8; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s82 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: -4pt; }
 .s83 { color: #FDB53C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s84 { color: #DF1186; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s85 { color: #DF1186; font-family:Cambria, serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s87 { color: #D7590B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s88 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; }
 .s89 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: -1pt; }
 .s90 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: 1pt; }
 .s91 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: 7pt; }
 .s92 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: 5pt; }
 .s93 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: 4pt; }
 .s94 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: -3pt; }
 .s95 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: 3pt; }
 .s96 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 4pt; }
 .s97 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 3pt; }
 .s98 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 4.5pt; vertical-align: 2pt; }
 .s99 { color: #32B4AA; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s100 { color: #F5950B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s101 { color: #32B4AA; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s102 { color: #F5950B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s103 { color: #19AA3C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s104 { color: #E6412E; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s105 { color: #5DAFD9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s106 { color: #FDB53C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s107 { color: #5DAFD9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s108 { color: #727476; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; }
 .s109 { color: #19AA3C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s110 { color: #5DAFD9; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s111 { color: #5DAFD9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s112 { color: #5DAFD9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; }
 .s113 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s114 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s115 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; }
 .s116 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; vertical-align: 1pt; }
 .s117 { color: #19AA3C; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s118 { color: #19AA3C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s119 { color: #19AA3C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; vertical-align: 1pt; }
 .s120 { color: #E6412E; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s121 { color: #E6412E; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s122 { color: #E6412E; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s123 { color: #E6412E; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1.5pt; vertical-align: 1pt; }
 .s124 { color: #5DAFD9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s125 { color: #E6412E; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s126 { color: #FDB53C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s127 { color: #D1D3D4; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s128 { color: #D1D3D4; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s129 { color: #41B915; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s130 { color: #77C6E5; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s131 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s132 { color: #19AA3C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; }
 .s133 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 4.5pt; }
 .s134 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2pt; vertical-align: 1pt; }
 .s135 { color: #3279B9; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s136 { color: #DE1285; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s137 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; vertical-align: 2pt; }
 .s138 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 4.5pt; }
 .s139 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s140 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3pt; vertical-align: 3pt; }
 .s141 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: -1pt; }
 .s143 { color: #EE68A4; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s144 { color: #2F7AB5; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s145 { color: #F069A4; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s146 { color: #2F7AB5; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 1pt; }
 .s147 { color: #357ABA; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s148 { color: #F39200; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s149 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s150 { color: black; font-family:Arial, sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -1pt; }
 .s151 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; vertical-align: -2pt; }
 .s152 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 1pt; }
 .s153 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; }
 .s154 { color: black; font-family:Ebrima; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 1pt; }
 .s155 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 3pt; }
 .s156 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -4pt; }
 .s157 { color: #EB5014; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s158 { color: #EB5014; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; vertical-align: 2pt; }
 .s159 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 2.5pt; vertical-align: 2pt; }
 .s160 { color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .p, p { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; margin:0pt; }
 .s161 { color: black; font-family:"Palatino Linotype", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s162 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 7.5pt; }
 .a, a { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s163 { color: black; font-family:Cambria, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s164 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 7pt; }
 .s165 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s169 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s170 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -1pt; }
 .s171 { color: black; font-family:Garamond, serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 1pt; }
 .s172 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s173 { color: black; font-family:"Palatino Linotype", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; }
 .s174 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s175 { color: black; font-family:"Book Antiqua", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 5pt; }
 .s176 { color: black; font-family:"Book Antiqua", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s177 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt; }
 .s178 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt; }
 .s179 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 5pt; }
 .s180 { color: black; font-family:"Bookman Old Style", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s181 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: 3pt; }
 .s182 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 10pt; }
 .s183 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 1pt; }
 .s184 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 5pt; }
 .s185 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 1pt; }
 .s186 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -3pt; }
 .s187 { color: black; font-family:"Book Antiqua", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -10pt; }
 .s188 { color: black; font-family:"Times New Roman", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -5pt; }
 .s189 { color: black; font-family:"Times New Roman", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s190 { color: black; font-family:Symbol, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s191 { color: black; font-family:"Palatino Linotype", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: -1pt; }
 .s192 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s193 { color: black; font-family:"Trebuchet MS", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s194 { color: black; font-family:"Palatino Linotype", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s195 { color: black; font-family:Cambria, serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s196 { color: black; font-family:Constantia, serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -5pt; }
 .s197 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 7.5pt; }
 .s198 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 7.5pt; }
 .s199 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -4pt; }
 .s200 { color: black; font-family:Garamond, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5pt; vertical-align: -5pt; }
 .s201 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -4pt; }
 .s202 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: 6pt; }
 .s203 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s204 { color: black; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; vertical-align: -3pt; }
 .s205 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s206 { color: #3B699E; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s207 { color: #E91C23; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 40.5pt; vertical-align: 2pt; }
 .s208 { color: #646B64; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 40.5pt; }
 .s209 { color: #0A0A0A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s210 { color: #646B64; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s211 { color: #0A0A0A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 8pt; }
 .s212 { color: #0A0A0A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 6pt; }
 .s213 { color: #0A0A0A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 7pt; }
 .s214 { color: #212121; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s215 { color: #E91C23; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 18.5pt; }
 .s216 { color: #0A0A0A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s218 { color: #086697; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s219 { color: #E91C23; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 14pt; }
 .s220 { color: #646B64; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s221 { color: #3A3B3B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 13pt; }
 .s222 { color: #0A0A0A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s223 { color: #0A0A0A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8.5pt; }
 .s224 { color: #646B64; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s225 { color: #3A3B3B; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s228 { color: #086697; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s229 { color: #0A0A0A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s231 { color: #646B64; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s234 { color: #086697; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6.5pt; }
 .s235 { color: #0A0A0A; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s236 { color: #0A0A0A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s237 { color: #0A0A0A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s238 { color: #3A3B3B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s239 { color: #ED1F26; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 16.5pt; }
 .s240 { color: #646E69; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 23.5pt; }
 .s241 { color: #111; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s242 { color: #2D2D2D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s243 { color: #111; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; }
 .s244 { color: #3F413F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s245 { color: #646E69; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s246 { color: #2D2D2D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s247 { color: #646E69; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14.5pt; }
 .s248 { color: #79807C; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s250 { color: #3B89AF; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 7pt; }
 .s252 { color: #646E69; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s254 { color: #111; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s255 { color: #111; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s256 { color: #971128; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s257 { color: #971128; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt; }
 .s258 { color: #3F413F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s259 { color: #646E69; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s260 { color: #646E69; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 21.5pt; }
 .s261 { color: #111; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11pt; }
 .s262 { color: #3F413F; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s263 { color: #646E69; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s264 { color: #ED1F26; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 12pt; }
 .s265 { color: #0F6E9E; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: underline; font-size: 7pt; }
 .s266 { color: #2D2D2D; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; vertical-align: 1pt; }
 .s267 { color: #111; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; }
 .s268 { color: #151515; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s269 { color: #2F2F2F; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s271 { color: #151515; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s273 { color: #626B66; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s275 { color: #626B66; font-family:宋体; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 li {display: block; }
 #l1 {padding-left: 0pt;counter-reset: c1 24; }
 #l1> li:before {counter-increment: c1; content: counter(c1, decimal)" "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l1> li:first-child:before {counter-increment: c1 0;  }
 #l2 {padding-left: 0pt;counter-reset: c2 4; }
 #l2> li:before {counter-increment: c2; content: counter(c1, decimal)"."counter(c2, decimal)" "; color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 #l2> li:first-child:before {counter-increment: c2 0;  }
 li {display: block; }
 #l3 {padding-left: 0pt;counter-reset: d1 1; }
 #l3> li:before {counter-increment: d1; content: counter(d1, decimal)". "; color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 #l3> li:first-child:before {counter-increment: d1 0;  }
 li {display: block; }
 #l4 {padding-left: 0pt; }
 #l4> li:before {content: "- "; color: #3A3B3B; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
</style></head><body><p style="text-indent: 0pt;text-align: left;"><span><img width="151" height="64" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_001.png"/></span></p><p style="padding-top: 6pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s1" style=" background-color: #96171A;">                                                                                            Articles  </span></p><p class="s2" style="padding-left: 360pt;text-indent: 0pt;text-align: left;">https://doi.org/10.1038/s41591-020-1125-8</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 446pt;text-indent: 0pt;text-align: left;"><span><img width="94" height="18" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_002.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s3" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a name="bookmark0">Single-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma</a></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_003.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_004.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_005.png"/></span></p><p class="s4" style="padding-top: 13pt;padding-left: 7pt;text-indent: 0pt;line-height: 109%;text-align: left;">Ruiping Wang<span class="s5">1</span>, Minghao Dang<span class="s5">1</span>, Kazuto Harada <span class="s5">2,12</span>, Guangchun Han <span class="s5">1</span>, Fang Wang<span class="s5">3</span>, Melissa Pool Pizzi<span class="s5">2</span>, Meina Zhao<span class="s5">2</span>, Ghia Tatlonghari<span class="s5">2</span>, Shaojun Zhang <span class="s5">1</span>, Dapeng Hao<span class="s5">1</span>,</p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_006.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_007.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_008.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_009.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_010.png"/></span></p><p class="s4" style="padding-left: 7pt;text-indent: 0pt;line-height: 109%;text-align: left;">Yang Lu <span class="s5">4</span>, Shuangtao Zhao<span class="s5">1</span>, Brian D. Badgwell<span class="s5">5</span>, Mariela Blum Murphy<span class="s5">2</span>, Namita Shanbhag<span class="s5">2</span>, Jeannelyn S. Estrella<span class="s5">6</span>, Sinchita Roy-Chowdhuri<span class="s5">6</span>, Ahmed Adel Fouad Abdelhakeem<span class="s5">2</span>, Yuanxin Wang<span class="s5">1</span>, Guang Peng <span class="s5">7</span>, Samir Hanash <span class="s5">7</span>, George A. Calin <span class="s5">8</span>, Xingzhi Song<span class="s5">1</span>, Yanshuo Chu<span class="s5">1</span>, Jianhua Zhang <span class="s5">1</span>,</p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_011.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_012.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_013.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_014.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_015.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_016.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="12" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_017.png"/></span></p><p class="s4" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;line-height: 83%;text-align: left;">Mingyao Li <span class="s5">9</span>, Ken Chen <span class="s5">3</span>, Alexander J. Lazar <span class="s5">6,10</span>, Andrew Futreal <span class="s5">1</span>, Shumei Song <span class="s5">2</span>, Jaffer A. Ajani <span class="s5">2 </span><span class="s6">✉ </span>and Linghua Wang <span class="s5">1,11 </span><span class="s6">✉</span></p><h1 style="padding-top: 13pt;padding-left: 7pt;text-indent: 0pt;line-height: 91%;text-align: justify;">Intratumoral heterogeneity (ITH) is a fundamental property of cancer; however, the origins of ITH remain poorly understood. We performed single-cell transcriptome profiling of peritoneal carcinomatosis (PC) from 15 patients with gastric adenocar- cinoma (GAC), constructed a map of 45,048 PC cells, profiled the transcriptome states of tumor cell populations, incisively explored ITH of malignant PC cells and identified significant correlates with patient survival. The links between tumor cell lin- eage/state compositions and ITH were illustrated at transcriptomic, genotypic, molecular and phenotypic levels. We uncovered the diversity in tumor cell lineage/state compositions in PC specimens and defined it as a key contributor to ITH. Single-cell analysis of ITH classified PC specimens into two subtypes that were prognostically independent of clinical variables, and a 12-gene prognostic signature was derived and validated in multiple large-scale GAC cohorts. The prognostic signature appears fundamental to GAC carcinogenesis and progression and could be practical for patient stratification.</h1><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s7" style="text-indent: 0pt;line-height: 53pt;text-align: left;">G</p><p style="text-indent: 0pt;text-align: left;"/><p class="s8" style="padding-top: 5pt;padding-left: 34pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark31" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">AC remains a common and lethal disease with a poor prog- nosis</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">1</span><a href="#bookmark32" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. Often diagnosed at an advanced stage, GAC is fre- quently resistant to therapy</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">2</span>. A common site of metastases</p><p class="s13" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark33" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">is the peritoneal cavity (peritoneal carcinomatosis; PC) and there is a high unmet need for improved therapeutic interventions in patients with advanced GAC</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">3</span><a href="#bookmark34" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>4<span class="s8">. Patients with PC are highly symp- tomatic and can have an overall survival of </span><span class="s12">&lt;</span><a href="#bookmark35" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">6 months. Only a small fraction of patients benefits, often only transiently, from immune checkpoint blockade</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">5</span><a href="#bookmark36" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>6 <a href="#bookmark37" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">or HER2-directed therapies</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">7</span><a href="#bookmark38" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. Molecular understanding of advanced GAC is limited. Four geno- types defined by The Cancer Genome Atlas (TCGA) were based on analysis of primary GACs</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">8</span><a href="#bookmark39" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. The two clinically favorable subtypes, Epstein–Barr virus-positive and microsatellite instable GACs, are rare in advanced cases</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">9</span><span class="s8">. In the clinic, empiricism prevails as patients are not routinely stratified and rational therapeutics are exceedingly limited.</span></p><p class="s13" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark38" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">It is well recognized that GAC is endowed with extensive inter- tumoral heterogeneity and ITH</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">8</span><a href="#bookmark39" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>9<span class="s8">. ITH is fundamental for tumor cell survival as it confers therapy resistance and is a major obstacle</span></p><p class="s8" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark40" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">to improving patient outcomes. However, the origins of ITH are poorly understood. Deeper understanding of the cellular/molecular basis of ITH could influence how GACs are treated. Single-cell tran- scriptome sequencing (scRNA-seq) is a robust and unbiased tool to assess cellular and transcriptomic ITH</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">10</span>.</p><p class="s8" style="padding-left: 7pt;text-indent: 11pt;line-height: 11pt;text-align: justify;">Here, we performed scRNA-seq of PC cells from <span class="s14">ten long- term and ten short-term survivors with PC</span><span style=" color: #F00;">, </span><a href="#bookmark41" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">inferred tumor cell lineages and transcriptomic states at single-cell resolution by mapping the scRNA-seq data to the Human Cell Landscape (HCL) data-base</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">11</span>, constructed a single-cell map of malignant PC cells, com-prehensively characterized ITH of PC tumor cells via integrative approaches and identified significant correlates with</p><p class="s8" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;">patient survival. This study demonstrated that <span class="s14">the diversity in tumor cell lineage/state compositions is a key contributor to ITH. </span>A 12-gene funda-mental signature was discovered, which although derived from PC cells, retained its prognostic significance when applied to indepen-dent, localized and advanced large-scale GAC cohorts. These results provide an avenue for patient stratification and target discovery for future therapeutic</p><p class="s16" style="padding-left: 7pt;text-indent: 0pt;line-height: 10pt;text-align: left;">                                                                                                                    exploitation.                                                                                          </p><p class="s17" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">1<span class="s18">Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>2<span class="s18">Department of Gastrointestinal Medical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>3<span class="s18">Department of Bioinformatics and Computational Biology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>4<span class="s18">Department of Nuclear Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>5<span class="s18">Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>6<span class="s18">Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>7<span class="s18">Department of Clinical Cancer Prevention, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>8<span class="s18">Department of Experimental Therapeutics, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>9<span class="s18">Department of Biostatistics and Epidemiology, University of Pennsylvania School of Medicine, Philadelphia, PA, USA. </span>10<span class="s18">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>11<span class="s18">UTHealth Graduate School of Biomedical Sciences, The University of Texas MD Anderson Cancer Center, Houston, TX, USA. </span>12<span class="s18">Present address: Department of Gastroenterological Surgery,</span></p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 10pt;text-align: left;">Kumamoto University, Kumamoto, Japan. <span class="s19">✉</span><a href="mailto:jajani@mdanderson.org" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;" target="_blank">e-mail: </a><a href="mailto:jajani@mdanderson.org" class="s20" target="_blank">jajani@mdanderson.or</a><span style=" color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">g</span><a href="mailto:LWang22@mdanderson.org" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;" target="_blank">; </a><a href="mailto:LWang22@mdanderson.org" class="s20" target="_blank">LWang22@mdanderson.org</a></p><p class="s21" style="padding-top: 9pt;padding-left: 7pt;text-indent: 0pt;line-height: 12pt;text-align: left;"><a name="bookmark1">Results</a><a name="bookmark2">&zwnj;</a></p><p class="s8" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;"><b>A single-cell transcriptome map of PC. </b><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">scRNA-seq was performed on cryopreserved PC cells collected from 20 patients with GAC at advanced stages, including ten long-term and ten short-term survi- vors (Fig. </a><span style=" color: #3B699E;">1a</span><a href="#bookmark42" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">). The clinical and histopathological characteristics and radiology images are summarized in the Supplementary Information (Supplementary Table 1 and Supplementary Fig. 1). All primary tumors were of diffuse type and no microsatellite instability (MSI) was observed (see the Methods). Following quality filtering, 45,048 cells were retained for the subsequent analyses. The batch effects were minimal as statistically assessed by k-BET</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">12</span><span class="s13"> </span><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">(Supplementary Fig. 2). We captured five main nonmalignant cell types: B cells, CD4 T cells, CD8 T cells, myeloid cells and fibroblasts, each defined by canonical marker genes (Fig. </a><span style=" color: #3B699E;">1a</span>, Extended Data Fig. 1 and Supplementary Table 2). A multistep approach was then applied to distinguish malignant PC cells and to define cell transcriptome states (see the Methods). The immune cells from different patients clustered together by cell type, whereas PC malignant cells clustered distinctly by patient (Extended Data Fig. 1a). It is evident from Extended Data Fig. 2 that tumor cell clusters from short-term sur- vivors were relatively closer on both the uniform manifold approxi- mation and projection (UMAP) plot and the cluster dendrogram. Consistently, the Bhattacharyya pairwise distance between clusters of long-term and short-term survivors was significantly larger than that of the background distributions, indicating distinct transcrip- tomic profiles associated with survival. In this study, we focused on PC tumor cells (<span class="s23">n </span><span class="s12">= </span>31,131). Five patients with too few tumor cells (<span class="s12">&lt;</span>50) were excluded from subsequent analyses (Supplementary Tables 3 and 4). To profile the transcriptome landscape of PC tumor cells, unsupervised cell-clustering analysis was carried out and the results were illustrated using both <span class="s23">t</span>-distributed stochastic neigh-</p><p class="s8" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;">bor embedding (<span class="s23">t</span><a href="#bookmark43" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">-SNE) and UMAP</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">13</span><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, which uncovered 14 unique cell clusters, with differentially expressed genes (DEGs) specifically marking each cell cluster (Fig. </a><span style=" color: #3B699E;">1b </span>and Supplementary Fig. 3). Seven of 14 clusters were defined by complete patient specificity, and for the remaining seven clusters, each one was dominated mainly by cells from an individual patient (Supplementary Fig. 4). The distri- bution of each patient’s tumor cells across clusters was quantified using chi-squared tests (Supplementary Table 5), demonstrating a high degree of interpatient heterogeneity in PC tumor cells.</p><h2 style="padding-top: 9pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a name="bookmark3">The inferred tumor cell lineages. </a><a href="#bookmark41" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">To map each individual tumor cell, to determine its transcriptome state and the potential cells of origin of PC, we used HCL</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">11</span><span class="s8">, a valuable and well-annotated</span></h2><p class="s24" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark44" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">scRNA-seq resource for human biology, as a reference. Using a prelabeled public scRNA-seq dataset</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">14</span><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, we first tested the reli- ability of HCL and also evaluated the performance of our approach in cell lineage inference and obtained satisfactory results (Supplementary Fig. 5). We then analyzed PC cells using the same approach (see the Methods). Intriguingly, although all cases in this study were clinically diagnosed as PC from GAC, our transcriptome-based analysis revealed a high degree of cel- lular heterogeneity in inferred tumor cell lineages (Fig. </a>1c <a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and Supplementary Table 3). Only ~70% of mapped PC tumor cells were defined as cells of stomach origin, including pit cells (47%), mucosal cells (22%) and chief cells (0.5%). However, the expression profiles of a subset of PC tumor cells (26%) transcrip- tomically resembled cells of other gastrointestinal (GI) organs, particularly the intestine (21%). It is unlikely that these cells represent cell doublets (Supplementary Fig. 6). DEG analysis revealed lineage-specific gene expression features across major cell lineages including colorectal-like, duodenal-like and gas- tric cells, and between colorectal enterocytes and goblet cells (Fig. </a>1d,e<a href="https://www.proteinatlas.org/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">), which is supported by compelling evidence from the literature and public databases such as the Human Protein Atlas (</a>https://www.proteinatlas.org<span style=" color: #000;">; Supplementary Figs. 7 and 8).</span></p><p class="s24" style="padding-top: 1pt;padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">We showed that the difference in transcriptomic features was unlikely due to dropouts or technical noise of the scRNA-seq data, as we observed a good correlation in cell lineage assignment between mapping with the raw and imputed data (Supplementary Fig. 9). In addition, we redid HCL mapping after regressing out cell-cycle-related genes and our analysis demonstrated that the cell lineage assignment was not confounded by differences in cell-cycle states (Extended Data Fig. 3). For the two cases (IP-158 and IP-010) with mixed gastric and colonic epithelial cells including colonic goblet cells, we were able to retrieve the histology images of their corresponding primary GACs and confirmed that tumors arose in the setting of gastric intestinal metaplasia, which is character- ized by the presence of well-formed goblet cells in gastric mucosa (Fig. </a>1f<a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">). This finding is intriguing given the associated analyses showing mixed cellular populations of both gastric and colonic lin- eages. For case IP-070, our analysis suggested that none of the PC tumor cells was of GI origin; instead, the cells transcriptomically resembled breast luminal epithelial cells (Fig. </a>1c <span style=" color: #000;">and Supplementary Fig. 10). After re-reviewing the patient’s clinical records, we noted that this case was of breast cancer that metastasized to the stomach and formed PC. This vignette on the other hand reflected the accuracy of our cell lineage analysis.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="677" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_018.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark9"><b>Fig. 1 | </b></a>A single-cell transcriptome map of PC and the inferred tumor cell lineages. This study included ten short-term survivors and ten long-term survivors. <b>a</b>, Left, the Kaplan–Meier curve demonstrates a dramatic difference (<i>P </i><span class="s26">= </span>3 <span class="s26">× </span>10<span class="s27">−</span><span class="s28">06 </span>by log-rank test) in the survival time since PC diagnosis between the two groups of patients with GAC; middle, a schema of sample collection for scrNA-seq; right, <i>t</i>-SNE plot showing unbiased clustering analysis of 45,048 single cells that passed quality control in this study. Each dot of the <i>t</i>-SNE plot represents a single cell. Cells are color coded for their associated cell types. <b>b</b>, The <i>t</i>-SNE and UMAP plots of the 31,131 PC tumor cells (14 cell clusters) that were selected for subsequent analyses. Cells are color coded by their corresponding patient origins. <b>c</b>, The tumor cell lineage compositions inferred by mapping scrNA-seq data to the HCL database. The middle panel shows the HCL-defined cell lineages/types (rows) by patient (columns). The size of the circle represents, for each specific cell lineage/type, the fraction of tumor cells (among the total quality-control-passed tumor cells) in each individual PC. The circles are color coded by defined cell lineages/ types, the same as in the annotation track on the left. The histogram on the top shows, for each individual sample, the number of tumor cells accumulated on listed cell lineages/types (plus other unclassified or rare cell types). The histogram on the right shows, for each specific tumor cell lineage/type, the fraction of tumor cells (among the total quality-control-passed tumor cells) in this cohort. The bottom annotation tracks show (from top to bottom): the corresponding patient IDs, the survival groups to which the patients belong, the presence of intestinal metaplasia in their corresponding primary tumors, fractions of intestinal cells among the total quality-control-passed tumor cells in each individual PC and the PC subtypes. Classification of the PC subtypes was based on tumor cell lineage compositions (gastric-dominant if fraction of intestinal cells <span class="s26">&lt;</span>20% and GI-mixed if fraction of intestinal cells <span class="s26">≥</span>20%).<a name="bookmark26">&zwnj;</a></p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><b>d</b>, Bubble plot showing expression of lineage-specific marker genes across different cell lineages/types. <b>e</b>, Violin plots of representative lineage-specific marker genes. <b>f</b>, A representative histology image for IP-010 demonstrating well-formed goblet cells in gastric mucosa (indicated by blue arrow heads). <b>g</b>, UMAP plot showing unsupervised clustering of 26,401 PC tumor cells from 14 samples that underwent HCL mapping and cell lineage inference as in <b>c</b>. Cells are colored by their inferred cell lineages/types. Dashed circles highlight samples that formed two or more tumor cell clusters (as labeled in the left panel of Extended Data Fig. 4). <b>h</b>, <i>t</i>-SNE and UMAP plots of PC tumor cells generated from patient-level subclustering analysis, showing that gastric cells (pink, purple) were clustered distinctly from the colorectal-like cells (dark blue).</p><h2 style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a name="bookmark4">The diversity in tumor cell lineage compositions links to ITH at transcriptomic, genotypic and molecular levels. </a><span class="s8">To further study transcriptomic ITH and examine its relationship with tumor cell</span></h2><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 42pt;text-indent: 0pt;line-height: 7pt;text-align: left;">a</h4><p class="s13" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">lineage compositions, we performed unsupervised clustering analysis of PC tumor cells using Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">15</span> <a href="#bookmark46" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and Monocle</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">16</span> <span class="s8">and col- ored cells on the global UMAP plots by their inferred cell lineages</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">n <span class="s30">= 45,048 cells</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="331" height="97" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_019.png"/></span></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p class="s31" style="text-indent: 0pt;text-align: right;">+ ++</p><p class="s31" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">+</p><p class="s31" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">+</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">P  <span class="s30">= 3 × 10–6</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s31" style="padding-left: 9pt;text-indent: 0pt;text-align: center;">+</p><p class="s30" style="padding-top: 2pt;padding-left: 60pt;text-indent: 0pt;text-align: left;">GAC-PC discovery cohort (<i>n </i>= 20)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="119" height="95" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_020.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="31" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_021.png"/></span></p><p class="s32" style="padding-top: 3pt;padding-left: 66pt;text-indent: -6pt;line-height: 9pt;text-align: left;"><span><img width="6" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_022.png"/></span> <span class="s30">Tumor cell CD8 T cell</span></p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p class="s31" style="padding-top: 1pt;padding-left: 58pt;text-indent: 0pt;line-height: 112%;text-align: right;">Long <span style=" color: #D8580A;">Short</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 42pt;text-indent: 0pt;text-align: left;">b</h4><p class="s31" style="padding-left: 34pt;text-indent: 0pt;line-height: 3pt;text-align: left;">Long-term survivor</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s33" style="padding-bottom: 3pt;padding-left: 15pt;text-indent: 0pt;text-align: left;">Short-term survivor</p><p class="s31" style="text-indent: 0pt;text-align: left;">10 10</p><p class="s33" style="text-indent: 0pt;text-align: left;">10 0</p><p class="s31" style="text-indent: 0pt;text-align: left;">4</p><p class="s33" style="text-indent: 0pt;text-align: left;">0</p><p class="s31" style="text-indent: 0pt;text-align: left;">2</p><p class="s33" style="text-indent: 0pt;text-align: left;">0</p><p class="s31" style="text-indent: 0pt;text-align: left;">1</p><p class="s33" style="text-indent: 0pt;text-align: left;">0</p><p style="padding-left: 2pt;text-indent: 0pt;line-height: 10pt;text-align: left;"><span style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9.5pt;">			</span></p><p class="s30" style="padding-top: 3pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">0 10 20 30 40</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;line-height: 110%;text-align: center;">Time since peritoneal metastasis (months)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 29pt;text-indent: 0pt;text-align: left;">By patient</p><p class="s29" style="padding-left: 4pt;text-indent: 0pt;line-height: 4pt;text-align: left;">n <span class="s30">= 10</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 3pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">n <span class="s30">= 10</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 22pt;text-indent: 2pt;line-height: 110%;text-align: left;">Ascites drainage</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 10pt;text-indent: 4pt;line-height: 110%;text-align: left;">Cell isolation</p><p class="s30" style="padding-left: 43pt;text-indent: -2pt;line-height: 110%;text-align: left;">Ascites cells</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="136" height="136" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_023.png"/></span></p><p class="s30" style="padding-left: 39pt;text-indent: 0pt;text-align: left;">By patient</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: left;">scRNA-seq</p><p style="padding-left: 9pt;text-indent: 0pt;line-height: 5pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">t<span class="s30">-SNE_2</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="text-indent: 0pt;text-align: right;">t<span class="s30">-SNE_1</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h3 style="padding-top: 4pt;padding-left: 2pt;text-indent: 0pt;text-align: center;">d</h3><p class="s30" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">Tumor cells</p><p class="s30" style="padding-left: 26pt;text-indent: 0pt;line-height: 145%;text-align: left;">CD4 T cell B Cell Myeloid cell</p><p class="s32" style="padding-top: 2pt;padding-left: 20pt;text-indent: 0pt;text-align: left;"><span><img width="6" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_024.png"/></span> <span class="s30">Fibroblast</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="38" height="8" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_025.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="165" height="38" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_026.png"/></span></p><p class="s30" style="padding-left: 31pt;text-indent: 0pt;text-align: left;">Percentage of cells</p><p style="text-indent: 0pt;text-align: left;"><span><img width="133" height="131" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_027.png"/></span></p><p class="s29" style="text-indent: 0pt;text-align: left;">n <span class="s30">= 31,131</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 110%;text-align: left;">Tumor cells</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">Patient_ID</p><p class="s30" style="padding-top: 1pt;padding-left: 19pt;text-indent: 0pt;line-height: 110%;text-align: left;">Tumor cells</p><p class="s29" style="text-indent: 0pt;text-align: right;">n <span class="s30">= 31,131</span></p><p class="s30" style="text-indent: 0pt;text-align: right;">Gastric</p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">REG4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">FABP1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;text-indent: 0pt;line-height: 2pt;text-align: right;">Duodenal</p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">expressed</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">UMAP_2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="text-indent: 0pt;text-align: right;">t<span class="s30">-SNE_1</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="text-indent: 0pt;text-align: center;">c</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-top: 2pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">4,000</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Cell count</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">3,000</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s35" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">2,000 <span class="s34">1,896</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">1,000</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">2,421 <span class="s36">2,250</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">1,235</p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">IP 007</p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="85" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_028.png"/></span></p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">IP 009</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">IP 010</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 031</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 051</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 056</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 057</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 062</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 067</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 070</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="351" height="238" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_029.png"/></span></p><p class="s34" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">1,564</p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">IP 072</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">IP 073</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">IP 081</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 093</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 107</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 109</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 114</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 116</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 124</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">IP 158</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Gene expression</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-left: 132pt;text-indent: 0pt;line-height: 3pt;text-align: center;">3,297</p><p class="s34" style="padding-left: 132pt;text-indent: 0pt;line-height: 86%;text-align: center;">2,865 <u>          </u> <span class="s36">2,751 </span><span class="s38">2,582</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">3,735</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">4,016</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">1,534</p><p style="text-indent: 0pt;line-height: 6pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">UMAP_1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="28" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_030.png"/></span></p><p class="s34" style="padding-left: 10pt;text-indent: 0pt;line-height: 1pt;text-align: left;">Proportion of cells (%)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">Colorectal</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">t<span class="s30">-SNE_2</span></p><p style="text-indent: 0pt;text-align: left;"/><h3 style="text-indent: 0pt;text-align: right;">e</h3><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="67" height="26" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_031.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="37" height="34" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_032.png"/></span></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">PHGR1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">CDH17</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">DMBT1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 3pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">Cell lineage</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">PHGR1 CDH17</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">2</p><p class="s30" style="padding-top: 1pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">1</p><p class="s30" style="padding-top: 1pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="67" height="28" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_033.png"/></span></p><p class="s30" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">4</p><p class="s30" style="padding-top: 2pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">2</p><p class="s30" style="padding-top: 2pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="28" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_034.png"/></span></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">REG4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">4</p><p class="s30" style="padding-top: 1pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">2</p><p class="s30" style="padding-top: 2pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">0</p><p class="s30" style="padding-top: 1pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">0 20 40 60 80 100</p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">PSCA</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">CTSE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 2pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">Relative expression</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;padding-left: 113pt;text-indent: 0pt;text-align: center;">1.0      0       1.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">FABP1 KRT20</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 19pt;text-indent: 0pt;text-align: left;">Colorectal-like cells</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="49" height="22" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_035.png"/></span></p><p class="s34" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 073</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">1</p><p class="s30" style="padding-top: 3pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="24" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_036.png"/></span></p><p class="s34" style="padding-top: 3pt;padding-left: 33pt;text-indent: 0pt;line-height: 110%;text-align: center;">All samples</p><p class="s30" style="padding-left: 16pt;text-indent: 0pt;line-height: 4pt;text-align: left;">4</p><p class="s30" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">2</p><p class="s30" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="55" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_037.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 031</p><p style="text-indent: 0pt;text-align: left;"/><p class="s39" style="padding-left: 12pt;text-indent: 0pt;line-height: 10pt;text-align: left;"><span><img width="0" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_038.png"/></span> <span class="s30">4</span></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">REG4</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 16pt;text-indent: 0pt;line-height: 4pt;text-align: left;">2</p><p class="s30" style="padding-top: 2pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="15" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_039.png"/></span></p><p class="s34" style="padding-top: 2pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 067</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 1pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">4</p><p class="s30" style="padding-top: 2pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">2</p><p class="s34" style="padding-top: 3pt;padding-left: 79pt;text-indent: 0pt;text-align: left;">Inferred cell lineage <span class="s36">0</span></p><p class="s34" style="padding-top: 1pt;padding-left: 39pt;text-indent: 0pt;text-align: left;">96 104 71</p><p class="s34" style="padding-left: 65pt;text-indent: 0pt;line-height: 6pt;text-align: left;">0 10 20 30 40 <span class="s40">0</span></p><p class="s34" style="text-indent: 0pt;text-align: right;">22%</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">0.5%</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">FABP1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: center;">4</p><p class="s34" style="padding-left: 93pt;text-indent: -1pt;line-height: 183%;text-align: right;">Stomach_Mucosal cell Stomach_Pit cell Stomach_Chief cell Pancreas_Ductal cell</p><p class="s34" style="padding-left: 75pt;text-indent: -6pt;line-height: 183%;text-align: right;">GallbladderWall_Mucous epithelial cell Duodenum_Enterocyte progenitor</p><p class="s34" style="padding-left: 66pt;text-indent: 23pt;line-height: 183%;text-align: right;">Duodenum_Goblet cell TransverseColon_Goblet cell TransverseColon_Enterocyte TransverseColon_Enterocyte progenitor SigmoidColon_Enterocyte progenitor</p><p class="s34" style="padding-left: 75pt;text-indent: 19pt;line-height: 183%;text-align: right;">Rectum_Enterocyte Rectum_Inflammed epithelial cell Breast_Epithelium_luminal cell</p><p class="s34" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">073      056      158       010       031   067</p><p class="s34" style="padding-top: 1pt;text-indent: 0pt;line-height: 3pt;text-align: left;">IP IP IP IP IP IP</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">070</p><p class="s34" style="padding-top: 1pt;text-indent: 0pt;line-height: 3pt;text-align: left;">IP</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;text-align: right;">Other</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">4%</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">1%</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;text-align: left;">0.4%</p><p class="s34" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">1%</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;text-align: left;">10%</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;text-align: right;">2.7%</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: right;">1.2%</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: right;">3.3%</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: right;">1.3%</p><p class="s34" style="padding-top: 1pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">4.5%</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">1.3%</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s41" style="padding-top: 1pt;padding-left: 3pt;text-indent: 1pt;line-height: 196%;text-align: left;">Pancreas Gallbladder</p><p class="s34" style="padding-top: 2pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">Duodenum</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Colon</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: left;">Rectum</p><p class="s34" style="padding-top: 2pt;padding-left: 18pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Stomach <span class="s30">2</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="24" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_040.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 010</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">FCGBP WFDC2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;text-align: left;">47%</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-left: 19pt;text-indent: 0pt;text-align: left;">(69.5%) <span class="s42">0</span><span class="s30"> 1</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="67" height="30" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_041.png"/></span></p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;line-height: 4pt;text-align: center;">0</p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">CTSE PSCA DMBT1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 49pt;text-indent: 0pt;line-height: 4pt;text-align: left;">4</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="27" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_042.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 031</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 3pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">2 4</p><p class="s42" style="padding-top: 1pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">0<span class="s30"> 2</span></p><p class="s30" style="text-indent: 0pt;text-align: center;">0</p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="30" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_043.png"/></span></p><p class="s30" style="padding-left: 49pt;text-indent: 0pt;text-align: left;">6</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="48" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_044.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="8" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_045.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 067</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-left: 10pt;text-indent: 0pt;line-height: 64%;text-align: left;">Intestine <span class="s42">4</span></p><p class="s43" style="padding-left: 10pt;text-indent: 0pt;line-height: 74%;text-align: left;">(21.2%) <span class="s44">2</span><span class="s30"> 1</span></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">KLK1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 1pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">0 <span class="s45">0</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="32" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_046.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="8" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_047.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 158</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 2pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">4 <span class="s42">1</span></p><p class="s30" style="padding-top: 3pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">2 <span class="s45">0</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="21" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_048.png"/></span></p><p class="s34" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">IP 158</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">SPDEF</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">0 <span class="s42">1</span></p><p class="s34" style="text-indent: 0pt;line-height: 3pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;line-height: 3pt;text-align: left;">20</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;line-height: 3pt;text-align: left;">40</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;line-height: 3pt;text-align: left;">60</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;line-height: 3pt;text-align: left;">80</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;line-height: 3pt;text-align: left;">100</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">Proportion of cells (%)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">093      124       009       114       081       057 109 007</p><p class="s34" style="padding-top: 1pt;text-indent: 0pt;line-height: 3pt;text-align: left;">IP IP IP IP IP IP IP IP</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;text-align: right;">Survival</p><p class="s34" style="padding-left: 9pt;text-indent: 0pt;line-height: 147%;text-align: left;">Short-term Long-term</p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: center;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Colorectal-like</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Duodenal-like</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Gastric</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Enterocyte</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Goblet cell</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-left: 5pt;text-indent: 0pt;line-height: 2pt;text-align: left;">Yes</p><p class="s34" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">Intestinal metaplasia                  <span class="s35">No</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s46" style="padding-top: 1pt;padding-left: 150pt;text-indent: 0pt;text-align: left;">Gastric-dominant <span style=" color: #2F7AB5;">GI-mixed</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: right;">Non</p><p class="s34" style="text-indent: 0pt;text-align: right;">-GI</p><p class="s34" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">Percentage of intestinal cell       NA</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="padding-left: 123pt;text-indent: 0pt;text-align: center;">10   20   30  40</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">IP 010</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">50 <span class="s48">µ</span>m</p><p style="text-indent: 0pt;text-align: left;"/><h4 style="padding-bottom: 2pt;padding-left: 42pt;text-indent: 0pt;text-align: left;">f</h4><p style="padding-left: 52pt;text-indent: 0pt;text-align: left;"><span><img width="135" height="112" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_049.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="166" height="146" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_050.png"/></span></p><h3 style="text-indent: 0pt;text-align: left;">g</h3><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 6pt;padding-left: 93pt;text-indent: 0pt;text-align: left;">IP-010</p><p class="s49" style="padding-top: 9pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">h <span class="s30">IP 009 IP 158</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="383" height="245" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_051.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_052.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_053.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_054.png"/></span></p><p class="s29" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">t<span class="s30">-SNE_2</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 8pt;padding-left: 39pt;text-indent: 0pt;text-align: left;">t<span class="s30">-SNE_1</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 41pt;text-indent: 0pt;text-align: left;">IP 067 IP 010</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">IP 073</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 24pt;text-indent: 0pt;text-align: left;">Seurat</p><p class="s29" style="padding-left: 24pt;text-indent: 0pt;text-align: left;">t<span class="s30">-SNE</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><table style="border-collapse:collapse" cellspacing="0"><tr style="height:55pt"><td style="width:55pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:56pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:56pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:55pt"><td style="width:55pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:56pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:56pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr></table><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 24pt;text-indent: 0pt;line-height: 110%;text-align: left;">Seurat UMAP</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">IP-093</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">IP-007</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">IP-158</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">IP-056</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;padding-left: 2pt;text-indent: 0pt;text-align: left;">IP-081</p><p class="s30" style="padding-left: 2pt;text-indent: 0pt;text-align: left;">IP-109</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">IP-073</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">UMAP_2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">IP-067</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 17pt;text-indent: 0pt;text-align: left;">IP-031</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;padding-left: 11pt;text-indent: 0pt;text-align: left;">IP-114</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">UMAP_1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Stomach pit cell Colorectal epithelial cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 57pt;text-indent: 0pt;line-height: 110%;text-align: left;">Monocle UMAP</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">UMAP_2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">IP-009</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">Stomach mucosal cell</p><p class="s30" style="padding-top: 1pt;padding-left: 30pt;text-indent: 0pt;text-align: left;">Duodenal epithelial cell Other</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">UMAP_1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">IP-124</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 25pt;text-indent: 0pt;text-align: left;">IP-057</p><p class="s8" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;line-height: 11pt;text-align: justify;"><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">(Fig. </a><span style=" color: #3B699E;">1g,h</span><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">). As expected, we observed that gastric cells were clustered distinctlyfromthecolorectal-likecells,andthiswasmoreevidentonthe UMAPplotsgeneratedfromsubclusteringanalysisbypatient(Fig. </a><span style=" color: #3B699E;">1h</span><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, Extended Data Fig. 4 and Supplementary Table 6). The cluster that was highly enriched with colorectal-like cells was separated from the rest of the clusters that were mainly composed of cells of stom- ach origin. For case IP-009, the stomach pit cells clustered distinctly from stomach mucosal cells (Fig. </a><span style=" color: #3B699E;">1h</span><a href="#bookmark47" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">). We also performed unsuper- vised clustering analysis of tumor cells using single-cell consensus clustering (SC3)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">17</span><span class="s13"> </span>at both cohort and patient levels. In line with the results of Seurat and Monocle, the independent SC3 approach grouped cells into clusters that exhibited significant differences in the compositions of different cell lineages (Extended Data Fig. 5). For example, tumor cells of IP-067 were grouped into four clusters by SC3. Cells within the cluster C1 were mainly of the stomach origin (96.5%), and only 3.5% of cells were colorectal-like, whereas the cluster C4 was mainly composed of cells of the intestine origin, with 71.4% cells being colorectal-like (two-sided propor- tion test, C4 versus C1, <span class="s23">P </span><span class="s12">&lt; </span>2.2 <span class="s12">× </span>10<span class="s50">−</span><span class="s10">16</span>). Further examining DEGs between the two clusters showed that cells within C4 expressed the highest levels of marker genes of intestine origin such as <span class="s23">DMBT1</span>, <span class="s23">FCGBP</span>, <span class="s23">PIGR </span>and <span class="s23">WFDC2 </span><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">(Fig. </a><span style=" color: #3B699E;">1e</span>), whereas cells within C1 had the highest expression of marker genes of the stomach origin such as <span class="s23">PSCA </span>and <span class="s23">TFF1</span>. Together, our results demonstrate that the diver- sity in tumor cell lineage compositions is likely a contributor to the transcriptomic ITH.</p><p class="s8" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;">In addition, we used the Bhattacharyya distance metric to measure the similarity of gene expression distributions between inferred cell lineages. The Bhattacharyya pairwise distances between colorectal-like cells and stomach pit (or mucosal) cells were significantly larger than the distances between pairs of cells of the same lineages (Extended Data Fig. 6), indicating that the colorectal-like cells were transcriptomically distinct from cells of the stomach origin. We next quantified the extent to which lineage diversity explains variations within tumors. On average, lineage dif- ference explains 21%, 20% and 7% of variations, respectively, in the top three PCs (principal components) of a tumor (Supplementary Fig. 11), and, overall, we observed larger distances between lineages across than within patients (Supplementary Fig. 12). Moreover, we sought to identify unsupervised factors that can explain the vari- ances across tumors and performed SC3 unsupervised clustering of tumor cells from all patients, but detected no significant associa- tion between the clinical, histopathological or molecular variables and the SC3-defined cell clusters. We further performed pathway enrichment analyses to examine whether differences in certain molecular processes can partially explain the interpatient variances. The cells of several cases tended to cluster together and demon- strated elevated activity of the metabolic and oncogenic pathways (Supplementary Figs. 13 and 14), indicating that tumor-intrinsic signaling pathways may have contributed to the observed interpa- tient transcriptomic heterogeneity.</p><p class="s8" style="padding-top: 10pt;padding-left: 7pt;text-indent: 11pt;line-height: 11pt;text-align: justify;"><a href="#bookmark40" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">We next investigated the genotypic ITH of PC tumor cells and examined its association with inferred tumor cell lineages. Large-scale copy number variations (CNVs) were inferred from scRNA-seq as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">10</span><a href="#bookmark48" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a><span class="s13">18</span><a href="#bookmark49" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a><span class="s13">19</span>, followed by phyloge- netic reconstruction analysis (see the Methods). In a subset of patients (<span class="s23">n </span><span class="s12">= </span>6) whose genomic DNAs were available, the inferred large-scale CNVs showed an overall good correlation with the CNVs called from bulk whole-exome sequencing (WES) data, as exemplified in Supplementary Figs. 15 and 16. We performed unsu- pervised clustering of inferred CNVs at both levels, by cell lineage and by patient, and observed greater intertumoral heterogeneity than ITH in the inferred large-scale CNV profiles (Supplementary Figs. 17 and 18). This observation was also consistent with the large <span class="s23">F </span>statistic and significant <span class="s23">P </span>values from one-way analysis of vari- ance (ANOVA) of the expression profiles (the mean <span class="s23">F </span>statistic for the top ten principal components was 7,728.4, <span class="s23">P </span><span class="s12">&lt; </span>2.2 <span class="s12">× </span>10<span class="s50">−</span><span class="s10">16</span>), and the larger Bhattacharyya pairwise distances between lineages across than within patients (Supplementary Fig. 12).</p><p class="s24" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Some PC specimens exhibited a high level of ITH. A representa- tive example was the case IP-067 (Fig. </a>2a<a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">). Phylogenetic reconstruc- tion analysis of the inferred CNVs identified five subpopulations (B1–5) with distinct CNV profiles. The pattern of CNV subclonal structure aligned well with the inferred tumor cell lineages: the largest subpopulation, B5, that demonstrated colorectal-like tran- scriptomic profiles showed distinguished CNVs profiles at mul- tiple chromosomes from the subpopulation B2 that was purely composed of cells of stomach lineage. Consistently, B5 cells were enriched in the Monocle cell cluster C1, whereas B2 cells were mainly in the cluster C3, a cluster that was clearly separated from C1 on the Monocle UMAP plot (Fig. </a>2a<span style=" color: #000;">, top right). Cells from clus- ters B1, B3 and B4 showing shared CNV profiles with that of B2 or B5 were mainly enriched in the Monocle cell cluster C2 that links clusters C1 and C3. To further understand the ITH in this case, we performed somatic variant analysis focusing on the 3</span><span class="s12">′ </span><a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">untranslated region (UTR) using scRNA-seq data (see Methods and Extended Data Fig. 7). Mutation overlapping analysis revealed ITH at the genomic level: 36% of somatic mutations identified in cells of C1 was not detected in cells of C2 or C3; overall, only 26% of muta- tions were shared among all three clusters (Fig. </a>2a<a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, bottom right). However, for case IP-009 (Fig. </a>2b<a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">), although the Monocle cell clus- ter C1 that was mainly composed of stomach pit cells was separated from the cluster C2 that was mainly composed of stomach mucosal cells, we observed slight differences in their CNV profiles among the three subpopulations defined by phylogenetic reconstruction analysis. Somatic variant analysis showed that ~60% of mutations were shared between the two Monocle cell clusters (Fig. </a>2b<span style=" color: #000;">, bottom right), suggesting a relatively more similar genomic background.</span></p><p class="s13" style="padding-top: 1pt;padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark50" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">We next examined ITH in the tumor cell proliferative property and its relationship with the inferred tumor cell lineages. We compu- tationally assigned a cell-cycle state to each individual cell based on its expression profile of cell-cycle-related signature genes</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">20</span> <span class="s8">(see the</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="677" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_055.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark10"><b>Fig. 2 | </b></a>The diversity in tumor cell lineage compositions links to iTH at transcriptomic, genotypic and molecular levels. <b>a</b>, A representative sample, IP–067. Left, phylogenetic reconstruction analysis of inferred CNVs. B1–5 labels of five tumor cell subpopulations with distinct CNV profiles. Middle, heatmap showing the inferred larger-scale CNVs by chromosome; the annotation track on the left of the heatmap indicates the inferred cell lineages, and the annotation track on the right indicates Monocle-defined cell clusters. right, top, Monocle-defined cell clusters. For each Monocle-defined cell cluster, its tumor-cell-lineage composition is shown in the small pie chart next to it; right, bottom, the Venn diagram showing shared and unique somatic variants across Monocle-defined cell clusters. Somatic variants were called from scrNA-seq data, and only variants located at the 3<span class="s26">ʹ </span>UTr were counted. <b>b</b>, Another representative sample, IP–009. B1–3 labels of three tumor cell subpopulations with distinct CNV profiles. The annotations for the remainder of <b>b </b>are in<a name="bookmark27">&zwnj;</a></p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;">the same format as those of <b>a</b>. <b>c</b>, Comparison of tumor cell proliferative property across the inferred tumor cell lineages. Box, median <span class="s26">± </span>interquartile range. Whiskers, the minimum and maximum values. <i>P </i>values were calculated by a two-sided Wilcoxon rank-sum test with Benjamini–Hochberg correction.</p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 108%;text-align: left;"><b>d</b>, Proportion of cycling (cells in G2M or S phase) and non-cycling cells across the inferred cell lineages. <b>e</b>, The violin plots for representative cell-cycle-related genes that are differentially expressed across tumor cell lineages/types (<i>P </i><span class="s26">&lt; </span>2.2 <span class="s26">× </span>10<span class="s27">−</span><span class="s28">16</span>). <i>P </i>values were calculated by one-way Kruskal–Wallis rank-sum test.</p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 108%;text-align: left;"><i>P </i><span class="s26">&lt; </span>2.2 <span class="s26">× </span>10<span class="s27">−</span><span class="s28">16 </span>represents a <i>P </i>value approaching 0. Number of cells for <b>c </b>and <b>e</b>: colon goblet cells, <i>n </i><span class="s26">= </span>2,658; colon enterocyte cells, <i>n </i><span class="s26">= </span>1,042; rectum epithelial cells, <i>n </i><span class="s26">= </span>1,578; duodenum epithelial cells, <i>n </i><span class="s26">= </span>366; stomach pit cells, <i>n </i><span class="s26">= </span>12,341; stomach mucosal cells, <i>n </i><span class="s26">= </span>5,937. FC, fold change.</p><p class="s24" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Methods and Supplementary Table 7) and compared the tumor cell proliferative property across the inferred cell lineages (Fig. </a><a href="#bookmark27" class="s22">2c–e</a> <span style=" color: #000;">and Supplementary Fig. 19). Our analysis showed that the proliferative</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 41pt;text-indent: 0pt;text-align: left;">a</h4><p class="s24" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">property of stomach pit cells was the highest among all cell lin- eages, indicated by high G2M and S scores (Fig. </a>2c<a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">) and fraction of cycling cells (Fig. </a>2d<span style=" color: #000;">). On average, about 72% of stomach pit</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="575" height="238" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_056.png"/></span></p><p class="s30" style="padding-left: 47pt;text-indent: 1pt;line-height: 111%;text-align: left;">Phylogenetic reconstruction</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;line-height: 5pt;text-align: right;">Inferred cell lineage <span class="s51">Monocle</span></p><p class="s52" style="text-indent: 0pt;text-align: left;">Chr 1 3 5 7 9 11 13 15 17 19 21 X</p><p style="text-indent: 0pt;text-align: left;"/><p class="s52" style="text-indent: 0pt;text-align: left;">20 22</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 44pt;text-indent: 0pt;text-align: center;">Inferred copy number alterations</p><p class="s52" style="padding-top: 2pt;padding-left: 44pt;text-indent: 0pt;text-align: center;">2 4  6  8 10 12 14 16 18</p><p style="text-indent: 0pt;text-align: left;"/><p class="s53" style="text-indent: 0pt;line-height: 4pt;text-align: left;">IP 067</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: right;">cluster ID</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">C1 C2 C3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><table style="border-collapse:collapse" cellspacing="0"><tr style="height:98pt"><td style="width:58pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-left-style:solid;border-left-width:1pt;border-left-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p class="s54" style="padding-top: 4pt;padding-left: 2pt;padding-right: 14pt;text-indent: 0pt;text-align: center;">Monocle cell clusters</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 37pt;text-indent: 0pt;text-align: left;">C3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 2pt;padding-right: 14pt;text-indent: 0pt;text-align: center;">C1</p></td><td style="width:6pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:39pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s55" style="padding-left: 13pt;text-indent: 0pt;text-align: left;">IP 067</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 4pt;text-indent: 0pt;text-align: left;">C2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s56" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">UMAP_1</p></td></tr><tr style="height:18pt"><td style="width:58pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-left-style:solid;border-left-width:1pt;border-left-color:#808285"><p class="s54" style="padding-top: 2pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">Somatic variants (3<span class="s57">&#39; </span>UTR)</p></td><td style="width:6pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:39pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:15pt"><td style="width:58pt;border-left-style:solid;border-left-width:1pt;border-left-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s58" style="padding-right: 1pt;text-indent: 0pt;line-height: 92%;text-align: right;">50 <span class="s54">24</span></p></td><td style="width:6pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:39pt;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 3pt;text-indent: 0pt;line-height: 5pt;text-align: left;">40 <span class="s58">C1</span></p></td></tr><tr style="height:5pt"><td style="width:58pt;border-left-style:solid;border-left-width:1pt;border-left-color:#808285"><p class="s54" style="padding-left: 18pt;text-indent: 0pt;line-height: 3pt;text-align: left;">C2</p></td><td style="width:6pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:39pt;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:5pt"><td style="width:58pt;border-left-style:solid;border-left-width:1pt;border-left-color:#808285"><p class="s54" style="padding-right: 1pt;text-indent: 0pt;line-height: 3pt;text-align: right;">46</p></td><td style="width:6pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:39pt;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:7pt"><td style="width:58pt;border-left-style:solid;border-left-width:1pt;border-left-color:#808285"><p class="s54" style="padding-top: 1pt;padding-right: 10pt;text-indent: 0pt;line-height: 4pt;text-align: right;">4</p></td><td style="width:6pt"><p class="s54" style="padding-left: 1pt;text-indent: 0pt;line-height: 4pt;text-align: left;">1</p></td><td style="width:39pt;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:12pt"><td style="width:58pt;border-left-style:solid;border-left-width:1pt;border-left-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p class="s54" style="padding-right: 1pt;text-indent: 0pt;text-align: right;">12</p></td><td style="width:6pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:39pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p class="s54" style="padding-top: 2pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">C3</p></td></tr></table><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 77pt;text-indent: 0pt;text-align: left;">B1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 89pt;text-indent: 0pt;text-align: left;">B2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s59" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">UMAP_2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 3pt;padding-left: 72pt;text-indent: 0pt;text-align: left;">B3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s53" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">IP 067 (<span class="s29">n </span><span class="s30">= 1,534 tumor cells)</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 74pt;text-indent: 0pt;text-align: left;">B4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">B5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_057.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_058.png"/></span></p><p class="s30" style="padding-left: 64pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Stomach pit cell Colorectal cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="6" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_059.png"/></span></p><p class="s30" style="padding-left: 31pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Pancreas ductal cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 58pt;text-indent: 0pt;text-align: left;">Inferred relative copy number ratio</p><p class="s30" style="padding-top: 2pt;padding-left: 64pt;text-indent: 0pt;text-align: left;">Stomach mucosal cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="575" height="227" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_060.png"/></span></p><h4 style="text-indent: 0pt;text-align: center;">b</h4><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">Duodenal epithelial cell</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">Gallbladder epithelial cell</p><p class="s30" style="padding-left: 20pt;text-indent: 0pt;text-align: center;">Loss                             Gain</p><p style="text-indent: 0pt;text-align: left;"><span><img width="40" height="4" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_061.png"/></span></p><p class="s60" style="padding-top: 1pt;padding-left: 18pt;text-indent: 0pt;text-align: center;">0   0.5 1.0  1.5 2.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 44pt;text-indent: 0pt;line-height: 111%;text-align: left;">Phylogenetic reconstruction</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;line-height: 6pt;text-align: right;">Inferred cell lineage <span class="s40">Monocle</span></p><p class="s52" style="text-indent: 0pt;text-align: left;">18 20 22</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 39pt;text-indent: 0pt;text-align: center;">Inferred copy number alterations</p><p class="s52" style="padding-top: 2pt;padding-left: 36pt;text-indent: 0pt;text-align: center;">2 4  6 8 10 12 14 16</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: left;">IP-009</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s52" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">Chr 1</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: right;">cluster</p><p class="s52" style="text-indent: 0pt;text-align: left;">3 5 7 9 11 13 15 17 19 21 X</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">ID</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">C1 C2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><table style="border-collapse:collapse" cellspacing="0"><tr style="height:95pt"><td style="width:54pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-left-style:solid;border-left-width:1pt;border-left-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p class="s54" style="padding-top: 3pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">Monocle cell clusters</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 32pt;text-indent: 0pt;text-align: left;">C1</p></td><td style="width:8pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:41pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s55" style="padding-left: 12pt;text-indent: 0pt;text-align: left;">IP 009</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">C2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s56" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">UMAP_1</p></td></tr><tr style="height:55pt"><td style="width:54pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-left-style:solid;border-left-width:1pt;border-left-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p class="s54" style="padding-top: 2pt;padding-left: 3pt;text-indent: 0pt;text-align: center;">Somatic variants (3<span class="s57">&#39; </span>UTR)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 18pt;text-indent: 0pt;text-align: center;">26</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 10pt;text-indent: 0pt;text-align: center;">C1</p></td><td style="width:8pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="text-indent: 0pt;text-align: left;">69</p></td><td style="width:41pt;border-top-style:solid;border-top-width:1pt;border-top-color:#808285;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#808285;border-right-style:solid;border-right-width:1pt;border-right-color:#808285"><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 5pt;padding-right: 20pt;text-indent: 0pt;text-align: center;">21</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s54" style="padding-left: 9pt;padding-right: 16pt;text-indent: 0pt;text-align: center;">C2</p></td></tr></table><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 76pt;text-indent: 0pt;text-align: left;">B1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s53" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">IP 009 (<span class="s29">n </span><span class="s30">= 2,250 tumor cells)</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s59" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">UMAP_2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 3pt;padding-left: 77pt;text-indent: 0pt;text-align: left;">B2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 76pt;text-indent: 0pt;text-align: left;">B3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_062.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_063.png"/></span></p><p class="s61" style="padding-bottom: 1pt;padding-left: 41pt;text-indent: 0pt;line-height: 9pt;text-align: left;">c<u> </u><span class="s63">***</span><span class="s64"> </span>d</p><p class="s65" style="padding-left: 109pt;text-indent: 0pt;line-height: 3pt;text-align: left;"><span><img width="0" height="4" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_064.png"/></span>	<span><img width="0" height="4" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_065.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.8</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">G2M score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0.4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0.2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">G2M score</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="55" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_066.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="38" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_067.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="24" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_068.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="22" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_069.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="85" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_070.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="29" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_071.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="75" height="4" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_072.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="151" height="96" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_073.png"/></span></p><p class="s30" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">FC=1.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s64" style="padding-left: 2pt;text-indent: 0pt;text-align: left;">***</p><p class="s30" style="text-indent: 0pt;line-height: 3pt;text-align: left;">FC = 1.8</p><p class="s64" style="padding-left: 4pt;text-indent: 0pt;line-height: 6pt;text-align: left;">***</p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="12" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_074.png"/></span></p><p class="s30" style="padding-left: 1pt;text-indent: 0pt;line-height: 4pt;text-align: left;">FC = 2.0</p><p class="s64" style="padding-top: 5pt;padding-left: 3pt;text-indent: 0pt;line-height: 7pt;text-align: left;">***</p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">FC = 1.7</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 35pt;text-indent: 14pt;line-height: 249%;text-align: right;">Colon_Goblet cell Colon_Enterocyte Rectum_Epithelial cell Duodenum_Epithelial cell</p><p class="s30" style="padding-left: 40pt;text-indent: 10pt;line-height: 249%;text-align: right;">Stomach_Pit cell Stomach_Mucosal cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="65" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_075.png"/></span></p><h4 style="text-indent: 0pt;line-height: 7pt;text-align: center;">e</h4><p class="s30" style="padding-bottom: 4pt;padding-left: 132pt;text-indent: 0pt;line-height: 1pt;text-align: center;">Cycling cell                  Non-cycling cell</p><p style="text-indent: 0pt;text-align: left;"><span><img width="155" height="56" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_076.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="157" height="26" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_077.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;line-height: 2pt;text-align: left;"><span><img width="158" height="2" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_078.png"/></span></p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0                     0.25                  0.50                  0.75                 1.00</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Fraction</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">2.5</p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">2.0</p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">1.5</p><p class="s66" style="text-indent: 0pt;line-height: 10pt;text-align: left;">***</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">1.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 3pt;padding-left: 10pt;text-indent: 0pt;text-align: left;">MKI67</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="106" height="65" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_079.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="65" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_080.png"/></span></p><p class="s29" style="padding-left: 10pt;text-indent: 0pt;text-align: left;">BIRC5</p><p class="s66" style="padding-top: 8pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">***</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s66" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">***</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">S score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0.4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0.2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="65" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_081.png"/></span></p><p class="s30" style="text-indent: 0pt;text-align: left;">S score</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">FC = 1.5</p><p class="s51" style="padding-left: 53pt;text-indent: 0pt;text-align: left;">5 <i>HN1</i><span class="s30">***</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="85" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_082.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="50" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_083.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="30" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_084.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="31" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_085.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="57" height="95" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_086.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="18" height="41" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_087.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="168" height="98" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_088.png"/></span></p><p class="s30" style="padding-top: 2pt;padding-left: 2pt;text-indent: 0pt;text-align: center;">4</p><p class="s30" style="padding-top: 4pt;padding-left: 2pt;text-indent: 0pt;text-align: center;">3</p><p class="s30" style="padding-top: 4pt;padding-left: 2pt;text-indent: 0pt;text-align: center;">2</p><p class="s30" style="padding-top: 4pt;padding-left: 2pt;text-indent: 0pt;text-align: center;">1</p><p class="s30" style="padding-top: 4pt;padding-left: 2pt;text-indent: 0pt;text-align: center;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="123" height="77" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_089.png"/></span></p><p class="s30" style="padding-top: 3pt;padding-left: 2pt;text-indent: 0pt;line-height: 4pt;text-align: center;">3</p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Gene expression</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="text-indent: 0pt;line-height: 4pt;text-align: right;">NME1<span class="s30">***</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 2pt;text-indent: 0pt;text-align: center;">2</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">2.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">1.5</p><p class="s29" style="text-indent: 0pt;text-align: right;">SKP1<span class="s30">***</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="65" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_090.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="107" height="65" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_091.png"/></span></p><p class="s29" style="text-indent: 0pt;text-align: right;">CDK4<span class="s30">***</span></p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">0.5</p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 3pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">TK1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s66" style="padding-left: 12pt;text-indent: 0pt;text-align: left;">***</p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Colon_Goblet cell</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Colon_Enterocyte</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Rectum_Epithelial cell</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Stomach_Pit cell</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Stomach_Mucosal cell</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 2pt;padding-left: 56pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Duodenum_Epithelial cell</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">***<i>P </i>&lt; 2.2 × 10–16</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: center;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: center;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">Cell lineage (Inferred)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">1.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p class="s30" style="padding-top: 2pt;padding-left: 64pt;text-indent: 0pt;line-height: 143%;text-align: left;">Colon_Goblet cell Colon_Enterocyte</p><p style="text-indent: 0pt;text-align: left;"><span><img width="8" height="51" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_092.png"/></span></p><p class="s30" style="padding-top: 1pt;padding-left: 64pt;text-indent: 0pt;text-align: left;">Rectum_Epithelial cell</p><p class="s30" style="padding-top: 2pt;padding-left: 64pt;text-indent: 0pt;line-height: 134%;text-align: left;">Duodenum_Epithelial cell Stomach pit cell Stomach mucosal cell</p><p class="s30" style="padding-top: 4pt;padding-left: 56pt;text-indent: 0pt;text-align: left;">***<i>P </i>&lt; 2.2 × 10–16</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s67" style="padding-top: 5pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">a <span class="s68">Large-scale CNVs</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="461" height="285" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_093.png"/></span></p><p class="s68" style="padding-top: 3pt;padding-left: 106pt;text-indent: 0pt;text-align: left;">Chr 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 X</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 408pt;text-indent: 0pt;text-align: left;">Survival</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="15" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_094.png"/></span></p><p class="s68" style="padding-top: 2pt;padding-left: 416pt;text-indent: 0pt;line-height: 138%;text-align: right;">Long-term survivor Short-term survivor</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 408pt;text-indent: 0pt;text-align: left;">Subtype</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="17" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_095.png"/></span></p><p class="s68" style="padding-top: 2pt;padding-left: 417pt;text-indent: 0pt;line-height: 148%;text-align: left;">Gastric-dominant GI-mixed</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 408pt;text-indent: 0pt;text-align: left;">Inferred cell lineage</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="45" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_096.png"/></span></p><p class="s68" style="padding-top: 2pt;padding-left: 417pt;text-indent: 0pt;text-align: left;">Stomach_Mucosal cell Stomach_Pit cell</p><p class="s68" style="padding-top: 1pt;padding-left: 417pt;text-indent: 0pt;line-height: 109%;text-align: left;">Colon_Goblet cell Colon_Enterocyte</p><p class="s68" style="padding-left: 417pt;text-indent: 0pt;text-align: left;">Rectum_Epithelial cell Duodenum_Epithelial cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="13" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_097.png"/></span></p><p class="s68" style="padding-left: 417pt;text-indent: 0pt;text-align: left;">Pancreas_Ductal cell Gallbladder_Epithelial cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_098.png"/></span></p><p class="s68" style="padding-left: 417pt;text-indent: 0pt;line-height: 126%;text-align: left;">Breast_Epithelial cell Other</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 59pt;text-indent: 0pt;text-align: left;">b</h4><p class="s69" style="padding-top: 2pt;padding-left: 77pt;text-indent: 0pt;text-align: left;">YES <span style=" color: #784A24;">NO</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">IP−093</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">IP−009</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">IP−057</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−114</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: right;">Long-term survivors</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s71" style="padding-left: 5pt;text-indent: 0pt;line-height: 3pt;text-align: left;">MYC targets</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;padding-left: 15pt;text-indent: 0pt;text-align: left;">CNVs Gain Loss</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="84" height="36" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_099.png"/></span></p><h4 style="text-indent: 0pt;text-align: right;">c</h4><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">4</p><p style="text-indent: 0pt;text-align: left;"><span><img width="283" height="156" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_100.png"/></span></p><p class="s68" style="padding-top: 4pt;text-indent: 0pt;line-height: 4pt;text-align: right;">2</p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;line-height: 4pt;text-align: center;">17q gain</p><p class="s68" style="padding-top: 1pt;text-indent: 0pt;line-height: 5pt;text-align: right;">0</p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">Survival</p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="36" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_101.png"/></span></p><p class="s72" style="padding-top: 3pt;padding-left: 48pt;text-indent: 0pt;line-height: 89%;text-align: left;">HN1 <span class="s73">3</span></p><p class="s68" style="text-indent: 0pt;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s72" style="padding-left: 47pt;text-indent: 0pt;line-height: 5pt;text-align: left;">LGALS3BP</p><p class="s72" style="padding-left: 132pt;text-indent: 0pt;line-height: 4pt;text-align: center;">DCXR</p><p class="s74" style="padding-left: 47pt;text-indent: 0pt;line-height: 6pt;text-align: left;">ARHGDIA <span class="s75">1</span></p><p class="s72" style="padding-left: 47pt;text-indent: 0pt;line-height: 6pt;text-align: left;">NME1 <span class="s76">0</span></p><p class="s72" style="padding-top: 1pt;padding-left: 47pt;text-indent: 0pt;line-height: 125%;text-align: left;">SYNGR2 <span style=" color: #4BC676;">PHB</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="4" height="26" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_102.png"/></span></p><p class="s78" style="padding-top: 1pt;padding-left: 47pt;text-indent: 0pt;text-align: left;">SLC9A3R1 <span class="s73">2</span></p><p class="s79" style="padding-left: 47pt;text-indent: 0pt;text-align: left;">EIF1 <span class="s80">1</span></p><p class="s79" style="padding-left: 47pt;text-indent: 0pt;text-align: left;">SOCS3 <span class="s68">0</span></p><p class="s74" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">GRB2</p><p style="text-indent: 0pt;text-align: left;"><span><img width="95" height="36" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_103.png"/></span></p><p class="s72" style="padding-top: 1pt;padding-left: 47pt;text-indent: 0pt;text-align: left;">RNF213 <span class="s76">2</span></p><p class="s78" style="padding-top: 2pt;padding-left: 47pt;text-indent: 0pt;text-align: left;">ATP5MC1 <span class="s80">1</span></p><p class="s77" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">PSMB3</p><p class="s81" style="padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">PI3K/AKT/MTOR signaling                          <span class="s75">0</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Sample</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Survival</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Subtype</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Cell lineage</p><p style="text-indent: 0pt;text-align: left;"/><p class="s72" style="padding-left: 21pt;text-indent: 0pt;line-height: 5pt;text-align: left;">HN1</p><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: right;">2</p><p class="s68" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="0" height="18" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_104.png"/></span></p><p class="s72" style="text-indent: 0pt;text-align: right;">SOCS3 <span class="s73">2</span></p><p class="s68" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="77" height="33" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_105.png"/></span></p><p class="s72" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">ARHGDIA</p><p class="s68" style="text-indent: 0pt;text-align: right;">4</p><p class="s68" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">2</p><p class="s68" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0</p><p class="s68" style="padding-top: 5pt;text-indent: 0pt;line-height: 5pt;text-align: right;">3</p><p class="s72" style="text-indent: 0pt;line-height: 93%;text-align: right;">GRB2 <span class="s82">2</span></p><p class="s68" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s72" style="padding-left: 94pt;text-indent: 0pt;text-align: center;">SLC9A3R1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="85" height="36" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_106.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="87" height="36" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_107.png"/></span></p><p class="s72" style="padding-left: 102pt;text-indent: 0pt;text-align: center;">PHB</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="87" height="36" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_108.png"/></span></p><p class="s72" style="padding-left: 100pt;text-indent: 0pt;text-align: center;">EIF1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="95" height="41" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_109.png"/></span></p><p class="s72" style="padding-top: 4pt;padding-left: 103pt;text-indent: 0pt;text-align: center;">DCXR</p><p class="s68" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">(<i>n </i>= 7)</p><p class="s83" style="padding-top: 2pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">MTORC1 signaling</p><p class="s84" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">TNF<span class="s85">a </span>signaling via NF<span class="s85">K</span>B</p><p class="s68" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">17q gain + + +</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−093</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−009</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−114</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-left: 23pt;text-indent: 0pt;text-align: left;">Sample</p><p class="s68" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">– –</p><p style="text-indent: 0pt;text-align: left;"><span><img width="66" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_110.png"/></span></p><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−057</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−093</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−009</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−114</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 3pt;padding-left: 13pt;text-indent: -1pt;line-height: 111%;text-align: justify;"><span style=" color: #FFCA54;">Long-term survivors </span>(<i>n </i>= 7)</p><p class="s68" style="padding-left: 14pt;text-indent: 0pt;text-align: left;">+ + +</p><p class="s68" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">– –</p><p style="text-indent: 0pt;text-align: left;"><span><img width="61" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_111.png"/></span></p><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: left;">IP−057</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-left: 12pt;text-indent: -1pt;line-height: 111%;text-align: justify;"><span style=" color: #FFCA54;">Long-term survivors </span>(<i>n </i>= 7)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="282" height="135" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_112.png"/></span></p><p class="s68" style="padding-top: 2pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">GAC-PC validation cohort (<i>n </i>= 45, MDACC)</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 9pt;padding-left: 59pt;text-indent: 0pt;text-align: left;"><span class="s67">d </span>Primary GAC cohort (<i>n </i>= 411, TCGA) <span class="s67">e</span></p><p class="s87" style="text-indent: 0pt;line-height: 4pt;text-align: left;">Short-term survivors</p><p class="s87" style="padding-left: 33pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Short-term survivors</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s88" style="padding-left: 132pt;text-indent: 0pt;line-height: 68%;text-align: center;">+<span class="s89">+</span><span class="s73">+</span></p><p class="s90" style="padding-left: 132pt;text-indent: 0pt;line-height: 49%;text-align: center;">+<span class="s68">+</span><span class="s89">+</span></p><p class="s75" style="padding-left: 11pt;text-indent: 0pt;line-height: 50%;text-align: left;">+<b>+</b><b>+</b><span class="s68">+</span></p><p class="s68" style="text-indent: 0pt;text-align: left;">+ +</p><p style="text-indent: 0pt;text-align: left;"/><p class="s91" style="padding-left: 14pt;text-indent: 0pt;line-height: 6pt;text-align: left;">+<span class="s92">+</span><span class="s93">+</span><span class="s75">+</span><span class="s88">++</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">OS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 30pt;text-indent: 0pt;line-height: 2pt;text-align: left;">Median survival</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 3pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s88" style="text-indent: 0pt;line-height: 5pt;text-align: center;">+</p><p class="s68" style="text-indent: 0pt;text-align: left;">+<b>+</b></p><p style="text-indent: 0pt;text-align: left;"/><p class="s88" style="text-indent: 0pt;line-height: 4pt;text-align: center;">+</p><p class="s76" style="padding-left: 132pt;text-indent: 0pt;line-height: 83%;text-align: center;">+<span class="s80">+</span><span class="s68">+</span><b>+</b></p><p class="s76" style="padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">+<span class="s68">++</span></p><p class="s88" style="padding-left: 14pt;text-indent: 0pt;line-height: 3pt;text-align: left;">+<span class="s89">+</span>+<span class="s68">+</span><span class="s94">+</span><span class="s68">+</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">PFI</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="178" height="131" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_113.png"/></span></p><p class="s68" style="text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s88" style="padding-left: 6pt;text-indent: 0pt;line-height: 5pt;text-align: left;">+</p><p class="s93" style="text-indent: 0pt;line-height: 9pt;text-align: right;">+<span class="s75">+ </span><span class="s68">++</span></p><p class="s68" style="text-indent: 0pt;line-height: 3pt;text-align: right;">+</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 26pt;text-indent: 0pt;text-align: left;">Median survival</p><p class="s68" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;line-height: 1pt;text-align: left;">9.6 months</p><p class="s95" style="padding-top: 2pt;padding-left: 101pt;text-indent: 0pt;line-height: 4pt;text-align: left;">+ <span class="s88">+ </span><span class="s76">21.1 months</span></p><p class="s90" style="padding-left: 35pt;text-indent: 0pt;line-height: 6pt;text-align: left;"><span class="s96">+</span>+<span class="s88">+ </span><span class="s97">+</span><span class="s98">+</span>++++++++ <span class="s68">+</span></p><p class="s68" style="padding-top: 1pt;padding-left: 97pt;text-indent: 0pt;line-height: 4pt;text-align: left;">0.75 <span class="s73">+</span></p><p class="s68" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s88" style="text-indent: 0pt;text-align: left;">+</p><p style="text-indent: 0pt;text-align: left;"/><p class="s90" style="padding-left: 103pt;text-indent: 0pt;line-height: 4pt;text-align: left;">+<span class="s68">+ </span><span class="s98">+</span>++<span class="s68">+</span><span class="s88">+ </span><span class="s76">+ </span><span class="s97">+</span></p><p class="s68" style="padding-left: 157pt;text-indent: 0pt;line-height: 5pt;text-align: left;">4.8 months</p><p class="s68" style="text-indent: 0pt;line-height: 1pt;text-align: center;">+</p><p class="s68" style="text-indent: 0pt;line-height: 5pt;text-align: right;">0.50</p><p class="s88" style="text-indent: 0pt;line-height: 5pt;text-align: right;">++ <span class="s68">+++</span></p><p class="s88" style="padding-left: 132pt;text-indent: 0pt;line-height: 1pt;text-align: center;">+<span class="s68">+</span></p><p class="s68" style="padding-left: 19pt;text-indent: 0pt;line-height: 5pt;text-align: left;">38.4 months</p><p class="s68" style="padding-left: 9pt;text-indent: 0pt;line-height: 4pt;text-align: left;">0.50</p><p class="s68" style="padding-left: 18pt;text-indent: 0pt;line-height: 6pt;text-align: left;"><b>+</b>+<b>++ </b>++<b>+</b></p><p class="s68" style="padding-top: 1pt;padding-left: 12pt;text-indent: 0pt;line-height: 4pt;text-align: left;">+<b>+</b>+ +++ + + + <span class="s80">+</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0.25</p><p class="s68" style="padding-left: 25pt;text-indent: 0pt;line-height: 5pt;text-align: left;">+++<b>+</b>+</p><p class="s88" style="text-indent: 0pt;line-height: 5pt;text-align: right;">+</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s72" style="padding-left: 11pt;text-indent: 0pt;line-height: 4pt;text-align: left;">P <span class="s68">= 0.047</span></p><p class="s80" style="padding-top: 3pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">++<b>+</b><span class="s68">+ +++ + + +</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;line-height: 1pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s72" style="padding-left: 9pt;text-indent: 0pt;line-height: 4pt;text-align: left;">P <span class="s68">= 0.036</span></p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">+ + + +</p><p class="s68" style="padding-top: 3pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">Median survival</p><p class="s68" style="padding-top: 2pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">18.5 months</p><p class="s68" style="padding-top: 1pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">56.2 months</p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.25</p><p class="s88" style="padding-top: 2pt;padding-left: 26pt;text-indent: 0pt;text-align: left;">+ <span class="s68">+ +</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">+</p><p class="s68" style="padding-top: 1pt;padding-left: 57pt;text-indent: 0pt;text-align: left;">+</p><p class="s68" style="text-indent: 0pt;text-align: center;">0</p><p class="s68" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0           25          50          75         100     125</p><p class="s68" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><table style="border-collapse:collapse" cellspacing="0"><tr style="height:11pt"><td style="width:19pt"><p class="s99" style="padding-top: 1pt;text-indent: 0pt;text-align: center;">No gain</p></td><td style="width:5pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:3pt;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:13pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698;border-left-style:solid;border-left-width:1pt;border-left-color:#949698"><p class="s99" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">228</p></td><td style="width:17pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">65</p></td><td style="width:18pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-left: 4pt;padding-right: 5pt;text-indent: 0pt;text-align: center;">13</p></td><td style="width:16pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;text-indent: 0pt;text-align: center;">5</p></td><td style="width:17pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">3</p></td><td style="width:13pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p class="s99" style="padding-top: 2pt;padding-right: 3pt;text-indent: 0pt;text-align: right;">0</p></td><td style="width:10pt;border-left-style:solid;border-left-width:1pt;border-left-color:#949698"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:5pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:3pt;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:14pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698;border-left-style:solid;border-left-width:1pt;border-left-color:#949698"><p class="s99" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: center;">229</p></td><td style="width:16pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-left: 3pt;padding-right: 5pt;text-indent: 0pt;text-align: center;">57</p></td><td style="width:18pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-left: 4pt;padding-right: 6pt;text-indent: 0pt;text-align: center;">13</p></td><td style="width:17pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-right: 2pt;text-indent: 0pt;text-align: center;">5</p></td><td style="width:16pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698"><p class="s99" style="padding-top: 2pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">3</p></td><td style="width:13pt;border-top-style:solid;border-top-width:1pt;border-top-color:#949698;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p class="s99" style="padding-top: 2pt;text-indent: 0pt;text-align: center;">0</p></td></tr><tr style="height:9pt"><td style="width:19pt"><p class="s100" style="padding-left: 6pt;text-indent: 0pt;text-align: center;">Gain</p></td><td style="width:5pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:3pt;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:13pt;border-left-style:solid;border-left-width:1pt;border-left-color:#949698;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="text-indent: 0pt;text-align: left;">104</p></td><td style="width:17pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">16</p></td><td style="width:18pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="text-indent: 0pt;text-align: center;">3</p></td><td style="width:16pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="text-indent: 0pt;text-align: center;">0</p></td><td style="width:17pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0</p></td><td style="width:13pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p class="s100" style="padding-right: 3pt;text-indent: 0pt;text-align: right;">0</p></td><td style="width:10pt;border-left-style:solid;border-left-width:1pt;border-left-color:#949698"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:5pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:3pt;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:14pt;border-left-style:solid;border-left-width:1pt;border-left-color:#949698;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-right: 5pt;text-indent: 0pt;text-align: center;">105</p></td><td style="width:16pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-left: 3pt;padding-right: 5pt;text-indent: 0pt;text-align: center;">12</p></td><td style="width:18pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-right: 1pt;text-indent: 0pt;text-align: center;">2</p></td><td style="width:17pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-right: 2pt;text-indent: 0pt;text-align: center;">0</p></td><td style="width:16pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698"><p class="s100" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">0</p></td><td style="width:13pt;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#949698;border-right-style:solid;border-right-width:1pt;border-right-color:#949698"><p class="s100" style="text-indent: 0pt;text-align: center;">0</p></td></tr></table><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 3pt;padding-bottom: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Number at risk</p><p style="padding-left: 78pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="11" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_114.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 78pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="11" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_115.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">0          25         50         75        100       125</p><p class="s68" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><p class="s68" style="padding-top: 2pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">Number at risk</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 8pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="10" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_116.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 8pt;text-indent: 0pt;line-height: 5pt;text-align: left;"><span><img width="10" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_117.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="135" height="23" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_118.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_119.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="9" height="7" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_120.png"/></span></p><p class="s101" style="padding-left: 39pt;text-indent: -6pt;line-height: 163%;text-align: right;">No gain <span style=" color: #F5950B;">Gain</span></p><p class="s72" style="padding-top: 1pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">P <span class="s68">= 0.029</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">0 10 20 30 40</p><p class="s68" style="padding-top: 1pt;padding-bottom: 2pt;padding-left: 2pt;text-indent: 13pt;line-height: 134%;text-align: left;">Time from PC diagnosis (months) Number at risk</p><p class="s101" style="text-indent: 0pt;line-height: 4pt;text-align: left;">35</p><p class="s102" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">10</p><p class="s101" style="text-indent: 0pt;line-height: 4pt;text-align: left;">11</p><p class="s102" style="padding-top: 2pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">1</p><p class="s101" style="text-indent: 0pt;line-height: 4pt;text-align: left;">5</p><p class="s102" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">0</p><p class="s101" style="text-indent: 0pt;line-height: 4pt;text-align: left;">1</p><p class="s102" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">0</p><p class="s101" style="text-indent: 0pt;line-height: 4pt;text-align: left;">1</p><p class="s102" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">0</p><p style="padding-left: 4pt;text-indent: 0pt;text-align: left;"><span style=" color: black; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;">				</span></p><p class="s68" style="text-indent: 0pt;text-align: right;">17q copy number status</p><p class="s101" style="padding-left: 10pt;text-indent: 0pt;text-align: left;">No gain</p><p class="s102" style="padding-left: 14pt;text-indent: 0pt;text-align: left;">Gain</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><a name="bookmark11">Fig. 3 | </a><span class="s18">17q copy number gain is prevalent in cells of stomach origin and significantly associated with inferior survival. </span>a<span class="s18">, The landscape of inferred</span><a name="bookmark28">&zwnj;</a></h4><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a href="#bookmark26" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">large-scale CNVs for all of the tumor cells. The annotation tracks on the left indicate (from left to right) the corresponding sample IDs (the same colors as in Fig. </a><span style=" color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">1b</span>), survival groups, PC subtypes and the inferred cell lineages/types. Chromosome numbers are labeled on the top. The yellow rectangle highlights the 17q copy number gain that was nearly exclusively found in cells from the short-term survivors. <b>b</b>, The heatmap displays scaled expression values of genes upregulated in three short-term survivors (sample IDs labeled at the bottom) with evident 17q gain (annotated on the top track) and one short-term survivor and seven long-term survivors without detectable 17q changes. Biologically important genes are listed on the right, color coded by their related signaling pathways. <b>c</b>, The representative violin plots of eight genes selected from <b>b</b>. <b>d</b>,<b>e</b>, 17q copy number gain was associated with worse patient survival in the TCGA primary GAC cohort (<b>d</b>) (<i>n </i><span class="s26">= </span>411; only cases with survival data available were included) and an independent GAC-PC cohort (<b>e</b>) (<i>n </i><span class="s26">= </span>45). <i>P </i>values were calculated by a two-sided log-rank test. Median survival times (in months) are labeled on the plots. MDACC, MD Anderson Cancer Center; OS, overall survival; PFI, progression-free interval.</p><p class="s13" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark51" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">cells were cycling, and the fraction was much higher than that of stomach mucosa cells (27%) or intestinal-like cells: colon goblet cells (58%), colon enterocytes (34%), rectum (15%) and duodenum (18%). These results are in line with previous observations show- ing that the cellular turnover of stomach pit cells is faster than that of the mucus-secreting cells of the stomach</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">21</span><a href="#bookmark52" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>22<a href="#bookmark53" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, and with numer- ous reports showing that intestinal surface epithelium, including goblet cells, undergoes turnover rapidly</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">23</span><a href="#bookmark54" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>24<a href="#bookmark27" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. Consistently, some key cell-cycle regulatory genes were differentially expressed across tumor cell lineages (Fig. </a><span class="s24">2e</span><span class="s8">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a name="bookmark5"><b>17q copy number gain is nearly exclusive to cells of stomach ori- gin and is associated with worse survival. </b></a><a href="#bookmark28" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">We analyzed the inferred CNVs from all cases together and discovered 17q copy number gain as a unique event that was prevalent in tumor cells of stomach ori- gin and only present in tumor cells from the short-term survivors (Fig. </a><span style=" color: #3B699E;">3a</span><a href="#bookmark28" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">). By integrating genotypic and transcriptomic profiles, we identified a list of genes upregulated on 17q in tumor cells from cases with evident 17q gain (versus cases without) and associated with patient survival (Fig. </a><span style=" color: #3B699E;">3b,c </span>and Supplementary Table 8). Some of these upregulated genes involved in key signaling pathways (for example, PI3K/AKT/mTOR, mTORC1, MYC) are also poten- tial therapeutic targets (for example, <span class="s23">HN1</span>, <span class="s23">GRB2</span>, <span class="s23">PSMB3</span><a href="#bookmark55" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">), with a number of compounds being screened as active</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">25</span>. However, as the CNV profiles were inferred from scRNA-seq data the analysis was limited due to the low resolution. We performed validation analysis in two independent GAC cohorts: the TCGA primary GAC cohort (<span class="s23">n </span><span class="s12">= </span><a href="#bookmark28" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">411; Fig. </a><span style=" color: #3B699E;">3d</span>) and a cohort of metastatic GAC (<span class="s23">n </span><span class="s12">= </span><a href="#bookmark28" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">45; Fig. </a><span style=" color: #3B699E;">3e</span>). The data from both validation cohorts showed that patients with 17q gain in their tumors had significantly worse survival.</p><p class="s24" style="padding-top: 9pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a name="bookmark6"><span class="h2">Cell signaling heterogeneity correlates with tumor cell lineages/ states. </span></a><span style=" color: #000;">To examine the molecular consequences of transcriptomic and genotypic alterations described above and to better under- stand the biological programs associated with patient survival, we performed pathway enrichment analysis of </span><span class="s12">&gt;</span><a href="#bookmark29" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">900 curated gene sets (see the Methods). Among them, 80 pathways were differentially expressed across the inferred tumor cell lineages (Fig. </a>4a<a href="#bookmark29" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">), and, of these, 37 were also strongly associated with patient survival (Fig. </a>4b <a href="#bookmark29" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and Supplementary Fig. 20). These pathways were categorized into five major classes based on their biological functions: oncogenic signaling, cell cycle, DNA repair, metabolism and immune signal- ing. Pathway interaction analysis revealed that these biological pro- cesses are functionally connected (Fig. </a>4c<span style=" color: #000;">).</span></p><p class="s8" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;">Pathways that were significantly enriched in tumor cells of stomach origin and associated with shorter survival included cell cycle, DNA repair, PI3K/AKT/mTOR, mTORC1, Wnt, NF-<span class="s12">κ</span>B and metabolic reprogramming, which are predominantly oncogenic. The pathways that were enriched in colorectal-like tumor cells and associated with longer survival included defensins, IL-7 signaling,</p><p class="s24" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark29" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">complement cascade, IL6/JAK/STAT3 signaling and interferon alpha/gamma, which are all immune related (Fig. </a>4a,b<span style=" color: #000;">). These results indicated that different biological processes might have been implicated in tumor cells with different lineages or transcriptome states, contributing to their distinct molecular consequences and patient survival.</span></p><p class="s24" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark29" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">To assess whether the cellular composition of the tumor immune microenvironment differed between tumors with gastric-dominant features and those with GI-mixed features, we performed immune deconvolution analysis of the bulk expression data using public datasets (see the Methods). Our results in Fig. </a>4d <span style=" color: #000;">and Supplementary Fig. 21 show that the abundance scores of B cells increased significantly in tumors with GI-mixed features (ver- sus those with gastric-dominant features), and this observation was replicated in three independent cohorts. In addition, the fractions of M1-like macrophages (pro-inflammatory) were higher, and M2-like macrophages (anti-inflammatory) were lower, in tumors with GI-mixed features. There was also a significant difference in the abundance scores of cancer-associated fibroblasts, which were lower in tumors with GI-mixed features. Together, our analysis suggests that tumors with GI-mixed features are immunologically more active.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s24" style="padding-left: 7pt;text-indent: 0pt;line-height: 11pt;text-align: justify;"><a name="bookmark7"><span class="h2">Single-cell analysis of tumor cell lineage compositions classified PC cases into two subtypes with significant survival difference. </span></a><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">Based on tumor cell lineage compositions, we classified PC samples into two main subtypes: gastric-dominant (mainly gastric cell lin- eages) and GI-mixed (with mixed gastric and colorectal-like cells) (Fig. </a>1c<a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">), and performed correlation analysis with the clinical/histo- pathological variables and patient survival (Extended Data Fig. 8 and Supplementary Fig. 22). No significant difference was observed in the histopathological features between these two subtypes. Notably, 17q gain was highly enriched in the gastric-dominant group in the GAC-PC (PC specimens from patients with metastatic GAC) valida- tion cohort (Supplementary Fig. 23); the cell-of-origin-based clas- sification of PC showed a strong correlation with patient survival (Fig. </a>1c<span style=" color: #000;">): all six cases with a GI-mixed phenotype were long-term survivors, whereas six of eight cases with a gastric-dominant phe- notype were short-term survivors (Fisher’s exact test, </span><span class="s23">P </span><span class="s12">= </span><span style=" color: #000;">0.0097, log-rank </span><span class="s23">P </span><span class="s12">= </span><a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">0.05; Fig. </a>5a<span style=" color: #000;">). Currently, a validated and practical molecular signature for PC is lacking. These results suggest that the transcriptomic features of PC tumor cells could prognosticate patient survival.</span></p><h2 style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a name="bookmark8">Generation and validation of a 12-gene prognostic signature. </a><span class="s8">We next sought to generate a gene expression signature that could be practical. We performed single-cell DEG analysis on PC tumor cells between the gastric-dominant and GI-mixed subtypes, followed by filtering the DEGs list to identify the most significant DEGs, screening each of the DEGs based on their statistical correlation</span></h2><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="677" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_121.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark12"><b>Fig. 4 | </b></a>Molecular pathway-based dissection of the transcriptomic iTH and correlation with tumor cell lineage and patient survival. <b>a</b>, The transcriptomic ITH of curated gene sets, including cancer hallmark gene sets (<i>n </i><span class="s26">= </span>50) and gene sets from KEGG (<i>n </i><span class="s26">= </span>186) and reactome (<i>n </i><span class="s26">= </span>674) pathway databases.<a name="bookmark29">&zwnj;</a></p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;">Each column represents a single cell. Only the pathways (rows) that were differentially expressed across different tumor cell lineages are shown. The pathway names are labeled on the right and color coded by their biological functions. <b>b</b>, representative violin plots of six pathways selected from <b>a </b>and Supplementary Fig. 20 that showed significant correlation with patient survival. Number of cells: stomach mucosal cells, <i>n </i><span class="s26">= </span>5,937; stomach pit cells,</p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><i>n </i><span class="s26">= </span>12,341; pancreas ductal cells, <i>n </i><span class="s26">= </span>1,037; gallbladder mucosal cells, <i>n </i><span class="s26">= </span>285; duodenal epithelial cells, <i>n </i><span class="s26">= </span>366; colorectal epithelial cells, <i>n </i><span class="s26">= </span>5,278; long-term survivors, <i>n </i><span class="s26">= </span>18,428; short-term survivors, <i>n </i><span class="s26">= </span>6,816. Box, median <span class="s26">± </span>interquartile range. Whiskers, 1.5 <span class="s26">× </span>interquartile range. <i>P </i>values across different tumor cell lineages were calculated by one-way Kruskal–Wallis rank-sum test. <i>P </i>values between two patient groups were calculated by a</p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;">two-sided Wilcoxon rank-sum test. <i>P </i><span class="s26">&lt; </span>2.2 <span class="s26">× </span>10<span class="s27">−</span><span class="s28">16 </span>represents a <i>P </i>value approaching 0. GSVA, gene set variation analysis. <b>c</b>, The interaction networks of differentially expressed pathways displayed in <b>a</b>. The curated gene sets were colored by their biological functions. The weight of a line corresponds to its Jaccard index (a similarity metric) between each pathway pair connected by the line. <b>d</b>, Violin plots showing the differences in immune cell composition between the gastric-dominant and GI-mixed groups across multiple validation cohorts. The MCP-counter scores for a specific tumor cell lineage or the CIBErSOrT cell fractions, or normalized gene expression levels, are shown on the <i>y </i>axis. The black, bold, horizontal line with a dot indicates the median value of each group. <i>P </i>values were calculated by a two-sided Wilcoxon rank-sum test. mDC, myeloid dendritic cells.</p><p class="s24" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">with patient survival and testing gene combination using a forward selection method (Fig. </a>5b <a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and see the Methods). After a multistep process, a 12-gene signature was derived (Fig. </a>5c<span style=" color: #000;">).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s8" style="padding-left: 7pt;text-indent: 11pt;line-height: 93%;text-align: justify;">We then validated this signature in an independent GAC-PC cohort (<span class="s23">n </span><span class="s12">= </span>45). For each tumor sample, a signature score was com- puted using bulk RNA-sequencing (RNA-seq) data, and based on</p><p style="text-indent: 0pt;text-align: left;"><span><img width="373" height="335" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_122.png"/></span></p><p class="s82" style="padding-top: 6pt;padding-left: 127pt;text-indent: 0pt;line-height: 85%;text-align: center;">Gallbladder mucosal cell       <span class="s68">Duodenal epithelial cell</span></p><h4 style="padding-top: 3pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">a</h4><p class="s69" style="padding-left: 52pt;text-indent: 0pt;text-align: left;">Stomach mucosal cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s69" style="padding-top: 5pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">Stomach pit cell</p><p class="s68" style="padding-top: 3pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">Pancreas ductal cell</p><p class="s68" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">Colorectal epithelial cell</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 62pt;text-indent: 0pt;text-align: left;">Cell lineage</p><p class="s103" style="padding-top: 2pt;padding-left: 69pt;text-indent: 0pt;line-height: 108%;text-align: left;">R_IL_7_SIGNALING <span style=" color: #000;">H_KRAS_SIGNALING_UP </span>H_INFLAMMATORY_RESPONSE R_COMPLEMENT_CASCADE R_DEFENSINS R_INTERFERON_ALPHA_BETA_SIGNALING R_INTERFERON_GAMMA_SIGNALING H_IL6_JAK_STAT3_SIGNALING R_IL_6_SIGNALING</p><p class="s103" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">H_COMPLEMENT</p><p class="s34" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_ADHERENS_JUNCTIONS_INTERACTIONS R_CELL_CELL_COMMUNICATION</p><p class="s34" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">H_APOPTOSIS</p><p class="s104" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_GLOBAL_GENOMIC_NER_GG_NER R_DNA_REPAIR <span style=" color: #000;">R_TELOMERE_MAINTENANCE </span>K_MISMATCH_REPAIR</p><p class="s34" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_EXTENSION_OF_TELOMERES <span style=" color: #E6412E;">R_DOUBLE_STRAND_BREAK_REPAIR R_HR_REPAIR_REPLICATION_INDEPENDENT_DSB </span>R_ACTIVATION_PRE_REPLICATIVE_COMPLEX R_ACTIVATION_OF_ATR_UNDER_REPLICATION_STRESS <span style=" color: #5DAFD9;">R_G2_M_CHECKPOINTS</span></p><p class="s105" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;line-height: 142%;text-align: left;">R_E2F_REGULATION_OF_DNA_REPLICATION <span style=" color: #000;">R_FANCONI_ANEMIA_PATHWAY</span></p><p class="s105" style="padding-left: 69pt;text-indent: 0pt;line-height: 131%;text-align: left;">R_G2_M_DNA_DAMAGE_CHECKPOINT <span style=" color: #000;">K_OLFACTORY_TRANSDUCTION</span></p><p class="s106" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;line-height: 109%;text-align: left;">K_PYRUVATE_METABOLISM K_PURINE_METABOLISM <span style=" color: #E6412E;">H_DNA_REPAIR </span>H_OXIDATIVE_PHOSPHORYLATION H_CHOLESTEROL_HOMEOSTASIS <span style=" color: #000;">H_PI3K_AKT_MTOR_SIGNALING </span>K_UBIQUITIN_MEDIATED_PROTEOLYSIS</p><p class="s34" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_APOPTOSIS <span style=" color: #19AA3C;">R_DOWNSTREAM_SIGNALING_EVENTS_OF_BCR R_SIGNALING_BY_THE_B_CELL_RECEPTOR_BCR</span></p><p class="s34" style="padding-top: 1pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">R_P53_DEPENDENT_G1_DNA_DAMAGE_RESPONSE</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 74pt;text-indent: 0pt;line-height: 132%;text-align: center;">GSVA score 4</p><p style="text-indent: 0pt;text-align: left;"><span><img width="11" height="49" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_123.jpg"/></span></p><p class="s68" style="padding-top: 1pt;text-indent: 0pt;text-align: center;">2</p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: center;">0</p><p class="s68" style="padding-top: 2pt;padding-left: 75pt;text-indent: 0pt;text-align: center;">–2</p><p class="s68" style="padding-top: 2pt;padding-left: 75pt;text-indent: 0pt;text-align: center;">–4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="103" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_124.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="101" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_125.png"/></span></p><p class="s67" style="padding-top: 9pt;padding-left: 21pt;text-indent: 0pt;line-height: 9pt;text-align: left;">b <span class="s107">R_CELL_CYCLE_CHECKPOINTS </span><span class="s68">H_MTORC1_SIGNALING</span></p><p class="s34" style="padding-top: 2pt;padding-left: 40pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_P53_INDEPENDENT_G1/S_DNA_DAMAGE_CHECKPOINT R_SIGNALING_BY_WNT <span style=" color: #19AA3C;">R_ACTIVATION_OF_NF_KAPPAB_IN_B_CELLS</span></p><p class="s103" style="padding-top: 2pt;padding-left: 3pt;text-indent: 0pt;line-height: 155%;text-align: left;">Immune signaling <span style=" color: #000;">Oncogenic signaling </span><span style=" color: #E6412E;">DNA repair </span><span style=" color: #FDB53C;">Metabolism</span></p><p class="s105" style="padding-left: 3pt;text-indent: 0pt;line-height: 3pt;text-align: left;">Cell cycle</p><p class="s108" style="padding-top: 3pt;padding-left: 3pt;text-indent: 0pt;line-height: 151%;text-align: left;">H: Hallmark R: Reactome K: KEGG</p><p style="text-indent: 0pt;text-align: left;"/><p class="s34" style="padding-top: 3pt;padding-left: 40pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_REGULATION_OF_APOPTOSIS H_MTORC1_SIGNALING <span style=" color: #5DAFD9;">R_CELL_CYCLE_CHECKPOINTS R_M_G1_TRANSITION R_G1_S_TRANSITION</span></p><p class="s106" style="padding-top: 2pt;padding-left: 40pt;text-indent: 0pt;line-height: 109%;text-align: left;">K_INSULIN_SIGNALING_PATHWAY <span style=" color: #000;">K_P53_SIGNALING_PATHWAY</span></p><p class="s105" style="padding-top: 3pt;padding-left: 40pt;text-indent: 0pt;line-height: 109%;text-align: left;">R_MITOTIC_G2_G2_M_PHASES H_E2F_TARGETS H_G2M_CHECKPOINT K_CELL_CYCLE R_CELL_CYCLE_MITOTIC</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="102" height="67" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_126.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="31" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_127.png"/></span></p><p class="s109" style="padding-top: 4pt;padding-left: 21pt;text-indent: 0pt;line-height: 4pt;text-align: left;">R_DEFENSINS</p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">1.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">GSVA score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;text-align: right;">1.2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.8</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s110" style="padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s111">&lt; 2.2 × 10</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s112" style="text-indent: 0pt;text-align: left;">–16</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="31" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_128.png"/></span></p><p class="s110" style="padding-left: 19pt;text-indent: 0pt;text-align: left;">P <span class="s111">&lt; 2.2 × 10</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 11pt;text-indent: 0pt;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 11pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">−2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s112" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">–16</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">0.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">0.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">0.4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">0.3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s113" style="padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s114">&lt; 2.2 × 10</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s115" style="padding-top: 1pt;text-indent: 0pt;text-align: left;">–16</p><p class="s113" style="padding-top: 2pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">P <span class="s114">&lt; 2.2 × 10</span><span class="s116">–16</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="31" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_129.png"/></span></p><p class="s68" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">−2</p><p class="s68" style="padding-top: 3pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">0.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">0.3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">−0.3</p><p class="s117" style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s118">&lt; 2.2 × 10</span><span class="s119">–16</span></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;line-height: 4pt;text-align: left;">5.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">2.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">−2.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">−5.0</p><p class="s117" style="padding-top: 2pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">P <span class="s118">&lt; 2.2 × 10</span><span class="s119">–16</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_130.png"/></span></p><p class="s68" style="padding-left: 11pt;text-indent: 0pt;line-height: 121%;text-align: left;">Stomach mucosal cell Stomach pit cell</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_131.png"/></span></p><p class="s68" style="padding-top: 3pt;padding-left: 11pt;text-indent: 0pt;line-height: 119%;text-align: left;">Pancreas ductal cell Gallbladder mucosal cell</p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="13" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_132.png"/></span></p><p class="s68" style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;line-height: 107%;text-align: left;">Duodenal epithelial cell Colorectal epithelial cell</p><p style="text-indent: 0pt;text-align: left;"><span><img width="103" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_133.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="102" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_134.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="101" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_135.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="31" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_136.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="31" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_137.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="32" height="68" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_138.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="7" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_139.png"/></span></p><p class="s120" style="padding-top: 5pt;padding-left: 56pt;text-indent: 0pt;line-height: 3pt;text-align: left;">R_DNA_REPAIR <span style=" color: #000;">H_PI3K_AKT_MTOR_SIGNALING </span><span style=" color: #19AA3C;">H_IL6_JAK_STAT3_SIGNALING</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">GSVA score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;text-align: right;">1.6</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">1.2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.8</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.4</p><p class="s121" style="padding-top: 3pt;padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s122">&lt; 2.2 × 10</span><span class="s123">–16</span></p><p class="s76" style="padding-top: 1pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">4 <span class="s121">P </span><span class="s122">&lt; 2.2 × 10</span><span class="s123">–16</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 2pt;text-indent: 0pt;text-align: left;">−2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">0.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">0.4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">0.3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">0.2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s113" style="padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s114">&lt; 2.2 × 10</span></p><p class="s115" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">–16 <span class="s68">4</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">−2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">−4</p><p class="s113" style="padding-top: 3pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">P <span class="s114">&lt; 2.2 × 10</span><span class="s116">–16</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0.3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0.2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0.1</p><p class="s117" style="padding-top: 4pt;padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s118">&lt; 2.2 × 10</span><span class="s119">–16</span></p><p class="s68" style="padding-top: 1pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">5.0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">2.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">−2.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">−2</p><p class="s117" style="padding-top: 3pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">P <span class="s118">&lt; 2.2 × 10</span><span class="s119">–16</span></p><p class="s68" style="padding-top: 2pt;text-indent: 0pt;line-height: 121%;text-align: left;">Long-term survivor Short-term survivor</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="417" height="180" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_140.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="23" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_141.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="25" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_142.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="79" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_143.png"/></span></p><p class="s124" style="text-indent: 0pt;text-align: left;">R_MITOTIC_G2_G2_M_PHASES</p><p style="text-indent: 0pt;text-align: left;"/><h4 style="padding-left: 21pt;text-indent: 0pt;line-height: 9pt;text-align: left;">c <span class="s76">GAC-PC (</span><span class="s72">n </span><span class="s68">= 45, MDACC)</span></h4><p style="text-indent: 0pt;text-align: left;"><span><img width="53" height="80" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_144.png"/></span></p><p class="s125" style="text-indent: 0pt;text-align: left;">DNA repair</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s126" style="text-indent: 0pt;text-align: left;">Metabolism</p><p style="text-indent: 0pt;text-align: left;"/><p class="s127" style="text-indent: 0pt;text-align: left;">Jaccard index</p><p class="s128" style="padding-top: 3pt;text-indent: 0pt;text-align: left;">0.01 0.125 0.25 0.5 0.85</p><p style="text-indent: 0pt;text-align: left;"/><p class="s124" style="text-indent: 0pt;text-align: left;">H_G2M_CHECKPOINT</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;text-align: right;">B cell</p><p class="s124" style="text-indent: 0pt;text-align: left;">R_G2_M_DNA_DAMAGE_CHECKPOINT</p><p class="s124" style="text-indent: 0pt;text-align: left;">R_E2F_REGULATION_OF_DNA_REPLICATION</p><p class="s129" style="text-indent: 0pt;text-align: left;">Immune signal</p><p class="s60" style="padding-left: 118pt;text-indent: 0pt;line-height: 8pt;text-align: left;">		</p><p class="s130" style="text-indent: 0pt;text-align: left;">Cell cycle</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s131" style="text-indent: 0pt;text-align: left;">Oncogenic signal</p><p style="text-indent: 0pt;text-align: left;"/><p class="s124" style="text-indent: 0pt;text-align: left;">R_M_G1_TRANSITION</p><p style="text-indent: 0pt;text-align: left;"/><p class="s124" style="text-indent: 0pt;text-align: left;">H_E2F_TARGETS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">MCP-counter score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 1pt;padding-bottom: 1pt;text-indent: 0pt;text-align: right;">2</p><p class="s124" style="text-indent: 0pt;text-align: left;">R_CELL_CYCLE_CHECKPOINTS</p><p style="padding-left: 122pt;text-indent: 0pt;line-height: 4pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s132" style="text-indent: 0pt;text-align: left;">R_IL_7_SIGNALING</p><p style="padding-left: 80pt;text-indent: 0pt;line-height: 4pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">R_SIGNALING_BY_THE_B_CELL_RECEPTOR_BCR <span style=" color: #5DAFD9;">K_CELL_CYCLE</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s124" style="text-indent: 0pt;text-align: left;">R_G2_M_CHECKPOINTS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s124" style="text-indent: 0pt;text-align: left;">R_CELL_CYCLE_MITOTIC</p><p style="text-indent: 0pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">H_COMPLEMENT</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">1</p><p class="s124" style="text-indent: 0pt;text-align: left;">R_G1_S_TRANSITION</p><p style="text-indent: 0pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">R_COMPLEMENT_CASCADE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s125" style="text-indent: 0pt;text-align: left;">R_DOUBLE_STRAND_BREAK_REPAIR</p><p style="text-indent: 0pt;text-align: left;"/><p class="s133" style="padding-top: 1pt;text-indent: 0pt;line-height: 3pt;text-align: right;">   </p><p class="s73" style="padding-left: 13pt;text-indent: 0pt;text-align: left;">4 <span class="s68">mDC</span></p><p class="s68" style="padding-top: 6pt;padding-left: 13pt;text-indent: 0pt;text-align: left;">3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 13pt;text-indent: 0pt;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 13pt;text-indent: 0pt;line-height: 3pt;text-align: left;">1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s125" style="text-indent: 0pt;text-align: left;">K_MISMATCH_REPAIR</p><p style="padding-left: 229pt;text-indent: 0pt;line-height: 4pt;text-align: left;"/><p class="s34" style="text-indent: 0pt;line-height: 52%;text-align: right;">17 <span class="s35">28</span></p><p class="s125" style="text-indent: 0pt;text-align: right;">R_DNA_REPAIR</p><p class="s131" style="padding-top: 2pt;text-indent: 0pt;text-align: left;">R_ACTIVATION_OF_ATR_UNDER_REPLICATION_STRESS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">H_INFLAMMATORY_RESPONSE</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s132" style="padding-left: 33pt;text-indent: 0pt;text-align: left;">R_DOWNSTREAM_SIGNALING_EVENTS_OF_BCR</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: right;">0</p><p class="s125" style="text-indent: 0pt;text-align: left;">R_HR_REPAIR_REPLICATION_INDEPENDENT_DSB</p><p style="padding-left: 274pt;text-indent: 0pt;line-height: 4pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">R_REGULATION_OF_APOPTOSIS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s72" style="text-indent: 0pt;text-align: right;">P <span class="s68">= 0.059</span></p><p class="s34" style="padding-top: 1pt;padding-left: 34pt;text-indent: 0pt;line-height: 3pt;text-align: left;">17</p><p class="s34" style="padding-left: 40pt;text-indent: 0pt;line-height: 3pt;text-align: left;">28</p><p class="s68" style="padding-top: 1pt;padding-left: 13pt;text-indent: 0pt;text-align: left;">0</p><p class="s72" style="padding-left: 21pt;text-indent: 0pt;text-align: left;">P <span class="s68">= 0.028</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s125" style="text-indent: 0pt;text-align: left;">R_GLOBAL_GENOMIC_NER_GG_NER</p><p style="padding-left: 258pt;text-indent: 0pt;line-height: 4pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">R_INTERFERON_GAMMA_SIGNALING</p><p class="s132" style="text-indent: 0pt;text-align: left;">R_DEFENSINS</p><p class="s131" style="text-indent: 0pt;text-align: left;">R_P53_INDEPENDENT_G1/S_DNA_DAMAGE_CHECKPOINT</p><p class="s125" style="text-indent: 0pt;text-align: left;">H_DNA_REPAIR</p><p class="s60" style="padding-left: 33pt;text-indent: 0pt;line-height: 7pt;text-align: left;"><span style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 3.5pt;"> </span>			<span><img width="34" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_145.png"/></span></p><p class="s126" style="text-indent: 0pt;text-align: left;">K_PURINE_METABOLISM</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s126" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">K_UBIQUITIN_MEDIATED_PROTEOLYSIS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">R_FANCONI_ANEMIA_PATHWAY</p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">R_P53_DEPENDENT_G1_DNA_DAMAGE_RESPONSE</p><p style="text-indent: 0pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">R_ACTIVATION_OF_NF_KAPPAB_IN_B_CELLS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">Ooi et al.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: left;">Primary GAC</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="35" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_146.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="81" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_147.png"/></span></p><p class="s68" style="padding-top: 1pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">Cristescu et al.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="58" height="81" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_148.png"/></span></p><p class="s131" style="text-indent: 0pt;text-align: left;">R_CELL_CELL_COMMUNICATION <span class="s134">R_TELOMERE_MAINTENANCE</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s131" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">H_MTORC1_SIGNALING</p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">R_APOPTOSIS</p><p class="s131" style="padding-top: 2pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">H_KRAS_SIGNALING_UP</p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">K_P53_SIGNALING_PATHWAY</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s129" style="text-indent: 0pt;text-align: left;">R_INTERFERON_ALPHA_BETA_SIGNALING</p><p style="text-indent: 0pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">R_IL_6_SIGNALING</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">Neutrophils</p><p class="s126" style="text-indent: 0pt;text-align: left;">IVE_PHOSPHORYLATION</p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">R_EXTENSION_OF_TELOMERES <span style=" color: #FDB53C;">H_OXIDAT</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">H_PI3K_AKT_MTOR_SIGNALING</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s131" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">K_OLFACTORY_TRANSDUCTION</p><p style="text-indent: 0pt;text-align: left;"/><p class="s132" style="text-indent: 0pt;text-align: left;">H_IL6_JAK_STAT3_SIGNALING</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">MCP-counter score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 5pt;text-indent: 0pt;text-align: right;">8</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">MCP-counter score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;text-align: right;">1.6</p><p class="s68" style="padding-top: 3pt;padding-left: 21pt;text-indent: 0pt;text-align: left;">B cell</p><p class="s131" style="text-indent: 0pt;text-align: left;">R_ACTIVATION_PRE_REPLICATIVE_COMPLEX</p><p style="text-indent: 0pt;text-align: left;"/><p class="s126" style="text-indent: 10pt;line-height: 161%;text-align: left;">K_INSULIN_SIGNALING_PATHWAY K_PYRUVATE_METABOLISM</p><p style="text-indent: 0pt;text-align: left;"/><p class="s131" style="text-indent: 0pt;text-align: left;">R_SIGNALING_BY_WNT</p><p style="text-indent: 0pt;text-align: left;"/><p class="s97" style="padding-top: 7pt;padding-left: 360pt;text-indent: 0pt;text-align: left;">7 <span class="s68">1.4</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s131" style="text-indent: 0pt;text-align: left;">R_ADHERENS_JUNCTIONS_INTERACTIONS</p><p class="s131" style="text-indent: 0pt;text-align: left;">H_APOPTOSIS</p><p class="s60" style="padding-left: 120pt;text-indent: 0pt;line-height: 4pt;text-align: left;">	</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s39" style="text-indent: 0pt;text-align: right;"><span><img width="9" height="8" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_149.png"/></span> <span class="s135">GI-mixed</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s39" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><span><img width="8" height="8" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_150.png"/></span> <span class="s136">Gastric-dominant</span></p><p class="s80" style="padding-top: 4pt;padding-left: 57pt;text-indent: 0pt;text-align: left;">6 <span class="s34">107 </span><span class="s35">85</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s126" style="text-indent: 0pt;text-align: left;">H_CHOLESTEROL_HOMEOSTASIS</p><p style="text-indent: 0pt;text-align: left;"/><p class="s96" style="padding-left: 57pt;text-indent: 0pt;text-align: left;">5 <i>P </i><span class="s68">= 0.016</span></p><p class="s80" style="padding-top: 7pt;padding-left: 116pt;text-indent: 0pt;text-align: center;">1.2  <u>         </u><span class="s137">126 </span><span class="s37">174                </span></p><p class="s72" style="padding-top: 6pt;padding-left: 113pt;text-indent: 0pt;text-align: center;">P <span class="s68">= 0.061</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="76" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_151.png"/></span></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">CIBERSORT cell fraction</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 5pt;padding-left: 21pt;text-indent: 0pt;text-align: left;"><span class="s49">d </span>Primary GAC (<i>n </i>= 415, TCGA)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="62" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_152.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="87" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_153.png"/></span></p><p class="s68" style="padding-left: 72pt;text-indent: 0pt;text-align: left;">Primary GAC (<i>n </i>= 433, Cheong et al.)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="79" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_154.png"/></span></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">MCP-counter score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 3pt;padding-left: 27pt;text-indent: 16pt;line-height: 136%;text-align: left;">Fibroblasts <span class="s76">16 </span><i>P </i>= 3.8 × 10–7</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">14</p><p class="s68" style="padding-top: 3pt;padding-left: 22pt;text-indent: 0pt;line-height: 5pt;text-align: left;">Neutrophils</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="83" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_155.png"/></span></p><p class="s97" style="padding-left: 6pt;text-indent: 0pt;line-height: 8pt;text-align: left;">10 <i>P </i><span class="s68">= 8.8 × 10–4</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">8</p><p class="s68" style="padding-top: 3pt;padding-left: 29pt;text-indent: 0pt;text-align: left;">Macrophages M2</p><p style="text-indent: 0pt;text-align: left;"><span><img width="54" height="82" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_156.png"/></span></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">CIBERSORT cell fraction</p><p style="text-indent: 0pt;text-align: left;"/><p class="s72" style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;">P <span class="s68">= 4.7 × 10–4</span></p><p class="s68" style="padding-top: 2pt;padding-left: 17pt;text-indent: 0pt;text-align: left;">0.4   <span class="s138">                                </span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 17pt;text-indent: 0pt;text-align: left;">0.3   <span class="s138">                                </span></p><p class="s72" style="padding-top: 3pt;padding-left: 32pt;text-indent: 0pt;text-align: left;">PRF1</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="80" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_157.png"/></span></p><p class="s76" style="padding-top: 1pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">12.5 <i>P </i><span class="s68">= 0.052</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">10.0    <span class="s139">                               </span></p><p class="s68" style="padding-top: 3pt;padding-left: 9pt;text-indent: 16pt;line-height: 122%;text-align: left;">Fibroblasts<span class="s97"> 14 </span><i>P </i>= 2.6 × 10–9</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="81" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_158.png"/></span></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">MCP-counter score</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 6pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">12                    <u>   </u></p><p class="s68" style="padding-top: 3pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">B cell</p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="78" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_159.png"/></span></p><p class="s72" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">P <span class="s68">= 0.032</span></p><p class="s68" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">10</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.2</p><p class="s68" style="padding-top: 3pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Macrophages M1</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="83" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_160.png"/></span></p><p class="s72" style="padding-top: 4pt;padding-left: 3pt;text-indent: 0pt;text-align: left;">P <span class="s68">= 0.020</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Expression</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="padding-top: 3pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">12.5</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 7pt;text-indent: 0pt;line-height: 5pt;text-align: left;">10.0</p><p class="s72" style="padding-top: 3pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">GZMB</p><p style="text-indent: 0pt;text-align: left;"><span><img width="51" height="85" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_161.png"/></span></p><p class="s72" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">P <span class="s68">= 1.2 × 10–4</span></p><p class="s72" style="padding-top: 3pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">IFNG</p><p style="text-indent: 0pt;text-align: left;"><span><img width="44" height="81" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_162.png"/></span></p><p class="s68" style="padding-top: 2pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">12 <i>P </i>= 1.1 × 10–4</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="padding-left: 4pt;text-indent: 0pt;text-align: left;">10</p><p class="s68" style="padding-top: 2pt;padding-left: 27pt;text-indent: 0pt;text-align: left;">12</p><p class="s34" style="padding-left: 48pt;text-indent: 0pt;line-height: 3pt;text-align: left;">172 <span class="s140">243 </span><span class="s76">6</span><span class="s68">                  </span><span class="s133">   </span></p><p class="s138" style="padding-left: 35pt;text-indent: 0pt;text-align: left;">                                </p><p class="s68" style="padding-top: 1pt;padding-left: 24pt;text-indent: 0pt;line-height: 5pt;text-align: left;">0.2                    <span class="s138">    </span></p><p class="s68" style="padding-left: 9pt;text-indent: 0pt;line-height: 4pt;text-align: left;">7.5    <span class="s138">                               </span></p><p class="s68" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Expression</p><p style="text-indent: 0pt;text-align: left;"/><p class="s68" style="text-indent: 0pt;line-height: 4pt;text-align: right;">10</p><p class="s34" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">172  <span class="s38">243</span></p><p class="s34" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;line-height: 3pt;text-align: center;">264</p><p class="s34" style="padding-left: 16pt;text-indent: 0pt;line-height: 5pt;text-align: left;">169 <span class="s76">8</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">0.1</p><p class="s138" style="padding-top: 2pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">   <span class="s139">               </span><span class="s68">8</span></p><p class="s34" style="padding-top: 3pt;padding-left: 27pt;text-indent: 0pt;line-height: 0pt;text-align: left;">169</p><p class="s137" style="padding-top: 3pt;padding-left: 95pt;text-indent: 0pt;line-height: 1pt;text-align: left;">172 <span class="s34">243                                          </span><u>                                            </u></p><p class="s68" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;line-height: 2pt;text-align: left;">5.0    <span class="s138">                               </span></p><p class="s138" style="text-indent: 0pt;line-height: 4pt;text-align: right;">   <span class="s139">                           </span><span class="s68">7.5</span></p><p class="s34" style="padding-top: 1pt;text-indent: 0pt;line-height: 3pt;text-align: right;">264</p><p class="s34" style="padding-top: 4pt;padding-left: 25pt;text-indent: 0pt;line-height: 1pt;text-align: left;">169</p><p class="s97" style="padding-left: 27pt;text-indent: 0pt;line-height: 8pt;text-align: left;">10<span class="s68"> 0.1</span></p><p class="s141" style="padding-top: 3pt;padding-left: 30pt;text-indent: 0pt;text-align: left;">8 <span class="s68">4 </span><span class="s73">0</span></p><p class="s34" style="padding-top: 3pt;padding-left: 24pt;text-indent: 0pt;text-align: left;">243 <span class="s141">8</span></p><p class="s37" style="text-indent: 0pt;line-height: 3pt;text-align: center;">                172                     <span class="s34">                           </span>                                          </p><p class="s68" style="padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">2.5</p><p class="s80" style="padding-top: 3pt;padding-left: 27pt;text-indent: 0pt;text-align: left;">6 <span class="s34">169 264</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s34" style="text-indent: 0pt;text-align: right;">169 <span class="s36">264</span></p><p class="s68" style="padding-top: 1pt;text-indent: 0pt;text-align: center;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s68" style="text-indent: 0pt;text-align: right;">5.0</p><p class="s68" style="padding-left: 27pt;text-indent: 0pt;text-align: left;">6 <span class="s34">264</span></p><p class="s8" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">which the sample was categorized into either the gastric-dominant or the GI-mixed group for subsequent analysis (Supplementary Fig. 24 and see the Methods). The signature demonstrated an excellent power to prognosticate patient survival, and consistently, patients whose PCs were in the gastric-dominant group survived signifi- cantly shorter (7.8 versus 24.5 months) than those whose PCs were in the GI-mixed group (Fig. </a><span style=" color: #3B699E;">5d</span>, left). Multivariate Cox regression analysis showed that this signature was a strong prognosticator of short survival, with a hazard ratio of 12.7 (95% confidence interval, 3.2–51.0, <span class="s23">P </span><span class="s12">= </span>3.3 <span class="s12">× </span>10<span class="s50">−</span><span class="s10">4</span><a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">; Fig. </a><span style=" color: #3B699E;">5d</span>, right), and it was independent of clinical/histopathological variables (Supplementary Fig. 22).</p><p class="s13" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark49" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">We also evaluated its prognostic significance in four other large-scale localized GAC cohorts</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">19</span><a href="#bookmark56" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>26<a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">–</a>28<a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, totaling 1,336 patients. Notably, although this signature was derived from an advanced GAC cohort, it retained its prognostic significance in all four val- idation cohorts of localized GACs (Fig. </a><span class="s24">5e–h </span><a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and Extended Data Fig. 8a). Intriguingly, this signature was independent of other molecular and clinical subtypes (Extended Data Fig. 8b). The mul- tivariate Cox proportional-hazards model analysis revealed that its prognostic value was preserved after accounting for the previously defined molecular subtypes, including MSI and EMT (epithelial– mesenchymal transition) signatures by Cristescu et al.</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">28</span><a href="#bookmark57" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">; metabolic, proliferative and invasive signatures by Ooi et al.</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">27</span><a href="#bookmark38" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">; and other clini- cal and histopathological variables including age, sex and histol- ogy types (Extended Data Figs. 9 and 10). In addition, it correlated strongly with the risk of local recurrence/distant metastasis among the TCGA</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">8</span> <a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and Cristescu cohorts</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">28</span><a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, where both the expression and outcome data were available (Fig. </a><span class="s24">5e</span><span class="s8">, Extended Data Fig. 8c and Supplementary Fig. 25). These results further highlighted the value of this prognostic signature and its robustness in prognosticating patient survival.</span></p><p class="s21" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;line-height: 12pt;text-align: left;"><a name="bookmark13">Discussion</a></p><p class="s13" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark59" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">The progress against GAC has lagged behind other GI tumor types. Therapy resistance and the lack of rational therapeutic targets rep- resent the major obstacles in improving survival of patients with advanced GAC</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">29</span><a href="#bookmark60" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. It is widely appreciated that ITH is a fundamental property of cancer contributing to therapeutic failure, development of distant metastases</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">30</span> <a href="#bookmark61" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and hindrance to biomarker/target discover- ies</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">31</span><a href="#bookmark39" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. Studies of localized and advanced GACs identified multiple molecular subtypes and revealed a high degree of ITH, which is associated with poor clinical outcomes</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">9</span><a href="#bookmark62" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>32<a href="#bookmark63" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>33<span class="s8">. Therefore, deeper dis- section of ITH is critical for understanding the mechanisms driving poor prognosis of GAC and for overcoming therapeutic resistance. In this study, we dissected, at single-cell resolution, the cellular and transcriptomic ITH of PC tumor cells using the cutting-edge scRNA-seq technology, in combination with integrative computa- tional analyses.</span></p><p class="s13" style="padding-top: 10pt;padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark64" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">A key finding of this study is that the diversity in tumor cell lin- eage/state compositions appears to mirror and may even dictate the inherent ITH of PC tumor cells at multiple levels. The origins of ITH have been a subject of discussion, with multiple models being proposed</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">34</span><a href="#bookmark65" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>35<span class="s8">. The peritoneal cavity is a unique microenvironment where tumor cells can be in suspension in the peritoneal fluid as opposed to being localized in solid tumor tissues, and thus the PC cells we have sequenced may be a better representation of ITH. We observed that more than one transcriptomically distinct tumor cell subpopulation co-existed in most of the PC cases analyzed and could be distinguished by the inferred cell lineage characteristics. We discovered that 6 of 14 (43%) cases in our discovery cohort had a considerable fraction (~26%) of PC tumor cells that transcrip- tomically resembled cells of nonstomach GI lineages, particular the intestine. Notably, ITH defined by single-cell lineage/state compo- sitions is perpetuated at transcriptomic, genotypic, cell-cycle state, molecular signaling and phenotypic levels and strongly associated with patient survival. Tumor cell transcriptomic profiles and prolif- erative property also significantly differed across the inferred tumor cell lineages/states, as did the molecular signaling, suggesting that treatment strategies could potentially be tailored to these molecular features. It appears that the contributors to ITH are likely diverse and more complicated than original thought, and varied biologi- cal programs (for example, genomic/epigenomic) might have been engaged early in tumorigenesis. It is important to note that the dif- ferent tumor cell lineages were inferred by mapping scRNA-seq data to HCL and thus may only reflect changes at transcriptome level, instead of developmental lineages. Although we are unable to discern the precise cells of origin of each PC, we believe that the insights shared by this report will stimulate further studies in the field focusing on tumor cell of origin and lineage diversity/plasticity analyses of both gastric and other cancer types, to better elucidate the regulatory mechanisms, possible effects on tumor progression and therapy responses.</span></p><p class="s8" style="padding-top: 1pt;padding-left: 7pt;text-indent: 11pt;text-align: justify;">It is noteworthy that we discovered 17q gain as a nearly exclu- sive event associated with PC cells of gastric lineage. The 17q region harbors multiple potential therapeutic targets and, interestingly, all patients whose tumors had 17q gain in our discovery cohort were short-term survivors, and the association of 17q gain and inferior survival was validated in both the localized and advanced GAC cohorts. Our discovery of the intimate link between tumor cell lin- eage compositions and genomic ITH at single-cell resolution could be generalized to other cancer types and broaden our understand- ing of cancer biology in general.</p><p class="s8" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;">High genomic ITH in most cancers is associated with worse sur- vival. We observed an opposite phenomenon in this study: patients with the GI-mixed molecular features in their PC tumor cells survived significantly longer than those with the gastric-dominant</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span><img width="677" height="6" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_163.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark14"><b>Fig. 5 | </b></a>identification and validation of the 12-gene prognostic signature. <b>a</b>, Survival analysis of the discovery GAC-PC cohort. Left, histogram showing relative proportions of long- and short-term survivors between the gastric-dominant and GI-mixed groups. <i>P </i>value was calculated by the two-tailed Fisher’s exact test. right, Kaplan–Meier plots showing the survival time (in months) since PC diagnosis and survival time since ascites collection, respectively, between patients in gastric-dominant and GI-mixed groups. <i>P </i>values were calculated by two-sided log-rank tests. <b>b</b>, A schema that illustrates the bioinformatics flow for generation of the 12-gene signature (see details in the Methods). <b>c</b>, Differential expression of the 12 signature genes between the gastric-dominant and GI-mixed groups. <b>d</b>, Survival analysis of a second independent cohort of GAC-PC patients (<i>n </i><span class="s26">= </span>45). Left, the Kaplan–Meier curves showing significant differences in patient survival from PC diagnosis between the two PC subtypes (the colors are the same as in panels <b>a</b>–<b>c</b>). right, multivariate Cox proportional regression outcomes, with the 12-gene signature included. For each variable, the reference level is the first one. The block<a name="bookmark30">&zwnj;</a></p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 108%;text-align: left;">in the center of the error bars represents the weighted mean. Whiskers of error bars represent the 95% confidence intervals. Patients whose PC belongs to the gastric-dominant subtype as defined by the 12-gene signature are significantly associated (<i>P </i><span class="s26">= </span>3.31 <span class="s26">× </span>10<span class="s27">−</span><span class="s28">4</span>) with worse survival in this multivariate model. CI, confidence interval. <b>e</b>–<b>h</b>, Survival analysis of the 12-gene signature across three additional large-scale validation cohorts of localized GACs. For each cohort, the source of the dataset, the sample size, the log-rank <i>P </i>value and the median survival time (in months) are labeled on the Kaplan–Meier plot. <b>e</b><a href="#bookmark58" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">, The localized GAC cohort from Cristescu and colleagues</a><span style=" color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 2pt;">28</span>. The alluvial plots (right) show the relationships between PC subtypes (left strip) and the presence of local recurrence and/or distant metastases (right strip). The yellow band highlights the significant enrichment of local recurrence and/or</p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 108%;text-align: left;">distant metastases events in patients whose PCs belong to the gastric-dominant subtype. The <i>P </i><a href="#bookmark57" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">values for the alluvial plots were calculated by a two-sided Fisher’s exact test. f, The GAC cohort from Cheong and colleagues. g, The GAC cohort from Ooi and colleagues</a><span style=" color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4.5pt; vertical-align: 2pt;">27</span>. h, The GAC cohort from TCGA.</p><p class="s8" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;">features. We remain uncertain of the detailed mechanisms. A possi- ble explanation is that the intestinal-like cells in the GI-mixed tumors could be acquired from a process called intestinal metaplasia, which is the main precancerous lesion of the stomach. Intestinal metapla- sia is characterized by the presence of differentiated epithelium that resembles the small intestine (partial or complete transformation</p><p class="s8" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark66" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">of gastric gland epithelial cells into the intestinal type) on the basis of ultrastructural morphology, mucin patterns and enzyme histo- chemistry</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">36</span>. Consistent with this, two patients (IP-010 and IP-158) in this study had intestinal metaplasia confirmed in their primary tumors (diffuse type), although intestinal metaplasia is thought to be mainly associated with GAC of the intestinal type. In line with</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 142pt;text-indent: 0pt;line-height: 4pt;text-align: left;">GAC-PC (<i>n </i>= 14, MDACC, discovery cohort)</p><h4 style="padding-left: 49pt;text-indent: 0pt;line-height: 7pt;text-align: left;">a</h4><h4 style="padding-top: 5pt;text-indent: 0pt;text-align: right;">f</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="565" height="793" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_164.png"/></span></p><p class="s30" style="text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 3pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;padding-left: 20pt;text-indent: 2pt;line-height: 111%;text-align: left;">Primary GAC (<i>n </i>= 433) Cheong et al., GSE84437</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">OS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Proportion</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: right;">0.50</p><p class="s29" style="padding-top: 3pt;padding-left: 10pt;text-indent: 0pt;text-align: left;">P <span class="s30">= 0.0097</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 14pt;text-indent: 0pt;line-height: 163%;text-align: left;">Long-term survivor Short-term survivor</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;line-height: 2pt;text-align: right;">0</p><p class="s30" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">From peritoneal metastasis</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 5pt;text-indent: 0pt;line-height: 4pt;text-align: left;">P <span class="s30">= 0.054</span></p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">From ascites collection</p><p class="s143" style="padding-left: 14pt;text-indent: 0pt;text-align: left;">+ <span style=" color: #2F7AB5;">+</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s143" style="padding-left: 4pt;text-indent: 0pt;text-align: center;">+</p><p class="s144" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">+</p><p class="s29" style="padding-top: 5pt;padding-left: 14pt;text-indent: 0pt;line-height: 4pt;text-align: left;">P <span class="s30">= 0.034</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 9pt;text-indent: 0pt;text-align: left;">P <span class="s30">= 0.0042</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 12pt;text-indent: 0pt;text-align: left;">Median survival</p><p class="s144" style="padding-top: 1pt;padding-left: 26pt;text-indent: 0pt;text-align: left;">Not reached</p><p class="s145" style="padding-top: 1pt;padding-left: 26pt;text-indent: 0pt;text-align: left;">87.0 mo</p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;line-height: 1pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">Number at risk</p><p class="s146" style="padding-top: 1pt;padding-left: 56pt;text-indent: 0pt;line-height: 6pt;text-align: left;">GI-mixed <span class="s144">6 6 3 0 6 4 3 3 2</span></p><p class="s30" style="padding-top: 1pt;padding-left: 35pt;text-indent: 0pt;text-align: left;">Number at risk</p><p class="s144" style="padding-top: 3pt;padding-left: 45pt;text-indent: 0pt;text-align: left;">169 114 82 37 1</p><p class="s145" style="padding-top: 2pt;padding-left: 46pt;text-indent: 0pt;line-height: 1pt;text-align: left;">264 157 103 56 0</p><p class="s143" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">Gastric-dominant</p><p class="s143" style="padding-left: 11pt;text-indent: 0pt;text-align: left;">8 2 1 1 8 3 1 0 0</p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;line-height: 2pt;text-align: right;">(<i>n </i>= 8) (<i>n </i>= 6)</p><p class="s30" style="padding-top: 1pt;padding-left: 73pt;text-indent: 0pt;text-align: left;">0 10 20 30</p><p class="s30" style="padding-top: 1pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">0 2.5 5</p><p class="s30" style="padding-top: 1pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">7.5 10</p><p class="s30" style="padding-top: 1pt;padding-left: 46pt;text-indent: 0pt;text-align: left;">0 40 80 120 160</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 10pt;text-indent: 0pt;text-align: center;">b</h4><p class="s30" style="padding-top: 5pt;text-indent: 0pt;text-align: right;">scRNA-seq data</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 5pt;padding-left: 132pt;text-indent: 0pt;line-height: 254%;text-align: center;">Cell lineage inference Gastric</p><p class="s30" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Intestinal</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;line-height: 13pt;text-align: center;">Cellular composition <span style=" color: #EE68A4;">Gastric</span></p><p class="s143" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">-dominant</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s147" style="padding-left: 132pt;text-indent: 0pt;line-height: 4pt;text-align: center;">GI-mixed</p><p class="s30" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">Time (months)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 31pt;text-indent: -3pt;text-align: left;">Single-cell DEGs</p><p class="s30" style="padding-left: 29pt;text-indent: 0pt;text-align: left;">analysis</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 38pt;text-indent: 0pt;text-align: left;">Modeling, test gene combinations</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 12pt;text-indent: 0pt;text-align: left;">Seed gene <i>X</i></p><p style="padding-left: 43pt;text-indent: 0pt;line-height: 7pt;text-align: left;"/><p class="s30" style="padding-top: 1pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Time (months)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s148" style="padding-top: 4pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">12-gene signature</p><table style="border-collapse:collapse" cellspacing="0"><tr style="height:7pt"><td style="width:14pt;border-right-style:solid;border-right-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:33pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt"><p class="s54" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Add gene <i>g</i></p></td><td style="width:18pt;border-left-style:solid;border-left-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:5pt"><td style="width:29pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt" colspan="2"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:36pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr><tr style="height:19pt"><td style="width:65pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt" colspan="3"><p class="s54" style="padding-top: 1pt;padding-left: 4pt;text-indent: 0pt;text-align: left;">Candidate geneset evaluation</p><p class="s149" style="padding-left: 3pt;text-indent: 0pt;line-height: 7pt;text-align: left;"><span class="s150">L</span><span class="s151">g,j </span><span class="s58">= </span><span class="s152">1 × </span>I<span class="s54">(g), </span>V<span class="s153">g,j </span><span class="s154">::; </span><span class="s54">Median(</span>V<span class="s153">g</span><span class="s152">)</span></p><p class="s149" style="padding-left: 16pt;text-indent: 0pt;line-height: 4pt;text-align: left;"><span class="s152">–1 × </span>I<span class="s54">(g), </span>V<span class="s153">g,j </span><span class="s154">&gt; </span><span class="s54">Median(</span>V<span class="s153">g</span><span class="s152">)</span></p></td></tr><tr style="height:5pt"><td style="width:29pt;border-top-style:solid;border-top-width:1pt;border-right-style:solid;border-right-width:1pt" colspan="2"><p style="text-indent: 0pt;text-align: left;"><br/></p></td><td style="width:36pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt"><p style="text-indent: 0pt;text-align: left;"><br/></p></td></tr></table><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 4pt;padding-left: 52pt;text-indent: 0pt;text-align: left;">Tumor cells <i>n </i>= 31,131</p><p class="s30" style="text-indent: 0pt;text-align: right;">Other</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 67pt;text-indent: -14pt;line-height: 73%;text-align: left;">No C-index doesn’t increase</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 4pt;text-indent: 0pt;text-align: left;">Yes</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 1pt;text-indent: 0pt;text-align: left;">End</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 46pt;text-indent: -2pt;text-align: left;">Multi-cohort validation</p><p class="s49" style="padding-top: 8pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">c <span class="s29">BTF3</span></p><p class="s30" style="padding-top: 1pt;padding-left: 54pt;text-indent: 0pt;text-align: left;">3 <span class="s51">3</span></p><p class="s30" style="padding-top: 3pt;padding-left: 54pt;text-indent: 0pt;text-align: left;">2 <span class="s51">2</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="text-indent: 0pt;text-align: right;">CKB</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="text-indent: 0pt;text-align: right;">VPS28</p><p class="s30" style="padding-top: 2pt;text-indent: 0pt;text-align: center;">2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;line-height: 2pt;text-align: center;">1</p><p class="s29" style="padding-top: 14pt;padding-left: 35pt;text-indent: 0pt;text-align: left;">TM4SF1 <span class="s44">3</span></p><p class="s30" style="padding-top: 3pt;padding-left: 16pt;text-indent: 0pt;text-align: left;">4 <span class="s42">2</span></p><p class="s30" style="padding-top: 5pt;padding-left: 16pt;text-indent: 0pt;line-height: 0pt;text-align: left;">2</p><p class="s29" style="padding-top: 14pt;padding-left: 15pt;text-indent: 0pt;text-align: left;">EIF3E</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="text-indent: 0pt;text-align: right;">GPX4</p><p class="s30" style="text-indent: 0pt;text-align: center;">3</p><p class="s30" style="padding-top: 4pt;text-indent: 0pt;text-align: center;">2</p><h4 style="padding-top: 8pt;padding-left: 24pt;text-indent: 0pt;text-align: left;">g</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 24pt;text-indent: -1pt;text-align: left;">Primary GAC (<i>n </i>= 192) Ooi et al., GSE15459</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;line-height: 2pt;text-align: center;">OS</p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Gene expression</p><p style="text-indent: 0pt;text-align: left;"/><p class="s42" style="padding-left: 54pt;text-indent: 0pt;text-align: left;">1 <span class="s45">1 </span><span class="s30">1 1</span></p><p class="s30" style="padding-top: 3pt;padding-left: 54pt;text-indent: 0pt;text-align: left;">0 0 0 0 <span class="s51">0</span> <span class="s51">0</span></p><p class="s29" style="padding-top: 1pt;padding-left: 14pt;text-indent: 0pt;text-align: left;">FCGBP</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 14pt;text-indent: 0pt;text-align: center;">TPM2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 1pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">CDK6</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 2pt;padding-left: 13pt;text-indent: 0pt;text-align: left;">CXCL3</p><p style="text-indent: 0pt;text-align: left;"/><p class="s29" style="padding-top: 2pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">NCBP2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s51" style="padding-top: 3pt;padding-left: 208pt;text-indent: 0pt;line-height: 5pt;text-align: left;">4 4<span class="s30"> 4</span></p><p class="s30" style="padding-left: 54pt;text-indent: 0pt;line-height: 8pt;text-align: left;">2 4 <span class="s155">4</span></p><p class="s156" style="padding-top: 2pt;padding-left: 54pt;text-indent: 0pt;text-align: left;">1 2 <span class="s42">2 </span><span class="s30">2 2 2</span></p><p class="s30" style="padding-top: 7pt;padding-left: 54pt;text-indent: 0pt;text-align: left;">0 0 0 0 0 0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 2pt;padding-left: 12pt;text-indent: 0pt;text-align: center;">PIGR</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 48pt;text-indent: 0pt;text-align: left;">Median survival</p><p class="s144" style="padding-top: 1pt;padding-left: 60pt;text-indent: 0pt;text-align: left;">63.7 months</p><ol id="l1"><ol id="l2"><li style="padding-top: 1pt;padding-left: 69pt;text-indent: -8pt;text-align: left;"><p class="s145" style="display: inline;">months</p><p class="s143" style="padding-top: 2pt;text-indent: 0pt;text-align: right;">Gastric-dominant <span style=" color: #2F7AB5;">GI-mixed</span></p><p class="s42" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0         <i>P </i><span class="s30">= 0.0019</span></p><h4 style="padding-top: 4pt;text-indent: 0pt;text-align: right;">d</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;line-height: 4pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 19pt;text-indent: 0pt;text-align: left;">Survival from PC diagnosis</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 5pt;padding-left: 45pt;text-indent: 0pt;text-align: left;">Median survival</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;padding-left: 25pt;text-indent: 0pt;text-align: left;">GAC-PC (<i>n </i>= 45, MDACC, validation cohort)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: left;">GI-mixed (<i>n </i>= 17) –</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Subgroup</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s157" style="text-indent: 0pt;text-align: left;">12-gene signature</p><p style="padding-left: 22pt;text-indent: 0pt;line-height: 6pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;padding-left: 12pt;text-indent: 0pt;text-align: left;">Hazard ratio (95% CI)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-top: 3pt;padding-left: 22pt;text-indent: 0pt;text-align: left;">P <span class="s30">value</span></p><p class="s30" style="padding-top: 4pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">Number at risk</p><p class="s144" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">107           36              22              13             0</p><p class="s145" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">85            22              14               6              0</p><p class="s30" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0               40             80             120          160</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s157" style="padding-left: 3pt;text-indent: 0pt;text-align: left;">3.31 × 10<span class="s158">–4</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><h4 style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;line-height: 9pt;text-align: left;">h</h4><p class="s30" style="padding-left: 132pt;text-indent: 0pt;line-height: 4pt;text-align: center;">Primary GAC (<i>n </i>= 411, TCGA)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 4pt;text-indent: 0pt;text-align: left;">P <span class="s30">= 8.2 × 10</span><span class="s159">–4</span></p></li><li style="padding-left: 39pt;text-indent: -8pt;text-align: left;"><p class="s144" style="display: inline;">months</p></li></ol></ol><p class="s30" style="text-indent: 0pt;text-align: left;">(this study) Gastric-dominant (<i>n </i>= 28) <span class="s51">12.7 (3.2, 51.0)</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s143" style="padding-top: 1pt;padding-left: 30pt;text-indent: 0pt;text-align: left;">7.8 months</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 12pt;text-indent: 0pt;text-align: left;">Sex</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: left;">Age</p><p style="padding-left: 12pt;text-indent: 0pt;line-height: 6pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 12pt;text-indent: 0pt;text-align: left;">Subtype</p><p class="s30" style="padding-top: 2pt;padding-left: 31pt;text-indent: 5pt;line-height: 166%;text-align: right;">Male (<i>n </i>= 18) Female (<i>n </i>= 27)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: left;">Intestinal (<i>n </i>= 9) –</p><p class="s30" style="padding-top: 2pt;padding-left: 10pt;text-indent: -9pt;line-height: 152%;text-align: left;">Diffuse (<i>n </i>= 30)   2.8 (0.70, 11.3) NA (<i>n </i>= 6)</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: left;">Histology</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: left;">(<i>n </i>= 45) 1.0 (0.92, 1.00)</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-left: 37pt;text-indent: 0pt;line-height: 145%;text-align: right;">NOS (<i>n </i>= 20) SRC (<i>n </i>= 21) N/A (<i>n </i>= 4)</p><p class="s30" style="padding-top: 2pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">–</p><p class="s30" style="padding-top: 3pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">1.3 (0.45, 3.7)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 6pt;text-indent: 0pt;text-align: left;">–</p><p class="s30" style="padding-top: 3pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">0.9 (0.27, 2.9)</p><p class="s30" style="padding-top: 1pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">–</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.643</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">0.837</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">0.025</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p class="s30" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">OS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 45pt;text-indent: 0pt;text-align: left;">Median survival</p><p class="s144" style="padding-top: 2pt;padding-left: 60pt;text-indent: 0pt;text-align: left;">56.2 months</p><p class="s143" style="padding-top: 1pt;padding-left: 60pt;text-indent: 0pt;text-align: left;">26.0 months</p><p class="s30" style="padding-top: 2pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Number at risk</p><p class="s144" style="padding-top: 3pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">17 8 4 1 1</p><p class="s143" style="padding-top: 2pt;padding-left: 69pt;text-indent: 0pt;text-align: left;">28 4 1 0 0</p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0.147</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">P <span class="s30">= 0.0041</span></p><p class="s30" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0               10              20              30           40</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 49pt;text-indent: 0pt;text-align: left;">e</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: left;">Primary GAC (<i>n </i>= 300) Cristescu et al., GSE62254</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 37pt;text-indent: 0pt;text-align: left;">0.1 1 10 100</p><p class="s30" style="padding-top: 3pt;padding-left: 49pt;text-indent: 0pt;text-align: left;">Number at risk</p><p class="s144" style="padding-top: 2pt;padding-left: 58pt;text-indent: 0pt;text-align: left;">168 47 13 4 2 0</p><p class="s143" style="padding-top: 2pt;padding-left: 59pt;text-indent: 0pt;text-align: left;">241 57 10 2 2 0</p><p class="s30" style="padding-top: 4pt;padding-left: 60pt;text-indent: 0pt;text-align: left;">0 25 50 75 100 125</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 5pt;text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 5pt;text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">OS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 57pt;text-indent: -7pt;line-height: 158%;text-align: right;">Median survival <span style=" color: #2F7AB5;">Not reached</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 71pt;text-indent: 0pt;text-align: left;">DFS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;padding-left: 59pt;text-indent: -7pt;line-height: 138%;text-align: left;">Median survival <span style=" color: #2F7AB5;">Not reached</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 15pt;text-indent: 0pt;text-align: left;">GI-mixed</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: left;">Subtype</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 3pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">Local recurrence</p><p class="s30" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">and/or metastasis</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">No</p><p class="s30" style="padding-top: 1pt;text-indent: 0pt;text-align: right;">1.00</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-top: 1pt;padding-left: 1pt;text-indent: 0pt;text-align: left;">Survival probability</p><p style="text-indent: 0pt;text-align: left;"/><p class="s30" style="text-indent: 0pt;text-align: right;">0.75</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.50</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0.25</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">0</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 4pt;text-indent: 0pt;text-align: left;">P <span class="s30">= 4.8 × 10</span><span class="s159">–4</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 37pt;text-indent: 0pt;text-align: left;">DFS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 16pt;text-indent: 0pt;text-align: left;">Median survival</p><p class="s144" style="padding-top: 1pt;padding-left: 25pt;text-indent: 0pt;line-height: 139%;text-align: left;">Not reached <span style=" color: #EE68A4;">Not reached</span></p><p class="s45" style="text-indent: 0pt;line-height: 5pt;text-align: right;">0 <i>P </i><span class="s30">= 0.011</span></p><p class="s145" style="padding-left: 30pt;text-indent: 0pt;line-height: 4pt;text-align: left;">89.4 months</p><p class="s42" style="padding-left: 12pt;text-indent: 0pt;line-height: 73%;text-align: left;">0 <i>P </i><span class="s30">= 0.0076</span></p><p class="s145" style="padding-left: 32pt;text-indent: 0pt;line-height: 4pt;text-align: left;">38.4 months</p><p class="s30" style="padding-left: 62pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Number at risk</p><p class="s30" style="padding-top: 4pt;padding-left: 57pt;text-indent: 0pt;text-align: left;">Number at risk Number at risk</p><p class="s30" style="padding-left: 57pt;text-indent: 0pt;text-align: left;">Gastric- dominant</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="text-indent: 0pt;text-align: right;">Yes</p><p class="s144" style="padding-left: 57pt;text-indent: 0pt;text-align: left;">98 36 8 3 2 0</p><p class="s143" style="padding-top: 2pt;padding-left: 56pt;text-indent: 0pt;line-height: 4pt;text-align: left;">143 38 7 1 1 0</p><p class="s144" style="padding-left: 132pt;text-indent: 0pt;text-align: center;">126           93              77             31              3</p><p class="s145" style="padding-top: 3pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">174          111             88             55              4</p><p class="s30" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0              25             50             75            100</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><p class="s144" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">126           69              54              13              1</p><p class="s145" style="padding-top: 3pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">174           72              49              20              0</p><p class="s30" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0               25              50              75          100</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><p class="s160" style="padding-top: 3pt;text-indent: 0pt;text-align: right;">*</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">*<i>P </i>= 0.0013</p><p class="s30" style="padding-top: 5pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">0           25          50          75         100       125</p><p class="s30" style="padding-top: 1pt;padding-left: 132pt;text-indent: 0pt;text-align: center;">Time (months)</p><p class="s13" style="padding-top: 10pt;padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="#bookmark67" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">this, it is generally recognized that patients with intestinal-type GAC have better prognoses than those with a poorly differenti- ated or diffuse type histology</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">37</span><a href="#bookmark68" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">–</a>39<a href="#bookmark69" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">. In addition, immune deconvolu- tion analysis using public datasets suggested that the better clinical outcomes of GI-mixed tumors could be partially associated with a more engaged and effective immune response against the tumor, including higher levels of B cells (which are known to be associated with a protective immunity and better clinical outcomes)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">40</span><a href="#bookmark70" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">–</a>42 <a href="#bookmark71" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">and M1 polarization</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">43</span><a href="#bookmark72" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">,</a>44<a href="#bookmark73" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;">, lower levels of fibroblasts</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt; vertical-align: 3pt;">45</span><a href="#bookmark74" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 5.5pt;">–</a>47 <span class="s8">and M2-like macrophages, and elevated cytolytic activity. Nevertheless, further investigations are needed to elucidate the underlying mechanisms.</span></p><p class="s8" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;">Most intriguingly, based on tumor cell lineage/state composi- tions, PC cases were classified into two cellular subtypes that were prognostic independent of histopathological features. Further analyses led us to discover a 12-gene signature that appears to be fundamental to GAC carcinogenesis/propagation as it was not only highly prognostic in GAC-PC validation cohort but performed just as robustly in several large-scale localized GAC cohorts. Currently, to our knowledge there is no such signature in clinical use, and thus it has a high potential to stratify patients for more effective therapies as they become available.</p><p class="s21" style="padding-top: 9pt;padding-left: 7pt;text-indent: 0pt;line-height: 12pt;text-align: left;"><a name="bookmark15">Online content</a></p><p class="s24" style="padding-left: 7pt;text-indent: 0pt;text-align: justify;"><a href="https://doi.org/10.1038/s41591-020-1125-8" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">Any methods, additional references, Nature Research report- ing summaries, source data, extended data, supplementary infor- mation, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at </a><a href="https://doi.org/10.1038/s41591-020-1125-8" class="s22" target="_blank">https://doi.org/10.1038/ </a>s41591-020-1125-8<span style=" color: #000;">.</span></p><p class="s8" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 10pt;text-align: left;">Received: 4 August 2019; Accepted: 9 October 2020;</p><p class="s8" style="padding-left: 7pt;text-indent: 0pt;line-height: 10pt;text-align: left;">Published online: 4 January 2021</p><p class="s21" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 12pt;text-align: left;"><a name="bookmark31">References</a></p><ol id="l3"><li style="padding-left: 18pt;text-indent: -11pt;line-height: 89%;text-align: left;"><p style="display: inline;"><a name="bookmark32">Bray, F. et al. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. </a><span class="s161">CA Cancer J. Clin. </span><b>68</b>, 394–424 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark33">Ikoma, N. et al. Preoperative chemoradiation therapy induces primary-tumor complete response more frequently than chemotherapy alone in gastric cancer: analyses of the National Cancer Database 2006–2014 using propensity score matching. </a><span class="s161">Gastric Cancer </span><b>21</b>, 1004–1013 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 8pt;text-align: left;"><p style="display: inline;">Mizrak Kaya, D. et al. Risk of peritoneal metastases in patients who had</p><p style="padding-left: 18pt;text-indent: 0pt;line-height: 92%;text-align: left;"><a name="bookmark34">negative peritoneal staging and received therapy for localized gastric adenocarcinoma. </a><span class="s161">J. Surg. Oncol. </span><b>117</b>, 678–684 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark35">Shiozaki, H. et al. Prognosis of gastric adenocarcinoma patients with various burdens of peritoneal metastases. </a><span class="s161">J. Surg. Oncol. </span><b>113</b>, 29–35 (2016).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark36">Chen, C. et al. Efficacy and safety of immune checkpoint inhibitors in advanced gastric or gastroesophageal junction cancer: a systematic review and meta-analysis. </a><span class="s161">Oncoimmunology </span><b>8</b>, e1581547 (2019).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 89%;text-align: left;"><p style="display: inline;"><a name="bookmark37">Taieb, J. et al. Evolution of checkpoint inhibitors for the treatment of metastatic gastric cancers: current status and future perspectives. </a><span class="s161">Cancer Treat. Rev. </span><b>66</b>, 104–113 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark38">Bartley, A. N. et al. HER2 testing and clinical decision making in gastroesophageal adenocarcinoma: guideline from the College of American Pathologists, American Society for Clinical Pathology, and the American Society of Clinical Oncology. </a><span class="s161">J. Clin. Oncol. </span><b>35</b>, 446–464 (2017).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark39">Cancer Genome Atlas Research Network. Comprehensive molecular characterization of gastric adenocarcinoma. </a><span class="s161">Nature </span><b>513</b>, 202–209 (2014).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark40">Wang, R. et al. Multiplex profiling of peritoneal metastases from gastric adenocarcinoma identified novel targets and molecular subtypes that predict treatment response. </a><span class="s161">Gut </span><b>69</b>, 18–31 (2020).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark41">Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. </a><span class="s161">Science </span><b>352</b>, 189–196 (2016).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 8pt;text-align: left;"><p style="display: inline;">Han, X. et al. Construction of a human cell landscape at single-cell level.</p><p class="s161" style="padding-left: 18pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><a name="bookmark42">Nature </a><span class="s162">581</span><span class="p">, 303–309 (2020).</span></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark43">Buttner, M., Miao, Z., Wolf, F. A., Teichmann, S. A. &amp; Theis, F. J. A test metric for assessing single-cell RNA-seq batch correction. </a><span class="s161">Nat. Methods </span><b>16</b>, 43–49 (2019).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p class="s192" style="display: inline;"><a href="https://arxiv.org/abs/1802.03426" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank" name="bookmark44">McInnes, L., Healy, J., Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Preprint at </a><a href="https://arxiv.org/abs/1802.03426" class="a" target="_blank">https://arxiv.org/ abs/1802.03426</a> <span style=" color: #000;">(2018).</span></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;">Smillie, C. S. et al. Intra- and inter-cellular rewiring of the human colon during ulcerative colitis. <span class="s161">Cell </span><b>178</b>, 714–730.e22 (2019).</p><p style="text-indent: 0pt;text-align: left;"><br/></p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark45">Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. </a><span class="s161">Nat. Biotechnol. </span><b>36</b>, 411–420 (2018).<a name="bookmark46">&zwnj;</a></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark47">Cao, J. et al. The single-cell transcriptional landscape of mammalian organogenesis. </a><span class="s161">Nature </span><b>566</b>, 496–502 (2019).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 86%;text-align: left;"><p style="display: inline;"><a name="bookmark48">Kiselev, V. Y. et al. SC3: consensus clustering of single-cell RNA-seq data. </a><span class="s161">Nat. Methods </span><b>14</b>, 483–486 (2017).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark49">Puram, S. V. et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. </a><span class="s161">Cell </span><b>171</b>, 1611–1624.e24 (2017).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark50">Patel, A. P. et al. Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma. </a><span class="s161">Science </span><b>344</b>, 1396–1401 (2014).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark51">Jerby-Arnon, L. et al. A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade. </a><span class="s161">Cell </span><b>175</b>, 984–997.e24 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark52">Stevens, C. E. &amp; Leblond, C. P. Renewal of the mucous cells in the gastric mucosa of the rat. </a><span class="s161">Anat. Rec. </span><b>115</b>, 231–245 (1953).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 86%;text-align: left;"><p style="display: inline;"><a name="bookmark53">Karam, S. M. A focus on parietal cells as a renewing cell population. </a><span class="s161">World J. Gastroenterol. </span><b>16</b>, 538–546 (2010).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark54">Merzel, J. &amp; Leblond, C. P. Origin and renewal of goblet cells in the epithelium of the mouse small intestine. </a><span class="s161">Am. J. Anat. </span><b>124</b>, 281–305 (1969).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark55">Blanpain, C., Horsley, V. &amp; Fuchs, E. Epithelial stem cells: turning over new leaves. </a><span class="s161">Cell </span><b>128</b>, 445–458 (2007).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark56">Coker, E. A. et al. canSAR: update to the cancer translational research and drug discovery knowledgebase. </a><span class="s161">Nucleic Acids Res. </span><b>47</b>, D917–D922 (2019).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark57">Kim, H. K. et al. A gene expression signature of acquired chemoresistance to cisplatin and fluorouracil combination chemotherapy in gastric cancer patients. </a><span class="s161">PLoS ONE </span><b>6</b>, e16694 (2011).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark58">Ooi, C. H. et al. Oncogenic pathway combinations predict clinical prognosis in gastric cancer. </a><span class="s161">PLoS Genet. </span><b>5</b>, e1000676 (2009).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark59">Cristescu, R. et al. Molecular analysis of gastric cancer identifies subtypes associated with distinct clinical outcomes. </a><span class="s161">Nat. Med. </span><b>21</b>, 449–456 (2015).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark60">Mizrak Kaya, D. et al. Advanced gastric adenocarcinoma: optimizing therapy options. </a><span class="s161">Expert Rev. Clin. Pharmacol. </span><b>10</b>, 263–271 (2017).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark61">Dagogo-Jack, I. &amp; Shaw, A. T. Tumour heterogeneity and resistance to cancer therapies. </a><span class="s161">Nat. Rev. Clin. Oncol. </span><b>15</b>, 81–94 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 86%;text-align: left;"><p style="display: inline;"><a name="bookmark62">Hudler, P. Challenges of deciphering gastric cancer heterogeneity. </a><span class="s161">World J. Gastroenterol. </span><b>21</b>, 10510–10527 (2015).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark63">Gullo, I., Carneiro, F., Oliveira, C. &amp; Almeida, G. M. Heterogeneity in gastric cancer: from pure morphology to molecular classifications. </a><span class="s161">Pathobiology </span><b>85</b>, 50–63 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark64">Oh, S. C. et al. Clinical and genomic landscape of gastric cancer with a mesenchymal phenotype. </a><span class="s161">Nat. Commun. </span><b>9</b>, 1777 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark65">Merlo, L. M., Pepper, J. W., Reid, B. J. &amp; Maley, C. C. Cancer as an evolutionary and ecological process. </a><span class="s161">Nat. Rev. Cancer </span><b>6</b>, 924–935 (2006).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark66">Michor, F. &amp; Polyak, K. The origins and implications of intratumor heterogeneity. </a><span class="s161">Cancer Prev. Res. (Phila.) </span><b>3</b>, 1361–1364 (2010).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark67">Barros, R., Freund, J. N., David, L. &amp; Almeida, R. Gastric intestinal metaplasia revisited: function and regulation of CDX2. </a><span class="s161">Trends Mol. Med. </span><b>18</b>, 555–563 (2012).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;">Moghimi-Dehkordi, B., Safaee, A. &amp; Zali, M. R. Comparison of colorectal and gastric cancer: survival and prognostic factors. <span class="s161">Saudi J. Gastroenterol. </span><b>15</b>, 18–23 (2009).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark68">Qiu, M. Z. et al. Clinicopathological characteristics and prognostic analysis of Lauren classification in gastric adenocarcinoma in China. </a><span class="s161">J. Transl. Med. </span><b>11</b>, 58 (2013).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;">Petrelli, F. et al. Prognostic value of diffuse versus intestinal histotype in patients with gastric cancer: a systematic review and meta-analysis.</p><p class="s161" style="padding-left: 18pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><a name="bookmark69">J. Gastrointest. Oncol. </a><span class="s162">8</span><span class="p">, 148–163 (2017).</span></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;">Petitprez, F. et al. B cells are associated with survival and immunotherapy response in sarcoma. <span class="s161">Nature </span><b>577</b>, 556–560 (2020).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark70">Cabrita, R. et al. Tertiary lymphoid structures improve immunotherapy and survival in melanoma. </a><span class="s161">Nature </span><b>577</b>, 561–565 (2020).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark71">Helmink, B. A. et al. B cells and tertiary lymphoid structures promote immunotherapy response. </a><span class="s161">Nature </span><b>577</b>, 549–555 (2020).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 9pt;text-align: left;"><p style="display: inline;">Najafi, M. et al. Macrophage polarity in cancer: a review. <span class="s161">J. Cell. Biochem.</span></p><p class="s162" style="padding-left: 18pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a name="bookmark72">120</a><span class="p">, 2756–2765 (2019).</span></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 91%;text-align: left;"><p style="display: inline;"><a name="bookmark73">Kaneda, M. M. et al. PI3K</a><span class="s163">γ </span>is a molecular switch that controls immune suppression. <span class="s161">Nature </span><b>539</b>, 437–442 (2016).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 9pt;text-align: left;"><p style="display: inline;">Kalluri, R. The biology and function of fibroblasts in cancer. <span class="s161">Nat. Rev. Cancer</span></p><p class="s162" style="padding-left: 18pt;text-indent: 0pt;line-height: 8pt;text-align: left;">16<span class="p">, 582–598 (2016).</span></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark74">Calon, A. et al. Stromal gene expression defines poor-prognosis subtypes in colorectal cancer. </a><span class="s161">Nat. Genet. </span><b>47</b>, 320–329 (2015).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;">Fiori, M. E. et al. Cancer-associated fibroblasts as abettors of tumor progression at the crossroads of EMT and therapy resistance. <span class="s161">Mol. Cancer </span><b>18</b>, 70 (2019).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s164" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 108%;text-align: left;">Publisher’s note <span class="s165">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</span></p><p class="s165" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">© The Author(s), under exclusive licence to Springer Nature America, Inc. 2021</p><p class="s21" style="padding-top: 9pt;padding-left: 7pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Methods</p><p class="s162" style="padding-left: 7pt;text-indent: 0pt;line-height: 8pt;text-align: left;">Patient cohort, clinical characteristics and sample collection. <span class="p">A total of 20</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">patients with GAC with malignant ascites (PC) were included in this study. Based on the Lauren classification, all of the primary GACs were of diffuse type. The primary GAC diagnosis was confirmed through an endoscopic biopsy. Pathology results were verified independently by two experienced GI pathologists. In addition, computed tomography images of all patients were re-reviewed by an experienced imaging physician. Our reviews of pathology and radiology results confirmed that all cases except IP-070 (breast cancer metastatic to stomach determined by pathology and profiling) represented primary gastric cancer. None of the patients had a history or documented diagnosis of primary colon cancer, or imaging findings of colon cancer. The detailed clinical and histopathological characteristics are described in Supplementary Table 1, and representative computed tomography images are shown in Supplementary Fig. 1. GACs were</p><p class="s169" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark75" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">staged according to the American Joint Committee on Cancer Staging Manual (8th edition)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">48</span><a href="#bookmark76" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a>49<span class="p">. PC was confirmed by cytologic examination. This cohort included ten long-term survivors and ten short-term survivors. The long-term survivors were patients who survived more than 1 yr after the diagnosis of PC and the short-term survivors were patients who died within 6 months after the diagnosis of PC. Of the 20 patients, 16 had signet-ring cell carcinoma. Her2 staining was performed and all tumors were Her2 negative. PC specimens were collected at The University of</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Texas MD Anderson Cancer Center (Houston, USA) under an Institutional Review Board-approved protocol (no. LAB01-543) after obtaining written, informed consent from each participant. Patients with diagnosed GAC-PC with ascites were approached when they required a therapeutic paracentesis. No other selection criteria were applied. This project was in accordance with the policy advanced</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">by the Helsinki Declaration of 1964 and later versions. PC specimens were spun down for 20 min at 2,000<span class="s161">g </span>and pelleted cells (PC cells) were isolated, cryopreserved at <span class="s163">−</span>80 °C and used for scRNA-seq. All samples were processed using the same protocol and by the same research assistant.</p><p class="s161" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">MSI testing. <span class="p">The MSI test was not routinely done in clinic for gastric cancer when these patients were enrolled. For this study, only three patients were assessed for MSI test and reported as ‘microsatellite (MS) stable’. In six patients (all expired) there were no residual tissues for additional testing. For the remaining 13 patients, WES was performed on the same ascites samples or the ascites samples collected at a similar timepoint as the sample used for scRNA-seq, and was thus used for</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark39" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">an unbiased genomic analysis of microsatellites. WES data were processed and mapped to human reference genome as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">9</span><a href="#bookmark77" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">. MSIsensor</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">50</span>, a validated algorithm for deriving MSI status from genomic sequencing data, was applied to the aligned BAM files for detection of somatic microsatellite changes. The MSIsensor score was below the suggested threshold (score <span class="s163">&gt; </span>10 for matched tumor–normal pair, score <span class="s163">&gt; </span><a href="#bookmark77" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">20 for tumor-only sample)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">50</span><span class="s169"> </span>and all 13 patients were designated as MS stable.</p><p style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><b>scRNA-seq library preparation and sequencing. </b>Chromium single-cell sequencing technology from 10x Genomics was used to perform single-cell separation, complementary DNA amplification and library construction following the manufacturer’s guidelines. Briefly, the cellular suspensions were loaded on a 10x Chromium Single Cell Controller to generate single-cell gel bead-in-emulsions. The scRNA-seq libraries were constructed using the Chromium Single Cell 3<span class="s163">ʹ </span>Library &amp; Gel Bead Kit v.2 (PN-120237, 10x Genomics). The HS dsDNA</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Qubit Kit was used to determine the concentrations of both the cDNA and the libraries. The HS DNA Bioanalyzer was used for quality-tracking purposes and size determination for cDNA and lower-concentrated libraries. Sample libraries were normalized to 7.5 nM and equal volumes were added of each library for pooling. The concentration of the library pool was determined using the Library Quantification qPCR Kit (KAPA Biosystems) before sequencing. The barcoded library at the concentration of 275 pM was sequenced on the NovaSeq6000 (Illumina) S2 flow cell (100 cycle kit) using a 26 <span class="s163">× </span>91 run format with 8 bp index (read 1). To minimize batch effects, the libraries were constructed using the same versions of reagent kits and following the same protocols, and the libraries were sequenced on the same NovaSeq6000 flow cell and analyzed together.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">aberrantly high gene count. (3) Cluster marker gene expression: cells of a cluster express markers from distinct lineages (for example, cells in the T cell cluster showed expression of epithelial cell markers; cells in the B cell cluster showed expression of myeloid cell markers). We carefully reviewed canonical marker gene expression on UMAP plots and repeated the steps above a couple of times to ensure that we had filtered out most of the barcodes associated with cell doublets.</p><p class="s169" style="padding-left: 7pt;text-indent: 11pt;text-align: justify;"><a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Following removal of the poor-quality cells and doublets, a total of 45,048 cells were retained for downstream analyses. Library size normalization was performed in Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span> <a href="#bookmark78" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">on the filtered gene–cell matrix to obtain the normalized UMI (unique molecular identifier) count data as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">51</span><span class="p">. Statistical assessment</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">of possible batch effects was performed using the R package k-BET (a robust and sensitive <span class="s161">k</span><a href="#bookmark42" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">-nearest neighbor batch-effect test)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">12</span>. k-BET was run on major immune cell types including B, myeloid, CD4 and CD8 T cells separately with default parameters. A control dataset with known significant batch effects was included to assist with data interpretation. We chose the <span class="s161">k </span>input value from 1% to 100% of the sample size. In each run, the number of tested neighborhoods was 10% of the sample size. The mean and maximal rejection rates were then calculated based on a total of 100 repeated k-BET runs. A low rejection rate indicates homogeneous mixing of samples from different batches. k-BET results suggested minimal batch effects in this dataset (Supplementary Fig. 2).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s161" style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">Unsupervised cell clustering, dimensionality reduction and cluster relationship analysis. <a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><span class="s169"> </span><span class="p">was applied to the normalized gene–cell matrix to identify highly variable genes for unsupervised cell clustering. To identify highly variable genes, the </span>MeanVarPlot <a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">method in the Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><span class="s169"> </span><span class="p">package was used to establish the mean– variance relationship of the normalized counts of each gene across cells. We then chose genes whose log-mean was between 0.0125 and 3 and whose dispersion was above 0.5, resulting in 3,018 highly variable genes. The elbow plot was generated with the </span>PCElbowPlot <a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">function of Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><span class="p">, and based on which the numbers of significant principal components (PCs) were determined. Different resolution parameters for unsupervised clustering were then examined to determine the optimal number of clusters. For this study, the first ten principal components</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 94%;text-align: left;"><a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">and the highly variable genes identified by Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><span class="s169"> </span>were used for unsupervised clustering with a resolution set to 0.6, yielding a total of 20 cell clusters (Extended Data Fig. 1a). For visualization, the dimensionality was further reduced using either the <span class="s161">t</span><a href="#bookmark43" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">-SNE or UMAP</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">13</span><span class="s169"> </span>methods with <span class="s161">Seurat </span>functions <span class="s161">RunTSNE </span>and <span class="s161">RunUMAP</span>, respectively. The principal components used to calculate the embedding were the same as those used for clustering.</p><p style="padding-left: 18pt;text-indent: 0pt;line-height: 9pt;text-align: left;">In addition, <span class="s161">Monocle 3 </span><a href="http://cole-trapnell-lab.github.io/monocle-release/monocle3/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">alpha (</a><a href="http://cole-trapnell-lab.github.io/monocle-release/monocle3/" target="_blank">http://cole-trapnell-lab.github.io/</a></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 92%;text-align: left;"><span style=" color: #3B699E;">monocle-release/monocle3/</span><a href="#bookmark46" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">16</span><span class="s169"> </span>was applied as an independent tool for unsupervised clustering analysis (function <span class="s161">cluster_cells</span>) focusing on tumor cells, and UMAP</p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 8pt;text-align: left;">was used by default with the <span class="s161">Monocle </span>functions <span class="s161">reduce_dimension </span>and <span class="s161">plot_cells</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">for dimensionality reduction and visualization of the <span class="s161">Monocle </span>clustering results. <span class="s161">Monocle 3 </span>alpha was also used to construct the single-cell trajectories. The function <span class="s161">learn_graph </span><a href="#bookmark47" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">was run with default parameters. Moreover, we applied an additional unsupervised clustering approach, the single-cell consensus clustering (SC3) analysis</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">17</span>, on tumor cells from all patients and on tumor cells from each individual PC specimen. SC3 was run with default parameters and independent of cell lineage annotation. Furthermore, to study the hierarchical relationships among tumor cell clusters, we performed unsupervised cluster analysis. The dendrogram was drawn using Pearson correlation coefficient (PCC) with average principal component analysis (PCA) space (<span class="s161">Seurat </span>function <span class="s161">RunPCA</span><a href="#bookmark79" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">) for each tumor cell cluster with the R package denextend</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">52</span>.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s161" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Sample distribution analysis. <a href="#bookmark80" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">To quantify the distribution of each patient’s tumor cells across Seurat-defined cell clusters, we used the ratio of the observed to expected cell numbers in clusters to measure the enrichment of cells within a sample (tumor) across different cell clusters as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">53</span><span class="p">. Given a contingency table of samples by clusters, we first applied the chi-squared tests to evaluate whether the distribution of cells of a sample across clusters significantly deviates from random expectations. We then calculated the </span>R<span class="s170">o/e </span><span class="p">for each combination of samples and clusters as follows:</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s162" style="padding-left: 7pt;text-indent: 0pt;line-height: 86%;text-align: left;">scRNA-seq data processing and analysis. <span class="s161">Raw sequencing data processing, quality check, data filtering, doublets removal, batch-effect evaluation and data</span></p><p class="s172" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s171">R</span>o<span class="s173">=</span>e</p><p class="s174" style="padding-left: 55pt;text-indent: 0pt;line-height: 6pt;text-align: center;">observed</p><p style="text-indent: 0pt;text-align: left;"><span><img width="35" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_165.png"/></span></p><p class="s175" style="padding-left: 47pt;text-indent: 0pt;line-height: 12pt;text-align: center;">¼<span class="s176"> </span><span class="s174">expected</span></p><p class="s161" style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">normalization. <span class="p">The raw scRNA-seq data were preprocessed (demultiplex</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">cellular barcodes, read alignment and generation of gene count matrix) using the Cell Ranger Single Cell Software Suite provided by 10x Genomics. Detailed quality-control metrics were generated and evaluated. Genes detected in fewer than three cells and cells with low-complexity libraries (in which detected transcripts were aligned to less than 200 genes) were filtered out and excluded from subsequent analysis. Low-quality cells where <span class="s163">&gt;</span>15% of transcripts derived from the mitochondria were considered apoptotic and also excluded. Following the initial clustering, we removed likely cell doublets from all clusters. Doublets</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">were identified by the following methods: (1) Library complexity: cells are outliers in terms of library complexity. Cells in the top 1% of the distribution of genes detected per cell were removed. (2) Cluster distribution: doublets or multiplets likely form distinct clusters with hybrid expression features and exhibit an</p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 8pt;text-align: left;">where <span class="s161">R</span><span class="s170">o/e </span>is the ratio of the observed cell number to the expected cell number of</p><p style="text-indent: 0pt;text-align: left;"><span><img width="62" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_166.png"/></span></p><p class="s177" style="text-indent: 0pt;text-align: left;">2</p><p style="text-indent: 0pt;text-align: left;"/><p class="s172" style="text-indent: 0pt;text-align: left;">expected</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 7pt;text-indent: 0pt;line-height: 94%;text-align: left;">a given combination of cluster and sample. The expected cell numbers for each combination of clusters and samples were obtained from the chi-squared test. Different from the <span class="s163">χ</span><span class="s178">2 </span>values, which are defined as ð<span class="s174">observed</span>�<span class="s174">expected</span>Þ and could only indicate the divergence of observations from random expectations, <span class="s161">R</span><span class="s170">o/e </span>could indicate whether cells of a certain sample are enriched in a specific cluster. For example, if <span class="s161">R</span><span class="s170">o/e </span><span class="s163">&gt; </span>1, it suggests that cells of the sample are more frequently observed than random expectations in a specific cluster; that is, enriched. If <span class="s161">R</span><span class="s170">o/e </span><span class="s163">&lt; </span>1, it suggests that cells of a given sample are observed with less frequency than random expectations in a specific cluster.</p><p class="s161" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 86%;text-align: left;">Determination of major cell types and cell states. <span class="p">To define the major cell type of each single cell, DEGs were identified for each cell cluster using the </span>FindAllMarkers</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">analysis in the Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><span class="s169"> </span><a href="#bookmark81" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">package and the top 20 most significant DEGs were carefully reviewed. In parallel, feature plots were generated for the top 20 DEGs and a suggested set of canonical immune and stromal cell markers (Supplementary Table 2), a similar approach as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">54</span><a href="#bookmark82" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a><span class="s169">55</span>, followed by a manual review process. Enrichment of these markers (for example, <span class="s161">EPCAM </span>for epithelial cells, <span class="s161">PTPRC </span>for immune cells, <span class="s161">CD3D/E </span>for T cells, <span class="s161">CD8A/B </span>for CD8 T cells, <span class="s161">IL7R/ CD4/CD40LG </span>for CD4 T cells, <span class="s161">CD19/MS4A1/CD79A </span>for B cells, <span class="s161">COL1A1/COL1A2</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s169" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark84" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">clusters between the long- and short-term survivors. We embedded cell clusters in two-dimensional space with PCA using the highly variable genes and retained the top 50 principal components for subsequent analysis. We randomly sampled 500 cells from each tumor cell cluster of short-term survivors and long-term survivors, repeated 100 times and computed the Bhattacharyya pairwise distance between clusters as follows (a similar approach as described previously</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">57</span><a href="#bookmark85" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a>58<span class="p">):</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 3pt;text-align: left;">for fibroblasts, and so on) in certain clusters was considered a strong indication</p><p style="padding-bottom: 3pt;padding-left: 7pt;text-indent: 0pt;line-height: 3pt;text-align: left;">D<span class="s174">B </span><span class="s179">1 </span><span class="s180">μ </span>� <span class="s180">μ </span>Þ<span class="s181">T </span><b>Σ</b><span class="s181">�</span><span class="s172">1 </span>ð<span class="s180">μ </span>� <span class="s180">μ </span>Þþ <span class="s179">1 </span><span class="s174">ln</span><span class="s182"></span>p</p><p class="s183" style="padding-left: 26pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="5" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_167.png"/></span>	<span><img width="5" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_168.png"/></span></p><p class="s174" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">det<span class="s162">Σ </span><span class="s184"></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="54" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_169.png"/></span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">of the clusters representing the corresponding cell types (Extended Data Fig. 1). The two approaches were combined to infer major cell types for each cell cluster</p><p class="s185" style="padding-left: 7pt;text-indent: 0pt;line-height: 39%;text-align: left;">¼ <span class="s186">8 </span>ð <span class="s174">1 2</span></p><p class="s172" style="padding-left: 7pt;text-indent: 0pt;line-height: 42%;text-align: left;">1 2 <span class="s186">2</span></p><p class="s174" style="padding-left: 13pt;text-indent: 0pt;line-height: 9pt;text-align: left;">det<span class="s162">Σ</span>1det<span class="s162">Σ</span>2</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark82" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">according to the enrichment of marker genes and top-ranked DEGs in each cell cluster, as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">55</span>.</p><p class="s161" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">Inference of large-scale CNVs, phylogenetic tree construction and correlation analysis. <a href="https://github.com/broadinstitute/inferCNV" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">The tool inferCNV (</a><span class="s192">https://github.com/broadinstitute/inferCNV</span><a href="#bookmark40" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">) was applied to infer the large-scale CNVs from scRNA-seq data, and monocytes from this dataset were used as a control for CNV analysis. Initial CNVs were estimated by sorting the analyzed genes by their chromosomal locations and applying a moving average to the relative expression values, with a sliding window of 100 genes within each chromosome, as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">10</span><a href="#bookmark49" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a><span class="s169">19</span><span class="p">. In a subset of cases (</span>n <span class="s163">= </span><span class="p">6) whose WES data were available, the true CNVs called from WES were used as the positive control to assess the performance of the inferCNV analysis. Finally, malignant cells were distinguished from normal cells based on the information integrated from multiple sources, including cluster distribution of the cells, marker gene expression, inferred large-scale CNVs and aneuploidy status.</span></p><p style="padding-left: 7pt;text-indent: 11pt;text-align: left;">To construct a phylogenetic tree from the CNV calls in each tumor cell of a specific sample, the relative CNVs were calculated using the inferCNV outputs and the average CNV values were computed for nonoverlapping genomic bins, each consisting of 30 genes. For each cell within a bin, we calculated an integer copy number by multiplying relative CNV value by 2 (diploid) and rounding the results off to the closest integers. The R package phangorn was then used to construct</p><p class="s169" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark83" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">the phylogenetic maximal parsimony tree. The integer copy number profiles were re-segmented by the collection of breakpoints detected in each cell, so that each column in the data matrix corresponds to the longest interval uninterrupted by any variations across the cell population. The breakpoints in individual cells were determined by the R package copynumber</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">56</span> <span class="p">under default parameters.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s161" style="padding-left: 7pt;text-indent: 0pt;line-height: 93%;text-align: left;">Correlation analysis of the CNVs inferred from scRNA-seq data and that identified using bulk WES. <span class="p">Copy number analysis using scRNA-seq data is limited due to the low resolution; therefore, only the arm-level events were included. The arm-level copy number ratio was calculated as the weighted average of copy number ratio</span></p><p style="text-indent: 0pt;line-height: 79%;text-align: left;">P<span class="s187">¼</span></p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">of genes (scRNA-seq data, inferCNV output) or segments (WES data, the copy number segments) as follows:</p><p style="text-indent: 0pt;text-align: left;"><span><img width="44" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_170.png"/></span></p><p class="s189" style="padding-top: 5pt;padding-left: 143pt;text-indent: -38pt;line-height: 75%;text-align: left;"><span class="s188">CN </span>CNi <span class="s190"></span><span class="p"> </span>Li Li</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 82%;text-align: left;">where <span class="s161">CN</span><span class="s191">i </span>means the copy number ratio of the <span class="s161">i</span>th gene or segment. <span class="s161">L</span><span class="s191">i </span>means the length of the <span class="s161">i</span>th gene or segmentation.</p><p style="padding-left: 7pt;text-indent: 11pt;text-align: left;">The Pearson correlation and Spearman correlation analyses were then conducted on the resultant arm-level copy number ratio.</p><p class="s192" style="padding-top: 7pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s161">Cell-of-origin inference. </span><a href="http://bis.zju.edu.cn/HCL" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">Origins of tumor cells were inferred by mapping our scRNA-seq data to the well-annotated single-cell database of HCL (</a><a href="http://bis.zju.edu.cn/HCL" class="a" target="_blank">http://bis.zju. </a>edu.cn/HCL<a href="#bookmark41" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">11</span><span class="s169"> </span><a href="https://github.com/ggjlab/scHCL" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">using the R package scHCL (</a>https://github.com/ggjlab/scHCL<span style=" color: #000;">). First, gene expression normalization was performed using the following formula:</span></p><p class="s193" style="padding-top: 4pt;padding-left: 132pt;text-indent: 0pt;line-height: 5pt;text-align: center;">Count</p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 7pt;text-align: left;">where <i>μ</i><span class="s170">1 </span>and <i>μ</i><span class="s170">2 </span>are the mean vectors of each distribution, and <b>Σ Σ</b><span class="s174">1 </span><b>Σ</b><span class="s174">2 </span><span class="s194">=</span><span class="s174">2</span>.</p><p style="text-indent: 0pt;line-height: 7pt;text-align: left;">¼ ð þ Þ</p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 7pt;text-indent: 11pt;text-align: left;">For comparison, we also generated background distributions of clusters of short-term survivors (Short) by randomly sampling 100 pairs of cells and computing the Bhattacharyya distance between each pair of cells. In addition,</p><p class="s192" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">we also generated the Bhattacharyya distance between cells randomly sampled independent of survival status (Random). We only evaluated clusters that had 500 or more cells. Similarly, we computed the Bhattacharyya pairwise distance between different cell lineages (inferred by HCL) within and across patients. For the patient-level analysis, we randomly sampled 100 cells from each inferred cell lineage (for example, stomach pit cells, stomach mucosal cells, colorectal-like cells; Fig. </a>1h<span style=" color: #000;">) and repeated 100 times, and computed the Bhattacharyya pairwise distance between different lineages as described above. For comparison, we</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">also generated background distributions of each lineage by randomly sampling 100 cells twice of the same lineage and computing the Bhattacharyya distance between each pair of cells, and also generated the Bhattacharyya distance between cells randomly sampled independent of lineage annotation (Random). We only</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">evaluated the major lineages (for example, stomach pit cells, stomach mucosal cells, colorectal-like cells) that had 500 or more cells.</p><p class="s161" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">scRNA-seq imputation. <a href="#bookmark86" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Markov affinity-based graph imputation of cells (MAGIC)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">59</span><span class="s169"> </span><span class="p">is a commonly used algorithm for denoising scRNA-seq data. It learns the manifold data and imputes likely gene expression in each cell by sharing information across similar cells. Here, we followed the concept of MAGIC and</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">simplified the imputation process as follows: (1) Computation of affinity matrix via the FindNeighbors function from Seurat. This step constructs a <span class="s161">k</span>-nearest neighbor graph based on the Euclidean distance in PCA space and refines the edge weights between any two cells based on the shared overlap in their local neighborhoods (Jaccard similarity). It takes default <span class="s161">k </span>and the same number of principal components as used for unsupervised clustering. (2) Symmetrization of the affinity matrix using an additive approach. (3) Row-stochastic Markov-normalization</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">of symmetric affinity matrix (so each row sums to 1) into Markov matrix, representing the probability distribution of transitioning from each cell to every other cell. (4) Imputation expression matrix by multiplying the Markov matrix by the original expression matrix.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s161" style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">Quantifying the contribution of cell lineage diversity to transcriptomic variation within tumors. <a href="#bookmark87" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Given that detection rate is a major source of cell-to-cell variation for scRNA-seq datasets</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">60</span><a href="#bookmark86" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">, we applied MAGIC</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">59</span><span class="s169"> </span><span class="p">to impute likely gene expression to reduce this unwanted source of variation. After MAGIC (</span>t <span class="s163">= </span><a href="#bookmark85" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">3), the top 15 principal components can explain 81% ~ 93% of the total variance in each patient. To quantify the extent to which lineage diversity explains transcriptomic variation within a tumor, we then performed one-way ANOVA tests on each of the top 15 principal components in each tumor, using a similar approach as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">58</span><span class="p">.</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">The one-way ANOVA tests were conducted using the <span class="s161">aov </span>function in the R stats package. ANOVA partitions the total variation into within-lineages variation and between-lineages variation. The percentages of variation that can be explained by lineage diversity in each PCA space were then calculated for each tumor.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="0" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_171.png"/></span></p><p style="text-indent: 0pt;line-height: 7pt;text-align: left;">¼</p><p style="text-indent: 0pt;text-align: left;"/><p class="s195" style="padding-left: 97pt;text-indent: 0pt;line-height: 7pt;text-align: left;">E</p><p class="s193" style="text-indent: 0pt;line-height: 7pt;text-align: right;">sum<span class="p">ð</span>Count<span class="p">Þ</span></p><p class="s190" style="text-indent: 0pt;text-align: left;"><span class="p"> </span><span class="s193">105</span></p><p class="s161" style="padding-top: 6pt;padding-left: 97pt;text-indent: 0pt;line-height: 94%;text-align: left;">Comparison of between- and within-patient variations. <span class="p">To compare the magnitudes of between- and within-patient variations in the transcriptomic profiles, we</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 7pt;text-align: left;">where <span class="s161">E </span>denotes the normalized gene expression value; ‘Count’ denotes the raw</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark41" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">UMI counts; and ‘sum (Count)’ is the sum of all raw UMI counts in one cell. The PCC between the normalized expression profile of each query cell and the expression profile of each annotated cell type from the HCL reference dataset was calculated using the scHCL software package. The PCC was estimated using 6,075 signature genes provided by scHCL</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">11</span>. Cell lineage/type was subsequently assigned for each query cell based on the following criteria: PCC<span class="s163">&gt; </span>0.3; and the</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">best-matched stomach-derived cell lineage/type or the best-matched cell lineage/ type if there was no stomach-derived cell lineage among the top five hits. Cells that did not have a good correlation coefficient (PCC <span class="s163">&lt; </span>0.3) with any cell lineage/type in the HCL database were classified as ‘other’. To examine the similarity between colorectal-like cells and cell doublets, we applied a doublet simulation approach. Therein, a random sampling of cells (excluding intestinal-like cells) was taken to generate 500 simulated doublets and these cells were used for the UMAP clustering along with colorectal-like cells.</p><p class="s161" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 93%;text-align: left;">Quantifying the similarity of gene expression distributions of cell lineages and clusters within and across patients. <span class="p">The Bhattacharyya distance metric (a distance metric that is effective at comparing pairwise probability distributions) was used to measure the similarity of gene expression distributions for all pairs of cell</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">performed the one-way ANOVA tests on the top 15 principal components for tumor cells from patients that had 1,000 or more cells. The one-way ANOVA tests were conducted using the oneway.test() function in the R stats package. ANOVA partitions the total variation into between- and within-patient variation. In addition, we also performed the one-way ANOVA tests in the CNV profiles for each inferred lineage (for example, stomach pit cells, stomach mucosal cells, colorectal-like cells). The <span class="s161">F </span>statistic is the ratio of between-patient variation to within-patient variation, with <span class="s161">F </span>statistic <span class="s163">&gt;</span>1 indicating that the between-patient variation is greater than the within-patient variation.</p><p style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><span class="s161">Inferring cell-cycle stage, hierarchical clustering, DEGs and pathway enrichment analysis. </span><a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">The cell-cycle stage was computationally assigned for each individual cell by the function CellCycleScoring that is implemented in Seurat</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><a href="#bookmark50" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">. Cell-cycle stage was inferred based on the expression profile of the cell-cycle-related signature genes, as previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">20</span>. Hierarchical clustering was performed at multiple levels (all tumor cells together, by cell lineage and by patient) using the Ward minimum variance method. DEGs were identified for each cluster using the <span class="s161">FindMarkers </span><a href="#bookmark45" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">function in Seurat R package</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">15</span><span class="s169"> </span>and DEG list was filtered with the following criteria: the gene should be expressed in 20% or more cells in the more abundant group; expression fold change <span class="s163">&gt; </span>1.5; and false discovery rate (FDR) <span class="s161">Q</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">value<span class="s163">&lt; </span>0.05. The heat map was then generated using the pheatmap function in pheatmap R package for filtered DEGs. For pathway analysis, the curated gene sets (including Hallmark, KEGG and Reactome gene sets, <span class="s161">n </span><span class="s163">= </span><a href="http://software.broadinstitute.org/gsea/msigdb/index.jsp" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">910) were downloaded from the Molecular Signature Database (MSigDB, </a><a href="http://software.broadinstitute.org/gsea/msigdb/index.jsp" class="a" target="_blank">http://software.broadinstitute. </a><span style=" color: #3B699E;">org/gsea/msigdb/index.jsp</span>), single-sample GSVA (ssGSVA) was applied to the scRNA-seq data and pathway scores were calculated for each cell using the <span class="s161">gsva </span><a href="#bookmark88" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">function in the GSVA software package</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">61</span>. Pathway enrichment analysis was done with the limma R software package. Significantly enriched signaling pathways were identified with an FDR <span class="s161">Q </span>value<span class="s163">&lt; </span>0.01.</p><p style="padding-left: 7pt;text-indent: 11pt;text-align: left;">To profile the interactions between biological pathways, the differentially expressed pathways were assembled into a network where nodes represent pathways and edges represent their interactions (if they have common genes). The weight of an interaction corresponds to its Jaccard index between each pathway pair. The Jaccard index is a measure of set (here it refers to pathway) similarity, and defines two sets (pathway A and pathway B) as the ratio of the size of their intersection over the size of their union (see the equation below):</p><p style="text-indent: 0pt;line-height: 7pt;text-align: left;">ð <span class="s163">; </span>Þ ¼ </p><p style="text-indent: 0pt;text-align: left;"/><p class="s197" style="padding-top: 6pt;padding-left: 136pt;text-indent: -31pt;line-height: 69%;text-align: left;"><span class="s196">J </span><span class="s174">A B</span> <span class="s198">j</span>A <span class="s198">\ </span>B<span class="s198">j</span><span class="p"> j</span><span class="s174">A </span><span class="s194">∪ </span><span class="s174">B</span><span class="p">j</span></p><p style="padding-top: 8pt;padding-left: 18pt;text-indent: 0pt;line-height: 9pt;text-align: left;">For each pathway, its entity membership was represented as the set</p><p class="s163" style="padding-left: 12pt;text-indent: 0pt;line-height: 1pt;text-align: left;">=</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">among one of the five genes that are associated with the GI-mixed feature and its expression in a sample was greater than the median value, we assigned a score of 1 for this gene for this specific sample, and we assigned a score of <span class="s163">−</span>1 if its expression was less than or equal to the median value (Supplementary Fig. 15). After that,</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">the scores of each sample were summed, which constituted the signature score. Finally, the samples were categorized into gastric-dominant or GI-mixed groups based on their corresponding signature scores: <span class="s163">≤</span>median or <span class="s163">&gt;</span>median, respectively.</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">For the bulk RNA-seq datasets, the signature scores were calculated using the</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">log-transformed FPKM (fragments per kilobase of transcript per million mapped reads) values. For the bulk expression microarray datasets, the signature scores were calculated using the normalized gene expression values.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s169" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s161">Immune cell deconvolution. </span><a href="#bookmark94" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">CIBERSORT</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">67</span> <a href="#bookmark95" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">was applied to the normalized bulk RNA-seq and microarray gene expression datasets with the LM22 gene signature to estimate the relative fractions of 22 immune cell types. In addition, the R package MCP-counter</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">68</span> <span class="p">was applied to infer the abundance of eight immune cell subpopulations including T cells, CD8 T cells, cytotoxic lymphocytes, NK cells, B lineage cells, monocytic lineage cells, myeloid dendritic cells and neutrophils, as well as endothelial cells and fibroblasts.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s161" style="padding-left: 7pt;text-indent: 0pt;line-height: 4pt;text-align: left;">Single-cell somatic variant analysis. <span class="p">For each sample, the reads were extracted from</span></p><p class="s170" style="padding-left: 7pt;text-indent: 0pt;line-height: 7pt;text-align: left;">Pi <span class="p">{</span><span class="s161">e</span>1<span class="p">, </span><span class="s161">e</span>2<span class="p">,</span><span class="s163">…</span><span class="s161">e</span>n <span class="p">}. The Jaccard index was computed between a pair of pathways</span></p><p style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;line-height: 3pt;text-align: left;">the original BAM file using the cell-specific barcodes and were aggregated to</p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 10pt;text-align: left;">A and B as <span class="s161">J</span><a href="#bookmark89" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">(A, B). Cytoscape</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">62</span><span class="s169"> </span>was then used to visualize the network.</p><p class="s161" style="padding-top: 7pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">Generation of the 12-gene prognostic signature. <a href="#bookmark30" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">To generate a gene expression signature that is clinically applicable, we performed multistep analysis (Fig. </a><span class="s192">5a</span><span class="p">). First, we compared the gene expression profiles of tumor cells between the gastric-dominant and GI-mixed groups and identified DEGs. The DEG list was then filtered based on the following criteria to select the most significant DEGs.</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Only the genes with an expression fold change <span class="s163">&gt;</span>1.5 or <span class="s163">&lt;−</span>1.5 and an FDR <span class="s163">&lt; </span>0.01 and highly expressed (normalized UMI count<span class="s163">&gt; </span>1, expressed in at least 50% of cells from one of the two groups) were selected and taken into subsequent analysis. We next screened each of the DEGs based on their statistical correlation with patient survival using a univariate Cox proportional-hazards regression model, and only the DEGs that showed consistency with patient survival were selected (for example,</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">DEGs highly expressed in the gastric-dominant group with a hazard ratio<span class="s163">&gt; </span>1 or DEGs highly expressed in the GI-mixed group with a hazard ratio<span class="s163">&lt; </span>1). We identified 149 significant DEGs in total (that met the above criteria). After that,</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark90" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">the Harrell concordance index (C-index)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">63</span><span class="s169"> </span>was applied to quantify the predictive accuracy of the prognosis-related DEGs. C-index was calculated using a univariate Cox proportional-hazards regression model and the R package <span class="s161">survcomp</span>. A</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark91" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">C-index value of 0.5 indicates no predictive ability, whereas a value of 1 represents perfect predictive ability, similar to that previously described</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">64</span><a href="#bookmark92" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a><span class="s169">65</span>. Finally, we performed a forward selection process to search for a set of DEGs that can achieve the largest C-index value based on the following procedures. We chose the gene that had a C-index <span class="s163">&gt; </span>0.7 and expression fold change <span class="s163">&gt;</span>2.5 or <span class="s163">&lt;−</span>2.5 as a seed (refers <span class="s161">TM4SF1</span>). The rest of the DEGs (<span class="s161">n </span><span class="s163">= </span>148) were then added to the signature, one at a time. Each time, we screened all of the rest of the genes one by one, then evaluated the C-index of the potential signature after adding a specific gene, and finally picked the gene that reached the highest C-index to the signature. We repeated this process until the C-index had reached a plateau and did not increase any more. For each DEG added to the signature, we labeled each sample based on the expression level of the corresponding gene using the following equation:</p><p class="s172" style="text-indent: 0pt;line-height: 5pt;text-align: left;">;</p><p style="text-indent: 0pt;text-align: left;"/><p class="s174" style="text-indent: 0pt;line-height: 9pt;text-align: left;"><span class="p">�</span>1 <span class="s190"></span><span class="p"> </span><span class="s180">I</span><span class="p">ð</span>g<span class="p">Þ</span>; <span class="s180">V</span>g;j &gt; median <span class="s180">V</span>g</p><p style="text-indent: 0pt;text-align: left;"/><p class="s174" style="padding-left: 57pt;text-indent: 0pt;text-align: left;"><span class="s199">For</span> gene g; <span class="s180">L</span><span class="s200">g</span><span class="s172"> j  </span><span class="s201">¼</span><span class="p"> </span><span class="s202">｛</span><span class="s203"> </span>1 <span class="s190"></span><span class="p"> </span><span class="s180">I</span><span class="p">ð</span>g<span class="p">Þ</span>; <span class="s180">V</span>g;j <span class="s176">≤</span>median<span class="s202">（</span><span class="s204">（</span><span class="s180">V</span>g<span class="s202">）</span><span class="s204">）</span></p><p class="s169" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark96" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">generate a sub-BAM file for each Monocle-defined cell cluster. Mutect2 (v.4.1.0.0)</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">69</span> <span class="p">was then applied to the sub-BAM files to identify somatic point variants. The Mutect2 outputs were run through our pipeline for filtering and annotation.</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Briefly, only Mutect2 calls located at the 3<span class="s163">ʹ </span>UTR and marked as ‘PASS’ were selected and taken into the next step. Variants with total read coverage <span class="s163">&lt; </span>30, variant read coverage <span class="s163">&lt; </span>6 or variant allelic fraction <span class="s163">&lt; </span>0.1 were removed. After that, common variants reported by the Phase-3 1000 Genomes Project or ExAC (the Exome Aggregation Consortium) with minor allele frequency greater than 0.5% were further removed. Additionally, we included a virtual normal panel of 33 germline samples from GAP-PC patients to help remove artifacts related to sequencing and mapping errors as well as common single nucleotide polymorphisms. The events that overlapped with variants called from this virtual normal panel were further excluded. Finally, the remaining somatic variants were carefully reviewed on the Integrative Genomics Viewer and variants with noisy background were further discarded. For variant-overlapping analysis at cluster level, we first made a unique list of variants by aggregating all quality-control-passed variants from the Monocle-defined tumor cell clusters of a sample. Then we queried their corresponding sub-BAM files for each unique variant site by chromosome and coordinates and the numbers of reference and variant alleles were counted, which were subsequently used to identify shared and unique variants among tumor cell clusters.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s162" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Public datasets. <a href="#bookmark39" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">In addition to the scRNA-seq dataset generated internally for the discovery GAC-PC cohort, we included the bulk transcriptome sequencing (RNA-seq) data generated on an independent GAC-PC cohort from our recent study</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">9</span><span class="s169"> </span><a href="https://gdc.cancer.gov/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">to validate the 12-gene prognostic signature. Moreover, we downloaded the normalized bulk RNA-seq data generated by TCGA on primary stomach adenocarcinoma (STAD) from the NCI Cancer Genomic Data Commons (NCI-GDC: </a><span class="s192">https://gdc.cancer.gov</span><span class="p">). The RNA-seq data were processed and normalized by the NCI-GDC bioinformatics team using their transcriptome analysis pipeline. The clinical annotation of the TCGA STAD cohort was</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="316" height="48" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_172.png"/></span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 86%;text-align: left;"><a href="#bookmark97" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">downloaded from a recent PanCanAtlas study</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">70</span>. The TCGA STAD cohort (<span class="s161">n </span><span class="s163">= </span>411) included both intestinal (<span class="s161">n </span><span class="s163">= </span>176) and diffuse type (<span class="s161">n </span><span class="s163">= </span>69) tumors.</p><p class="s169" style="text-indent: 11pt;text-align: left;"><a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">Furthermore, we downloaded three other large-scale primary GAC datasets (GSE62254 (ref. </a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">28</span><a href="#bookmark57" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">), GSE15459 (refs. </a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">27</span><a href="#bookmark98" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a>71<a href="#bookmark99" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">), GSE84437 (ref. </a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">72</span><span class="p">)) from</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"/><p style="padding-left: 7pt;text-indent: 0pt;line-height: 7pt;text-align: left;">where <span class="s161">I</span>(<span class="s161">g</span>)<span class="s163">= </span>1 if gene <span class="s161">g </span>is highly expressed in samples of the gastric-dominant</p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">group; otherwise, <span class="s161">I</span>(<span class="s161">g</span>)<span class="s163">= −</span>1.</p><p class="s161" style="padding-left: 18pt;text-indent: 0pt;line-height: 9pt;text-align: left;">L<span class="p">, </span>g<span class="p">, </span>j <span class="p">and </span>V <span class="p">denote label, gene, </span>j<span class="p">th sample and gene expression, respectively.</span></p><p style="padding-left: 7pt;text-indent: 11pt;text-align: left;">We then summed the scores for each sample and quantified the predictive performance of each derived signature based on its corresponding C-index value. Among all of the signatures derived from the seed, a 12-gene signature showed the highest C-index and was chosen for subsequent validation analyses. As we started from a relatively small number of carefully filtered DEGs (<span class="s161">n </span><span class="s163">= </span>149), no further statistic was applied to limit the size of the gene set.</p><p style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s161">Validation of the 12-gene prognostic signature. </span><a href="#bookmark93" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">The signature was then subject to validation with both internally generated and publicly available datasets. Briefly, the signature score was calculated for each sample using a similar approach as that used by Kang et al.</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">66</span>. The workflow is illustrated in Supplementary Fig. 15. First, a sample–gene expression matrix (for 12 signature genes) was extracted from each normalized bulk RNA-seq or expression microarray dataset. Second, for each sample, a score of 1 or <span class="s163">−</span>1 was assigned for each of the 12 signature genes based on its relative expression (<span class="s163">&gt; </span>or <span class="s163">≤ </span>median value) and whether the signature gene was associated with gastric-dominant or GI-mixed features. Briefly, if the gene was among one of the seven genes that are associated with the gastric-dominant feature and its expression in a sample was less than or equal to the median value, we assigned a score of 1 for this gene for this specific sample, and we assigned a score of <span class="s163">−</span>1 if its expression was greater than the median value. If the gene was</p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 7pt;text-align: left;"><a href="https://www.ncbi.nlm.nih.gov/geo/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">the Gene Expression Omnibus database (GEO, </a><a href="https://www.ncbi.nlm.nih.gov/geo/" target="_blank">https://www.ncbi.nlm.nih.gov/</a></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><span style=" color: #3B699E;">geo/</span>) to further evaluate the prognostic power of the 12-gene signature. The raw gene expression values from microarray experiments were preprocessed (background-corrected and log<span class="s170">2</span><a href="#bookmark100" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">-transformed) and quantile-normalized using the robust multi-array average algorithm</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">73</span>. For each sample, the expression</p><p class="s192" style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62254" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">measurements of all probes corresponding to the same gene ID were averaged to obtain a single measurement. For datasets </a>GSE62254 <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15459" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">and </a>GSE15459<a href="#bookmark57" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">, the clinical, histopathological and survival data as well as molecular signatures defined by each study</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">27</span><a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a><span class="s169">28</span><a href="#bookmark98" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a><span class="s169">71 </span><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62254" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">were downloaded and used for the multivariate Cox regression analysis. These primary GAC datasets included both intestinal and diffuse types of GACs. The dataset </a>GSE62254 <span style=" color: #000;">(</span><span class="s161">n </span><span class="s163">= </span><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15459" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">300) included 134 diffuse types, 146 intestinal types and 20 mixed types of GACs. The dataset </a>GSE15459 <span style=" color: #000;">(</span><span class="s161">n </span><span class="s163">= </span><span style=" color: #000;">192) included 75 diffuse types, 99 intestinal types and 18 mixed types of GACs.</span></p><p class="s192" style="padding-left: 7pt;text-indent: 11pt;text-align: left;"><a href="https://data.humancellatlas.org/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">Additionally, to test the reliability of the HCL resource as a reference dataset and to evaluate the performance of our approach in cell lineage inference, we downloaded from the Data Portal of Human Cell Atlas (SCP259, </a><a href="https://data.humancellatlas.org/" class="a" target="_blank">https://data. </a>humancellatlas.org<a href="#bookmark44" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">) the scRNA-seq dataset generated on normal human colon tissues using a SMART-Seq2 protocol by a recent study</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">14</span><span style=" color: #000;">. The same approach as outlined in the section ‘Cell-of-origin inference’ was applied to the SCP259 dataset for cell lineage inference.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s162" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Statistical analysis. <span class="p">In addition to the bioinformatics approaches described above for scRNA-seq data analysis, all other statistical analyses were performed using</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">statistical software R v.3.5.2. Analysis of differences on a continuous variable (for example, gene expression, pathway score) across two groups (a categorical</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">independent variable, such as gastric-dominant versus GI-mixed) was performed  by the nonparametric Mann–Whitney <span class="s161">U </span>test. The nonparametric Kruskal–Wallis test was applied to assess the significant difference on a continuous variable by a categorical independent variable with multiple groups (for example, across different tumor cell lineages/types). For survival analyses, including overall survival (OS), progression-free interval, disease-free survival (DFS), disease-specific survival (DSS), disease-free interval (DFI) and survival time from peritoneal metastasis, we used the log-rank test to calculate <span class="s161">P </span>values between groups, and the Kaplan–Meier method to plot survival curves. For the TCGA dataset, the clinical annotation</p><p class="s169" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark97" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">and the times calculated for OS, DFS, DSS and DFI were downloaded from the PanCanAtlas study</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">70</span><a href="#bookmark57" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">. For other large-scale primary GAC datasets downloaded from GEO, the OS times were downloaded from their corresponding published studies</a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">27</span><a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a>28<a href="#bookmark98" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a>71<a href="#bookmark99" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a>72<span class="p">. The hazard ratios were calculated using the multivariate Cox proportional-hazards model. All statistical significance testing in this study was two-sided. To control for multiple hypothesis testing, we applied the Benjamini– Hochberg method to correct </span><span class="s161">P </span><span class="p">values and the FDR </span><span class="s161">Q </span><span class="p">values were calculated.</span></p><p style="padding-left: 7pt;text-indent: 0pt;line-height: 9pt;text-align: left;">Results were considered statistically significant at <span class="s161">P </span>value or FDR <span class="s161">Q </span>value of <span class="s163">&lt;</span>0.05.</p><p style="padding-top: 7pt;padding-left: 7pt;text-indent: 0pt;line-height: 93%;text-align: left;"><b>Statistics and reproducibility. </b>Supplementary Fig. 2b was generated from <span class="s161">n </span><span class="s163">= </span>100 repeated k-BET runs. A random sampling of cells (<span class="s161">n </span><span class="s163">= </span>100 times) was performed to generate the simulated doublets in Supplementary Fig. 6b, to calculate the Bhattacharyya pairwise distance between tumor cell clusters from samples of</p><p class="s192" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><a href="#bookmark26" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">long- and short-term survivors in Extended Data Fig. 2c and to calculate the Bhattacharyya distance between and within inferred cell lineages in Extended Data Fig. 6. The statistical methods used for each analysis are described within the figure legends. The key findings of this study were validated by analyzing large-scale public datasets as described above in the section ‘Public datasets’. For the histology image shown in Fig. </a>1f<span style=" color: #000;">: because of the nature of clinical care in this disease,</span></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">only one diagnostic biopsy specimen per patient was available. Obviously, this is representative of the tumor and sufficient for making clinical decisions, but is also used here for analysis that parallels how we practice in the clinic. Keeping with the theme of our report on ITH, it is likely that metaplasia has been overgrown</p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;">by tumor cells; however, the phenotypic appearance of metaplasia and the classic appearance of goblet cells are highly reliable patterns and true representations of the presence of a premalignant element in a particular GAC. Lack of metaplasia in the primary specimen, however, does not exclude its previous existence (sampling errors or abolition by eventual GAC).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="328" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_173.png"/></span></p><p class="s192" style="padding-left: 1pt;text-indent: 0pt;line-height: 7pt;text-align: left;">org/<a href="https://ega-archive.org/studies/EGAS00001004443" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">). The data can be accessed under the accession number </a>EGAS00001004443<span style=" color: #000;">.</span></p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 1pt;text-indent: 0pt;line-height: 8pt;text-align: left;">All single-cell RNA-sequencing data generated by this study have been be</p><p style="text-indent: 0pt;text-align: left;"/><p class="s162" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Reporting Summary. <span class="p">Further information on research design is available in the Nature Research Reporting Summary linked to this article.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s21" style="padding-left: 1pt;text-indent: 0pt;line-height: 11pt;text-align: left;">Data availability</p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 11pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 1pt;text-indent: 0pt;text-align: left;"><a href="https://ega-archive.org/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">deposited in the European Genome-Phenome Archive (EGA, </a><a href="https://ega-archive.org/" target="_blank">https://ega-archive.</a></p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 10pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 1pt;text-indent: 0pt;text-align: left;">Bulk mRNA-seq expression data (normalized) generated by The Cancer Genome</p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 9pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s192" style="padding-left: 1pt;text-indent: 0pt;text-align: left;"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84437" class="a" target="_blank">GSE84437</a> <a href="#bookmark99" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">(ref. </a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">72</span><span style=" color: #000;">)) were downloaded from the Gene Expression Omnibus (GEO)</span></p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 10pt;text-align: left;"/><p class="s192" style="padding-left: 1pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a href="https://www.ncbi.nlm.nih.gov/geo/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">database (</a>https://www.ncbi.nlm.nih.gov/geo/<span style=" color: #000;">).</span></p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 9pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><span><img width="328" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_174.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="328" height="14" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_175.png"/></span></p><p class="s192" style="padding-left: 1pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62254" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">large-scale primary GAC datasets (</a><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62254" class="a" target="_blank">GSE62254</a> <a href="#bookmark58" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">(ref. </a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">28</span><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15459" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">) and </a><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15459" class="a" target="_blank">GSE15459</a> <a href="#bookmark57" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;">(refs. </a><span style=" color: #3B699E; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; vertical-align: 2pt;">27</span><a href="#bookmark98" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt;">,</a><span class="s169">71</span><span style=" color: #000;">),</span></p><p style="text-indent: 0pt;text-align: left;"/><p class="s192" style="padding-left: 1pt;text-indent: 0pt;line-height: 7pt;text-align: left;"><a href="https://gdc.cancer.gov/" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">Cancer Genomic Data Commons (NCI-GDC: </a>https://gdc.cancer.gov<span style=" color: #000;">). Three</span></p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-left: 1pt;text-indent: 0pt;line-height: 8pt;text-align: left;">Atlas (TCGA) on primary stomach adenocarcinoma were downloaded from NCI</p><p style="text-indent: 0pt;text-align: left;"/><p class="s21" style="padding-top: 8pt;padding-left: 7pt;text-indent: 0pt;line-height: 11pt;text-align: left;"><a name="bookmark75">References</a></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 9pt;text-align: left;"><p style="display: inline;"><a name="bookmark76">Amin, M. B. et al. </a><span class="s161">AJCC Cancer Staging Manual </span>8th edn (Springer, 2017).</p></li><li style="padding-left: 18pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark77">Amin, M. B. et al. The Eighth Edition AJCC Cancer staging manual: continuing to build a bridge from a population-based to a more ‘personalized’ approach to cancer staging. </a><span class="s161">CA Cancer J. Clin. </span><b>67</b>, 93–99 (2017).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark78">Niu, B. et al. MSIsensor: microsatellite instability detection using paired tumor-normal sequence data. </a><span class="s161">Bioinformatics </span><b>30</b>, 1015–1016 (2014).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 8pt;text-align: left;"><p style="display: inline;">Savas, P. et al. Single-cell profiling of breast cancer T cells reveals a</p><p style="padding-left: 18pt;text-indent: 0pt;line-height: 9pt;text-align: left;">tissue-resident memory subset associated with improved prognosis. <span class="s161">Nat. Med.</span></p><p class="s162" style="padding-left: 18pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a name="bookmark79">24</a><span class="p">, 986–993 (2018).</span></p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark80">Galili, T. dendextend: an R package for visualizing, adjusting and comparing trees of hierarchical clustering. </a><span class="s161">Bioinformatics </span><b>31</b>, 3718–3720 (2015).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark81">Zhang, L. et al. Lineage tracking reveals dynamic relationships of T cells in colorectal cancer. </a><span class="s161">Nature </span><b>564</b>, 268–272 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark82">Lambrechts, D. et al. Phenotype molding of stromal cells in the lung tumor microenvironment. </a><span class="s161">Nat. Med. </span><b>24</b>, 1277–1289 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark83">Sade-Feldman, M. et al. Defining T cell states associated with response to checkpoint immunotherapy in melanoma. </a><span class="s161">Cell </span><b>175</b>, 998–1013.e20 (2018).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark84">Nilsen, G. et al. Copynumber: efficient algorithms for single- and multi-track copy number segmentation. </a><span class="s161">BMC Genomics </span><b>13</b>, 591 (2012).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark85">Cillo, A. R. et al. Immune landscape of viral- and carcinogen-driven head and neck cancer. </a><span class="s161">Immunity </span><b>52</b>, 183–199.e9 (2020).</p></li><li style="padding-left: 18pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;">Azizi, E. et al. Single-cell map of diverse immune phenotypes in the breast tumor microenvironment. <span class="s161">Cell </span><b>174</b>, 1293–1308.e6 (2018).</p><p style="text-indent: 0pt;text-align: left;"><br/></p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark86">van Dijk, D. et al. Recovering gene interactions from single-cell data using data diffusion. </a><span class="s161">Cell </span><b>174</b>, 716–729.e27 (2018).<a name="bookmark87">&zwnj;</a></p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark88">Hicks, S. C., Townes, F. W., Teng, M. &amp; Irizarry, R. A. Missing data and technical variability in single-cell RNA-sequencing experiments. </a><span class="s161">Biostatistics </span><b>19</b>, 562–578 (2018).</p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark89">Hanzelmann, S., Castelo, R. &amp; Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. </a><span class="s161">BMC Bioinforma. </span><b>14</b>, 7 (2013).</p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark90">Cline, M. S. et al. Integration of biological networks and gene expression data using Cytoscape. </a><span class="s161">Nat. Protoc. </span><b>2</b>, 2366–2382 (2007).</p></li><li style="padding-left: 17pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark91">Harrell, F. E. Jr., Lee, K. L. &amp; Mark, D. B. Multivariable prognostic models: issues in developing models, evaluating assumptions and adequacy, and measuring and reducing errors. </a><span class="s161">Stat. Med. </span><b>15</b>, 361–387 (1996).</p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;">Peng, F. et al. Differential expression analysis at the individual level reveals a lncRNA prognostic signature for lung adenocarcinoma. <span class="s161">Mol. Cancer </span><b>16</b>,</p><p style="padding-left: 17pt;text-indent: 0pt;line-height: 8pt;text-align: left;"><a name="bookmark92">98 (2017).</a></p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark93">Lau, S. K. et al. Three-gene prognostic classifier for early-stage non small-cell lung cancer. </a><span class="s161">J. Clin. Oncol. </span><b>25</b>, 5562–5569 (2007).</p></li><li style="padding-left: 17pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark94">Kang, J., D’Andrea, A. D. &amp; Kozono, D. A DNA repair pathway–focused score for prediction of outcomes in ovarian cancer treated with platinum-based chemotherapy. </a><span class="s161">J. Natl Cancer Inst. </span><b>104</b>, 670–681 (2012).</p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark95">Newman, A. M. et al. Robust enumeration of cell subsets from tissue expression profiles. </a><span class="s161">Nat. Methods </span><b>12</b>, 453–457 (2015).</p></li><li style="padding-left: 17pt;text-indent: -11pt;line-height: 93%;text-align: left;"><p style="display: inline;"><a name="bookmark96">Becht, E. et al. Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression. </a><span class="s161">Genome Biol. </span><b>17</b>, 218 (2016).</p></li><li style="padding-left: 16pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark97">Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. </a><span class="s161">Nat. Biotechnol. </span><b>31</b>, 213–219 (2013).</p></li><li style="padding-left: 16pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark98">Liu, J. et al. An integrated TCGA pan-cancer clinical data resource to drive high-quality survival outcome analytics. </a><span class="s161">Cell </span><b>173</b>, 400–416.e11 (2018).</p></li><li style="padding-left: 16pt;text-indent: -11pt;text-align: left;"><p style="display: inline;"><a name="bookmark99">Lei, Z. et al. Identification of molecular subtypes of gastric cancer with different responses to PI3-kinase inhibitors and 5-fluorouracil. </a><span class="s161">Gastroenterology </span><b>145</b>, 554–565 (2013).</p></li><li style="padding-left: 16pt;text-indent: -11pt;line-height: 92%;text-align: left;"><p style="display: inline;"><a name="bookmark100">Yoon, S. J. et al. Deconvolution of diffuse gastric cancer and the suppression of CD34 on the BALB/c nude mice model. </a><span class="s161">BMC Cancer </span><b>20</b>, 314 (2020).</p></li><li style="padding-left: 16pt;text-indent: -11pt;line-height: 8pt;text-align: left;"><p style="display: inline;">Irizarry, R. A. et al. Summaries of Affymetrix GeneChip probe level data.</p></li></ol><p class="s161" style="padding-left: 16pt;text-indent: 0pt;line-height: 10pt;text-align: left;">Nucleic Acids Res. <span class="s162">31</span><span class="p">, e15 (2003).</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s21" style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Acknowledgements</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">This study was supported by the start-up research fund provided to L.W. by the UT MD Anderson Cancer Center (MDACC); the Andrew Sabin Family Fellowship Program</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">to L.W. by the Andrew Sabin Family Foundation; the DOD grants no. CA150334 and no. CA160445 to J.A.A.; the DOD grants no. CA160433 and no. CA170906 to S.S.; and the generous support from the Caporella, Dallas, Sultan, Park, Smith, Frazier, Oaks, Vanstekelenberg, Planjery, McNeil, Hyland and Cantu families; as well as from the Schecter Private Foundation, the Rivercreek Foundation, the Kevin Fund, the Myer Fund, the Stupid Strong Foundation, the V. Foundation, the Dio Fund, the Milrod Fund and the MDACC multidisciplinary grant programs. This study was also supported by the NIH grant no. 1S10OD024977-01 Award to the Advanced Technology Genomics Core (ATGC) and the Core grant no. CA016672 (ATGC). We thank E. J. Thompson and</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">D. P. Pollock from the ATGC for their excellent technical assistance. We thank all of the patients who participated in this study.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s21" style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Author contributions</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">L.W. and J.A.A. conceived and jointly supervised the study. S.S., K.H., M.P.P., M.Z., G.T., N.S., A.A.F.A., B.D.B. and M.B.M. contributed to sample collection and processing and collection of patient clinical information. A.J.L., J.S.E. and S.R.-C. contributed to pathology review. Y.L. reviewed the CT images. L.W. supervised the bioinformatics</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">data analysis, data integration and interpretation. R.W. contributed to sequencing data processing, integrative analyses and generation of figures and tables for the manuscript. M.D., G.H., F.W., S. Zhang., D.H., S. Zhao., Y.W., X.S., Y.C., J.Z., M.L. and K.C. assisted</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">with data processing and analysis. L.W., J.A.A. and R.W. wrote the manuscript. L.W., J.A.A., R.W., A.J.L., A.F., S.H., G.A.C. and G.P. revised the manuscript.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s21" style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Competing interests</p><p class="s165" style="padding-left: 5pt;text-indent: 0pt;line-height: 8pt;text-align: left;">The authors declare no competing interests.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s21" style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Additional information</p><p class="s164" style="padding-left: 5pt;text-indent: 0pt;line-height: 8pt;text-align: left;">Extended data <a href="https://doi.org/10.1038/s41591-020-1125-8" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank">is available for this paper at </a><span class="s206">https://doi.org/10.1038/s41591-020-1125-8</span><span class="s165">.</span></p><p class="s164" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">Supplementary information <a href="https://doi.org/10.1038/s41591-020-1125-8" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank">is available for this paper at </a><a href="https://doi.org/10.1038/s41591-020-1125-8" class="s205" target="_blank">https://doi.org/10.1038/ </a><span class="s206">s41591-020-1125-8</span><span class="s165">.</span></p><p class="s164" style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Correspondence and requests for materials <span class="s165">should be addressed to J.A.A. or L.W.</span></p><p class="s164" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 108%;text-align: left;">Peer review information <span class="s165">Javier Carmona was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</span></p><p class="s164" style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Reprints and permissions information <a href="http://www.nature.com/reprints" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;" target="_blank">is available at </a><span class="s206">www.nature.com/reprints</span><span class="s165">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 18pt;text-indent: 0pt;text-align: left;"><span><img width="651" height="866" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_176.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">Extended Data Fig. 1 | See next page for caption.</h4><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark16">Extended Data Fig. 1 | </a><span class="s18">A single cell transcriptome map of PC. </span>a<span class="s18">, t-SNE (t-distributed stochastic neighbor embedding) plots showing unbiased clustering analysis of 45,048 single cells that passed quality control in this study. Each dot represents a single cell. Cells are color coded for (left to right): the associated cell types, cell clusters, the corresponding patient origins, and survival status. </span>b<span class="s18">, t-SNE as in a, showing expression of canonical marker genes used for cell types assignment.</span></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 51pt;text-indent: 0pt;text-align: left;"><span><img width="565" height="688" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_177.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark17"><b>Extended Data Fig. 2 | </b></a>Relationships between tumor cell clusters and correlation with patient survival. <b>a</b>, the UMAP (uniform manifold approximation and projection) plot of PC tumor cells, showing the global data structure. Tumor cell clusters from short-term survivors appeared closer to each other on the UMAP plot than to cell clusters from long-term survivors. <b>b</b>, the dendrogram showing relationships between tumor cell clusters. <b>c</b>, the Bhattacharyya pairwise distance between tumor cell clusters from samples of long and short-term survivors. Overall, the pairwise distance between clusters of long and short survivors was significantly larger than that within the clusters of Short or random, indicating distinct transcriptomic profiles associated with survival. Each dot represents one sampling, in totally 100 times. Box, median <span class="s26">± </span>interquartile range. Whiskers, the minimum and maximum values. <i>P </i>values were calculated by a two-sided Wilcoxon rank sum test with Benjamini-Hochberg correction. <i>P </i><span class="s26">&lt; </span>2.2e-16 represents a P value approaching 0.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 20pt;text-indent: 0pt;text-align: left;"><span><img width="646" height="348" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_178.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark18">Extended Data Fig. 3 | </a><span class="s18">Cell lineage assignment was not confounded by differences in cell cycle states. The histograms showing tumor cell lineage compositions before (top) and after (bottom) regressing out cell cycle-related genes, respectively.</span></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;"><span><img width="609" height="302" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_179.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark19">Extended Data Fig. 4 | </a><a href="#bookmark26" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">unsupervised clustering analysis revealed inter-patient and intra-tumoral transcriptome heterogeneity in PC tumor cells. The UMAP plots showing unsupervised clustering analysis of tumor cells (using Seurat) from 14 samples underwent HCL mapping and cell lineage inference as in Fig. </a><span style=" color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">1g</span><a href="#bookmark26" style=" color: black; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">. Cells are colored by their corresponding cluster IDs (left) and sample origins (right). Dashed circles highlight samples that formed two or more tumor cell clusters (related to Fig. </a><span style=" color: #3B699E; font-family:Calibri, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt;">1g</span><span class="s18">).</span></h4><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><span><img width="699" height="810" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_180.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark20"><b>Extended Data Fig. 5 | </b></a>SC3 unsupervised clustering analysis of PC tumor cells by patient. SC3 results of 3 representative patients are shown. Each column represents a cell. The lineage annotation is shown in the top annotation track. The fractions of intestinal cells (IP-067, IP-073) or stomach pit cells (IP-009) in each SC3 defined cell clusters are labelled at the top. Some of the representative marker genes of intestine and stomach origins are labelled on the right. Two-sided proportion tests were performed between C1 and C4 (IP-067), C1 and C3 (IP-073), and C1 and C2 (IP-009), and all are significant (<i>P </i><span class="s26">&lt; </span>2.2e-16).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 8pt;text-indent: 0pt;text-align: left;"><span><img width="676" height="812" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_181.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark21"><b>Extended Data Fig. 6 | </b></a>The Bhattacharyya distance between and within inferred cell lineages. The Bhattacharyya pairwise distance between different tumor cell lineages was computed as previously described (see Methods). Only the major lineages that had 500 or more cells were included in the analysis. The Bhattacharyya distance between cells of the same lineage and the Bhattacharyya distance between cells randomly sampled independent of lineage annotation (random) was also computed to provide background distributions for statistical comparison. Each dot represents one sampling, in total 100 times. Box, median <span class="s26">± </span>interquartile range. Whiskers, the minimum and maximum values. P values were calculated by a two-sided Wilcoxon rank-sum test with Benjamini-Hochberg correction. <i>P </i><span class="s26">&lt; </span>2.2e-16 represents a P value approaching 0.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 133pt;text-indent: 0pt;text-align: left;"><span><img width="344" height="291" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_182.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark22">Extended Data Fig. 7 | </a><span class="s18">Representative examples of somatic variants identified on 3’uTR using scRNA-seq data. Integrative Genomics Viewer (IGV) was used for visualization of the QC-passed somatic variants. The Bam files of Monocle defined cell clusters C1, C2, C3 of sample IP-067 were loaded to IGV and snapshots of 3’UTr mutations are shown for representative events: somatic mutations shared by PC tumor cells from all three clusters (top); mutations shared by only two of the three clusters (bottom left and middle), and mutations that were unique to one of the three clusters (bottom right)</span></h4><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;">are shown. For each representative mutation across Monocle cell clusters, the gene name, chromosome, start position, base change, total read coverage, and tumor variant allele fraction (TAF) are shown. Total_dp: total read depth.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 42pt;text-indent: 0pt;text-align: left;"><span><img width="586" height="770" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_183.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark23">Extended Data Fig. 8 | </a><span class="s18">Prognostic significance of 12-gene signature in TCGA primary gastric cancer cohort and correlation with molecular subtypes and clinical variables. </span>a<span class="s18">, Disease-specific survival (DSS, left) and progression-free interval (PFI, right) of patients whose PCs were in the GI-mixed and gastric- dominant groups defined by expression of the 12-gene signature. The analyses were performed with the Kaplan–Meier estimates and two-sided log-rank tests. Twenty-five out of 411 patients whose DSS information were not available were excluded from survival analysis. </span>b<span class="s18">, the alluvial plots display relationships between the PC subtypes defined by the 12-gene signature (left strip) and the molecular subtypes defined by TCGA multi-omic analysis (left), tumor stages (middle), histology types (right), and presence of local recurrence and/or distant metastasis (c). N.S., not statistically significant.</span></h4><p class="s25" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">P <span class="s18">value for alluvial plots were calculated by a two-sided Fisher’s Exact test.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 67pt;text-indent: 0pt;text-align: left;"><span><img width="520" height="644" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_184.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark24">Extended Data Fig. 9 | </a><span class="s18">Validation of the 12-gene signature in a large-scale localized GAC cohort from Cristescu R, et al. </span>a<span class="s18">, The multivariate Cox proportional hazard model analysis. The 12-gene signature, clinical and histopathological variables as well as the molecular signatures defined by the original study were included. For each variable, the reference level is the first one. Block in center of error bars represent the weighted mean. Whiskers of error bars represent the 95% confidence interval. </span>b<span class="s18">, (left) Alluvial plot shows the relationships between the PC subtypes (left strip) and the molecular signatures (right strip). The two-sided Fisher’s Exact test was used to calculate the P values and asterisks indicate significant enrichment events.</span></h4><p class="s18" style="padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;">(right) The 12-gene signature scores were calculated and compared across the four molecular groups defined by the original the study. Box, median <span class="s26">± </span>interquartile range. Whiskers, 1.5X interquartile range. <i>P </i>value was calculated by one-way Kruskal-Wallis rank-sum test.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 105pt;text-indent: 0pt;text-align: left;"><span><img width="419" height="520" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_185.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s18" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;line-height: 107%;text-align: left;"><a name="bookmark25"><b>Extended Data Fig. 10 | </b></a>Validation of the 12-gene signature in a large-scale localized GAC cohort from Ooi CH, et al. <b>a</b>, The multivariate Cox proportional hazard model analysis. The 12-gene signature, clinical and histopathological variables as well as the molecular signatures defined by the original study were included. For each variable, the reference level is the first one. Block in center of error bars represent the weighted mean. Whiskers of error bars represent the 95% confidence interval. <b>b</b>, (left) Alluvial plot shows the relationships between the PC subtypes (left strip) and the molecular signatures (right strip). The two-sided Fisher’s Exact test was used to calculate the P values and asterisks indicate significant enrichment events. (right) The 12-gene signature scores were calculated and compared across the four molecular groups defined by the original the study. Box, median <span class="s26">± </span>interquartile range.</p><p class="s18" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Whiskers, 1.5X interquartile range. <i>P </i>value was calculated by one-way Kruskal-Wallis rank-sum test.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="29" height="1055" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_186.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s211" style="padding-top: 3pt;padding-left: 6pt;text-indent: 0pt;text-align: left;"><span class="s207">nature </span><span class="s208">research </span><span class="s209">Corresponding author(s) </span><span class="s210">: </span>Jaffer A. A<span class="s212">」</span>ani <span class="s213">&amp; </span>Linghua Wang                     </p><p class="s209" style="padding-top: 2pt;padding-left: 292pt;text-indent: 0pt;text-align: left;">Last updated by author(s): Sep <span style=" color: #212121;">24, </span>2020</p><p style="padding-left: 390pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="203" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_187.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s215" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">Reporting Summary                                                                          </p><p class="s209" style="padding-top: 8pt;padding-left: 8pt;text-indent: 0pt;line-height: 115%;text-align: left;">Nature Research wishes to improve the reproducib <span class="s216">巾 </span>ty of the work that we publish<span style=" color: #3A3B3B;">. </span>This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see our <span style=" color: #086697;">Editorial Policies </span>and the <span style=" color: #086697;">Editorial Policy Checklist</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s219" style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: left;"> Statistics                                                                                                                                         </p><p class="s209" style="padding-top: 4pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">For all statistical analyses, confirm that the following items are present in the figure <span style=" color: #212121;">legend, </span>table legend, main text, or Methods section</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s209" style="padding-left: 7pt;text-indent: 0pt;text-align: left;">n/a Confirmed</p><p class="s220" style="padding-top: 2pt;padding-left: 7pt;text-indent: 0pt;line-height: 17pt;text-align: left;">口 <span style=" color: #3A3B3B;">因 </span><span class="s209">The exact sample size (n) for each experimental group/condition, given as a discrete number and unit of measurement</span></p><p class="s220" style="padding-left: 7pt;text-indent: 0pt;line-height: 17pt;text-align: left;">口 <span style=" color: #3A3B3B;">因</span><span class="s209">A statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly</span></p><p class="s220" style="padding-left: 8pt;text-indent: 0pt;line-height: 16pt;text-align: left;">口<span style=" color: #3A3B3B;">因 </span><span class="s209">The statistical test(s) used AND whether they are one- or two-sided</span></p><p class="s209" style="padding-left: 41pt;text-indent: 0pt;line-height: 8pt;text-align: left;">Only common tests should be described solely by name; describe more complex techniques in the Methods <span style=" color: #212121;">section</span></p><p class="s209" style="padding-top: 1pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s220">口 </span><span class="s221">因</span>A descrip.t1on o<span class="s222">f </span>a<span class="s222">II </span>covariates tested</p><p class="s209" style="padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s220">口</span><span class="s221">因</span>A description of any assumptions or corrections, such as tests of normality and ad<span class="s223">」</span>ust ment for multiple comparisons</p><p class="s220" style="padding-top: 1pt;padding-left: 41pt;text-indent: -33pt;text-align: left;">口 <span style=" color: #3A3B3B;">因 </span><span class="s209">A full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence </span><span class="s214">intervals)</span></p><p class="s209" style="padding-top: 4pt;padding-left: 41pt;text-indent: -33pt;line-height: 87%;text-align: left;"><span class="s224">口</span><span class="s225">凶 </span>For null hypothesis testing, the test statistic (e. g<span style=" color: #3A3B3B;">. </span>F, t, r) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable.</p><p class="s221" style="padding-top: 3pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">因<span style=" color: #646B64;">口 </span><span class="s209">For Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings</span></p><p class="s220" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">口 <span style=" color: #3A3B3B;">因</span><span class="s209">For hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes</span></p><p class="s220" style="padding-left: 8pt;text-indent: 0pt;text-align: left;">口 <span style=" color: #212121;">因 </span><span class="s214">Esti mates </span><span class="s209">of effect sizes (e.g. Cohen&#39;s d, Pearson&#39;s r), indicating how they were calculated</span></p><p class="s210" style="padding-top: 3pt;text-indent: 0pt;text-align: center;">Our web collection on <span style=" color: #1A72A0;">statistics for bioloq1sts </span>contains articles on many of the points above</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s219" style="padding-left: 5pt;text-indent: 0pt;text-align: left;"> Software and code                                                                                                                              </p><p class="s218" style="padding-top: 6pt;padding-left: 8pt;text-indent: 0pt;text-align: left;"><span style=" color: #646B64;">Policy information about </span>availa b<span class="s228">山</span>ty of computer code</p><p class="s209" style="padding-top: 5pt;padding-left: 81pt;text-indent: -66pt;text-align: left;">Data collection All <span style=" color: #212121;">single </span>cell RNA <span style=" color: #212121;">sequencing </span>data were generated locally in the ATGC Core at <span style=" color: #212121;">U.T. </span>MD Anderson <span style=" color: #212121;">Cancer </span>Center. Chromium Single cell <span style=" color: #212121;">sequencing </span>technology from lOX Genomics was used for <span style=" color: #212121;">library </span>construction following the manufacturer&#39;s guidelines and all libraries were <span style=" color: #212121;">sequenced </span>on the <span style=" color: #212121;">same </span>Nova5eq6000 flow cell.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s209" style="padding-left: 81pt;text-indent: -66pt;text-align: left;">Data analysis Data analysis were performed using CellRanger v2.l.1, R v3.6.0, Seurat v3<span style=" color: #3A3B3B;">.</span>0.2 (R package), k-BET(R package), <span style=" color: #212121;">scHCL </span>v0.1.1, <span style=" color: #212121;">inferCNV </span>vl.0.3, pheatmap (R package), SC3(R package), phangorn (R package), copynumbe <span class="s229">「 </span>(R package), <span style=" color: #212121;">survcomp(R </span>package), CIBERSORT, and MCP-counter (R package)</p><p class="s210" style="padding-top: 4pt;padding-left: 7pt;text-indent: 0pt;line-height: 91%;text-align: left;">For manuscripts utilizing cus<span style=" color: #7E857E;">t </span>om algorithms or softwa<span class="s231">「 </span>e that are cent<span class="s231">「</span>al to the<span class="s231">「 </span>esearch but not yet described in publ<span style=" color: #959795;">i</span>shed <span style=" color: #7E857E;">l</span>iterature<span style=" color: #7E857E;">, </span>softwa<span class="s231">「 </span>e must be made ava <span style=" color: #7E857E;">i</span>lab<span style=" color: #7E857E;">l</span>e to edito<span class="s231">「</span>sa nd reviewers. We st<span class="s231">「</span>onglyencourage code deposition in a community repository (e.g. GitHub)<span style=" color: #7E857E;">. </span>See the Nature Research <span style=" color: #1A72A0;">guide</span><span style=" color: #5295B8;">l</span><span style=" color: #1A72A0;">inesfor submitting code &amp; software </span>fo<span class="s231">「 </span>furthe<span class="s231">「 </span><span style=" color: #7E857E;">i</span>nfo<span class="s231">「</span>mat <span style=" color: #7E857E;">1</span>on</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s219" style="padding-left: 5pt;text-indent: 0pt;text-align: left;"> Data                                                                                                                                     </p><p class="s218" style="padding-top: 6pt;padding-left: 8pt;text-indent: 0pt;text-align: left;"><span style=" color: #646B64;">Policy information about </span>availab <span class="s228">山</span>ty of data</p><p class="s218" style="padding-top: 2pt;padding-left: 12pt;text-indent: 0pt;text-align: left;"><span style=" color: #0A0A0A;">All manuscripts must include a </span>data availab <span class="s234">巾 </span>ty statement. <span style=" color: #0A0A0A;">This statement should provide the following information, where applicable </span><span style=" color: #3A3B3B;">:</span></p><p class="s209" style="padding-top: 2pt;padding-left: 22pt;text-indent: 0pt;text-align: left;">-Accession codes, unique identifiers, or web links for publicly available datasets</p><ul id="l4"><li style="padding-left: 27pt;text-indent: -4pt;text-align: left;"><p class="s209" style="display: inline;">A list of figures that have associated raw data</p></li><li style="padding-left: 27pt;text-indent: -4pt;text-align: left;"><p class="s209" style="display: inline;">A description of any restrictions on data availa b<span class="s235">巾</span>ty</p></li></ul><p class="s209" style="padding-top: 7pt;padding-left: 10pt;text-indent: 0pt;text-align: left;">All <span style=" color: #212121;">single cell </span>RNA <span style=" color: #212121;">sequencing </span>data generated by this <span style=" color: #212121;">study </span>has been deposited in the European <span style=" color: #212121;">Gen ome</span><span style=" color: #3A3B3B;">-</span>PhenomeArchive (EGA, https<span style=" color: #3A3B3B;">:</span>// ega<span style=" color: #3A3B3B;">-</span>archive.o <span class="s229">「</span><span class="s236">g/). </span><span style=" color: #212121;">The </span>data can be accessed under the accession number EGAS 00001004443<span style=" color: #3A3B3B;">. </span>Bulk mRNA-seq expression data (normalized) gene<span class="s229">「</span>ated by The Cancer Genome Atlas (TCGA) on primary <span style=" color: #212121;">stomach </span>adenocarcinoma was downloaded from NCI Cancer Genomic Data Commons (NCI <span style=" color: #3A3B3B;">-</span>GDC: https://gdc.cancer.gov). <span style=" color: #212121;">Three </span>large<span style=" color: #3A3B3B;">-</span><a href="http://www.ncbi.nlm.nih.gov/geo/)" class="s237" target="_blank">scale primary GAC datasets (GSE62254, GSE15459, GSE84437) were downloaded from the Gene Expression Omnibus (GEO) database (GEO, https:// www.ncbi.nlm</a><a href="http://www.ncbi.nlm.nih.gov/geo/)" class="s238" target="_blank">.</a><a href="http://www.ncbi.nlm.nih.gov/geo/)" class="s237" target="_blank">nih.gov/geo/)</a></p><p style="text-indent: 0pt;text-align: left;"><span><img width="28" height="1054" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_188.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s239" style="padding-top: 12pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">Field-specific reporting                                                                                        </p><p class="s240" style="text-indent: 0pt;line-height: 26pt;text-align: left;">□</p><p style="text-indent: 0pt;text-align: left;"/><p class="s241" style="padding-top: 7pt;padding-left: 8pt;text-indent: 0pt;line-height: 8pt;text-align: left;">Please <span style=" color: #2D2D2D;">s</span>elect the <span style=" color: #2D2D2D;">o </span>ne belo <span style=" color: #2D2D2D;">w </span>that is the be<span style=" color: #2D2D2D;">s</span>t fit for <span style=" color: #2D2D2D;">y </span>our <span class="s243">「 </span>esear <span style=" color: #2D2D2D;">c</span>h<span style=" color: #2D2D2D;">. </span>If <span style=" color: #2D2D2D;">y</span>ou are n<span style=" color: #2D2D2D;">o</span>t sure, read the app <span class="s243">「 </span>op <span class="s243">「 </span>iat e <span style=" color: #2D2D2D;">s</span>ection <span style=" color: #2D2D2D;">s </span>before making <span style=" color: #2D2D2D;">y</span>our sele <span style=" color: #2D2D2D;">c</span>tion</p><p class="s241" style="padding-left: 8pt;text-indent: -1pt;line-height: 118%;text-align: left;"><span class="s244">区 </span><span class="s245">] </span>Life science <span style=" color: #2D2D2D;">s </span>Beha <span style=" color: #2D2D2D;">v</span>ioural <span class="s246">&amp; </span><span style=" color: #2D2D2D;">s</span>ocial sciences <span class="s247">口 </span>Ecol <span style=" color: #2D2D2D;">o </span>gic al, e<span style=" color: #2D2D2D;">vo </span>lutionar <span style=" color: #2D2D2D;">y </span><span class="s246">&amp; </span>en <span style=" color: #2D2D2D;">v</span>iro nment al sciences <span style=" color: #79807C;">For a reference copy of the document with all secti ons</span><span style=" color: #A3A8A5;">, </span><span style=" color: #79807C;">see </span><span class="s250">nature.com/documents/nr-reporting-summary-flat.pd</span><span style=" color: #3B89AF;">f</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s239" style="padding-top: 5pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">Life sciences study design                                                                            </p><p class="s252" style="padding-top: 8pt;padding-bottom: 3pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">All studies must disclose on these points even <span style=" color: #79807C;">when </span>the disclosure is negative</p><p style="padding-left: 82pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="607" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_189.png"/></span></p><p class="s241" style="padding-top: 2pt;padding-left: 82pt;text-indent: -66pt;line-height: 119%;text-align: left;"><span style=" color: #2D2D2D;">S</span>am ple size <span style=" color: #2D2D2D;">sc</span>RN <span style=" color: #2D2D2D;">A-se</span>q <span style=" color: #2D2D2D;">was </span>p<span style=" color: #2D2D2D;">e</span>rform <span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">o</span>n cr<span style=" color: #2D2D2D;">yo</span>p<span class="s243">「</span><span style=" color: #2D2D2D;">ese</span><span class="s243">「</span><span style=" color: #2D2D2D;">ve</span>d P<span style=" color: #2D2D2D;">C ce</span>ll<span style=" color: #2D2D2D;">s </span>fr<span style=" color: #2D2D2D;">o</span>m <span style=" color: #2D2D2D;">20 GAC </span>p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #3F413F;">s, </span>in cludin <span style=" color: #2D2D2D;">g 10 </span>l<span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">g-</span>t <span style=" color: #2D2D2D;">e</span><span class="s243">「</span>m <span style=" color: #2D2D2D;">a</span>nd 10 <span style=" color: #2D2D2D;">s</span>h<span style=" color: #2D2D2D;">o</span>rt <span style=" color: #3F413F;">-</span>t<span style=" color: #2D2D2D;">e</span>rm <span style=" color: #2D2D2D;">s</span>ur<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">v</span>o<span class="s243">「</span><span style=" color: #2D2D2D;">s. </span>F<span style=" color: #2D2D2D;">o</span>ll<span style=" color: #2D2D2D;">ow</span>in<span style=" color: #2D2D2D;">g q</span>u<span style=" color: #2D2D2D;">a</span>lit <span style=" color: #2D2D2D;">y </span>filt <span style=" color: #2D2D2D;">e</span>rin<span style=" color: #2D2D2D;">g, 45,0</span>4<span style=" color: #2D2D2D;">8 ce</span>ll<span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>r<span style=" color: #2D2D2D;">e</span>t<span style=" color: #2D2D2D;">a</span>in <span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">fo</span>r th<span style=" color: #2D2D2D;">e s</span>ub<span style=" color: #2D2D2D;">se</span>qu<span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">yses, </span>includin<span style=" color: #2D2D2D;">g </span>n<span style=" color: #3F413F;">=3</span>1<span style=" color: #3F413F;">,13</span>1 tum<span style=" color: #2D2D2D;">o</span>r <span style=" color: #2D2D2D;">ce</span>ll<span style=" color: #2D2D2D;">s</span></p><p style="padding-left: 82pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="607" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_190.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="617" height="58" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_191.png"/></span></p><p class="s241" style="padding-left: 81pt;text-indent: -66pt;line-height: 118%;text-align: left;">Data e<span style=" color: #2D2D2D;">x</span>clusion <span style=" color: #2D2D2D;">s Ge </span>n<span style=" color: #2D2D2D;">es </span>d<span style=" color: #2D2D2D;">e</span>t <span style=" color: #2D2D2D;">ec</span>t<span style=" color: #2D2D2D;">e</span>d in <span style=" color: #2D2D2D;">&lt;3 ce</span>ll<span style=" color: #3F413F;">s </span><span style=" color: #2D2D2D;">a</span>nd <span style=" color: #2D2D2D;">ce</span>ll<span style=" color: #2D2D2D;">s w</span>h<span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">e &lt; 2</span>00 <span style=" color: #2D2D2D;">ge</span>n<span style=" color: #2D2D2D;">es </span>h<span style=" color: #2D2D2D;">a</span>d n<span style=" color: #2D2D2D;">o</span>n<span style=" color: #3F413F;">zero </span><span style=" color: #2D2D2D;">co</span>unt <span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>filt <span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">ed </span>out <span style=" color: #2D2D2D;">a</span>nd <span style=" color: #2D2D2D;">ex</span>clud<span style=" color: #2D2D2D;">e</span>d fr<span style=" color: #2D2D2D;">o</span>m <span style=" color: #2D2D2D;">s</span>ub<span style=" color: #2D2D2D;">se</span>qu<span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">yses</span>. L<span style=" color: #2D2D2D;">ow q</span>u<span style=" color: #2D2D2D;">a</span>lit <span style=" color: #2D2D2D;">y ce</span>ll<span style=" color: #2D2D2D;">s w</span>h<span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">e </span>&gt;15<span style=" color: #2D2D2D;">% </span>of th <span style=" color: #2D2D2D;">e </span>r<span style=" color: #2D2D2D;">ea</span>d <span style=" color: #2D2D2D;">co</span>unt <span style=" color: #3F413F;">s </span>d <span style=" color: #2D2D2D;">e</span>ri<span style=" color: #2D2D2D;">ve</span>d fr<span style=" color: #2D2D2D;">o</span>m th<span style=" color: #2D2D2D;">e </span>mit<span style=" color: #2D2D2D;">oc</span>h<span style=" color: #2D2D2D;">o</span>nd<span class="s254">「</span>i<span style=" color: #2D2D2D;">a</span>l <span style=" color: #2D2D2D;">ge</span>n<span style=" color: #2D2D2D;">o</span>m<span style=" color: #2D2D2D;">e we</span>r<span style=" color: #2D2D2D;">e a</span>l<span style=" color: #2D2D2D;">so </span>di<span style=" color: #3F413F;">sca </span>rd <span style=" color: #2D2D2D;">e</span>d. Lik<span style=" color: #2D2D2D;">e</span>l<span style=" color: #2D2D2D;">y ce</span>ll d<span style=" color: #2D2D2D;">e</span>bri <span style=" color: #3F413F;">s </span><span style=" color: #2D2D2D;">a</span>nd d<span style=" color: #2D2D2D;">o</span>ubl<span style=" color: #2D2D2D;">e</span>t <span style=" color: #3F413F;">s </span><span style=" color: #2D2D2D;">we</span>r<span style=" color: #2D2D2D;">e </span>furth <span style=" color: #2D2D2D;">e</span>r r<span style=" color: #2D2D2D;">e</span>mo<span style=" color: #2D2D2D;">ve</span>d fr<span style=" color: #2D2D2D;">o</span>m <span style=" color: #2D2D2D;">a</span>ll clu<span style=" color: #2D2D2D;">s</span>t<span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">s</span>. In <span style=" color: #2D2D2D;">a</span>dditi <span style=" color: #2D2D2D;">o</span>n, fi<span style=" color: #2D2D2D;">ve </span>p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s w</span>ith t<span style=" color: #2D2D2D;">o</span>o f<span style=" color: #2D2D2D;">ew </span>P<span style=" color: #2D2D2D;">C </span>tumor <span style=" color: #2D2D2D;">ce</span>ll<span style=" color: #2D2D2D;">s (&lt;5</span>0 <span style=" color: #2D2D2D;">ce</span>ll<span style=" color: #2D2D2D;">s</span>) <span style=" color: #2D2D2D;">we</span>r<span style=" color: #2D2D2D;">e ex</span>clud<span style=" color: #2D2D2D;">e</span>d fr<span style=" color: #2D2D2D;">o</span>m <span style=" color: #2D2D2D;">s</span>ub<span style=" color: #2D2D2D;">se</span>qu<span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">yses </span>Thi<span style=" color: #3F413F;">s </span><span style=" color: #2D2D2D;">c</span>rit <span style=" color: #2D2D2D;">e</span>ri<span style=" color: #2D2D2D;">a was </span>pr<span style=" color: #2D2D2D;">e-es</span>t <span style=" color: #2D2D2D;">a</span>bli <span style=" color: #2D2D2D;">s</span>h<span style=" color: #2D2D2D;">e</span>d<span style=" color: #2D2D2D;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="617" height="84" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_192.png"/></span></p><p class="s241" style="padding-left: 81pt;text-indent: -66pt;line-height: 118%;text-align: left;">Repli <span style=" color: #2D2D2D;">c</span>ation N<span style=" color: #2D2D2D;">o re</span>pli<span style=" color: #2D2D2D;">ca</span>ti <span style=" color: #2D2D2D;">o</span>n <span style=" color: #2D2D2D;">was </span>in clud<span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">o</span>n indi<span style=" color: #2D2D2D;">v</span>idu<span style=" color: #2D2D2D;">a</span>l p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s</span>p<span style=" color: #2D2D2D;">ec</span>im <span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">s g</span>i<span style=" color: #3F413F;">ve</span>n limit <span style=" color: #2D2D2D;">e</span>d ti <span style=" color: #2D2D2D;">ss</span>u<span style=" color: #2D2D2D;">e ava</span>il <span style=" color: #2D2D2D;">a</span>b<span class="s255">巾</span>t<span style=" color: #2D2D2D;">y. </span>In <span style=" color: #2D2D2D;">a s</span>ub <span style=" color: #2D2D2D;">se </span>t <span style=" color: #2D2D2D;">o</span>f p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s </span>(n<span style=" color: #3F413F;">=6) </span><span style=" color: #2D2D2D;">w</span>h<span style=" color: #2D2D2D;">ose ge</span>n<span style=" color: #2D2D2D;">o</span>mi<span style=" color: #2D2D2D;">c </span>DN<span style=" color: #2D2D2D;">As we</span>r<span style=" color: #2D2D2D;">e ava</span>il <span style=" color: #2D2D2D;">a</span>bl <span style=" color: #2D2D2D;">e</span>, <span style=" color: #2D2D2D;">w</span>h<span style=" color: #2D2D2D;">o</span>l<span style=" color: #2D2D2D;">e-exo </span>m<span style=" color: #2D2D2D;">e se</span>qu<span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">c</span>in<span style=" color: #2D2D2D;">g (WES</span>) <span style=" color: #2D2D2D;">was </span>p<span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">fo</span>rm<span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">a</span>t 1<span style=" color: #2D2D2D;">5</span>0<span style=" color: #2D2D2D;">x a</span>nd th <span style=" color: #2D2D2D;">e WES </span>d<span style=" color: #2D2D2D;">a</span>t<span style=" color: #2D2D2D;">a was </span>u<span style=" color: #2D2D2D;">se</span>d t<span style=" color: #2D2D2D;">o va</span>lid <span style=" color: #2D2D2D;">a</span>t<span style=" color: #2D2D2D;">e </span>th<span style=" color: #2D2D2D;">e </span>inf<span style=" color: #2D2D2D;">e</span>rr<span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">C</span>N<span style=" color: #2D2D2D;">Vs </span>from <span style=" color: #2D2D2D;">sc</span>RN <span style=" color: #2D2D2D;">A­ se</span>q<span style=" color: #2D2D2D;">. We a</span>ppli<span style=" color: #2D2D2D;">e</span>d thr <span style=" color: #2D2D2D;">ee </span>ind <span style=" color: #2D2D2D;">e</span>p<span style=" color: #2D2D2D;">e</span>nd<span style=" color: #2D2D2D;">e</span>nt un<span style=" color: #2D2D2D;">s</span>u p<span style=" color: #2D2D2D;">erv</span>i<span style=" color: #2D2D2D;">se</span>d clu<span style=" color: #2D2D2D;">s</span>t<span style=" color: #2D2D2D;">e</span>rin <span style=" color: #2D2D2D;">g a</span>pp <span style=" color: #2D2D2D;">roac</span>h<span style=" color: #2D2D2D;">es </span>for <span style=" color: #2D2D2D;">cross va</span>lid <span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">o</span>n <span style=" color: #2D2D2D;">o</span>f th <span style=" color: #2D2D2D;">e </span>int <span style=" color: #2D2D2D;">es</span>ti n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #3F413F;">-</span>lik <span style=" color: #2D2D2D;">e ce</span>ll p<span style=" color: #2D2D2D;">o</span>pul<span style=" color: #2D2D2D;">a</span>tion <span style=" color: #2D2D2D;">s we o</span>b<span style=" color: #2D2D2D;">se</span>r<span style=" color: #2D2D2D;">ve</span>d In <span style=" color: #2D2D2D;">a</span>dditi <span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">, </span><span style=" color: #3F413F;">we </span>r<span style=" color: #2D2D2D;">e</span>p<span style=" color: #2D2D2D;">ea</span>t <span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">o</span>ur <span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">ys</span>i<span style=" color: #2D2D2D;">s o</span>f <span style=" color: #2D2D2D;">ce</span>ll lin<span style=" color: #2D2D2D;">eage </span>inf <span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">ce </span>multipl<span style=" color: #2D2D2D;">e </span>tim <span style=" color: #2D2D2D;">es </span>und<span style=" color: #2D2D2D;">e</span>r diff <span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">co</span>nditi <span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">s </span>t <span style=" color: #2D2D2D;">o e</span>n<span style=" color: #2D2D2D;">s</span>ur<span style=" color: #2D2D2D;">e </span>that <span style=" color: #2D2D2D;">w</span>h<span style=" color: #2D2D2D;">a</span>t <span style=" color: #2D2D2D;">we </span>h<span style=" color: #2D2D2D;">ave o</span>b<span style=" color: #2D2D2D;">se</span>r<span style=" color: #2D2D2D;">ve</span>d <span style=" color: #2D2D2D;">a</span>r<span style=" color: #2D2D2D;">e ve</span>r<span style=" color: #2D2D2D;">y </span>lik<span style=" color: #2D2D2D;">e</span>l<span style=" color: #2D2D2D;">y </span>th <span style=" color: #2D2D2D;">e </span>tru <span style=" color: #2D2D2D;">e eve</span>nt <span style=" color: #2D2D2D;">s</span>. M <span style=" color: #2D2D2D;">o</span>r<span style=" color: #2D2D2D;">eove</span>r<span style=" color: #2D2D2D;">, </span>th <span style=" color: #2D2D2D;">e </span>k<span style=" color: #2D2D2D;">ey </span>findin <span style=" color: #2D2D2D;">gs o</span>f thi <span style=" color: #2D2D2D;">s s</span>t ud<span style=" color: #2D2D2D;">y </span>includin<span style=" color: #2D2D2D;">g </span>th <span style=" color: #2D2D2D;">e </span>1<span style=" color: #2D2D2D;">2-ge</span>n<span style=" color: #2D2D2D;">e </span>p<span style=" color: #2D2D2D;">rog</span>n<span style=" color: #2D2D2D;">os</span>ti <span style=" color: #2D2D2D;">c s</span>i<span style=" color: #2D2D2D;">g</span>n <span style=" color: #2D2D2D;">a</span>tu <span style=" color: #2D2D2D;">re we</span>r<span style=" color: #2D2D2D;">e va</span>lid <span style=" color: #2D2D2D;">a</span>t<span style=" color: #2D2D2D;">e</span>d in multipl <span style=" color: #2D2D2D;">e </span>l<span style=" color: #2D2D2D;">a</span>r<span style=" color: #2D2D2D;">ge-sca </span>l<span style=" color: #2D2D2D;">e GAC co</span>hort <span style=" color: #2D2D2D;">s </span>in cludin<span style=" color: #2D2D2D;">g </span>b<span style=" color: #2D2D2D;">o</span>th <span style=" color: #2D2D2D;">o</span>ur in<span style=" color: #3F413F;">-</span>h<span style=" color: #2D2D2D;">o</span>u<span style=" color: #2D2D2D;">se a</span>nd publi<span style=" color: #2D2D2D;">c </span>d<span style=" color: #2D2D2D;">a</span>t <span style=" color: #2D2D2D;">ase </span>t<span style=" color: #2D2D2D;">s</span>, thu <span style=" color: #2D2D2D;">s ex</span>cludin<span style=" color: #2D2D2D;">g </span>r<span style=" color: #2D2D2D;">e</span>produ <span style=" color: #2D2D2D;">c</span>ib <span class="s255">小</span>t <span style=" color: #2D2D2D;">y </span>i<span style=" color: #2D2D2D;">ss</span>u<span style=" color: #2D2D2D;">es</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 82pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="607" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_193.png"/></span></p><p class="s241" style="padding-top: 3pt;padding-left: 15pt;text-indent: 0pt;text-align: left;">Randomization P<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>di<span style=" color: #2D2D2D;">c</span>h<span style=" color: #2D2D2D;">o</span>tomi <span style=" color: #3F413F;">ze</span>d into l<span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">g-</span>t<span style=" color: #2D2D2D;">e</span>rm <span style=" color: #2D2D2D;">a</span>nd <span style=" color: #2D2D2D;">s</span>h<span style=" color: #2D2D2D;">o</span>rt <span style=" color: #3F413F;">-</span>t<span style=" color: #2D2D2D;">er</span>m <span style=" color: #2D2D2D;">s</span>ur<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">v</span>or <span style=" color: #2D2D2D;">s </span>ba<span style=" color: #2D2D2D;">se </span>d <span style=" color: #2D2D2D;">o</span>n <span style=" color: #2D2D2D;">s</span>ur <span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">va</span>l ti m<span style=" color: #2D2D2D;">e</span>. Th<span style=" color: #2D2D2D;">e </span>10 l<span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">g-</span>t<span style=" color: #2D2D2D;">e</span>rm <span style=" color: #2D2D2D;">s</span>u r<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">vo</span>r<span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>pati<span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s w</span>ho</p><p class="s241" style="padding-top: 1pt;padding-left: 82pt;text-indent: 0pt;line-height: 122%;text-align: left;"><span style=" color: #2D2D2D;">s</span>u r<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">ve</span>d mor<span style=" color: #2D2D2D;">e </span>th<span style=" color: #2D2D2D;">a</span>n 1 <span style=" color: #2D2D2D;">ye</span>ar <span style=" color: #2D2D2D;">a</span>ft <span style=" color: #2D2D2D;">e</span>r th<span style=" color: #2D2D2D;">e </span>di<span style=" color: #2D2D2D;">ag</span>n<span style=" color: #2D2D2D;">os</span>i<span style=" color: #2D2D2D;">s </span>of P<span style=" color: #2D2D2D;">C a</span>nd th <span style=" color: #2D2D2D;">e </span>10 <span style=" color: #2D2D2D;">s</span>h<span style=" color: #2D2D2D;">o</span>rt <span style=" color: #3F413F;">-</span>t<span style=" color: #2D2D2D;">e</span>rm <span style=" color: #2D2D2D;">s</span>ur<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">v</span>or <span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s w</span>h<span style=" color: #2D2D2D;">o </span>p<span style=" color: #2D2D2D;">asse </span>d <span style=" color: #2D2D2D;">away w</span>ithin <span style=" color: #2D2D2D;">6 </span>m<span style=" color: #2D2D2D;">o</span>nth <span style=" color: #2D2D2D;">s a</span>ft <span style=" color: #2D2D2D;">e</span>r th <span style=" color: #2D2D2D;">e </span>di<span style=" color: #2D2D2D;">ag</span>n <span style=" color: #2D2D2D;">os</span>i <span style=" color: #2D2D2D;">s o</span>f P<span style=" color: #2D2D2D;">C.</span></p><p style="padding-left: 82pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="607" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_194.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 82pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="607" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_195.png"/></span></p><p class="s241" style="padding-top: 3pt;padding-left: 81pt;text-indent: -66pt;line-height: 119%;text-align: left;">Blinding H<span style=" color: #2D2D2D;">C</span>L m<span style=" color: #2D2D2D;">a</span>ppin <span style=" color: #2D2D2D;">g a</span>nd tum <span style=" color: #2D2D2D;">o</span>r <span style=" color: #2D2D2D;">ce</span>ll lin <span style=" color: #2D2D2D;">eage </span>inf <span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">ce, s</span>in<span style=" color: #2D2D2D;">g</span>l<span style=" color: #2D2D2D;">e-ce</span>ll <span style=" color: #2D2D2D;">c</span>lu <span style=" color: #2D2D2D;">s</span>t <span style=" color: #2D2D2D;">e</span>rin<span style=" color: #2D2D2D;">g, </span>1TH <span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">yses, as </span><span style=" color: #3F413F;">we</span>ll <span style=" color: #2D2D2D;">as sa</span>mpl<span style=" color: #2D2D2D;">e </span>cl<span style=" color: #2D2D2D;">ass</span>ifi <span style=" color: #2D2D2D;">ca</span>ti <span style=" color: #2D2D2D;">o</span>n (<span style=" color: #2D2D2D;">Gas</span>t ri<span style=" color: #2D2D2D;">c-</span>d<span style=" color: #2D2D2D;">o</span>min<span style=" color: #2D2D2D;">a</span>nt <span style=" color: #2D2D2D;">a</span>nd <span style=" color: #2D2D2D;">G</span>I<span style=" color: #3F413F;">­ </span>mi<span style=" color: #2D2D2D;">xe</span>d) <span style=" color: #2D2D2D;">a</span>nd <span style=" color: #2D2D2D;">s</span>u r<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">va</span>l <span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">ys</span>i <span style=" color: #2D2D2D;">s, </span><span style=" color: #3F413F;">we</span>r<span style=" color: #2D2D2D;">e a</span>ll <span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">a </span>un <span style=" color: #2D2D2D;">s</span>up<span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">se</span>d <span style=" color: #2D2D2D;">o</span>r blind<span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">ap </span>pro<span style=" color: #2D2D2D;">ac</span>h<span style=" color: #2D2D2D;">es</span></p><p style="padding-left: 82pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="607" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_196.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s239" style="padding-top: 6pt;padding-left: 7pt;text-indent: 0pt;text-align: left;">Reporting for specific materials, systems and methods                         </p><p class="s248" style="padding-top: 4pt;padding-left: 8pt;text-indent: 0pt;line-height: 119%;text-align: left;">We <span style=" color: #646E69;">require information from authors </span>about some <span style=" color: #646E69;">types </span>of <span style=" color: #646E69;">materials, </span>experimental systems and <span style=" color: #646E69;">methods used in many </span>studies. <span style=" color: #646E69;">Here, indicate </span>whether each <span style=" color: #646E69;">material, </span>system or <span style=" color: #646E69;">method listed 1s relevant to your </span>study. <span style=" color: #646E69;">If you are not </span>sure <span style=" color: #646E69;">if a list item applies to </span>your <span style=" color: #646E69;">research, read the appropriate </span>section <span style=" color: #646E69;">before </span>selecting <span style=" color: #646E69;">a response</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="250" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_197.png"/></span></p><p class="s241" style="padding-left: 9pt;text-indent: 1pt;line-height: 151%;text-align: left;"><span style=" color: #971128;">Materials </span><span class="s257">&amp; </span><span style=" color: #971128;">experimental systems </span>n<span style=" color: #2D2D2D;">/</span>a In<span style=" color: #2D2D2D;">v</span>ol <span style=" color: #2D2D2D;">v</span>ed in the st ud <span style=" color: #2D2D2D;">y</span></p><p class="s242" style="padding-left: 12pt;text-indent: 0pt;line-height: 11pt;text-align: left;"><span class="s258">区</span><span class="s259">口 </span>A<span style=" color: #111;">ntib </span>o <span style=" color: #111;">di </span>es</p><p class="s242" style="padding-left: 12pt;text-indent: 0pt;line-height: 14pt;text-align: left;"><span class="s258">区</span><span class="s259">口 </span>E<span style=" color: #111;">uk</span>a<span style=" color: #111;">r</span>yo<span style=" color: #111;">t1</span>c ce<span style=" color: #111;">ll lin</span>es</p><p class="s241" style="padding-left: 11pt;text-indent: 0pt;text-align: left;"><span class="s258">因 </span><span class="s259">口 </span>P<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">aeo </span>nt<span style=" color: #2D2D2D;">o</span>l<span style=" color: #2D2D2D;">ogy a</span>nd <span style=" color: #2D2D2D;">a</span>r<span style=" color: #2D2D2D;">c</span>ha<span style=" color: #2D2D2D;">eo</span>l<span style=" color: #2D2D2D;">ogy</span></p><p class="s260" style="text-indent: 0pt;line-height: 24pt;text-align: left;">□</p><p style="text-indent: 0pt;text-align: left;"/><p class="s241" style="padding-left: 12pt;text-indent: 0pt;text-align: left;"><span class="s258">区</span><span class="s259">口 </span><span class="s261">础</span>m<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">s a</span>nd <span style=" color: #2D2D2D;">o</span>th <span style=" color: #2D2D2D;">e</span>r or<span style=" color: #2D2D2D;">ga</span>ni <span style=" color: #2D2D2D;">s</span>m<span style=" color: #2D2D2D;">s</span></p><p class="s241" style="text-indent: 0pt;line-height: 13pt;text-align: center;"><span class="s262">区 </span>Hum <span style=" color: #2D2D2D;">a</span>n r<span style=" color: #2D2D2D;">esea</span>r<span style=" color: #2D2D2D;">c</span>h p<span style=" color: #2D2D2D;">a</span>rti <span style=" color: #2D2D2D;">c</span>ip <span style=" color: #2D2D2D;">a</span>nt <span style=" color: #2D2D2D;">s</span></p><p class="s242" style="padding-left: 12pt;text-indent: 0pt;line-height: 15pt;text-align: left;"><span class="s262">区 </span><span class="s263">D </span>C<span style=" color: #111;">lini </span>ca<span style=" color: #111;">l d</span>a<span style=" color: #111;">ta</span></p><p class="s241" style="padding-left: 12pt;text-indent: 0pt;line-height: 14pt;text-align: left;"><span class="s258">区</span><span class="s259">口 </span>Du<span style=" color: #2D2D2D;">a</span>l u<span style=" color: #2D2D2D;">se </span>r<span style=" color: #2D2D2D;">esea</span>r<span style=" color: #2D2D2D;">c</span>h <span style=" color: #2D2D2D;">o</span>f <span style=" color: #2D2D2D;">co</span>n<span style=" color: #2D2D2D;">ce</span>rn</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s256" style="padding-bottom: 1pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">Methods</p><p style="padding-left: 3pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="250" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_198.png"/></span></p><p class="s241" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">n<span style=" color: #2D2D2D;">/</span>a In<span style=" color: #2D2D2D;">vo </span>l<span style=" color: #2D2D2D;">v</span>ed in the stud <span style=" color: #2D2D2D;">y</span></p><p class="s241" style="padding-top: 1pt;padding-left: 7pt;text-indent: 0pt;text-align: left;"><span class="s258">因 </span><span class="s259">口 </span><span style=" color: #2D2D2D;">C</span>hIP<span style=" color: #3F413F;">-s e</span>q</p><p class="s241" style="padding-left: 7pt;text-indent: 0pt;line-height: 14pt;text-align: left;"><span class="s258">因</span><span class="s259">口 </span>Fl <span style=" color: #2D2D2D;">ow cy</span>t<span style=" color: #2D2D2D;">o</span>m<span style=" color: #2D2D2D;">e</span>tr <span style=" color: #2D2D2D;">y</span></p><p class="s241" style="padding-left: 7pt;text-indent: 0pt;line-height: 14pt;text-align: left;"><span class="s258">因 </span><span class="s259">口 </span>M RI<span style=" color: #3F413F;">- </span>b <span style=" color: #2D2D2D;">ase </span>d n<span style=" color: #2D2D2D;">e</span>u<span style=" color: #2D2D2D;">ro</span>1m<span style=" color: #2D2D2D;">ag</span>in<span style=" color: #2D2D2D;">g</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s264" style="padding-top: 4pt;padding-left: 5pt;text-indent: 0pt;text-align: left;"> Human research participants                                                                                                                       </p><p style="text-indent: 0pt;text-align: left;"><span><img width="545" height="135" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_199.png"/></span></p><p class="s265" style="padding-top: 8pt;padding-left: 8pt;text-indent: 0pt;text-align: left;"><span class="s252">Policy information about </span>studies in<span style=" color: #3B89AF;">v</span>ol<span style=" color: #3B89AF;">v</span>ing human research participants</p><p class="s241" style="padding-top: 7pt;padding-left: 135pt;text-indent: -120pt;line-height: 107%;text-align: left;">Population <span style=" color: #2D2D2D;">c</span>haract eri <span style=" color: #2D2D2D;">s</span>t ic <span style=" color: #2D2D2D;">s              </span><span class="s266">A</span> t <span style=" color: #2D2D2D;">ota </span>l <span style=" color: #2D2D2D;">o</span>f <span style=" color: #2D2D2D;">20 GAC </span>p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s </span><span style=" color: #3F413F;">w</span>ith m<span style=" color: #2D2D2D;">a</span>li<span style=" color: #2D2D2D;">g</span>n<span style=" color: #2D2D2D;">a</span>nt <span style=" color: #2D2D2D;">asc</span>1t <span style=" color: #2D2D2D;">es </span>(p <span style=" color: #2D2D2D;">e</span>rit <span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">ea</span>l <span style=" color: #2D2D2D;">ca</span>r<span style=" color: #2D2D2D;">c</span>in<span style=" color: #2D2D2D;">o</span>m<span style=" color: #2D2D2D;">a</span>t <span style=" color: #2D2D2D;">os</span>i<span style=" color: #2D2D2D;">s</span>, <span style=" color: #2D2D2D;">PC</span>) <span style=" color: #3F413F;">we</span>r<span style=" color: #2D2D2D;">e </span>includ <span style=" color: #2D2D2D;">e</span>d in thi<span style=" color: #2D2D2D;">s s</span>tud <span style=" color: #2D2D2D;">y</span>. Th<span style=" color: #2D2D2D;">e </span>prim<span style=" color: #2D2D2D;">a</span>r<span style=" color: #2D2D2D;">y GAC </span>d1a<span style=" color: #2D2D2D;">g</span>n<span style=" color: #2D2D2D;">os</span>1<span style=" color: #2D2D2D;">s was co</span>nfirm <span style=" color: #2D2D2D;">e</span>d th<span style=" color: #2D2D2D;">ro</span>u<span style=" color: #2D2D2D;">g</span>h <span style=" color: #2D2D2D;">a</span>n <span style=" color: #2D2D2D;">e</span>nd<span style=" color: #2D2D2D;">osco</span>pi<span style=" color: #2D2D2D;">c </span>bi<span style=" color: #2D2D2D;">o</span>p<span style=" color: #2D2D2D;">sy</span>. P<span style=" color: #2D2D2D;">a</span>th <span style=" color: #2D2D2D;">o </span>l<span style=" color: #2D2D2D;">ogy  </span>r<span style=" color: #2D2D2D;">es</span>ult <span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e ve</span>rifi <span style=" color: #2D2D2D;">e</span>d ind<span style=" color: #2D2D2D;">e</span>p<span style=" color: #2D2D2D;">e</span>nd <span style=" color: #2D2D2D;">e</span>ntl <span style=" color: #2D2D2D;">y </span>b<span style=" color: #2D2D2D;">y </span>t <span style=" color: #2D2D2D;">wo</span></p><p class="s241" style="padding-top: 1pt;padding-left: 135pt;text-indent: 0pt;line-height: 119%;text-align: left;"><span style=" color: #2D2D2D;">ex</span>p <span style=" color: #2D2D2D;">e</span>ri<span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">ce </span>d <span style=" color: #2D2D2D;">G</span>I p<span style=" color: #2D2D2D;">a</span>th <span style=" color: #2D2D2D;">o</span>l<span style=" color: #2D2D2D;">og</span>i<span style=" color: #3F413F;">s</span>t <span style=" color: #2D2D2D;">s</span>. In <span style=" color: #2D2D2D;">a</span>dd<span class="s267">巾</span><span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">, C</span>T im<span style=" color: #2D2D2D;">ages o </span>f <span style=" color: #2D2D2D;">a</span>ll p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>r<span style=" color: #2D2D2D;">e-</span>r<span style=" color: #2D2D2D;">ev</span>i<span style=" color: #2D2D2D;">ewe</span>d b<span style=" color: #2D2D2D;">y a</span>n <span style=" color: #2D2D2D;">expe </span>ri <span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">ce</span>d im<span style=" color: #2D2D2D;">ag</span>in <span style=" color: #2D2D2D;">g </span>ph<span style=" color: #2D2D2D;">ys</span>i<span style=" color: #2D2D2D;">c</span>i<span style=" color: #2D2D2D;">a</span>n <span style=" color: #2D2D2D;"> GACs </span><span style=" color: #3F413F;">we</span>r<span style=" color: #2D2D2D;">e </span><span style=" color: #3F413F;">s</span>t <span style=" color: #2D2D2D;">age </span>d a<span style=" color: #2D2D2D;">cco</span>rdin<span style=" color: #2D2D2D;">g </span>t<span style=" color: #2D2D2D;">o t</span>h<span style=" color: #2D2D2D;">e A</span>m<span style=" color: #2D2D2D;">e</span>ri <span style=" color: #2D2D2D;">ca</span>n J<span style=" color: #2D2D2D;">o </span>int   <span style=" color: #2D2D2D;">Co</span>mmitt <span style=" color: #2D2D2D;">ee o</span>n <span style=" color: #2D2D2D;">Ca</span>n <span style=" color: #2D2D2D;">ce</span>r <span style=" color: #2D2D2D;">Stag </span>in <span style=" color: #2D2D2D;">g </span>M <span style=" color: #2D2D2D;">a</span>nu <span style=" color: #2D2D2D;">a</span>l (<span style=" color: #2D2D2D;">8t</span>h <span style=" color: #2D2D2D;">e</span>d1t1 <span style=" color: #2D2D2D;">o</span>n)<span style=" color: #3F413F;">. </span>P<span style=" color: #2D2D2D;">C was co</span>nfirm <span style=" color: #2D2D2D;">e</span>d b<span style=" color: #2D2D2D;">y cy</span>t<span style=" color: #2D2D2D;">o</span>l<span style=" color: #2D2D2D;">og</span>1<span style=" color: #2D2D2D;">c exa</span>min<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">o</span>n<span style=" color: #3F413F;">. </span>Thi<span style=" color: #2D2D2D;">s co</span>h<span style=" color: #2D2D2D;">o</span>rt includ<span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">10 </span>l<span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">g-</span>t <span style=" color: #2D2D2D;">e</span>rm  <span style=" color: #2D2D2D;">s</span>ur<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">vo</span>r<span style=" color: #2D2D2D;">s a</span>nd  10 <span style=" color: #2D2D2D;">s</span>h <span style=" color: #2D2D2D;">o</span>rt <span style=" color: #3F413F;">-</span>t<span style=" color: #2D2D2D;">e</span>rm  <span style=" color: #2D2D2D;">s</span>u r<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">vo</span>r<span style=" color: #2D2D2D;">s </span>(L<span style=" color: #2D2D2D;">og  </span>r<span style=" color: #2D2D2D;">a</span>nk p <span style=" color: #3F413F;">= </span><span style=" color: #2D2D2D;">3e-</span>0<span style=" color: #2D2D2D;">6)</span>. Th<span style=" color: #2D2D2D;">e </span>l<span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">g-</span>t <span style=" color: #2D2D2D;">e</span>rm <span style=" color: #2D2D2D;">s</span>u r<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">vo</span>r<span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>p<span style=" color: #2D2D2D;">at </span>i<span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s w</span>h<span style=" color: #2D2D2D;">o s</span>ur <span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">ve</span>d m<span style=" color: #2D2D2D;">o</span>r<span style=" color: #2D2D2D;">e t</span>h<span style=" color: #2D2D2D;">a</span>n 1 <span style=" color: #2D2D2D;">yea</span>r <span style=" color: #2D2D2D;">a</span>ft <span style=" color: #2D2D2D;">e</span>r th<span style=" color: #2D2D2D;">e </span>di<span style=" color: #2D2D2D;">ag</span>n<span style=" color: #2D2D2D;">os</span>i<span style=" color: #2D2D2D;">s o </span>f P<span style=" color: #2D2D2D;">C a</span>nd  th <span style=" color: #2D2D2D;">e s</span>h<span style=" color: #2D2D2D;">o</span>rt <span style=" color: #3F413F;">-</span>t<span style=" color: #2D2D2D;">e</span>rm  <span style=" color: #2D2D2D;">s</span>ur<span style=" color: #2D2D2D;">v</span>i<span style=" color: #2D2D2D;">vo</span>r<span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e </span>p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s w</span>h<span style=" color: #2D2D2D;">o </span>p<span style=" color: #2D2D2D;">asse </span>d <span style=" color: #2D2D2D;">away w</span>i<span style=" color: #2D2D2D;">t</span>hin <span style=" color: #2D2D2D;">6 </span>m<span style=" color: #2D2D2D;">o</span>n<span style=" color: #2D2D2D;">t</span>h<span style=" color: #2D2D2D;">s a</span>ft <span style=" color: #2D2D2D;">e</span>r th <span style=" color: #2D2D2D;">e </span>di<span style=" color: #2D2D2D;">ag </span>n<span style=" color: #2D2D2D;">os</span>i<span style=" color: #2D2D2D;">s o</span>f P<span style=" color: #2D2D2D;">C. </span>B<span style=" color: #2D2D2D;">ase </span>d <span style=" color: #2D2D2D;">o</span>n th<span style=" color: #2D2D2D;">e </span>L<span style=" color: #2D2D2D;">a</span>ur <span style=" color: #2D2D2D;">e</span>n<span style=" color: #2D2D2D;">&#39;s </span>cl<span style=" color: #2D2D2D;">ass</span>ifi <span style=" color: #2D2D2D;">ca</span>ti <span style=" color: #2D2D2D;">o</span>n <span style=" color: #2D2D2D;">o</span>f th <span style=" color: #2D2D2D;">e p</span>rim<span style=" color: #2D2D2D;">a</span>r<span style=" color: #2D2D2D;">y GAC, a</span>ll tu m<span style=" color: #2D2D2D;">o</span>r<span style=" color: #2D2D2D;">s we</span>r<span style=" color: #2D2D2D;">e o</span>f diffu <span style=" color: #2D2D2D;">se </span>t<span style=" color: #2D2D2D;">y</span>p<span style=" color: #2D2D2D;">e. S</span>i <span style=" color: #2D2D2D;">x</span>t <span style=" color: #2D2D2D;">ee</span>n <span style=" color: #2D2D2D;">o</span>ut <span style=" color: #2D2D2D;">o</span>f t <span style=" color: #2D2D2D;">we</span>nt<span style=" color: #2D2D2D;">y </span>p<span style=" color: #2D2D2D;">a</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s </span>h<span style=" color: #2D2D2D;">a</span>d <span style=" color: #2D2D2D;">S</span>i<span style=" color: #2D2D2D;">g </span>n<span style=" color: #2D2D2D;">e</span>t<span style=" color: #2D2D2D;">-</span>rin<span style=" color: #2D2D2D;">g </span><span style=" color: #3F413F;">ce</span>ll <span style=" color: #2D2D2D;">ca</span>r<span style=" color: #2D2D2D;">c</span>in<span style=" color: #2D2D2D;">o</span>m<span style=" color: #2D2D2D;">a. </span>H<span style=" color: #2D2D2D;">e</span>r <span style=" color: #2D2D2D;">2 </span>p<span style=" color: #2D2D2D;">os</span>iti <span style=" color: #2D2D2D;">v</span>it <span style=" color: #2D2D2D;">y was pe</span>r <span style=" color: #2D2D2D;">fo</span>rm<span style=" color: #2D2D2D;">e</span>d bu<span style=" color: #2D2D2D;">t </span>no H<span style=" color: #2D2D2D;">e</span>r<span style=" color: #2D2D2D;">2 </span>p<span style=" color: #2D2D2D;">os</span>it 1<span style=" color: #2D2D2D;">v</span>1t <span style=" color: #2D2D2D;">y was </span>d<span style=" color: #2D2D2D;">e</span>t <span style=" color: #2D2D2D;">ecte </span>d in th<span style=" color: #2D2D2D;">ese pa</span>ti <span style=" color: #2D2D2D;">e</span>nt <span style=" color: #2D2D2D;">s. </span>Th<span style=" color: #2D2D2D;">ese </span>includ <span style=" color: #2D2D2D;">e</span>d <span style=" color: #2D2D2D;">8 </span>m<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">es a</span>n<span style=" color: #2D2D2D;">d </span>1<span style=" color: #2D2D2D;">2 </span>f<span style=" color: #2D2D2D;">e</span>m<span style=" color: #2D2D2D;">a</span>l<span style=" color: #2D2D2D;">es w</span>ith <span style=" color: #2D2D2D;">a</span>n <span style=" color: #2D2D2D;">ave </span>r<span style=" color: #2D2D2D;">age </span>a<span style=" color: #2D2D2D;">ge o</span>f <span style=" color: #2D2D2D;">5</span>4 <span style=" color: #2D2D2D;">(</span>r<span style=" color: #2D2D2D;">a</span>n<span style=" color: #2D2D2D;">ge 32-72)</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="28" height="1054" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_200.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 136pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="535" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_201.png"/></span></p><p class="s268" style="padding-top: 3pt;padding-bottom: 2pt;padding-left: 15pt;text-indent: 0pt;text-align: left;">Re <span style=" color: #2F2F2F;">c </span>ruitm ent n<span style=" color: #2F2F2F;">o</span>t <span style=" color: #2F2F2F;">ap </span>pli<span style=" color: #2F2F2F;">ca</span>bl<span style=" color: #2F2F2F;">e, </span>thi <span style=" color: #2F2F2F;">s </span>i<span style=" color: #2F2F2F;">s a </span>r<span style=" color: #2F2F2F;">e</span>t <span style=" color: #2F2F2F;">ros</span>p <span style=" color: #2F2F2F;">ec</span>ti <span style=" color: #2F2F2F;">ve s</span>tud <span style=" color: #2F2F2F;">y, </span>n<span style=" color: #2F2F2F;">ot a </span>clini<span style=" color: #2F2F2F;">ca</span><span style=" color: #010101;">l </span>t ri<span style=" color: #2F2F2F;">a</span><span style=" color: #010101;">l</span></p><p style="padding-left: 136pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="535" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_202.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 136pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="535" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_203.png"/></span></p><p class="s269" style="padding-top: 1pt;padding-bottom: 1pt;padding-left: 135pt;text-indent: -120pt;line-height: 106%;text-align: left;"><span style=" color: #010101;">Et hic </span>s <span style=" color: #151515;">o </span>v<span style=" color: #151515;">er </span>s<span style=" color: #151515;">ight P</span>C s<span style=" color: #151515;">p</span>ec<span style=" color: #151515;">im </span>e<span style=" color: #151515;">n</span>s we<span style=" color: #151515;">r</span>e co<span style=" color: #010101;">ll</span>ec<span style=" color: #151515;">t</span>e<span style=" color: #151515;">d </span>a<span style=" color: #151515;">t Th</span>e <span style=" color: #151515;">Uni</span>ve<span style=" color: #151515;">r</span>s<span style=" color: #151515;">it </span>y o<span style=" color: #151515;">f T</span>exas <span style=" color: #151515;">MD </span>A<span style=" color: #151515;">nd </span>e<span style=" color: #151515;">r</span>so<span style=" color: #151515;">n </span>Ca<span style=" color: #151515;">n </span>ce<span style=" color: #151515;">r </span>Ce<span style=" color: #151515;">nt</span>e<span style=" color: #151515;">r </span><span class="s271">旧 </span>o<span style=" color: #151515;">u</span>s<span style=" color: #151515;">t </span>o<span style=" color: #151515;">n</span>, <span style=" color: #151515;">U</span>SA <span style=" color: #151515;">) und</span>e<span style=" color: #151515;">r </span>a<span style=" color: #151515;">n </span><span style=" color: #010101;">In</span>s<span style=" color: #151515;">tituti </span>o<span style=" color: #151515;">n</span>a<span style=" color: #010101;">l </span><span style=" color: #151515;">R</span>ev<span style=" color: #151515;">i</span>ew <span style=" color: #151515;">B</span>oa <span style=" color: #151515;">rd (IRB) </span>a<span style=" color: #151515;">pp </span>rove<span style=" color: #151515;">d prot </span>oco<span style=" color: #010101;">l </span>(#<span style=" color: #151515;">L</span>A<span style=" color: #151515;">BOl </span><span style=" color: #424242;">-5</span><span style=" color: #151515;">4</span>3) a<span style=" color: #151515;">ft </span>e<span style=" color: #151515;">r </span>o<span style=" color: #151515;">bt</span>a<span style=" color: #151515;">inin </span>g w<span style=" color: #151515;">ritt </span>e<span style=" color: #151515;">n inform </span>e<span style=" color: #151515;">d </span>co<span style=" color: #151515;">n</span>se<span style=" color: #151515;">nt fr</span>o<span style=" color: #151515;">m </span>eac<span style=" color: #151515;">h p</span>a<span style=" color: #151515;">rti </span>c<span style=" color: #151515;">ip </span>a<span style=" color: #151515;">nt</span></p><p style="padding-left: 136pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="535" height="1" alt="image" src="腹膜转移SCRNASingle-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma/Image_204.png"/></span></p><p class="s273" style="padding-top: 4pt;text-indent: 0pt;text-align: center;">Note that full information <span style=" color: #747C77;">on </span>the <span style=" color: #747C77;">approval of </span>the <span style=" color: #747C77;">study </span>protocol must <span style=" color: #747C77;">also </span>be p<span class="s275">「</span>ovided in the manuscript</p></body></html>
